<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Nucleosome structures</title>

<script src="site_libs/header-attrs-2.6/header-attrs.js"></script>
<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/simplex.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<script src="site_libs/navigation-1.1/sourceembed.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="site_libs/plotly-binding-4.9.3/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
#rmd-source-code {
  display: none;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 41px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 46px;
  margin-top: -46px;
}
.section h2 {
  padding-top: 46px;
  margin-top: -46px;
}
.section h3 {
  padding-top: 46px;
  margin-top: -46px;
}
.section h4 {
  padding-top: 46px;
  margin-top: -46px;
}
.section h5 {
  padding-top: 46px;
  margin-top: -46px;
}
.section h6 {
  padding-top: 46px;
  margin-top: -46px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Insights from the PDB</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="nucleosome-structures.html">Nucleosome structures</a>
</li>
<li>
  <a href="sars-cov-2.html">SARS-CoV-2</a>
</li>
<li>
  <a href="dna-length-in-protein-dna-complexes.html">DNA length in protein-DNA complexes</a>
</li>
<li>
  <a href="free-dna.html">Free DNA</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/Guillawme/insights-from-the-pdb">GitHub repository</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">

<div class="btn-group pull-right">
<button type="button" class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
<li role="separator" class="divider"></li>
<li><a id="rmd-download-source" href="#">Download Rmd</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Nucleosome structures</h1>

</div>


<p><strong>Last updated on Tue Feb 9 12:51:01 2021.</strong></p>
<div id="how-to-re-use-this-work" class="section level2">
<h2>How to re-use this work</h2>
<p>If you use the following figures in your own work, please cite:</p>
<ul>
<li>Zhou K, Gaullier G &amp; Luger K (2019) Nucleosome structure and dynamics are coming of age. <em>Nature Structural &amp; Molecular Biology</em> <strong>26</strong>: 3–13. <a href="https://doi.org/10.1038/s41594-018-0166-x" class="uri">https://doi.org/10.1038/s41594-018-0166-x</a></li>
<li>This website: <a href="https://doi.org/10.5281/zenodo.3470119" class="uri">https://doi.org/10.5281/zenodo.3470119</a></li>
</ul>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load required packages</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(magrittr)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(purrr)</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(jsonlite)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(forcats)</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(plotly)</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(stringr)</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="co"># We need two queries: one for uppercase titles, the other for lowercase ones</span></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a>pdb_queries <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">uppercase =</span> <span class="st">&#39;https://www.ebi.ac.uk/pdbe/search/pdb/select?q=title:*UCLEOSOM*%20AND%20status:REL&amp;fl=pdb_id,citation_year,title,experimental_method,resolution,organism_scientific_name,molecule_name,molecule_type,number_of_protein_chains&amp;rows=1000000&amp;wt=json&#39;</span>,</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">lowercase =</span> <span class="st">&#39;https://www.ebi.ac.uk/pdbe/search/pdb/select?q=title:*ucleosom*%20AND%20status:REL&amp;fl=pdb_id,citation_year,title,experimental_method,resolution,organism_scientific_name,molecule_name,molecule_type,number_of_protein_chains&amp;rows=1000000&amp;wt=json&#39;</span>,</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">ncp =</span> <span class="st">&#39;https://www.ebi.ac.uk/pdbe/search/pdb/select?q=title:*NCP*%20AND%20status:REL&amp;fl=pdb_id,citation_year,title,experimental_method,resolution,organism_scientific_name,molecule_name,molecule_type,number_of_protein_chains&amp;rows=1000000&amp;wt=json&#39;</span></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="co"># The following PDB entries have the word &quot;nucleosome in their title, but do not</span></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="co"># actually contain a nucleosome, so we need to exclude them from the analysis.</span></span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Unfortunately, there is no good way to automate this.</span></span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a>non_nucleosome_structures <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;1hst&quot;</span>,</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;2z2r&quot;</span>,</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5x7v&quot;</span>,</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;3uv2&quot;</span>,</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;3fs3&quot;</span>,</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;1wg3&quot;</span>,</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;1nw3&quot;</span>,</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5ikf&quot;</span>,</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;3gyw&quot;</span>,</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;3gyv&quot;</span>,</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;1ofc&quot;</span>,</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;2ayu&quot;</span>,</span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;2iw5&quot;</span>,</span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;3hfd&quot;</span>,</span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;6uch&quot;</span>,</span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4k&quot;</span>,</span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4m&quot;</span>,</span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4l&quot;</span>,</span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4o&quot;</span>,</span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4g&quot;</span>,</span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4h&quot;</span>,</span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4j&quot;</span>,</span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4i&quot;</span>,</span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;5r4n&quot;</span>,</span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;1bj6&quot;</span>,</span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;6qds&quot;</span>,</span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;2iwj&quot;</span>,</span>
<span id="cb1-50"><a href="#cb1-50" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;4dvk&quot;</span>,</span>
<span id="cb1-51"><a href="#cb1-51" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;1a6b&quot;</span>,</span>
<span id="cb1-52"><a href="#cb1-52" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;6qdu&quot;</span>,</span>
<span id="cb1-53"><a href="#cb1-53" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;6ne8&quot;</span>,</span>
<span id="cb1-54"><a href="#cb1-54" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;1tsu&quot;</span>,</span>
<span id="cb1-55"><a href="#cb1-55" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;1esk&quot;</span>,</span>
<span id="cb1-56"><a href="#cb1-56" aria-hidden="true" tabindex="-1"></a>    <span class="st">&quot;7c4j&quot;</span></span>
<span id="cb1-57"><a href="#cb1-57" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-58"><a href="#cb1-58" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-59"><a href="#cb1-59" aria-hidden="true" tabindex="-1"></a><span class="co"># This is a helper pipeline to extract data</span></span>
<span id="cb1-60"><a href="#cb1-60" aria-hidden="true" tabindex="-1"></a>dig_up_data <span class="ot">&lt;-</span> . <span class="sc">%&gt;%</span></span>
<span id="cb1-61"><a href="#cb1-61" aria-hidden="true" tabindex="-1"></a>    .<span class="sc">$</span>response <span class="sc">%&gt;%</span></span>
<span id="cb1-62"><a href="#cb1-62" aria-hidden="true" tabindex="-1"></a>    .<span class="sc">$</span>docs <span class="sc">%&gt;%</span></span>
<span id="cb1-63"><a href="#cb1-63" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as_tibble</span>()</span>
<span id="cb1-64"><a href="#cb1-64" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-65"><a href="#cb1-65" aria-hidden="true" tabindex="-1"></a><span class="co"># This is a helper function to detect the presence of a binding factor</span></span>
<span id="cb1-66"><a href="#cb1-66" aria-hidden="true" tabindex="-1"></a>has_binding_factor_one <span class="ot">&lt;-</span> <span class="cf">function</span>(number_of_protein_chains, title) {</span>
<span id="cb1-67"><a href="#cb1-67" aria-hidden="true" tabindex="-1"></a>    <span class="co"># It takes 8 histone chains to make a nucleosome, so if the number of</span></span>
<span id="cb1-68"><a href="#cb1-68" aria-hidden="true" tabindex="-1"></a>    <span class="co"># protein chains is not divisible by 8, this means there is a binding factor,</span></span>
<span id="cb1-69"><a href="#cb1-69" aria-hidden="true" tabindex="-1"></a>    <span class="co"># unless we&#39;re seeing the overlapping dinucleosome (14 proteins chains, but</span></span>
<span id="cb1-70"><a href="#cb1-70" aria-hidden="true" tabindex="-1"></a>    <span class="co"># they are all histones... one octamer + one hexamer)</span></span>
<span id="cb1-71"><a href="#cb1-71" aria-hidden="true" tabindex="-1"></a>    oldn <span class="ot">&lt;-</span> <span class="fu">str_detect</span>(title, <span class="st">&quot;unusual&quot;</span>)</span>
<span id="cb1-72"><a href="#cb1-72" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (number_of_protein_chains <span class="sc">%%</span> <span class="dv">8</span> <span class="sc">!=</span> <span class="dv">0</span> <span class="sc">&amp;</span> <span class="sc">!</span>oldn) {</span>
<span id="cb1-73"><a href="#cb1-73" aria-hidden="true" tabindex="-1"></a>        <span class="fu">return</span>(<span class="cn">TRUE</span>)</span>
<span id="cb1-74"><a href="#cb1-74" aria-hidden="true" tabindex="-1"></a>    } <span class="cf">else</span> {</span>
<span id="cb1-75"><a href="#cb1-75" aria-hidden="true" tabindex="-1"></a>        <span class="co"># But for multiple of 8 chains &gt; 8 (i.e. 16, 24, 32, 40), we can have this</span></span>
<span id="cb1-76"><a href="#cb1-76" aria-hidden="true" tabindex="-1"></a>        <span class="co"># number of chains by chance even with binding factors. There is no good</span></span>
<span id="cb1-77"><a href="#cb1-77" aria-hidden="true" tabindex="-1"></a>        <span class="co"># way to automatically find these cases, unfortunately</span></span>
<span id="cb1-78"><a href="#cb1-78" aria-hidden="true" tabindex="-1"></a>        compass <span class="ot">&lt;-</span> <span class="fu">str_detect</span>(title, <span class="st">&quot;COMPASS&quot;</span>)</span>
<span id="cb1-79"><a href="#cb1-79" aria-hidden="true" tabindex="-1"></a>        corest <span class="ot">&lt;-</span> <span class="fu">str_detect</span>(title, <span class="st">&quot;LSD1/CoREST&quot;</span>)</span>
<span id="cb1-80"><a href="#cb1-80" aria-hidden="true" tabindex="-1"></a>        binding_factor <span class="ot">&lt;-</span> compass <span class="sc">||</span> corest</span>
<span id="cb1-81"><a href="#cb1-81" aria-hidden="true" tabindex="-1"></a>        <span class="cf">if</span> (binding_factor) {</span>
<span id="cb1-82"><a href="#cb1-82" aria-hidden="true" tabindex="-1"></a>            <span class="fu">return</span>(<span class="cn">TRUE</span>)</span>
<span id="cb1-83"><a href="#cb1-83" aria-hidden="true" tabindex="-1"></a>        } <span class="cf">else</span> {</span>
<span id="cb1-84"><a href="#cb1-84" aria-hidden="true" tabindex="-1"></a>            <span class="fu">return</span>(<span class="cn">FALSE</span>)</span>
<span id="cb1-85"><a href="#cb1-85" aria-hidden="true" tabindex="-1"></a>        }</span>
<span id="cb1-86"><a href="#cb1-86" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb1-87"><a href="#cb1-87" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb1-88"><a href="#cb1-88" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-89"><a href="#cb1-89" aria-hidden="true" tabindex="-1"></a><span class="co"># Vectorize the above function</span></span>
<span id="cb1-90"><a href="#cb1-90" aria-hidden="true" tabindex="-1"></a>has_binding_factor <span class="ot">&lt;-</span> <span class="cf">function</span>(number_of_protein_chains_vector, title) {</span>
<span id="cb1-91"><a href="#cb1-91" aria-hidden="true" tabindex="-1"></a>    <span class="fu">map2_lgl</span>(number_of_protein_chains_vector, title, has_binding_factor_one)</span>
<span id="cb1-92"><a href="#cb1-92" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb1-93"><a href="#cb1-93" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-94"><a href="#cb1-94" aria-hidden="true" tabindex="-1"></a><span class="co"># Query the PDB and clean up data</span></span>
<span id="cb1-95"><a href="#cb1-95" aria-hidden="true" tabindex="-1"></a>pdb_data <span class="ot">&lt;-</span> pdb_queries <span class="sc">%&gt;%</span> </span>
<span id="cb1-96"><a href="#cb1-96" aria-hidden="true" tabindex="-1"></a>    <span class="fu">map</span>(fromJSON) <span class="sc">%&gt;%</span> </span>
<span id="cb1-97"><a href="#cb1-97" aria-hidden="true" tabindex="-1"></a>    <span class="fu">map</span>(dig_up_data) <span class="sc">%&gt;%</span> </span>
<span id="cb1-98"><a href="#cb1-98" aria-hidden="true" tabindex="-1"></a>    <span class="fu">bind_rows</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb1-99"><a href="#cb1-99" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(<span class="sc">!</span>(pdb_id <span class="sc">%in%</span> non_nucleosome_structures)) <span class="sc">%&gt;%</span> </span>
<span id="cb1-100"><a href="#cb1-100" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(</span>
<span id="cb1-101"><a href="#cb1-101" aria-hidden="true" tabindex="-1"></a>      <span class="at">has_binding_factor    =</span> <span class="fu">has_binding_factor</span>(number_of_protein_chains, title),</span>
<span id="cb1-102"><a href="#cb1-102" aria-hidden="true" tabindex="-1"></a>      <span class="at">experimental_method   =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(experimental_method)),</span>
<span id="cb1-103"><a href="#cb1-103" aria-hidden="true" tabindex="-1"></a>      <span class="at">citation_year         =</span> <span class="fu">as.integer</span>(citation_year),</span>
<span id="cb1-104"><a href="#cb1-104" aria-hidden="true" tabindex="-1"></a>      <span class="at">molecule_name         =</span> <span class="fu">as.character</span>(molecule_name),</span>
<span id="cb1-105"><a href="#cb1-105" aria-hidden="true" tabindex="-1"></a>      <span class="at">molecule_type         =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(molecule_type))</span>
<span id="cb1-106"><a href="#cb1-106" aria-hidden="true" tabindex="-1"></a>      ) <span class="sc">%&gt;%</span> </span>
<span id="cb1-107"><a href="#cb1-107" aria-hidden="true" tabindex="-1"></a>    <span class="fu">distinct</span>(pdb_id, <span class="at">.keep_all =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</div>
<div id="nucleosome-structures-by-year" class="section level2">
<h2>Nucleosome structures by year</h2>
<p>All figures are interactive (you can zoom in, and hovering over elements will show more information).</p>
<div id="nucleosome-structures-by-year-and-experimental-method" class="section level3">
<h3>Nucleosome structures by year and experimental method</h3>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>nucleosome_structures_year <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> citation_year, <span class="at">fill =</span> experimental_method)) <span class="sc">+</span></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Experimental method&quot;</span>)) <span class="sc">+</span></span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of nucleosomes by year&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Publication year&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_structures_year)</span></code></pre></div>
<div id="htmlwidget-24fbcb24f5da33e54533" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-24fbcb24f5da33e54533">{"x":{"data":[{"orientation":"v","width":[0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,5,21,71,39],"x":[1997,2000,2001,2002,2003,2004,2005,2006,2007,2008,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020],"y":[1,2,1,3,3,12,3,2,1,5,14,21,3,14,4,20,15,23,7,14,13],"text":["count:  1<br />citation_year: 1997<br />experimental_method: X-ray diffraction","count:  2<br />citation_year: 2000<br />experimental_method: X-ray diffraction","count:  1<br />citation_year: 2001<br />experimental_method: X-ray diffraction","count:  3<br />citation_year: 2002<br />experimental_method: X-ray diffraction","count:  3<br />citation_year: 2003<br />experimental_method: X-ray diffraction","count: 12<br />citation_year: 2004<br />experimental_method: X-ray diffraction","count:  3<br />citation_year: 2005<br />experimental_method: X-ray diffraction","count:  2<br />citation_year: 2006<br />experimental_method: X-ray diffraction","count:  1<br />citation_year: 2007<br />experimental_method: X-ray diffraction","count:  5<br />citation_year: 2008<br />experimental_method: X-ray diffraction","count: 14<br />citation_year: 2010<br />experimental_method: X-ray diffraction","count: 21<br />citation_year: 2011<br />experimental_method: X-ray diffraction","count:  3<br />citation_year: 2012<br />experimental_method: X-ray diffraction","count: 14<br />citation_year: 2013<br />experimental_method: X-ray diffraction","count:  4<br />citation_year: 2014<br />experimental_method: X-ray diffraction","count: 20<br />citation_year: 2015<br />experimental_method: X-ray diffraction","count: 15<br />citation_year: 2016<br />experimental_method: X-ray diffraction","count: 23<br />citation_year: 2017<br />experimental_method: X-ray diffraction","count:  7<br />citation_year: 2018<br />experimental_method: X-ray diffraction","count: 14<br />citation_year: 2019<br />experimental_method: X-ray diffraction","count: 13<br />citation_year: 2020<br />experimental_method: X-ray diffraction"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"X-ray diffraction","legendgroup":"X-ray diffraction","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091],"base":[0,0,0,0,0,0],"x":[2016,2017,2018,2019,2020,2021],"y":[1,5,21,71,39,1],"text":["count:  1<br />citation_year: 2016<br />experimental_method: Electron Microscopy","count:  5<br />citation_year: 2017<br />experimental_method: Electron Microscopy","count: 21<br />citation_year: 2018<br />experimental_method: Electron Microscopy","count: 71<br />citation_year: 2019<br />experimental_method: Electron Microscopy","count: 39<br />citation_year: 2020<br />experimental_method: Electron Microscopy","count:  1<br />citation_year: 2021<br />experimental_method: Electron Microscopy"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Electron Microscopy","legendgroup":"Electron Microscopy","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":40.1826484018265,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Structures of nucleosomes by year","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[1995.305,2022.695],"tickmode":"array","ticktext":["2000","2005","2010","2015","2020"],"tickvals":[2000,2005,2010,2015,2020],"categoryorder":"array","categoryarray":["2000","2005","2010","2015","2020"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Publication year","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-4.25,89.25],"tickmode":"array","ticktext":["0","20","40","60","80"],"tickvals":[0,20,40,60,80],"categoryorder":"array","categoryarray":["0","20","40","60","80"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Experimental method","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a650f7467":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a650f7467","visdat":{"406a650f7467":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-structures-by-year.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
<div id="nucleosome-structures-by-year-and-presence-of-a-binding-factor" class="section level3">
<h3>Nucleosome structures by year and presence of a binding factor</h3>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>nucleosome_structures_year_binding_factor <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> citation_year, <span class="at">fill =</span> has_binding_factor)) <span class="sc">+</span></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Contains a binding factor&quot;</span>)) <span class="sc">+</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of nucleosomes by year&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Publication year&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_structures_year_binding_factor)</span></code></pre></div>
<div id="htmlwidget-9e43f498b6bdf22c3a11" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-9e43f498b6bdf22c3a11">{"x":{"data":[{"orientation":"v","width":[0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091],"base":[0,0,0,0,0,0,0,1,0,0,1,1,0,4,1,0,4,8,15,60,43],"x":[1997,2000,2001,2002,2003,2004,2005,2006,2007,2008,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020],"y":[1,2,1,3,3,12,3,1,1,5,13,20,3,10,3,20,12,20,13,25,9],"text":["count:  1<br />citation_year: 1997<br />has_binding_factor: FALSE","count:  2<br />citation_year: 2000<br />has_binding_factor: FALSE","count:  1<br />citation_year: 2001<br />has_binding_factor: FALSE","count:  3<br />citation_year: 2002<br />has_binding_factor: FALSE","count:  3<br />citation_year: 2003<br />has_binding_factor: FALSE","count: 12<br />citation_year: 2004<br />has_binding_factor: FALSE","count:  3<br />citation_year: 2005<br />has_binding_factor: FALSE","count:  1<br />citation_year: 2006<br />has_binding_factor: FALSE","count:  1<br />citation_year: 2007<br />has_binding_factor: FALSE","count:  5<br />citation_year: 2008<br />has_binding_factor: FALSE","count: 13<br />citation_year: 2010<br />has_binding_factor: FALSE","count: 20<br />citation_year: 2011<br />has_binding_factor: FALSE","count:  3<br />citation_year: 2012<br />has_binding_factor: FALSE","count: 10<br />citation_year: 2013<br />has_binding_factor: FALSE","count:  3<br />citation_year: 2014<br />has_binding_factor: FALSE","count: 20<br />citation_year: 2015<br />has_binding_factor: FALSE","count: 12<br />citation_year: 2016<br />has_binding_factor: FALSE","count: 20<br />citation_year: 2017<br />has_binding_factor: FALSE","count: 13<br />citation_year: 2018<br />has_binding_factor: FALSE","count: 25<br />citation_year: 2019<br />has_binding_factor: FALSE","count:  9<br />citation_year: 2020<br />has_binding_factor: FALSE"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"FALSE","legendgroup":"FALSE","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091],"base":[0,0,0,0,0,0,0,0,0,0,0],"x":[2006,2010,2011,2013,2014,2016,2017,2018,2019,2020,2021],"y":[1,1,1,4,1,4,8,15,60,43,1],"text":["count:  1<br />citation_year: 2006<br />has_binding_factor:  TRUE","count:  1<br />citation_year: 2010<br />has_binding_factor:  TRUE","count:  1<br />citation_year: 2011<br />has_binding_factor:  TRUE","count:  4<br />citation_year: 2013<br />has_binding_factor:  TRUE","count:  1<br />citation_year: 2014<br />has_binding_factor:  TRUE","count:  4<br />citation_year: 2016<br />has_binding_factor:  TRUE","count:  8<br />citation_year: 2017<br />has_binding_factor:  TRUE","count: 15<br />citation_year: 2018<br />has_binding_factor:  TRUE","count: 60<br />citation_year: 2019<br />has_binding_factor:  TRUE","count: 43<br />citation_year: 2020<br />has_binding_factor:  TRUE","count:  1<br />citation_year: 2021<br />has_binding_factor:  TRUE"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"TRUE","legendgroup":"TRUE","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":40.1826484018265,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Structures of nucleosomes by year","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[1995.305,2022.695],"tickmode":"array","ticktext":["2000","2005","2010","2015","2020"],"tickvals":[2000,2005,2010,2015,2020],"categoryorder":"array","categoryarray":["2000","2005","2010","2015","2020"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Publication year","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-4.25,89.25],"tickmode":"array","ticktext":["0","20","40","60","80"],"tickvals":[0,20,40,60,80],"categoryorder":"array","categoryarray":["0","20","40","60","80"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Contains a binding factor","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a379d33f2":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a379d33f2","visdat":{"406a379d33f2":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-structures-by-year-binding-factor.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
</div>
<div id="nucleosome-structures-by-experimental-method" class="section level2">
<h2>Nucleosome structures by experimental method</h2>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>nucleosome_structures_method <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> experimental_method, <span class="at">fill =</span> experimental_method)) <span class="sc">+</span></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Experimental method&quot;</span>)) <span class="sc">+</span></span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of nucleosomes by experimental method&quot;</span>) <span class="sc">+</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;&quot;</span>) <span class="sc">+</span></span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_structures_method)</span></code></pre></div>
<div id="htmlwidget-865d48868f39d8c86f41" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-865d48868f39d8c86f41">{"x":{"data":[{"orientation":"v","width":0.9,"base":0,"x":[1],"y":[188],"text":"count: 188<br />experimental_method: X-ray diffraction<br />experimental_method: X-ray diffraction","type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"X-ray diffraction","legendgroup":"X-ray diffraction","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[2],"y":[145],"text":"count: 145<br />experimental_method: Electron Microscopy<br />experimental_method: Electron Microscopy","type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Electron Microscopy","legendgroup":"Electron Microscopy","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":25.5707762557078,"l":43.1050228310502},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Structures of nucleosomes by experimental method","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["X-ray diffraction","Electron Microscopy"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["X-ray diffraction","Electron Microscopy"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-9.4,197.4],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[0,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Experimental method","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a74e97c65":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a74e97c65","visdat":{"406a74e97c65":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-structures-by-method.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
<div id="nucleosome-structures-by-presence-of-a-binding-factor" class="section level2">
<h2>Nucleosome structures by presence of a binding factor</h2>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>nucleosome_binding_factors_methods <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> experimental_method, <span class="at">fill =</span> has_binding_factor)) <span class="sc">+</span></span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Presence of a binding factor&quot;</span>)) <span class="sc">+</span></span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of nucleosomes by presence of a binding factor&quot;</span>) <span class="sc">+</span></span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Experimental method&quot;</span>) <span class="sc">+</span></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_binding_factors_methods)</span></code></pre></div>
<div id="htmlwidget-2e26b97d96f0d922f13d" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-2e26b97d96f0d922f13d">{"x":{"data":[{"orientation":"v","width":[0.9,0.9],"base":[25,118],"x":[1,2],"y":[163,27],"text":["count: 163<br />experimental_method: X-ray diffraction<br />has_binding_factor: FALSE","count:  27<br />experimental_method: Electron Microscopy<br />has_binding_factor: FALSE"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"FALSE","legendgroup":"FALSE","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.9,0.9],"base":[0,0],"x":[1,2],"y":[25,118],"text":["count:  25<br />experimental_method: X-ray diffraction<br />has_binding_factor:  TRUE","count: 118<br />experimental_method: Electron Microscopy<br />has_binding_factor:  TRUE"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"TRUE","legendgroup":"TRUE","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":40.1826484018265,"l":43.1050228310502},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Structures of nucleosomes by presence of a binding factor","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["X-ray diffraction","Electron Microscopy"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["X-ray diffraction","Electron Microscopy"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Experimental method","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-9.4,197.4],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[0,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Presence of a binding factor","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a281bd15e":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a281bd15e","visdat":{"406a281bd15e":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-binding-factors-methods.svg"><strong>Download figure in SVG format</strong></a></p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>nucleosome_binding_factors <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> has_binding_factor, <span class="at">fill =</span> has_binding_factor)) <span class="sc">+</span></span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Presence of a binding factor&quot;</span>)) <span class="sc">+</span></span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of nucleosomes by presence of a binding factor&quot;</span>) <span class="sc">+</span></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Presence of a binding factor&quot;</span>) <span class="sc">+</span></span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_binding_factors)</span></code></pre></div>
<div id="htmlwidget-6cfda6ff776f30e2791a" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-6cfda6ff776f30e2791a">{"x":{"data":[{"orientation":"v","width":0.9,"base":0,"x":[1],"y":[190],"text":"count: 190<br />has_binding_factor: FALSE<br />has_binding_factor: FALSE","type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"FALSE","legendgroup":"FALSE","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[2],"y":[143],"text":"count: 143<br />has_binding_factor:  TRUE<br />has_binding_factor:  TRUE","type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"TRUE","legendgroup":"TRUE","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":40.1826484018265,"l":43.1050228310502},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Structures of nucleosomes by presence of a binding factor","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["FALSE","TRUE"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["FALSE","TRUE"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Presence of a binding factor","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-9.5,199.5],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[0,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Presence of a binding factor","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a6126df1c":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a6126df1c","visdat":{"406a6126df1c":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-binding-factors.svg"><strong>Download figure in SVG format</strong></a></p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>nucleosome_binding_factors_methods_2 <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> has_binding_factor, <span class="at">fill =</span> experimental_method)) <span class="sc">+</span></span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Experimental method&quot;</span>)) <span class="sc">+</span></span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of nucleosomes by presence of a binding factor&quot;</span>) <span class="sc">+</span></span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Presence of a binding factor&quot;</span>) <span class="sc">+</span></span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_binding_factors_methods_2)</span></code></pre></div>
<div id="htmlwidget-807ccf7f4cbc77b1ddb1" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-807ccf7f4cbc77b1ddb1">{"x":{"data":[{"orientation":"v","width":[0.9,0.9],"base":[27,118],"x":[1,2],"y":[163,25],"text":["count: 163<br />has_binding_factor: FALSE<br />experimental_method: X-ray diffraction","count:  25<br />has_binding_factor:  TRUE<br />experimental_method: X-ray diffraction"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"X-ray diffraction","legendgroup":"X-ray diffraction","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.9,0.9],"base":[0,0],"x":[1,2],"y":[27,118],"text":["count:  27<br />has_binding_factor: FALSE<br />experimental_method: Electron Microscopy","count: 118<br />has_binding_factor:  TRUE<br />experimental_method: Electron Microscopy"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Electron Microscopy","legendgroup":"Electron Microscopy","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":40.1826484018265,"l":43.1050228310502},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Structures of nucleosomes by presence of a binding factor","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["FALSE","TRUE"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["FALSE","TRUE"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Presence of a binding factor","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-9.5,199.5],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[0,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Experimental method","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a7758b2e8":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a7758b2e8","visdat":{"406a7758b2e8":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-binding-factors-methods-2.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
<div id="nucleosome-structures-by-histone-species" class="section level2">
<h2>Nucleosome structures by histone species</h2>
<div id="crystal-structures-of-nucleosomes-by-histone-species" class="section level3">
<h3>Crystal structures of nucleosomes by histone species</h3>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>nucleosome_xtal_species <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(experimental_method <span class="sc">==</span> <span class="st">&quot;X-ray diffraction&quot;</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">is_histone =</span> <span class="fu">str_detect</span>(molecule_name, <span class="at">pattern =</span> <span class="st">&quot;Histone H&quot;</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(is_histone <span class="sc">==</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">organism_scientific_name =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(organism_scientific_name))) <span class="sc">%&gt;%</span> </span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> organism_scientific_name,</span>
<span id="cb8-8"><a href="#cb8-8" aria-hidden="true" tabindex="-1"></a>                           <span class="at">fill =</span> organism_scientific_name)) <span class="sc">+</span></span>
<span id="cb8-9"><a href="#cb8-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Histone species&quot;</span>)) <span class="sc">+</span></span>
<span id="cb8-10"><a href="#cb8-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Crystal structures of nucleosomes by histone species&quot;</span>) <span class="sc">+</span></span>
<span id="cb8-11"><a href="#cb8-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;&quot;</span>) <span class="sc">+</span></span>
<span id="cb8-12"><a href="#cb8-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb8-13"><a href="#cb8-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb8-14"><a href="#cb8-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">30</span>, <span class="at">hjust =</span> <span class="dv">1</span>))</span>
<span id="cb8-15"><a href="#cb8-15" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_xtal_species)</span></code></pre></div>
<div id="htmlwidget-cb5abd41fea8f1f957e0" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-cb5abd41fea8f1f957e0">{"x":{"data":[{"orientation":"v","width":0.9,"base":0,"x":[1],"y":[2],"text":"count:  2<br />organism_scientific_name: Drosophila melanogaster<br />organism_scientific_name: Drosophila melanogaster","type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Drosophila melanogaster","legendgroup":"Drosophila melanogaster","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[2],"y":[1],"text":"count:  1<br />organism_scientific_name: Gallus gallus<br />organism_scientific_name: Gallus gallus","type":"bar","marker":{"autocolorscale":false,"color":"rgba(196,154,0,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Gallus gallus","legendgroup":"Gallus gallus","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[3],"y":[80],"text":"count: 80<br />organism_scientific_name: Homo sapiens<br />organism_scientific_name: Homo sapiens","type":"bar","marker":{"autocolorscale":false,"color":"rgba(83,180,0,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Homo sapiens","legendgroup":"Homo sapiens","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[4],"y":[4],"text":"count:  4<br />organism_scientific_name: Mus musculus<br />organism_scientific_name: Mus musculus","type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,192,148,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Mus musculus","legendgroup":"Mus musculus","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[5],"y":[53],"text":"count: 53<br />organism_scientific_name: Xenopus laevis<br />organism_scientific_name: Xenopus laevis","type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,182,235,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Xenopus laevis","legendgroup":"Xenopus laevis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[6],"y":[5],"text":"count:  5<br />organism_scientific_name: Arabidopsis thaliana<br />organism_scientific_name: Arabidopsis thaliana","type":"bar","marker":{"autocolorscale":false,"color":"rgba(165,138,255,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Arabidopsis thaliana","legendgroup":"Arabidopsis thaliana","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[7],"y":[1],"text":"count:  1<br />organism_scientific_name: Saccharomyces cerevisiae S288C<br />organism_scientific_name: Saccharomyces cerevisiae S288C","type":"bar","marker":{"autocolorscale":false,"color":"rgba(251,97,215,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Saccharomyces cerevisiae S288C","legendgroup":"Saccharomyces cerevisiae S288C","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":25.5707762557078,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Crystal structures of nucleosomes by histone species","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,7.6],"tickmode":"array","ticktext":["Drosophila melanogaster","Gallus gallus","Homo sapiens","Mus musculus","Xenopus laevis","Arabidopsis thaliana","Saccharomyces cerevisiae S288C"],"tickvals":[1,2,3,4,5,6,7],"categoryorder":"array","categoryarray":["Drosophila melanogaster","Gallus gallus","Homo sapiens","Mus musculus","Xenopus laevis","Arabidopsis thaliana","Saccharomyces cerevisiae S288C"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-30,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-4,84],"tickmode":"array","ticktext":["0","20","40","60","80"],"tickvals":[0,20,40,60,80],"categoryorder":"array","categoryarray":["0","20","40","60","80"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Histone species","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a20ec2bea":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a20ec2bea","visdat":{"406a20ec2bea":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-xtal-structures-by-histone-species.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
<div id="cryo-em-structures-of-nucleosomes-by-histone-species" class="section level3">
<h3>Cryo-EM structures of nucleosomes by histone species</h3>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>nucleosome_cryoem_species <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(experimental_method <span class="sc">==</span> <span class="st">&quot;Electron Microscopy&quot;</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">is_histone =</span> <span class="fu">str_detect</span>(molecule_name, <span class="at">pattern =</span> <span class="st">&quot;Histone H&quot;</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(is_histone <span class="sc">==</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">organism_scientific_name =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(organism_scientific_name))) <span class="sc">%&gt;%</span> </span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> organism_scientific_name,</span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a>                           <span class="at">fill =</span> organism_scientific_name)) <span class="sc">+</span></span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Histone species&quot;</span>)) <span class="sc">+</span></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Cryo-EM structures of nucleosomes by histone species&quot;</span>) <span class="sc">+</span></span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;&quot;</span>) <span class="sc">+</span></span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">30</span>, <span class="at">hjust =</span> <span class="dv">1</span>))</span>
<span id="cb9-15"><a href="#cb9-15" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_cryoem_species)</span></code></pre></div>
<div id="htmlwidget-32f25edcbcbd686399d5" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-32f25edcbcbd686399d5">{"x":{"data":[{"orientation":"v","width":0.9,"base":0,"x":[1],"y":[54],"text":"count: 54<br />organism_scientific_name: Xenopus laevis<br />organism_scientific_name: Xenopus laevis","type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Xenopus laevis","legendgroup":"Xenopus laevis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[2],"y":[38],"text":"count: 38<br />organism_scientific_name: Homo sapiens<br />organism_scientific_name: Homo sapiens","type":"bar","marker":{"autocolorscale":false,"color":"rgba(124,174,0,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Homo sapiens","legendgroup":"Homo sapiens","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[3],"y":[2],"text":"count:  2<br />organism_scientific_name: Saccharomyces cerevisiae S288C<br />organism_scientific_name: Saccharomyces cerevisiae S288C","type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Saccharomyces cerevisiae S288C","legendgroup":"Saccharomyces cerevisiae S288C","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[4],"y":[3],"text":"count:  3<br />organism_scientific_name: Drosophila melanogaster<br />organism_scientific_name: Drosophila melanogaster","type":"bar","marker":{"autocolorscale":false,"color":"rgba(199,124,255,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Drosophila melanogaster","legendgroup":"Drosophila melanogaster","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":25.5707762557078,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Cryo-EM structures of nucleosomes by histone species","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,4.6],"tickmode":"array","ticktext":["Xenopus laevis","Homo sapiens","Saccharomyces cerevisiae S288C","Drosophila melanogaster"],"tickvals":[1,2,3,4],"categoryorder":"array","categoryarray":["Xenopus laevis","Homo sapiens","Saccharomyces cerevisiae S288C","Drosophila melanogaster"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-30,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-2.7,56.7],"tickmode":"array","ticktext":["0","20","40"],"tickvals":[0,20,40],"categoryorder":"array","categoryarray":["0","20","40"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Histone species","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a2ca6d3c4":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a2ca6d3c4","visdat":{"406a2ca6d3c4":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-cryoem-structures-by-histone-species.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
</div>
<div id="resolution-of-nucleosome-structures-by-histone-species" class="section level2">
<h2>Resolution of nucleosome structures by histone species</h2>
<div id="crystal-structures" class="section level3">
<h3>Crystal structures</h3>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>pdb_data <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">filter</span>(experimental_method <span class="sc">==</span> <span class="st">&quot;X-ray diffraction&quot;</span>) <span class="sc">%&gt;%</span> .<span class="sc">$</span>resolution <span class="sc">%&gt;%</span> <span class="fu">summary</span>()</span></code></pre></div>
<pre><code>##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##   1.580   2.450   2.750   2.898   3.000   9.700</code></pre>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>nucleosome_xtal_resolution_species <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(experimental_method <span class="sc">==</span> <span class="st">&quot;X-ray diffraction&quot;</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">is_histone =</span> <span class="fu">str_detect</span>(molecule_name, <span class="at">pattern =</span> <span class="st">&quot;Histone H&quot;</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(is_histone <span class="sc">==</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">organism_scientific_name =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(organism_scientific_name))) <span class="sc">%&gt;%</span> </span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">select</span>(resolution, organism_scientific_name) <span class="sc">%&gt;%</span> </span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb12-8"><a href="#cb12-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_histogram</span>(<span class="fu">aes</span>(<span class="at">x =</span> resolution, <span class="at">fill =</span> organism_scientific_name),</span>
<span id="cb12-9"><a href="#cb12-9" aria-hidden="true" tabindex="-1"></a>                   <span class="at">binwidth =</span> <span class="fl">0.2</span>) <span class="sc">+</span></span>
<span id="cb12-10"><a href="#cb12-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Histone species&quot;</span>)) <span class="sc">+</span></span>
<span id="cb12-11"><a href="#cb12-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Resolution of nucleosome crystal structures by histone species&quot;</span>) <span class="sc">+</span></span>
<span id="cb12-12"><a href="#cb12-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Resolution (Å)&quot;</span>) <span class="sc">+</span></span>
<span id="cb12-13"><a href="#cb12-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb12-14"><a href="#cb12-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb12-15"><a href="#cb12-15" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_xtal_resolution_species)</span></code></pre></div>
<div id="htmlwidget-f0a846384868f2533f84" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-f0a846384868f2533f84">{"x":{"data":[{"orientation":"v","width":[0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003],"base":[2,1,6,15,23,25,24,14,13,8,4,2,1,0,1,1,0,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1],"x":[1.6,1.8,2,2.2,2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6],"y":[0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  1<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  1<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster","count:  0<br />resolution:    NA<br />organism_scientific_name: Drosophila melanogaster"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Drosophila melanogaster","legendgroup":"Drosophila melanogaster","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003],"base":[2,1,6,15,22,25,24,14,13,8,4,2,1,0,1,1,0,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1],"x":[1.6,1.8,2,2.2,2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6],"y":[0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  1<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus","count:  0<br />resolution: 2.500<br />organism_scientific_name: Gallus gallus"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(196,154,0,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Gallus gallus","legendgroup":"Gallus gallus","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003],"base":[2,1,3,5,11,11,9,4,8,4,0,1,0,0,1,0,0,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1.6,1.8,2,2.2,2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6],"y":[0,0,3,10,11,14,15,10,5,4,4,1,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1],"text":["count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  3<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count: 10<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count: 11<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count: 14<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count: 15<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count: 10<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  5<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  4<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  4<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  1<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  1<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  1<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  0<br />resolution:    NA<br />organism_scientific_name: Homo sapiens","count:  1<br />resolution:    NA<br />organism_scientific_name: Homo sapiens"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(83,180,0,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Homo sapiens","legendgroup":"Homo sapiens","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003],"base":[2,1,3,5,9,11,8,4,8,3,0,1,0,0,1,0,0,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1.6,1.8,2,2.2,2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6],"y":[0,0,0,0,2,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  2<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  1<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  1<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus","count:  0<br />resolution:    NA<br />organism_scientific_name: Mus musculus"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,192,148,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Mus musculus","legendgroup":"Mus musculus","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003],"base":[2,1,1,0,0,0,0,1,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1.6,1.8,2,2.2,2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6],"y":[0,0,2,5,9,11,8,3,7,3,0,1,0,0,1,0,0,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  2<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  5<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  9<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count: 11<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  8<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  3<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  7<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  3<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  1<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  1<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  1<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  1<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  1<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis","count:  0<br />resolution:    NA<br />organism_scientific_name: Xenopus laevis"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,182,235,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Xenopus laevis","legendgroup":"Xenopus laevis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003],"base":[0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1.6,1.8,2,2.2,2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6],"y":[2,1,1,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count:  2<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  1<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  1<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  1<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana","count:  0<br />resolution:    NA<br />organism_scientific_name: Arabidopsis thaliana"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(165,138,255,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Arabidopsis thaliana","legendgroup":"Arabidopsis thaliana","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.2,0.2,0.2,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1.6,1.8,2,2.2,2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6],"y":[0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  1<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count:  0<br />resolution: 3.203<br />organism_scientific_name: Saccharomyces cerevisiae S288C"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(251,97,215,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Saccharomyces cerevisiae S288C","legendgroup":"Saccharomyces cerevisiae S288C","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":40.1826484018265,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Resolution of nucleosome crystal structures by histone species","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[1.09,10.11],"tickmode":"array","ticktext":["2.5","5.0","7.5","10.0"],"tickvals":[2.5,5,7.5,10],"categoryorder":"array","categoryarray":["2.5","5.0","7.5","10.0"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Resolution (Å)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1.25,26.25],"tickmode":"array","ticktext":["0","5","10","15","20","25"],"tickvals":[0,5,10,15,20,25],"categoryorder":"array","categoryarray":["0","5","10","15","20","25"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Histone species","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a1e669de9":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a1e669de9","visdat":{"406a1e669de9":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-xtal-resolution-by-histone-species.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
<div id="cryo-em-structures" class="section level3">
<h3>Cryo-EM structures</h3>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a>pdb_data <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">filter</span>(experimental_method <span class="sc">==</span> <span class="st">&quot;Electron Microscopy&quot;</span>) <span class="sc">%&gt;%</span> .<span class="sc">$</span>resolution <span class="sc">%&gt;%</span> <span class="fu">summary</span>()</span></code></pre></div>
<pre><code>##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##   2.500   3.500   4.000   4.734   4.610  20.000</code></pre>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>nucleosome_cryoem_resolution_species <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(experimental_method <span class="sc">==</span> <span class="st">&quot;Electron Microscopy&quot;</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">is_histone =</span> <span class="fu">str_detect</span>(molecule_name, <span class="at">pattern =</span> <span class="st">&quot;Histone H&quot;</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(is_histone <span class="sc">==</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb15-5"><a href="#cb15-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">organism_scientific_name =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(organism_scientific_name))) <span class="sc">%&gt;%</span> </span>
<span id="cb15-6"><a href="#cb15-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">select</span>(resolution, organism_scientific_name) <span class="sc">%&gt;%</span> </span>
<span id="cb15-7"><a href="#cb15-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb15-8"><a href="#cb15-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_histogram</span>(<span class="fu">aes</span>(<span class="at">x =</span> resolution, <span class="at">fill =</span> organism_scientific_name),</span>
<span id="cb15-9"><a href="#cb15-9" aria-hidden="true" tabindex="-1"></a>                   <span class="at">binwidth =</span> <span class="fl">0.2</span>) <span class="sc">+</span></span>
<span id="cb15-10"><a href="#cb15-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Histone species&quot;</span>)) <span class="sc">+</span></span>
<span id="cb15-11"><a href="#cb15-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Resolution of nucleosome cryo-EM structures by histone species&quot;</span>) <span class="sc">+</span></span>
<span id="cb15-12"><a href="#cb15-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Resolution (Å)&quot;</span>) <span class="sc">+</span></span>
<span id="cb15-13"><a href="#cb15-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb15-14"><a href="#cb15-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb15-15"><a href="#cb15-15" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(nucleosome_cryoem_resolution_species)</span></code></pre></div>
<div id="htmlwidget-137cc4272dc6bc6b208a" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-137cc4272dc6bc6b208a">{"x":{"data":[{"orientation":"v","width":[0.2,0.199999999999999,0.2,0.2,0.199999999999999,0.2,0.199999999999999,0.2,0.2,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.200000000000001,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003],"base":[0,3,1,4,2,5,4,5,6,3,4,0,2,1,0,0,0,0,0,0,0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1],"x":[2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6,9.8,10,10.2,10.4,10.6,10.8,11,11.2,11.4,11.6,11.8,12,12.2,12.4,12.6,12.8,13,13.2,13.4,13.6,13.8,14,14.2,14.4,14.6,14.8,15,15.2,15.4,15.6,15.8,16,16.2,16.4,16.6,16.8,17,17.2,17.4,17.6,17.8,18,18.2,18.4,18.6,18.8],"y":[1,0,0,4,5,6,7,8,6,0,5,2,2,1,0,1,0,0,0,1,0,0,0,0,2,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count: 1<br />resolution:  2.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  2.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  2.8<br />organism_scientific_name: Xenopus laevis","count: 4<br />resolution:  3.0<br />organism_scientific_name: Xenopus laevis","count: 5<br />resolution:  3.2<br />organism_scientific_name: Xenopus laevis","count: 6<br />resolution:  3.4<br />organism_scientific_name: Xenopus laevis","count: 7<br />resolution:  3.6<br />organism_scientific_name: Xenopus laevis","count: 8<br />resolution:  3.8<br />organism_scientific_name: Xenopus laevis","count: 6<br />resolution:  4.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  4.2<br />organism_scientific_name: Xenopus laevis","count: 5<br />resolution:  4.4<br />organism_scientific_name: Xenopus laevis","count: 2<br />resolution:  4.6<br />organism_scientific_name: Xenopus laevis","count: 2<br />resolution:  4.8<br />organism_scientific_name: Xenopus laevis","count: 1<br />resolution:  5.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  5.2<br />organism_scientific_name: Xenopus laevis","count: 1<br />resolution:  5.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  5.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  5.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  6.0<br />organism_scientific_name: Xenopus laevis","count: 1<br />resolution:  6.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  6.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  6.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  6.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  7.0<br />organism_scientific_name: Xenopus laevis","count: 2<br />resolution:  7.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  7.4<br />organism_scientific_name: Xenopus laevis","count: 1<br />resolution:  7.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  7.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  8.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  8.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  8.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  8.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  8.8<br />organism_scientific_name: Xenopus laevis","count: 1<br />resolution:  9.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  9.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  9.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  9.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution:  9.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 10.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 10.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 10.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 10.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 10.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 11.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 11.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 11.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 11.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 11.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 12.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 12.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 12.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 12.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 12.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 13.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 13.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 13.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 13.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 13.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 14.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 14.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 14.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 14.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 14.8<br />organism_scientific_name: Xenopus laevis","count: 1<br />resolution: 15.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 15.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 15.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 15.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 15.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 16.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 16.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 16.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 16.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 16.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 17.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 17.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 17.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 17.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 17.8<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 18.0<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 18.2<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 18.4<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 18.6<br />organism_scientific_name: Xenopus laevis","count: 0<br />resolution: 18.8<br />organism_scientific_name: Xenopus laevis"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Xenopus laevis","legendgroup":"Xenopus laevis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.199999999999999,0.2,0.2,0.199999999999999,0.2,0.199999999999999,0.2,0.2,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.200000000000001,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003],"base":[0,1,0,1,0,0,1,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6,9.8,10,10.2,10.4,10.6,10.8,11,11.2,11.4,11.6,11.8,12,12.2,12.4,12.6,12.8,13,13.2,13.4,13.6,13.8,14,14.2,14.4,14.6,14.8,15,15.2,15.4,15.6,15.8,16,16.2,16.4,16.6,16.8,17,17.2,17.4,17.6,17.8,18,18.2,18.4,18.6,18.8],"y":[0,2,1,3,2,5,3,5,4,3,4,0,2,1,0,0,0,0,0,0,0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1],"text":["count: 0<br />resolution:  2.4<br />organism_scientific_name: Homo sapiens","count: 2<br />resolution:  2.6<br />organism_scientific_name: Homo sapiens","count: 1<br />resolution:  2.8<br />organism_scientific_name: Homo sapiens","count: 3<br />resolution:  3.0<br />organism_scientific_name: Homo sapiens","count: 2<br />resolution:  3.2<br />organism_scientific_name: Homo sapiens","count: 5<br />resolution:  3.4<br />organism_scientific_name: Homo sapiens","count: 3<br />resolution:  3.6<br />organism_scientific_name: Homo sapiens","count: 5<br />resolution:  3.8<br />organism_scientific_name: Homo sapiens","count: 4<br />resolution:  4.0<br />organism_scientific_name: Homo sapiens","count: 3<br />resolution:  4.2<br />organism_scientific_name: Homo sapiens","count: 4<br />resolution:  4.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  4.6<br />organism_scientific_name: Homo sapiens","count: 2<br />resolution:  4.8<br />organism_scientific_name: Homo sapiens","count: 1<br />resolution:  5.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  5.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  5.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  5.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  5.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  6.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  6.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  6.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  6.6<br />organism_scientific_name: Homo sapiens","count: 1<br />resolution:  6.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  7.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  7.2<br />organism_scientific_name: Homo sapiens","count: 1<br />resolution:  7.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  7.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  7.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  8.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  8.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  8.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  8.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  8.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  9.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  9.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  9.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  9.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution:  9.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 10.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 10.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 10.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 10.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 10.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 11.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 11.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 11.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 11.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 11.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 12.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 12.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 12.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 12.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 12.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 13.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 13.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 13.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 13.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 13.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 14.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 14.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 14.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 14.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 14.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 15.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 15.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 15.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 15.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 15.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 16.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 16.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 16.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 16.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 16.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 17.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 17.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 17.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 17.6<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 17.8<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 18.0<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 18.2<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 18.4<br />organism_scientific_name: Homo sapiens","count: 0<br />resolution: 18.6<br />organism_scientific_name: Homo sapiens","count: 1<br />resolution: 18.8<br />organism_scientific_name: Homo sapiens"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(124,174,0,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Homo sapiens","legendgroup":"Homo sapiens","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.199999999999999,0.2,0.2,0.199999999999999,0.2,0.199999999999999,0.2,0.2,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.200000000000001,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003],"base":[0,0,0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6,9.8,10,10.2,10.4,10.6,10.8,11,11.2,11.4,11.6,11.8,12,12.2,12.4,12.6,12.8,13,13.2,13.4,13.6,13.8,14,14.2,14.4,14.6,14.8,15,15.2,15.4,15.6,15.8,16,16.2,16.4,16.6,16.8,17,17.2,17.4,17.6,17.8,18,18.2,18.4,18.6,18.8],"y":[0,1,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count: 0<br />resolution:  2.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 1<br />resolution:  2.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  2.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  3.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  3.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  3.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 1<br />resolution:  3.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  3.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  4.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  4.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  4.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  4.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  4.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  5.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  5.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  5.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  5.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  5.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  6.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  6.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  6.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  6.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  6.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  7.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  7.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  7.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  7.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  7.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  8.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  8.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  8.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  8.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  8.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  9.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  9.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  9.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  9.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution:  9.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 10.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 10.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 10.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 10.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 10.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 11.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 11.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 11.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 11.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 11.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 12.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 12.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 12.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 12.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 12.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 13.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 13.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 13.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 13.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 13.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 14.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 14.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 14.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 14.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 14.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 15.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 15.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 15.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 15.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 15.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 16.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 16.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 16.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 16.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 16.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 17.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 17.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 17.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 17.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 17.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 18.0<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 18.2<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 18.4<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 18.6<br />organism_scientific_name: Saccharomyces cerevisiae S288C","count: 0<br />resolution: 18.8<br />organism_scientific_name: Saccharomyces cerevisiae S288C"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,196,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Saccharomyces cerevisiae S288C","legendgroup":"Saccharomyces cerevisiae S288C","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":[0.2,0.199999999999999,0.2,0.2,0.199999999999999,0.2,0.199999999999999,0.2,0.2,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.200000000000001,0.199999999999999,0.199999999999999,0.200000000000001,0.199999999999999,0.200000000000002,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.199999999999999,0.199999999999999,0.200000000000003,0.199999999999999,0.200000000000003,0.200000000000001,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003,0.200000000000003],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,4.8,5,5.2,5.4,5.6,5.8,6,6.2,6.4,6.6,6.8,7,7.2,7.4,7.6,7.8,8,8.2,8.4,8.6,8.8,9,9.2,9.4,9.6,9.8,10,10.2,10.4,10.6,10.8,11,11.2,11.4,11.6,11.8,12,12.2,12.4,12.6,12.8,13,13.2,13.4,13.6,13.8,14,14.2,14.4,14.6,14.8,15,15.2,15.4,15.6,15.8,16,16.2,16.4,16.6,16.8,17,17.2,17.4,17.6,17.8,18,18.2,18.4,18.6,18.8],"y":[0,0,0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"text":["count: 0<br />resolution:  2.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  2.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  2.8<br />organism_scientific_name: Drosophila melanogaster","count: 1<br />resolution:  3.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  3.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  3.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  3.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  3.8<br />organism_scientific_name: Drosophila melanogaster","count: 2<br />resolution:  4.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  4.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  4.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  4.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  4.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  5.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  5.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  5.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  5.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  5.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  6.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  6.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  6.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  6.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  6.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  7.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  7.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  7.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  7.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  7.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  8.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  8.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  8.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  8.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  8.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  9.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  9.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  9.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  9.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution:  9.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 10.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 10.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 10.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 10.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 10.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 11.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 11.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 11.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 11.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 11.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 12.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 12.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 12.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 12.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 12.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 13.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 13.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 13.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 13.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 13.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 14.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 14.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 14.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 14.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 14.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 15.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 15.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 15.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 15.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 15.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 16.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 16.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 16.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 16.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 16.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 17.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 17.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 17.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 17.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 17.8<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 18.0<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 18.2<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 18.4<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 18.6<br />organism_scientific_name: Drosophila melanogaster","count: 0<br />resolution: 18.8<br />organism_scientific_name: Drosophila melanogaster"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(199,124,255,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Drosophila melanogaster","legendgroup":"Drosophila melanogaster","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":40.1826484018265,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Resolution of nucleosome cryo-EM structures by histone species","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[1.47,19.73],"tickmode":"array","ticktext":["5","10","15"],"tickvals":[5,10,15],"categoryorder":"array","categoryarray":["5","10","15"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Resolution (Å)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.65,13.65],"tickmode":"array","ticktext":["0","5","10"],"tickvals":[0,5,10],"categoryorder":"array","categoryarray":["0","5","10"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Histone species","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"406a78c91213":{"x":{},"fill":{},"type":"bar"}},"cur_data":"406a78c91213","visdat":{"406a78c91213":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/nucleosome-cryoem-resolution-by-histone-species.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
</div>
<div id="dataset" class="section level2">
<h2>Dataset</h2>
<p>The graphs presented above are derived from the following dataset:</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Format table for display</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>pdb_table <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">arrange</span>(<span class="fu">desc</span>(citation_year)) <span class="sc">%&gt;%</span> </span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">select</span>(<span class="st">`</span><span class="at">PDB code</span><span class="st">`</span> <span class="ot">=</span> pdb_id,</span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a>           <span class="st">`</span><span class="at">Citation year</span><span class="st">`</span> <span class="ot">=</span> citation_year,</span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a>           <span class="st">`</span><span class="at">Experimental method</span><span class="st">`</span> <span class="ot">=</span> experimental_method,</span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a>           <span class="at">Title =</span> title)</span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>pdb_table</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["PDB code"],"name":[1],"type":["chr"],"align":["left"]},{"label":["Citation year"],"name":[2],"type":["int"],"align":["right"]},{"label":["Experimental method"],"name":[3],"type":["fct"],"align":["left"]},{"label":["Title"],"name":[4],"type":["chr"],"align":["left"]}],"data":[{"1":"6wkr","2":"2021","3":"Electron Microscopy","4":"PRC2-AEBP2-JARID2 bound to H2AK119ub1 nucleosome"},{"1":"7a08","2":"2020","3":"Electron Microscopy","4":"CryoEM Structure of cGAS Nucleosome complex"},{"1":"6y5e","2":"2020","3":"Electron Microscopy","4":"Structure of human cGAS (K394E) bound to the nucleosome (focused refinement of cGAS-NCP subcomplex)"},{"1":"6usj","2":"2020","3":"Electron Microscopy","4":"Structure of two nucleosomes bridged by human PARP2"},{"1":"6ven","2":"2020","3":"Electron Microscopy","4":"Yeast COMPASS in complex with a ubiquitinated nucleosome"},{"1":"6t79","2":"2020","3":"Electron Microscopy","4":"Structure of a human nucleosome at 3.2 A resolution"},{"1":"7crr","2":"2020","3":"Electron Microscopy","4":"Native NSD3 bound to 187-bp nucleosome"},{"1":"6x59","2":"2020","3":"Electron Microscopy","4":"The mouse cGAS catalytic domain binding to human assembled nucleosome"},{"1":"7bp6","2":"2020","3":"X-ray diffraction","4":"Structural insights into nucleosome reorganization by NAP1-RELATED PROTEIN 1 (NRP1)"},{"1":"7bp4","2":"2020","3":"X-ray diffraction","4":"Structural insights into nucleosome reorganization by NAP1-RELATED PROTEIN 1 (NRP1)"},{"1":"7at8","2":"2020","3":"Electron Microscopy","4":"Histone H3 recognition by nucleosome-bound PRC2 subunit EZH2."},{"1":"7bp2","2":"2020","3":"X-ray diffraction","4":"Structural mechanism directing nucleosome reorganization by NAP1-RELATED PROTEIN 1 (NRP1)"},{"1":"6xjd","2":"2020","3":"Electron Microscopy","4":"Two mouse cGAS catalytic domain binding to human assembled nucleosome"},{"1":"7c0m","2":"2020","3":"Electron Microscopy","4":"Human cGAS-nucleosome complex"},{"1":"6vz4","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of Sth1-Arp7-Arp9-Rtt102 bound to the nucleosome in ADP Beryllium Fluoride state"},{"1":"6yov","2":"2020","3":"Electron Microscopy","4":"OCT4-SOX2-bound nucleosome - SHL+6"},{"1":"6tda","2":"2020","3":"Electron Microscopy","4":"Structure of SWI/SNF chromatin remodeler RSC bound to a nucleosome"},{"1":"6vyp","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the LSD1/CoREST histone demethylase bound to its nucleosome substrate"},{"1":"6uph","2":"2020","3":"Electron Microscopy","4":"Structure of a Yeast Centromeric Nucleosome at 2.7 Angstrom resolution"},{"1":"6uxw","2":"2020","3":"Electron Microscopy","4":"SWI/SNF nucleosome complex with ADP-BeFx"},{"1":"6ryu","2":"2020","3":"Electron Microscopy","4":"Nucleosome-CHD4 complex structure (two CHD4 copies)"},{"1":"6le9","2":"2020","3":"X-ray diffraction","4":"The Human Telomeric Nucleosome Displays Distinct Structural and Dynamic Properties"},{"1":"6m44","2":"2020","3":"X-ray diffraction","4":"355 bp di-nucleosome harboring cohesive DNA termini (high cryoprotectant)"},{"1":"6la9","2":"2020","3":"X-ray diffraction","4":"349 bp di-nucleosome harboring cohesive DNA termini assembled with linker histone H1.0 (high cryoprotectant)"},{"1":"6la8","2":"2020","3":"X-ray diffraction","4":"349 bp di-nucleosome harboring cohesive DNA termini assembled with linker histone H1.0"},{"1":"6ltj","2":"2020","3":"Electron Microscopy","4":"Structure of nucleosome-bound human BAF complex"},{"1":"6m3v","2":"2020","3":"X-ray diffraction","4":"355 bp di-nucleosome harboring cohesive DNA termini"},{"1":"6l9h","2":"2020","3":"X-ray diffraction","4":"The Human Telomeric Nucleosome Displays Distinct Structural and Dynamic Properties"},{"1":"6ke9","2":"2020","3":"X-ray diffraction","4":"The Human Telomeric Nucleosome Displays Distinct Structural and Dynamic Properties"},{"1":"6pa7","2":"2020","3":"Electron Microscopy","4":"The cryo-EM structure of the human DNMT3A2-DNMT3B3 complex bound to nucleosome."},{"1":"6t7c","2":"2020","3":"Electron Microscopy","4":"Structure of two copies of human Sox11 transcription factor in complex with a nucleosome"},{"1":"6t93","2":"2020","3":"Electron Microscopy","4":"Nucleosome with OCT4-SOX2 motif at SHL-6"},{"1":"6t9l","2":"2020","3":"Electron Microscopy","4":"SAGA DUB module bound to a ubiqitinated nucleosome"},{"1":"6tem","2":"2020","3":"Electron Microscopy","4":"CENP-A nucleosome core particle with 145 base pairs of the Widom 601 sequence by cryo-EM"},{"1":"6t7b","2":"2020","3":"Electron Microscopy","4":"Structure of human Sox2 transcription factor in complex with a nucleosome"},{"1":"6t7a","2":"2020","3":"Electron Microscopy","4":"Structure of human Sox11 transcription factor in complex with a nucleosome"},{"1":"6ryr","2":"2020","3":"Electron Microscopy","4":"Nucleosome-CHD4 complex structure (single CHD4 copy)"},{"1":"6t7d","2":"2020","3":"Electron Microscopy","4":"Structure of human Sox11 transcription factor in complex with a nucleosome"},{"1":"6t90","2":"2020","3":"Electron Microscopy","4":"OCT4-SOX2-bound nucleosome - SHL-6"},{"1":"7ccr","2":"2020","3":"Electron Microscopy","4":"Structure of the 2:2 cGAS-nucleosome complex"},{"1":"7joa","2":"2020","3":"Electron Microscopy","4":"2:1 cGAS-nucleosome complex"},{"1":"7ccq","2":"2020","3":"Electron Microscopy","4":"Structure of the 1:1 cGAS-nucleosome complex"},{"1":"6y5d","2":"2020","3":"Electron Microscopy","4":"Structure of human cGAS (K394E) bound to the nucleosome"},{"1":"6zhx","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the regulatory linker of ALC1 bound to the nucleosome's acidic patch: nucleosome class."},{"1":"7c7x","2":"2020","3":"X-ray diffraction","4":"Structural insights into nucleosome reorganization by NAP1-RELATED PROTEIN 1 (NRP1)"},{"1":"7k61","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of 197bp nucleosome aided by scFv"},{"1":"7bp5","2":"2020","3":"X-ray diffraction","4":"Structural insights into nucleosome reorganization by NAP1-RELATED PROTEIN 1 (NRP1)"},{"1":"7jo9","2":"2020","3":"Electron Microscopy","4":"1:1 cGAS-nucleosome complex"},{"1":"6x5a","2":"2020","3":"Electron Microscopy","4":"The mouse cGAS catalytic domain binding to human nucleosome that purified from HEK293T cells"},{"1":"6zhy","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the regulatory linker of ALC1 bound to the nucleosome's acidic patch: hexasome class."},{"1":"7cro","2":"2020","3":"Electron Microscopy","4":"NSD2 bearing E1099K/T1150A dual mutation in complex with 187-bp NCP"},{"1":"7crp","2":"2020","3":"Electron Microscopy","4":"NSD3 bearing E1181K/T1232A dual mutation in complex with 187-bp NCP (1:1 binding mode)"},{"1":"7crq","2":"2020","3":"Electron Microscopy","4":"NSD3 bearing E1181K/T1232A dual mutation in complex with 187-bp NCP (2:1 binding mode)"},{"1":"5z3u","2":"2019","3":"Electron Microscopy","4":"Structure of Snf2-nucleosome complex at shl2 in ADP BeFx state"},{"1":"5zbx","2":"2019","3":"X-ray diffraction","4":"The crystal structure of the nucleosome containing histone H3.1 CATD(V76Q, K77D)"},{"1":"6upk","2":"2019","3":"Electron Microscopy","4":"Structure of FACT_subnucleosome complex 1"},{"1":"6uh5","2":"2019","3":"Electron Microscopy","4":"Structural basis of COMPASS eCM recognition of the H2Bub nucleosome"},{"1":"6iy2","2":"2019","3":"Electron Microscopy","4":"Structure of Snf2-MMTV-A nucleosome complex at shl2 in ADP state"},{"1":"6iy3","2":"2019","3":"Electron Microscopy","4":"Structure of Snf2-MMTV-A nucleosome complex at shl-2 in ADP state"},{"1":"6e0p","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of the centromeric nucleosome (Native alpha satellite DNA) in complex with a single chain antibody fragment"},{"1":"5z23","2":"2019","3":"X-ray diffraction","4":"Crystal structure of the nucleosome containing a chimeric histone H3/CENP-A CATD"},{"1":"5z3v","2":"2019","3":"Electron Microscopy","4":"Structure of Snf2-nucleosome complex at shl-2 in ADP BeFx state"},{"1":"5z3o","2":"2019","3":"Electron Microscopy","4":"Structure of Snf2-nucleosome complex in ADP state"},{"1":"5z3l","2":"2019","3":"Electron Microscopy","4":"Structure of Snf2-nucleosome complex in apo state"},{"1":"6j4x","2":"2019","3":"Electron Microscopy","4":"RNA polymerase II elongation complex bound with Elf1 and Spt4/5, stalled at SHL(-1) of the nucleosome (+1A)"},{"1":"6jou","2":"2019","3":"X-ray diffraction","4":"Crystal structure of the human nucleosome containing H2A.Z.1 S42R"},{"1":"6pwf","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of the ATPase domain of chromatin remodeling factor ISWI bound to the nucleosome"},{"1":"6kvd","2":"2019","3":"X-ray diffraction","4":"Crystal structure of human nucleosome containing H2A.J"},{"1":"6iro","2":"2019","3":"Electron Microscopy","4":"the crosslinked complex of ISWI-nucleosome in the ADP-bound state"},{"1":"6jr0","2":"2019","3":"X-ray diffraction","4":"Crystal structure of the human nucleosome phased with 12 selenium atoms"},{"1":"6k1p","2":"2019","3":"Electron Microscopy","4":"The complex of ISWI-nucleosome in the ADP.BeF-bound state"},{"1":"6muo","2":"2019","3":"Electron Microscopy","4":"CENP-A nucleosome bound by two copies of CENP-C(CD) and one copy CENP-N(NT)"},{"1":"6mup","2":"2019","3":"Electron Microscopy","4":"CENP-A nucleosome bound by two copies of CENP-C(CD) and two copies CENP-N(NT)"},{"1":"6jr1","2":"2019","3":"X-ray diffraction","4":"Crystal structure of the human nucleosome phased with 16 selenium atoms"},{"1":"6k1k","2":"2019","3":"X-ray diffraction","4":"Human nucleosome core particle with H2A.X S139E variant"},{"1":"6nzo","2":"2019","3":"Electron Microscopy","4":"Set2 bound to nucleosome"},{"1":"6jma","2":"2019","3":"Electron Microscopy","4":"cryo-EM structure of DOT1L bound to H2B ubiquitinated nucleosome"},{"1":"6iq4","2":"2019","3":"X-ray diffraction","4":"Nucleosome core particle cross-linked with a hetero-binuclear molecule possessing RAPTA and gold(I) 4-(diphenylphosphino)benzoic acid groups."},{"1":"6j99","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of human DOT1L in complex with an H2B-monoubiquitinated nucleosome"},{"1":"6j4w","2":"2019","3":"Electron Microscopy","4":"RNA polymerase II elongation complex bound with Elf1 and Spt4/5, stalled at SHL(-5) of the nucleosome"},{"1":"6j4z","2":"2019","3":"Electron Microscopy","4":"RNA polymerase II elongation complex bound with Spt4/5 and foreign DNA, stalled at SHL(-1) of the nucleosome"},{"1":"6kxv","2":"2019","3":"X-ray diffraction","4":"Crystal structure of a nucleosome containing Leishmania histone H3"},{"1":"6k1j","2":"2019","3":"X-ray diffraction","4":"Human nucleosome core particle with H2A.X variant"},{"1":"6l4a","2":"2019","3":"Electron Microscopy","4":"H3-H3-H3 tri-nucleosome with the 22 base-pair linker DNA"},{"1":"6jyl","2":"2019","3":"Electron Microscopy","4":"The crosslinked complex of ISWI-nucleosome in the ADP.BeF-bound state"},{"1":"6ipu","2":"2019","3":"X-ray diffraction","4":"Human nucleosome core particle containing 145 bp of DNA"},{"1":"6j50","2":"2019","3":"Electron Microscopy","4":"RNA polymerase II elongation complex bound with Spt4/5 and foreign DNA, stalled at SHL(-1) of the nucleosome (tilted conformation)"},{"1":"6j4y","2":"2019","3":"Electron Microscopy","4":"RNA polymerase II elongation complex bound with Elf1 and Spt4/5, stalled at SHL(-1) of the nucleosome (+1B)"},{"1":"6pww","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of MLL1 in complex with RbBP5 and WDR5 bound to the nucleosome"},{"1":"6nj9","2":"2019","3":"Electron Microscopy","4":"Active state Dot1L bound to the H2B-Ubiquitinated nucleosome, 2-to-1 complex"},{"1":"6k1i","2":"2019","3":"X-ray diffraction","4":"Human nucleosome core particle with gammaH2A.X variant"},{"1":"6ne3","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of singly-bound SNF2h-nucleosome complex with SNF2h bound at SHL-2"},{"1":"6jm9","2":"2019","3":"Electron Microscopy","4":"cryo-EM structure of DOT1L bound to unmodified nucleosome"},{"1":"6j51","2":"2019","3":"Electron Microscopy","4":"RNA polymerase II elongation complex bound with Spt4/5 and foreign DNA, stalled at SHL(-1) of the nucleosome, weak Elf1 (+1 position)"},{"1":"6ir9","2":"2019","3":"Electron Microscopy","4":"RNA polymerase II elongation complex bound with Elf1 and Spt4/5, stalled at SHL(-1) of the nucleosome"},{"1":"6px1","2":"2019","3":"Electron Microscopy","4":"Set2 bound to nucleosome"},{"1":"6dzt","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of nucleosome in complex with a single chain antibody fragment"},{"1":"6l49","2":"2019","3":"Electron Microscopy","4":"H3-CA-H3 tri-nucleosome with the 22 base-pair linker DNA"},{"1":"6jxd","2":"2019","3":"X-ray diffraction","4":"Human nucleosome core particle with cohesive end DNA termini"},{"1":"6kw3","2":"2019","3":"Electron Microscopy","4":"The ClassA RSC-Nucleosome Complex"},{"1":"6nn6","2":"2019","3":"Electron Microscopy","4":"Structure of Dot1L-H2BK120ub nucleosome complex"},{"1":"6px3","2":"2019","3":"Electron Microscopy","4":"Set2 bound to nucleosome"},{"1":"6pwv","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of MLL1 core complex bound to the nucleosome"},{"1":"6r1u","2":"2019","3":"Electron Microscopy","4":"Structure of LSD2/NPAC-linker/nucleosome core particle complex: Class 2"},{"1":"6see","2":"2019","3":"Electron Microscopy","4":"Class2A : CENP-A nucleosome in complex with CENP-C central region"},{"1":"6se0","2":"2019","3":"Electron Microscopy","4":"Class 1 : CENP-A nucleosome"},{"1":"6nog","2":"2019","3":"Electron Microscopy","4":"Poised-state Dot1L bound to the H2B-Ubiquitinated nucleosome"},{"1":"6pwx","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of RbBP5 bound to the nucleosome"},{"1":"6ugm","2":"2019","3":"Electron Microscopy","4":"Structural basis of COMPASS eCM recognition of an unmodified nucleosome"},{"1":"6r1t","2":"2019","3":"Electron Microscopy","4":"Structure of LSD2/NPAC-linker/nucleosome core particle complex: Class 1, free nuclesome"},{"1":"6upl","2":"2019","3":"Electron Microscopy","4":"Structure of FACT_subnucleosome complex 2"},{"1":"6r25","2":"2019","3":"Electron Microscopy","4":"Structure of LSD2/NPAC-linker/nucleosome core particle complex: Class 3"},{"1":"6s01","2":"2019","3":"Electron Microscopy","4":"Structure of LEDGF PWWP domain bound H3K36 methylated nucleosome"},{"1":"6seg","2":"2019","3":"Electron Microscopy","4":"Class1: CENP-A nucleosome in complex with CENP-C central region"},{"1":"6pwe","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of nucleosome core particle"},{"1":"6rny","2":"2019","3":"Electron Microscopy","4":"PFV intasome - nucleosome strand transfer complex"},{"1":"6o1d","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of the centromeric nucleosome with native alpha satellite DNA"},{"1":"6o96","2":"2019","3":"Electron Microscopy","4":"Dot1L bound to the H2BK120 Ubiquitinated nucleosome"},{"1":"6kw4","2":"2019","3":"Electron Microscopy","4":"The ClassB RSC-Nucleosome Complex"},{"1":"6e0c","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of the CENP-A nucleosome (W601) in complex with a single chain antibody fragment"},{"1":"6nqa","2":"2019","3":"Electron Microscopy","4":"Active state Dot1L bound to the H2B-Ubiquitinated nucleosome, 1-to-1 complex"},{"1":"6r0c","2":"2019","3":"Electron Microscopy","4":"Human-D02 Nucleosome Core Particle with biotin-streptavidin label"},{"1":"6om3","2":"2019","3":"X-ray diffraction","4":"Crystal structure of the Orc1 BAH domain in complex with a nucleosome core particle"},{"1":"6sef","2":"2019","3":"Electron Microscopy","4":"Class2C : CENP-A nucleosome in complex with CENP-C central region"},{"1":"6se6","2":"2019","3":"Electron Microscopy","4":"Class2 : CENP-A nucleosome in complex with CENP-C central region"},{"1":"6kiz","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of human MLL1-NCP complex, binding mode2"},{"1":"6kix","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of human MLL1-NCP complex, binding mode1"},{"1":"6kiu","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of human MLL1-ubNCP complex (3.2 angstrom)"},{"1":"6kiv","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of human MLL1-ubNCP complex (4.0 angstrom)"},{"1":"6kiw","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of human MLL3-ubNCP complex (4.0 angstrom)"},{"1":"6r8z","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of NCP_THF2(-1)-UV-DDB"},{"1":"6r93","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of NCP-6-4PP"},{"1":"6r90","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of NCP-THF2(+1)-UV-DDB class A"},{"1":"6r94","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of NCP_THF2(-3)"},{"1":"6r91","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of NCP_THF2(-3)-UV-DDB"},{"1":"6r92","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of NCP-THF2(+1)-UV-DDB class B"},{"1":"6r8y","2":"2019","3":"Electron Microscopy","4":"Cryo-EM structure of NCP-6-4PP(-1)-UV-DDB"},{"1":"6v92","2":"2019","3":"Electron Microscopy","4":"RSC-NCP"},{"1":"6a5t","2":"2018","3":"Electron Microscopy","4":"RNA polymerase II elongation complex stalled at SHL(-1) of the nucleosome"},{"1":"5y0c","2":"2018","3":"X-ray diffraction","4":"Crystal Structure of the human nucleosome at 2.09 angstrom resolution"},{"1":"6a5u","2":"2018","3":"Electron Microscopy","4":"RNA polymerase II elongation complex stalled at SHL(-1) of the nucleosome, with foreign DNA, tilt conformation"},{"1":"5xm1","2":"2018","3":"X-ray diffraction","4":"The mouse nucleosome structure containing H2A, H2B type3-A, H3mm7, and H4"},{"1":"5y0d","2":"2018","3":"X-ray diffraction","4":"Crystal Structure of the human nucleosome containing the H2B E76K mutant"},{"1":"6hkt","2":"2018","3":"X-ray diffraction","4":"Structure of an H1-bound 6-nucleosome array"},{"1":"6esf","2":"2018","3":"Electron Microscopy","4":"Nucleosome : Class 1"},{"1":"6fml","2":"2018","3":"Electron Microscopy","4":"CryoEM Structure INO80core Nucleosome complex"},{"1":"5wcu","2":"2018","3":"X-ray diffraction","4":"Crystal structure of 167 bp nucleosome bound to the globular domain of linker histone H5"},{"1":"5z30","2":"2018","3":"X-ray diffraction","4":"The crystal structure of the nucleosome containing a cancer-associated histone H2A.Z R80C mutant"},{"1":"6esh","2":"2018","3":"Electron Microscopy","4":"Nucleosome breathing : Class 3"},{"1":"6a5l","2":"2018","3":"Electron Microscopy","4":"RNA polymerase II elongation complex stalled at SHL(-1) of the nucleosome, with foreign DNA"},{"1":"6a5r","2":"2018","3":"Electron Microscopy","4":"RNA polymerase II elongation complex stalled at SHL(-2) of the nucleosome"},{"1":"6fq5","2":"2018","3":"Electron Microscopy","4":"Class 1 : canonical nucleosome"},{"1":"5xm0","2":"2018","3":"X-ray diffraction","4":"The mouse nucleosome structure containing H2A, H2B type3-A, H3.3, and H4"},{"1":"6a5p","2":"2018","3":"Electron Microscopy","4":"RNA polymerase II elongation complex stalled at SHL(-5) of the nucleosome"},{"1":"6a5o","2":"2018","3":"Electron Microscopy","4":"RNA polymerase II elongation complex stalled at SHL(-6) of the nucleosome"},{"1":"6fq8","2":"2018","3":"Electron Microscopy","4":"Class 3 : translocated nucleosome"},{"1":"6esi","2":"2018","3":"Electron Microscopy","4":"Nucleosome breathing : Class 4"},{"1":"6gej","2":"2018","3":"Electron Microscopy","4":"Chromatin remodeller-nucleosome complex at 3.6 A resolution."},{"1":"6g0l","2":"2018","3":"Electron Microscopy","4":"Structure of two molecules of the chromatin remodelling enzyme Chd1 bound to a nucleosome"},{"1":"6gen","2":"2018","3":"Electron Microscopy","4":"Chromatin remodeller-nucleosome complex at 4.5 A resolution."},{"1":"6inq","2":"2018","3":"Electron Microscopy","4":"RNA polymerase II elongation complex stalled at SHL(-1) of the nucleosome, with foreign DNA (+1 position)"},{"1":"6hts","2":"2018","3":"Electron Microscopy","4":"Cryo-EM structure of the human INO80 complex bound to nucleosome"},{"1":"6i84","2":"2018","3":"Electron Microscopy","4":"Structure of transcribing RNA polymerase II-nucleosome complex"},{"1":"6esg","2":"2018","3":"Electron Microscopy","4":"Nucleosome breathing : Class 2"},{"1":"6ftx","2":"2018","3":"Electron Microscopy","4":"Structure of the chromatin remodelling enzyme Chd1 bound to a ubiquitinylated nucleosome"},{"1":"6fq6","2":"2018","3":"Electron Microscopy","4":"Class 2 : distorted nucleosome"},{"1":"5xf5","2":"2017","3":"X-ray diffraction","4":"Nucleosome core particle with an adduct of a binuclear RAPTA (Ru-arene-phosphaadamantane) compound having a 1,2-diphenylethylenediamine linker (R,S-configuration)"},{"1":"5x0y","2":"2017","3":"Electron Microscopy","4":"Complex of Snf2-Nucleosome complex with Snf2 bound to SHL2 of the nucleosome"},{"1":"5x0x","2":"2017","3":"Electron Microscopy","4":"Complex of Snf2-Nucleosome complex with Snf2 bound to position +6 of the nucleosome"},{"1":"5xf6","2":"2017","3":"X-ray diffraction","4":"Nucleosome core particle with an adduct of a binuclear RAPTA (Ru-arene-phosphaadamantane) compound having an ethylenediamine linker"},{"1":"5xf3","2":"2017","3":"X-ray diffraction","4":"Nucleosome core particle with an adduct of a binuclear RAPTA (Ru-arene-phosphaadamantane) compound having a 1,2-diphenylethylenediamine linker (R,R-configuration)"},{"1":"6c0w","2":"2017","3":"Electron Microscopy","4":"Cryo-EM structure of human kinetochore protein CENP-N with the centromeric nucleosome containing CENP-A"},{"1":"5omx","2":"2017","3":"X-ray diffraction","4":"X-ray Structure of the H2A-N38C Nucleosome Core Particle"},{"1":"6buz","2":"2017","3":"Electron Microscopy","4":"Cryo-EM structure of CENP-A nucleosome in complex with kinetochore protein CENP-N"},{"1":"5x7x","2":"2017","3":"X-ray diffraction","4":"The crystal structure of the nucleosome containing H3.3 at 2.18 angstrom resolution"},{"1":"5oy7","2":"2017","3":"X-ray diffraction","4":"Structure of the 4_601_157 tetranucleosome (P1 form)"},{"1":"5b1l","2":"2017","3":"X-ray diffraction","4":"The mouse nucleosome structure containing H3t"},{"1":"5gt0","2":"2017","3":"X-ray diffraction","4":"Crystal structure of nucleosome complex with human testis-specific histone variants, Th2a"},{"1":"5gtc","2":"2017","3":"X-ray diffraction","4":"Crystal structure of complex between DMAP-SH conjugated with a Kaposi's sarcoma herpesvirus LANA peptide (5-15) and nucleosome core particle"},{"1":"5gt3","2":"2017","3":"X-ray diffraction","4":"Crystal structure of nucleosome particle in the presence of human testis-specific histone variant, hTh2b"},{"1":"5gxq","2":"2017","3":"X-ray diffraction","4":"The crystal structure of the nucleosome containing H3.6"},{"1":"5dnm","2":"2017","3":"X-ray diffraction","4":"Nucleosome core particle containing adducts of ruthenium(II)-toluene PTA complex"},{"1":"5gse","2":"2017","3":"X-ray diffraction","4":"Crystal structure of unusual nucleosome"},{"1":"5jrg","2":"2017","3":"X-ray diffraction","4":"Crystal structure of the nucleosome containing the DNA with tetrahydrofuran (THF)"},{"1":"5gsu","2":"2017","3":"X-ray diffraction","4":"Crystal structure of nucleosome core particle consisting of human testis-specific histone variants, Th2A and Th2B"},{"1":"5b1m","2":"2017","3":"X-ray diffraction","4":"The mouse nucleosome structure containing H3.1"},{"1":"5dnn","2":"2017","3":"X-ray diffraction","4":"Nucleosome core particle containing adducts of gold(I)-triethylphosphane and ruthenium(II)-toluene PTA complexes"},{"1":"5onw","2":"2017","3":"X-ray diffraction","4":"X-Ray crystal structure of a nucleosome core particle with its DNA site-specifically crosslinked to the histone octamer and the two H2A/H2B dimers crosslinked via H2A N38C"},{"1":"5mlu","2":"2017","3":"X-ray diffraction","4":"Crystal structure of the PFV GAG CBS bound to a mononucleosome"},{"1":"5oxv","2":"2017","3":"X-ray diffraction","4":"Structure of the 4_601_157 tetranucleosome (C2 form)"},{"1":"5ong","2":"2017","3":"X-ray diffraction","4":"X-Ray crystal structure of a nucleosome core particle with its DNA site-specifically crosslinked to the histone octamer"},{"1":"5o9g","2":"2017","3":"Electron Microscopy","4":"Structure of nucleosome-Chd1 complex"},{"1":"5nl0","2":"2017","3":"X-ray diffraction","4":"Crystal structure of a 197-bp palindromic 601L nucleosome in complex with linker histone H1"},{"1":"5xf4","2":"2017","3":"X-ray diffraction","4":"Nucleosome core particle with an adduct of a binuclear RAPTA (Ru-arene-phosphaadamantane) compound having a 1,2-diphenylethylenediamine linker (S,S-configuration)"},{"1":"4z5t","2":"2016","3":"X-ray diffraction","4":"The nucleosome containing human H3.5"},{"1":"5f99","2":"2016","3":"X-ray diffraction","4":"X-ray Structure of the MMTV-A Nucleosome Core Particle"},{"1":"5b2j","2":"2016","3":"X-ray diffraction","4":"Human nucleosome containing CpG methylated DNA"},{"1":"5b32","2":"2016","3":"X-ray diffraction","4":"The crystal structure of the heterotypic H2AZ/H2A nucleosome with H3.3."},{"1":"5ay8","2":"2016","3":"X-ray diffraction","4":"Crystal structure of human nucleosome containing H3.Y"},{"1":"5b0y","2":"2016","3":"X-ray diffraction","4":"Crystal structure of the nucleosome containing histone H3 with the crotonylated lysine 122"},{"1":"4zux","2":"2016","3":"X-ray diffraction","4":"SAGA DUB module Ubp8/Sgf11/Sus1/Sgf73 bound to ubiqitinated nucleosome"},{"1":"5kgf","2":"2016","3":"Electron Microscopy","4":"Structural model of 53BP1 bound to a ubiquitylated and methylated nucleosome, at 4.5 A resolution"},{"1":"5cp6","2":"2016","3":"X-ray diffraction","4":"Nucleosome Core Particle with Adducts from the Anticancer Compound, [(eta6-5,8,9,10-tetrahydroanthracene)Ru(ethylenediamine)Cl][PF6]"},{"1":"5hq2","2":"2016","3":"X-ray diffraction","4":"Structural model of Set8 histone H4 Lys20 methyltransferase bound to nucleosome core particle"},{"1":"5e5a","2":"2016","3":"X-ray diffraction","4":"Crystal structure of the chromatin-tethering domain of Human cytomegalovirus IE1 protein bound to the nucleosome core particle"},{"1":"5b0z","2":"2016","3":"X-ray diffraction","4":"The crystal structure of the nucleosome containing H3.2, at 1.98 A resolution"},{"1":"5b2i","2":"2016","3":"X-ray diffraction","4":"Human nucleosome containing CpG unmethylated DNA"},{"1":"5b31","2":"2016","3":"X-ray diffraction","4":"The crystal structure of the heterotypic H2AZ/H2A nucleosome with H3.1."},{"1":"5b24","2":"2016","3":"X-ray diffraction","4":"The crystal structure of the nucleosome containing cyclobutane pyrimidine dimer"},{"1":"5b33","2":"2016","3":"X-ray diffraction","4":"The crystal structure of the H2AZ nucleosome with H3.3."},{"1":"4xzq","2":"2015","3":"X-ray diffraction","4":"Nucleosome disassembly by RSC and SWI/SNF is enhanced by H3 acetylation near the nucleosome dyad axis"},{"1":"5av9","2":"2015","3":"X-ray diffraction","4":"human nucleosome core particle"},{"1":"5avb","2":"2015","3":"X-ray diffraction","4":"human nucleosome core particle"},{"1":"4ys3","2":"2015","3":"X-ray diffraction","4":"Nucleosome disassembly by RSC and SWI/SNF is enhanced by H3 acetylation near the nucleosome dyad axis"},{"1":"5cpj","2":"2015","3":"X-ray diffraction","4":"Nucleosome containing methylated Sat2R DNA"},{"1":"4ym5","2":"2015","3":"X-ray diffraction","4":"Crystal structure of the human nucleosome containing 6-4PP (inside)"},{"1":"3x1u","2":"2015","3":"X-ray diffraction","4":"Crystal structure of nucleosome core particle in the presence of histone variants involved in reprogramming"},{"1":"3x1t","2":"2015","3":"X-ray diffraction","4":"Crystal structure of nucleosome core particle consisting of mouse testis specific histone variants H2aa and H2ba"},{"1":"3x1v","2":"2015","3":"X-ray diffraction","4":"Crystal structure of nucleosome core particle in the presence of histone variant involved in reprogramming"},{"1":"5cpk","2":"2015","3":"X-ray diffraction","4":"Nucleosome containing methylated Sat2L DNA"},{"1":"5cpi","2":"2015","3":"X-ray diffraction","4":"Nucleosome containing unmethylated Sat2R DNA"},{"1":"5avc","2":"2015","3":"X-ray diffraction","4":"human nucleosome core particle"},{"1":"5av8","2":"2015","3":"X-ray diffraction","4":"human nucleosome core particle"},{"1":"4z66","2":"2015","3":"X-ray diffraction","4":"Nucleosome disassembly by RSC and SWI/SNF is enhanced by H3 acetylation near the nucleosome dyad axis"},{"1":"5av6","2":"2015","3":"X-ray diffraction","4":"human nucleosome core particle"},{"1":"5av5","2":"2015","3":"X-ray diffraction","4":"human nucleosome core particle"},{"1":"4ym6","2":"2015","3":"X-ray diffraction","4":"Crystal structure of the human nucleosome containing 6-4PP (outside)"},{"1":"3x1s","2":"2015","3":"X-ray diffraction","4":"Crystal structure of the nucleosome core particle"},{"1":"4wu9","2":"2015","3":"X-ray diffraction","4":"Structure of cisPtNAP-NCP145"},{"1":"4wu8","2":"2015","3":"X-ray diffraction","4":"Structure of trPtNAP-NCP145"},{"1":"3wtp","2":"2014","3":"X-ray diffraction","4":"Crystal Structure of the heterotypic nucleosome containing human CENP-A and H3.3"},{"1":"4kgc","2":"2014","3":"X-ray diffraction","4":"Nucleosome Core Particle Containing (ETA6-P-CYMENE)-(1, 2-ETHYLENEDIAMINE)-RUTHENIUM"},{"1":"4r8p","2":"2014","3":"X-ray diffraction","4":"Crystal structure of the Ring1B/Bmi1/UbcH5c PRC1 ubiquitylation module bound to the nucleosome core particle"},{"1":"3wkj","2":"2014","3":"X-ray diffraction","4":"The nucleosome containing human TSH2B"},{"1":"4x23","2":"2013","3":"X-ray diffraction","4":"CRYSTAL STRUCTURE OF CENP-C IN COMPLEX WITH THE NUCLEOSOME CORE PARTICLE"},{"1":"3wa9","2":"2013","3":"X-ray diffraction","4":"The nucleosome containing human H2A.Z.1"},{"1":"3waa","2":"2013","3":"X-ray diffraction","4":"The nucleosome containing human H2A.Z.2"},{"1":"3w98","2":"2013","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle lacking H3.1 N-terminal region"},{"1":"3w99","2":"2013","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle lacking H4 N-terminal region"},{"1":"4kud","2":"2013","3":"X-ray diffraction","4":"Crystal structure of N-terminal acetylated Sir3 BAH domain D205N mutant in complex with yeast nucleosome core particle"},{"1":"4ld9","2":"2013","3":"X-ray diffraction","4":"Crystal structure of the N-terminally acetylated BAH domain of Sir3 bound to the nucleosome core particle"},{"1":"4jjn","2":"2013","3":"X-ray diffraction","4":"Crystal structure of heterochromatin protein Sir3 in complex with a silenced yeast nucleosome"},{"1":"3w97","2":"2013","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle lacking H2B N-terminal region"},{"1":"3w96","2":"2013","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle lacking H2A N-terminal region"},{"1":"4j8u","2":"2013","3":"X-ray diffraction","4":"X-ray structure of NCP145 with chlorido(eta-6-p-cymene)(N-phenyl-2-pyridinecarbothioamide)osmium(II)"},{"1":"4j8w","2":"2013","3":"X-ray diffraction","4":"X-ray structure of NCP145 with chlorido(eta-6-p-cymene)(N-fluorophenyl-2-pyridinecarbothioamide)osmium(II)"},{"1":"4j8v","2":"2013","3":"X-ray diffraction","4":"X-ray structure of NCP145 with bound chlorido(eta-6-p-cymene)(N-phenyl-2-pyridinecarbothioamide)ruthenium(II)"},{"1":"4j8x","2":"2013","3":"X-ray diffraction","4":"X-ray structure of NCP145 with bound chlorido(eta-6-p-cymene)(N-fluorophenyl-2-pyridinecarbothioamide)ruthenium(II)"},{"1":"3uta","2":"2012","3":"X-ray diffraction","4":"Crystal Structure of Nucleosome Core Particle Assembled with an Alpha-Satellite Sequence Containing Two TTAAA elements (NCP-TA2)"},{"1":"3ut9","2":"2012","3":"X-ray diffraction","4":"Crystal Structure of Nucleosome Core Particle Assembled with a Palindromic Widom '601' Derivative (NCP-601L)"},{"1":"3utb","2":"2012","3":"X-ray diffraction","4":"Crystal Structure of Nucleosome Core Particle Assembled with the 146b Alpha-Satellite Sequence (NCP146b)"},{"1":"3tu4","2":"2011","3":"X-ray diffraction","4":"Crystal structure of the Sir3 BAH domain in complex with a nucleosome core particle."},{"1":"3azh","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H3K122Q mutation"},{"1":"3av1","2":"2011","3":"X-ray diffraction","4":"The human nucleosome structure containing the histone variant H3.2"},{"1":"3azk","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H4K59Q mutation"},{"1":"3reh","2":"2011","3":"X-ray diffraction","4":"2.5 Angstrom Crystal Structure of the Nucleosome Core Particle Assembled with a 145 bp Alpha-Satellite DNA (NCP145)"},{"1":"3mnn","2":"2011","3":"X-ray diffraction","4":"A Ruthenium Antitumour Agent Forms Specific Histone Protein Adducts in the Nucleosome Core"},{"1":"3ayw","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H3K56Q mutation"},{"1":"3azi","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H4K31Q mutation"},{"1":"3azj","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H4K44Q mutation"},{"1":"3azm","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H4K79Q mutation"},{"1":"3azg","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H3K115Q mutation"},{"1":"3aze","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H3K64Q mutation"},{"1":"3azl","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H4K77Q mutation"},{"1":"3av2","2":"2011","3":"X-ray diffraction","4":"The human nucleosome structure containing the histone variant H3.3"},{"1":"3azf","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H3K79Q mutation"},{"1":"3an2","2":"2011","3":"X-ray diffraction","4":"The structure of the centromeric nucleosome containing CENP-A"},{"1":"3azn","2":"2011","3":"X-ray diffraction","4":"Crystal Structure of Human Nucleosome Core Particle Containing H4K91Q mutation"},{"1":"3rel","2":"2011","3":"X-ray diffraction","4":"2.7 Angstrom Crystal Structure of the Nucleosome Core Particle Assembled with a 146 bp Alpha-Satellite DNA (NCP146b) Derivatized with Triamminechloroplatinum(II) Chloride"},{"1":"3rek","2":"2011","3":"X-ray diffraction","4":"2.6 Angstrom Crystal Structure of the Nucleosome Core Particle Assembled with a 146 bp Alpha-Satellite DNA (NCP146b) Derivatized with Oxaliplatin"},{"1":"3rei","2":"2011","3":"X-ray diffraction","4":"2.65 Angstrom Crystal Structure of the Nucleosome Core Particle Assembled with a 145 bp Alpha-Satellite DNA (NCP145) Derivatized with Triamminechloroplatinum(II) Chloride"},{"1":"3rej","2":"2011","3":"X-ray diffraction","4":"2.55 Angstrom Crystal Structure of the Nucleosome Core Particle Assembled with a 146 bp Alpha-Satellite DNA (NCP146b)"},{"1":"3mgr","2":"2010","3":"X-ray diffraction","4":"Binding of Rubidium ions to the Nucleosome Core Particle"},{"1":"3lja","2":"2010","3":"X-ray diffraction","4":"Using Soft X-Rays for a Detailed Picture of Divalent Metal Binding in the Nucleosome"},{"1":"3lz0","2":"2010","3":"X-ray diffraction","4":"Crystal Structure of Nucleosome Core Particle Composed of the Widom 601 DNA Sequence (orientation 1)"},{"1":"3o62","2":"2010","3":"X-ray diffraction","4":"Nucleosome core particle modified with a cisplatin 1,3-cis-{Pt(NH3)2}2+-d(GpTpG) intrastrand cross-link"},{"1":"3mgs","2":"2010","3":"X-ray diffraction","4":"Binding of Cesium ions to the Nucleosome Core particle"},{"1":"3mgp","2":"2010","3":"X-ray diffraction","4":"Binding of Cobalt ions to the Nucleosome Core Particle"},{"1":"3lel","2":"2010","3":"X-ray diffraction","4":"Structural Insight into the Sequence-Dependence of Nucleosome Positioning"},{"1":"3kuy","2":"2010","3":"X-ray diffraction","4":"DNA Stretching in the Nucleosome Facilitates Alkylation by an Intercalating Antitumor Agent"},{"1":"3lz1","2":"2010","3":"X-ray diffraction","4":"Crystal Structure of Nucleosome Core Particle Composed of the Widom 601 DNA Sequence (orientation 2)"},{"1":"3kxb","2":"2010","3":"X-ray diffraction","4":"Structural characterization of H3K56Q nucleosomes and nucleosomal arrays"},{"1":"3mgq","2":"2010","3":"X-ray diffraction","4":"Binding of Nickel ions to the Nucleosome Core Particle"},{"1":"3mvd","2":"2010","3":"X-ray diffraction","4":"Crystal structure of the chromatin factor RCC1 in complex with the nucleosome core particle"},{"1":"3a6n","2":"2010","3":"X-ray diffraction","4":"The nucleosome containing a testis-specific histone variant, human H3T"},{"1":"3afa","2":"2010","3":"X-ray diffraction","4":"The human nucleosome structure"},{"1":"3b6f","2":"2008","3":"X-ray diffraction","4":"Nucleosome core particle treated with cisplatin"},{"1":"3c1b","2":"2008","3":"X-ray diffraction","4":"The effect of H3 K79 dimethylation and H4 K20 trimethylation on nucleosome and chromatin structure"},{"1":"2pyo","2":"2008","3":"X-ray diffraction","4":"Drosophila nucleosome core"},{"1":"3b6g","2":"2008","3":"X-ray diffraction","4":"Nucleosome core particle treated with oxaliplatin"},{"1":"3c1c","2":"2008","3":"X-ray diffraction","4":"The effect of H3 K79 dimethylation and H4 K20 trimethylation on nucleosome and chromatin structure"},{"1":"2nzd","2":"2007","3":"X-ray diffraction","4":"Nucleosome core particle containing 145 bp of DNA"},{"1":"1zla","2":"2006","3":"X-ray diffraction","4":"X-ray Structure of a Kaposi's sarcoma herpesvirus LANA peptide bound to the nucleosomal core"},{"1":"2fj7","2":"2006","3":"X-ray diffraction","4":"Crystal structure of Nucleosome Core Particle Containing a Poly (dA.dT) Sequence Element"},{"1":"1u35","2":"2005","3":"X-ray diffraction","4":"Crystal structure of the nucleosome core particle containing the histone domain of macroH2A"},{"1":"1zbb","2":"2005","3":"X-ray diffraction","4":"Structure of the 4_601_167 Tetranucleosome"},{"1":"2cv5","2":"2005","3":"X-ray diffraction","4":"Crystal structure of human nucleosome core particle"},{"1":"1p3p","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p3b","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p3f","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p34","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p3i","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1s32","2":"2004","3":"X-ray diffraction","4":"Molecular Recognition of the Nucleosomal 'Supergroove'"},{"1":"1p3a","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p3o","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p3l","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p3m","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p3g","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1p3k","2":"2004","3":"X-ray diffraction","4":"Crystallographic Studies of Nucleosome Core Particles containing Histone 'Sin' Mutants"},{"1":"1m18","2":"2003","3":"X-ray diffraction","4":"LIGAND BINDING ALTERS THE STRUCTURE AND DYNAMICS OF NUCLEOSOMAL DNA"},{"1":"1m19","2":"2003","3":"X-ray diffraction","4":"LIGAND BINDING ALTERS THE STRUCTURE AND DYNAMICS OF NUCLEOSOMAL DNA"},{"1":"1m1a","2":"2003","3":"X-ray diffraction","4":"LIGAND BINDING ALTERS THE STRUCTURE AND DYNAMICS OF NUCLEOSOMAL DNA"},{"1":"1kx5","2":"2002","3":"X-ray diffraction","4":"X-Ray Structure of the Nucleosome Core Particle, NCP147, at 1.9 A Resolution"},{"1":"1kx3","2":"2002","3":"X-ray diffraction","4":"X-Ray Structure of the Nucleosome Core Particle, NCP146, at 2.0 A Resolution"},{"1":"1kx4","2":"2002","3":"X-ray diffraction","4":"X-Ray Structure of the Nucleosome Core Particle, NCP146b, at 2.6 A Resolution"},{"1":"1id3","2":"2001","3":"X-ray diffraction","4":"CRYSTAL STRUCTURE OF THE YEAST NUCLEOSOME CORE PARTICLE REVEALS FUNDAMENTAL DIFFERENCES IN INTER-NUCLEOSOME INTERACTIONS"},{"1":"1eqz","2":"2000","3":"X-ray diffraction","4":"X-RAY STRUCTURE OF THE NUCLEOSOME CORE PARTICLE AT 2.5 A RESOLUTION"},{"1":"1f66","2":"2000","3":"X-ray diffraction","4":"2.6 A CRYSTAL STRUCTURE OF A NUCLEOSOME CORE PARTICLE CONTAINING THE VARIANT HISTONE H2A.Z"},{"1":"1aoi","2":"1997","3":"X-ray diffraction","4":"COMPLEX BETWEEN NUCLEOSOME CORE PARTICLE (H3,H4,H2A,H2B) AND 146 BP LONG DNA FRAGMENT"},{"1":"6v2k","2":"NA","3":"X-ray diffraction","4":"The nucleosome structure after H2A-H2B exchange"},{"1":"6m4h","2":"NA","3":"Electron Microscopy","4":"Structural mechanism of nucleosome dynamics governed by human histone variants H2A.B and H2A.Z.2.2"},{"1":"6m4g","2":"NA","3":"Electron Microscopy","4":"Structural mechanism of nucleosome dynamics governed by human histone variants H2A.B and H2A.Z.2.2"},{"1":"6kw5","2":"NA","3":"Electron Microscopy","4":"The ClassC RSC-Nucleosome Complex"},{"1":"6m4d","2":"NA","3":"Electron Microscopy","4":"Structural mechanism of nucleosome dynamics governed by human histone variants H2A.B and H2A.Z.2.2"},{"1":"7ksr","2":"NA","3":"Electron Microscopy","4":"PRC2:EZH1_A from a dimeric PRC2 bound to a nucleosome"},{"1":"7ktq","2":"NA","3":"Electron Microscopy","4":"Nucleosome from a dimeric PRC2 bound to a nucleosome"},{"1":"7ktp","2":"NA","3":"Electron Microscopy","4":"PRC2:EZH1_B from a dimeric PRC2 bound to a nucleosome"},{"1":"5b40","2":"NA","3":"X-ray diffraction","4":"The nucleosome structure containing H2B-K120 and H4-K31 monoubiquitinations"},{"1":"4xuj","2":"NA","3":"X-ray diffraction","4":"Nucleosome core particle containing adducts from treatment with a thiomorpholine-substituted [(eta-6-p-cymene)Ru(3-hydroxy-2-pyridone)Cl] compound"},{"1":"2f8n","2":"NA","3":"X-ray diffraction","4":"2.9 Angstrom X-ray structure of hybrid macroH2A nucleosomes"},{"1":"2nqb","2":"NA","3":"X-ray diffraction","4":"Drosophila Nucleosome Structure"},{"1":"3kwq","2":"NA","3":"X-ray diffraction","4":"Structural characterization of H3K56Q nucleosomes and nucleosomal arrays"},{"1":"6t7k","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of Prolyl-tRNA synthetase (ProRS, Proline--tRNA ligase) from Plasmodium falciparum in complex with NCP-26 and L-Proline"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p><a href="datasets/nucleosome-structures.json"><strong>Download raw dataset in JSON format</strong></a></p>
</div>

<div id="rmd-source-code">---
title: "Nucleosome structures"
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(message = FALSE, warning = FALSE)
```

**Last updated on `r date()`.**

## How to re-use this work

If you use the following figures in your own work, please cite:

- Zhou K, Gaullier G & Luger K (2019) Nucleosome structure and dynamics are
  coming of age. *Nature Structural & Molecular Biology* **26**: 3–13.
  <https://doi.org/10.1038/s41594-018-0166-x>
- This website: <https://doi.org/10.5281/zenodo.3470119>

```{r Load packages, download data and prepare dataset}
# Load required packages
library(magrittr)
library(dplyr)
library(purrr)
library(jsonlite)
library(forcats)
library(ggplot2)
library(plotly)
library(here)
library(stringr)

# We need two queries: one for uppercase titles, the other for lowercase ones
pdb_queries <- c(
    uppercase = 'https://www.ebi.ac.uk/pdbe/search/pdb/select?q=title:*UCLEOSOM*%20AND%20status:REL&fl=pdb_id,citation_year,title,experimental_method,resolution,organism_scientific_name,molecule_name,molecule_type,number_of_protein_chains&rows=1000000&wt=json',
    lowercase = 'https://www.ebi.ac.uk/pdbe/search/pdb/select?q=title:*ucleosom*%20AND%20status:REL&fl=pdb_id,citation_year,title,experimental_method,resolution,organism_scientific_name,molecule_name,molecule_type,number_of_protein_chains&rows=1000000&wt=json',
    ncp = 'https://www.ebi.ac.uk/pdbe/search/pdb/select?q=title:*NCP*%20AND%20status:REL&fl=pdb_id,citation_year,title,experimental_method,resolution,organism_scientific_name,molecule_name,molecule_type,number_of_protein_chains&rows=1000000&wt=json'
)

# The following PDB entries have the word "nucleosome in their title, but do not
# actually contain a nucleosome, so we need to exclude them from the analysis.
# Unfortunately, there is no good way to automate this.
non_nucleosome_structures <- c(
    "1hst",
    "2z2r",
    "5x7v",
    "3uv2",
    "3fs3",
    "1wg3",
    "1nw3",
    "5ikf",
    "3gyw",
    "3gyv",
    "1ofc",
    "2ayu",
    "2iw5",
    "3hfd",
    "6uch",
    "5r4k",
    "5r4m",
    "5r4l",
    "5r4o",
    "5r4g",
    "5r4h",
    "5r4j",
    "5r4i",
    "5r4n",
    "1bj6",
    "6qds",
    "2iwj",
    "4dvk",
    "1a6b",
    "6qdu",
    "6ne8",
    "1tsu",
    "1esk",
    "7c4j"
)

# This is a helper pipeline to extract data
dig_up_data <- . %>%
    .$response %>%
    .$docs %>%
    as_tibble()

# This is a helper function to detect the presence of a binding factor
has_binding_factor_one <- function(number_of_protein_chains, title) {
    # It takes 8 histone chains to make a nucleosome, so if the number of
    # protein chains is not divisible by 8, this means there is a binding factor,
    # unless we're seeing the overlapping dinucleosome (14 proteins chains, but
    # they are all histones... one octamer + one hexamer)
    oldn <- str_detect(title, "unusual")
    if (number_of_protein_chains %% 8 != 0 & !oldn) {
        return(TRUE)
    } else {
        # But for multiple of 8 chains > 8 (i.e. 16, 24, 32, 40), we can have this
        # number of chains by chance even with binding factors. There is no good
        # way to automatically find these cases, unfortunately
        compass <- str_detect(title, "COMPASS")
        corest <- str_detect(title, "LSD1/CoREST")
        binding_factor <- compass || corest
        if (binding_factor) {
            return(TRUE)
        } else {
            return(FALSE)
        }
    }
}

# Vectorize the above function
has_binding_factor <- function(number_of_protein_chains_vector, title) {
    map2_lgl(number_of_protein_chains_vector, title, has_binding_factor_one)
}

# Query the PDB and clean up data
pdb_data <- pdb_queries %>% 
    map(fromJSON) %>% 
    map(dig_up_data) %>% 
    bind_rows() %>% 
    filter(!(pdb_id %in% non_nucleosome_structures)) %>% 
    mutate(
      has_binding_factor    = has_binding_factor(number_of_protein_chains, title),
      experimental_method   = as_factor(as.character(experimental_method)),
      citation_year         = as.integer(citation_year),
      molecule_name         = as.character(molecule_name),
      molecule_type         = as_factor(as.character(molecule_type))
      ) %>% 
    distinct(pdb_id, .keep_all = TRUE)
```

## Nucleosome structures by year

All figures are interactive (you can zoom in, and hovering over elements will
show more information).

### Nucleosome structures by year and experimental method

```{r Nucleosome structures by year}
nucleosome_structures_year <- pdb_data %>% 
    ggplot() +
    geom_bar(mapping = aes(x = citation_year, fill = experimental_method)) +
    guides(fill = guide_legend(title = "Experimental method")) +
    ggtitle("Structures of nucleosomes by year") +
    xlab("Publication year") +
    ylab("Number of PDB entries") +
    theme_bw()
ggplotly(nucleosome_structures_year)
```

```{r Save SVG file of year graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-structures-by-year.svg",
       plot = nucleosome_structures_year,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-structures-by-year.svg)

### Nucleosome structures by year and presence of a binding factor

```{r Nucleosome structures by year and binding factor}
nucleosome_structures_year_binding_factor <- pdb_data %>% 
    ggplot() +
    geom_bar(mapping = aes(x = citation_year, fill = has_binding_factor)) +
    guides(fill = guide_legend(title = "Contains a binding factor")) +
    ggtitle("Structures of nucleosomes by year") +
    xlab("Publication year") +
    ylab("Number of PDB entries") +
    theme_bw()
ggplotly(nucleosome_structures_year_binding_factor)
```

```{r Save SVG file of year and binding factor graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-structures-by-year-binding-factor.svg",
       plot = nucleosome_structures_year_binding_factor,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-structures-by-year-binding-factor.svg)

## Nucleosome structures by experimental method

```{r Nucleosome structures by experimental method}
nucleosome_structures_method <- pdb_data %>% 
    ggplot() +
    geom_bar(mapping = aes(x = experimental_method, fill = experimental_method)) +
    guides(fill = guide_legend(title = "Experimental method")) +
    ggtitle("Structures of nucleosomes by experimental method") +
    xlab("") +
    ylab("Number of PDB entries") +
    theme_bw()
ggplotly(nucleosome_structures_method)
```

```{r Save SVG file of method graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-structures-by-method.svg",
       plot = nucleosome_structures_method,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-structures-by-method.svg)

## Nucleosome structures by presence of a binding factor

```{r Nucleosome structures by presence of a binding factor and method}
nucleosome_binding_factors_methods <- pdb_data %>% 
    ggplot() +
    geom_bar(mapping = aes(x = experimental_method, fill = has_binding_factor)) +
    guides(fill = guide_legend(title = "Presence of a binding factor")) +
    ggtitle("Structures of nucleosomes by presence of a binding factor") +
    xlab("Experimental method") +
    ylab("Number of PDB entries") +
    theme_bw()
ggplotly(nucleosome_binding_factors_methods)
```

```{r Save SVG file of binding factors and methods graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-binding-factors-methods.svg",
       plot = nucleosome_binding_factors_methods,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-binding-factors-methods.svg)

```{r Nucleosome structures by presence of a binding factor}
nucleosome_binding_factors <- pdb_data %>% 
    ggplot() +
    geom_bar(mapping = aes(x = has_binding_factor, fill = has_binding_factor)) +
    guides(fill = guide_legend(title = "Presence of a binding factor")) +
    ggtitle("Structures of nucleosomes by presence of a binding factor") +
    xlab("Presence of a binding factor") +
    ylab("Number of PDB entries") +
    theme_bw()
ggplotly(nucleosome_binding_factors)
```

```{r Save SVG file of binding factors graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-binding-factors.svg",
       plot = nucleosome_binding_factors,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-binding-factors.svg)

```{r Nucleosome structures by presence of a binding factor and method 2}
nucleosome_binding_factors_methods_2 <- pdb_data %>% 
    ggplot() +
    geom_bar(mapping = aes(x = has_binding_factor, fill = experimental_method)) +
    guides(fill = guide_legend(title = "Experimental method")) +
    ggtitle("Structures of nucleosomes by presence of a binding factor") +
    xlab("Presence of a binding factor") +
    ylab("Number of PDB entries") +
    theme_bw()
ggplotly(nucleosome_binding_factors_methods_2)
```

```{r Save SVG file of binding factors methods 2 graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-binding-factors-methods-2.svg",
       plot = nucleosome_binding_factors_methods_2,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-binding-factors-methods-2.svg)

## Nucleosome structures by histone species
### Crystal structures of nucleosomes by histone species

```{r Crystal structures of nucleosomes by histone species}
nucleosome_xtal_species <- pdb_data %>% 
    filter(experimental_method == "X-ray diffraction") %>% 
    mutate(is_histone = str_detect(molecule_name, pattern = "Histone H")) %>% 
    filter(is_histone == TRUE) %>% 
    mutate(organism_scientific_name = as_factor(as.character(organism_scientific_name))) %>% 
    ggplot() +
    geom_bar(mapping = aes(x = organism_scientific_name,
                           fill = organism_scientific_name)) +
    guides(fill = guide_legend(title = "Histone species")) +
    ggtitle("Crystal structures of nucleosomes by histone species") +
    xlab("") +
    ylab("Number of PDB entries") +
    theme_bw() +
    theme(axis.text.x = element_text(angle = 30, hjust = 1))
ggplotly(nucleosome_xtal_species)
```

```{r Save SVG file of crystal-species graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-xtal-structures-by-histone-species.svg",
       plot = nucleosome_xtal_species,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-xtal-structures-by-histone-species.svg)

### Cryo-EM structures of nucleosomes by histone species

```{r Cryo-EM structures of nucleosomes by histone species}
nucleosome_cryoem_species <- pdb_data %>% 
    filter(experimental_method == "Electron Microscopy") %>% 
    mutate(is_histone = str_detect(molecule_name, pattern = "Histone H")) %>% 
    filter(is_histone == TRUE) %>% 
    mutate(organism_scientific_name = as_factor(as.character(organism_scientific_name))) %>% 
    ggplot() +
    geom_bar(mapping = aes(x = organism_scientific_name,
                           fill = organism_scientific_name)) +
    guides(fill = guide_legend(title = "Histone species")) +
    ggtitle("Cryo-EM structures of nucleosomes by histone species") +
    xlab("") +
    ylab("Number of PDB entries") +
    theme_bw() +
    theme(axis.text.x = element_text(angle = 30, hjust = 1))
ggplotly(nucleosome_cryoem_species)
```

```{r Save SVG file of cryoem-species graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-cryoem-structures-by-histone-species.svg",
       plot = nucleosome_cryoem_species,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-cryoem-structures-by-histone-species.svg)

## Resolution of nucleosome structures by histone species
### Crystal structures

```{r Resolution of nucleosome crystal structures by histone species}
pdb_data %>% dplyr::filter(experimental_method == "X-ray diffraction") %>% .$resolution %>% summary()

nucleosome_xtal_resolution_species <- pdb_data %>% 
    filter(experimental_method == "X-ray diffraction") %>% 
    mutate(is_histone = str_detect(molecule_name, pattern = "Histone H")) %>% 
    filter(is_histone == TRUE) %>% 
    mutate(organism_scientific_name = as_factor(as.character(organism_scientific_name))) %>% 
    select(resolution, organism_scientific_name) %>% 
    ggplot() +
    geom_histogram(aes(x = resolution, fill = organism_scientific_name),
                   binwidth = 0.2) +
    guides(fill = guide_legend(title = "Histone species")) +
    ggtitle("Resolution of nucleosome crystal structures by histone species") +
    xlab("Resolution (Å)") +
    ylab("Number of PDB entries") +
    theme_bw()
ggplotly(nucleosome_xtal_resolution_species)
```

```{r Save SVG file of xtal-resolution-species graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-xtal-resolution-by-histone-species.svg",
       plot = nucleosome_xtal_resolution_species,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-xtal-resolution-by-histone-species.svg)

### Cryo-EM structures

```{r Resolution of nucleosome cryo-EM structures by histone species}
pdb_data %>% dplyr::filter(experimental_method == "Electron Microscopy") %>% .$resolution %>% summary()

nucleosome_cryoem_resolution_species <- pdb_data %>% 
    filter(experimental_method == "Electron Microscopy") %>% 
    mutate(is_histone = str_detect(molecule_name, pattern = "Histone H")) %>% 
    filter(is_histone == TRUE) %>% 
    mutate(organism_scientific_name = as_factor(as.character(organism_scientific_name))) %>% 
    select(resolution, organism_scientific_name) %>% 
    ggplot() +
    geom_histogram(aes(x = resolution, fill = organism_scientific_name),
                   binwidth = 0.2) +
    guides(fill = guide_legend(title = "Histone species")) +
    ggtitle("Resolution of nucleosome cryo-EM structures by histone species") +
    xlab("Resolution (Å)") +
    ylab("Number of PDB entries") +
    theme_bw()
ggplotly(nucleosome_cryoem_resolution_species)
```

```{r Save SVG file of cryoem-resolution-species graph, include=FALSE}
# Save figure for download
if (!dir.exists(here("figures"))) {
    dir.create(here("figures"))
}
ggsave(filename = "nucleosome-cryoem-resolution-by-histone-species.svg",
       plot = nucleosome_cryoem_resolution_species,
       device = "svg",
       path = here("figures"))
```

[**Download figure in SVG format**](figures/nucleosome-cryoem-resolution-by-histone-species.svg)

## Dataset

The graphs presented above are derived from the following dataset:

```{r Entire dataset}
# Format table for display
pdb_table <- pdb_data %>% 
    arrange(desc(citation_year)) %>% 
    select(`PDB code` = pdb_id,
           `Citation year` = citation_year,
           `Experimental method` = experimental_method,
           Title = title)
pdb_table
```

```{r Save dataset as JSON, include=FALSE}
# Save dataset for download
if (!dir.exists(here("datasets"))) {
    dir.create(here("datasets"))
}

write_json(pdb_data, here("datasets", "nucleosome-structures.json"))
```

[**Download raw dataset in JSON format**](datasets/nucleosome-structures.json)
</div>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeSourceEmbed("nucleosome-structures.Rmd");
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
