<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>SARS-CoV-2 structures</title>

<script src="site_libs/header-attrs-2.9/header-attrs.js"></script>
<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/simplex.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<script src="site_libs/navigation-1.1/sourceembed.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>


<style type="text/css">
#rmd-source-code {
  display: none;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Insights from the PDB</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="nucleosome-structures.html">Nucleosome structures</a>
</li>
<li>
  <a href="sars-cov-2.html">SARS-CoV-2</a>
</li>
<li>
  <a href="dna-length-in-protein-dna-complexes.html">DNA length in protein-DNA complexes</a>
</li>
<li>
  <a href="free-dna.html">Free DNA</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/Guillawme/insights-from-the-pdb">GitHub repository</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
<li role="separator" class="divider"></li>
<li><a id="rmd-download-source" href="#">Download Rmd</a></li>
</ul>
</div>



<h1 class="title toc-ignore">SARS-CoV-2 structures</h1>

</div>


<p><strong>Last updated on Mon Jul 5 13:23:51 2021.</strong></p>
<div id="how-to-re-use-this-work" class="section level2">
<h2>How to re-use this work</h2>
<p>If you use these figures in your own work, please cite this website: <a href="https://doi.org/10.5281/zenodo.3470119" class="uri">https://doi.org/10.5281/zenodo.3470119</a></p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load required packages</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(magrittr)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(jsonlite)</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(forcats)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(plotly)</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(lubridate)</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="co"># We need two queries: one for uppercase titles, the other for lowercase ones</span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a>pdb_query <span class="ot">&lt;-</span> <span class="st">&#39;https://www.ebi.ac.uk/pdbe/search/pdb/select?q=title:*SARS-CoV-2*%20AND%20status:REL&amp;fl=pdb_id,deposition_date,citation_year,title,experimental_method,resolution,organism_scientific_name,molecule_name,molecule_type&amp;rows=1000000&amp;wt=json&#39;</span></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="co"># This is a helper pipeline to extract data</span></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a>dig_up_data <span class="ot">&lt;-</span> . <span class="sc">%&gt;%</span></span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a>    .<span class="sc">$</span>response <span class="sc">%&gt;%</span></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a>    .<span class="sc">$</span>docs <span class="sc">%&gt;%</span></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as_tibble</span>()</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Query the PDB and clean up data</span></span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a>pdb_data <span class="ot">&lt;-</span> pdb_query <span class="sc">%&gt;%</span> </span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">fromJSON</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">dig_up_data</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a>    <span class="fu">bind_rows</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a>      <span class="at">experimental_method   =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(experimental_method)),</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a>      <span class="at">deposition_date       =</span> <span class="fu">as_date</span>(<span class="fu">ymd_hms</span>(deposition_date)),</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a>      <span class="at">citation_year         =</span> <span class="fu">as.integer</span>(citation_year),</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a>      <span class="at">molecule_name         =</span> <span class="fu">as.character</span>(molecule_name),</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a>      <span class="at">molecule_type         =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(molecule_type))</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a>      ) <span class="sc">%&gt;%</span> </span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a>    <span class="fu">distinct</span>(pdb_id, <span class="at">.keep_all =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</div>
<div id="about-this-section" class="section level2">
<h2>About this section</h2>
<p>With the ongoing <a href="https://en.wikipedia.org/wiki/COVID-19_pandemic">COVID-19 pandemic</a>, new structures of proteins from <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a> are deposited to the PDB at a fast pace. The visualizations presented here won’t help curb the pandemic (and are not medical advice, obviously), but I hope they will make curious and helpless minds (like myself) get a feel for the research effort currently being deployed against it.</p>
<p>The following resources are a lot more relevant and actionable:</p>
<ul>
<li><a href="https://insidecorona.net/">Coronavirus Structural Task Force</a></li>
<li><a href="https://www.ebi.ac.uk/pdbe/covid-19">PDBe COVID-19 Data Portal</a></li>
<li><a href="https://crowdfightcovid19.org/">Crowdfight COVID-19</a></li>
</ul>
</div>
<div id="sars-cov-2-structures-by-deposition-date" class="section level2">
<h2>SARS-CoV-2 structures by deposition date</h2>
<p>All figures are interactive (you can zoom in, and hovering over elements will show more information).</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>sars_cov2_structures_year <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> deposition_date)) <span class="sc">+</span></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of SARS-CoV-2 by deposition date&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Deposition date&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(sars_cov2_structures_year)</span></code></pre></div>
<div id="htmlwidget-6e727cbfacb40595dbff" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-6e727cbfacb40595dbff">{"x":{"data":[{"orientation":"v","width":[0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455,0.900000000001455],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[18307,18310,18313,18317,18320,18324,18325,18330,18331,18336,18337,18339,18340,18344,18349,18351,18352,18353,18354,18355,18358,18359,18365,18366,18367,18368,18375,18376,18379,18380,18381,18382,18384,18385,18388,18389,18390,18392,18393,18396,18397,18398,18399,18401,18402,18404,18407,18408,18409,18410,18411,18416,18418,18419,18421,18422,18423,18424,18425,18426,18427,18428,18429,18431,18432,18434,18439,18442,18443,18444,18445,18446,18448,18449,18450,18451,18452,18453,18454,18457,18458,18460,18463,18466,18468,18469,18470,18471,18472,18473,18474,18476,18477,18478,18479,18481,18484,18485,18486,18488,18490,18491,18492,18493,18494,18495,18498,18499,18500,18501,18503,18504,18505,18506,18507,18508,18509,18512,18513,18514,18516,18517,18518,18519,18520,18521,18522,18523,18524,18527,18528,18529,18532,18533,18534,18535,18537,18539,18541,18543,18544,18545,18547,18548,18549,18550,18551,18553,18556,18557,18558,18559,18560,18561,18562,18564,18565,18567,18568,18569,18570,18571,18572,18573,18575,18576,18577,18579,18580,18581,18583,18584,18585,18586,18587,18588,18589,18594,18596,18597,18599,18600,18603,18604,18606,18607,18608,18610,18611,18612,18616,18617,18618,18619,18620,18627,18629,18633,18634,18635,18637,18638,18639,18640,18641,18643,18645,18647,18649,18654,18655,18657,18659,18660,18661,18662,18663,18665,18666,18667,18668,18670,18673,18675,18676,18677,18679,18681,18682,18683,18685,18686,18687,18688,18689,18691,18693,18694,18695,18696,18697,18703,18704,18708,18711,18712,18713,18715,18716,18717,18718,18723,18725,18728,18730,18731,18732,18737,18739,18741,18744,18746,18747,18752,18755,18759,18766,18772,18773,18774,18778,18785,18786,18792],"y":[3,1,1,2,2,5,1,2,1,68,1,4,1,1,1,1,2,1,1,1,1,14,2,3,1,2,1,1,3,4,4,1,1,3,5,3,1,1,1,2,19,5,1,1,4,3,1,1,1,2,4,6,4,2,14,2,4,2,1,1,3,3,1,4,4,1,5,5,1,3,2,2,7,5,6,9,1,6,2,1,2,1,1,1,1,2,1,1,1,3,1,2,2,1,6,3,10,2,2,1,5,1,2,1,9,2,3,1,3,5,4,1,3,11,6,1,4,1,1,16,4,1,1,5,1,52,2,1,3,3,2,1,15,118,2,3,14,2,2,6,1,9,1,1,2,4,4,6,3,1,2,2,4,1,3,1,2,2,108,4,8,3,2,1,6,1,2,6,2,1,3,2,1,6,2,4,3,2,4,1,2,2,2,1,4,4,1,1,3,5,1,9,1,7,4,4,1,2,1,4,1,2,1,1,1,1,12,2,3,2,1,12,1,10,1,4,3,1,4,1,2,1,2,1,1,1,1,4,1,2,2,2,18,1,1,7,12,5,3,13,1,7,1,2,6,1,1,3,3,1,4,1,1,1,2,4,12,8,1,2,1,1,2,3,1,2,15,5,1,1,1,1,1,2],"text":["count:   3<br />deposition_date: 18307","count:   1<br />deposition_date: 18310","count:   1<br />deposition_date: 18313","count:   2<br />deposition_date: 18317","count:   2<br />deposition_date: 18320","count:   5<br />deposition_date: 18324","count:   1<br />deposition_date: 18325","count:   2<br />deposition_date: 18330","count:   1<br />deposition_date: 18331","count:  68<br />deposition_date: 18336","count:   1<br />deposition_date: 18337","count:   4<br />deposition_date: 18339","count:   1<br />deposition_date: 18340","count:   1<br />deposition_date: 18344","count:   1<br />deposition_date: 18349","count:   1<br />deposition_date: 18351","count:   2<br />deposition_date: 18352","count:   1<br />deposition_date: 18353","count:   1<br />deposition_date: 18354","count:   1<br />deposition_date: 18355","count:   1<br />deposition_date: 18358","count:  14<br />deposition_date: 18359","count:   2<br />deposition_date: 18365","count:   3<br />deposition_date: 18366","count:   1<br />deposition_date: 18367","count:   2<br />deposition_date: 18368","count:   1<br />deposition_date: 18375","count:   1<br />deposition_date: 18376","count:   3<br />deposition_date: 18379","count:   4<br />deposition_date: 18380","count:   4<br />deposition_date: 18381","count:   1<br />deposition_date: 18382","count:   1<br />deposition_date: 18384","count:   3<br />deposition_date: 18385","count:   5<br />deposition_date: 18388","count:   3<br />deposition_date: 18389","count:   1<br />deposition_date: 18390","count:   1<br />deposition_date: 18392","count:   1<br />deposition_date: 18393","count:   2<br />deposition_date: 18396","count:  19<br />deposition_date: 18397","count:   5<br />deposition_date: 18398","count:   1<br />deposition_date: 18399","count:   1<br />deposition_date: 18401","count:   4<br />deposition_date: 18402","count:   3<br />deposition_date: 18404","count:   1<br />deposition_date: 18407","count:   1<br />deposition_date: 18408","count:   1<br />deposition_date: 18409","count:   2<br />deposition_date: 18410","count:   4<br />deposition_date: 18411","count:   6<br />deposition_date: 18416","count:   4<br />deposition_date: 18418","count:   2<br />deposition_date: 18419","count:  14<br />deposition_date: 18421","count:   2<br />deposition_date: 18422","count:   4<br />deposition_date: 18423","count:   2<br />deposition_date: 18424","count:   1<br />deposition_date: 18425","count:   1<br />deposition_date: 18426","count:   3<br />deposition_date: 18427","count:   3<br />deposition_date: 18428","count:   1<br />deposition_date: 18429","count:   4<br />deposition_date: 18431","count:   4<br />deposition_date: 18432","count:   1<br />deposition_date: 18434","count:   5<br />deposition_date: 18439","count:   5<br />deposition_date: 18442","count:   1<br />deposition_date: 18443","count:   3<br />deposition_date: 18444","count:   2<br />deposition_date: 18445","count:   2<br />deposition_date: 18446","count:   7<br />deposition_date: 18448","count:   5<br />deposition_date: 18449","count:   6<br />deposition_date: 18450","count:   9<br />deposition_date: 18451","count:   1<br />deposition_date: 18452","count:   6<br />deposition_date: 18453","count:   2<br />deposition_date: 18454","count:   1<br />deposition_date: 18457","count:   2<br />deposition_date: 18458","count:   1<br />deposition_date: 18460","count:   1<br />deposition_date: 18463","count:   1<br />deposition_date: 18466","count:   1<br />deposition_date: 18468","count:   2<br />deposition_date: 18469","count:   1<br />deposition_date: 18470","count:   1<br />deposition_date: 18471","count:   1<br />deposition_date: 18472","count:   3<br />deposition_date: 18473","count:   1<br />deposition_date: 18474","count:   2<br />deposition_date: 18476","count:   2<br />deposition_date: 18477","count:   1<br />deposition_date: 18478","count:   6<br />deposition_date: 18479","count:   3<br />deposition_date: 18481","count:  10<br />deposition_date: 18484","count:   2<br />deposition_date: 18485","count:   2<br />deposition_date: 18486","count:   1<br />deposition_date: 18488","count:   5<br />deposition_date: 18490","count:   1<br />deposition_date: 18491","count:   2<br />deposition_date: 18492","count:   1<br />deposition_date: 18493","count:   9<br />deposition_date: 18494","count:   2<br />deposition_date: 18495","count:   3<br />deposition_date: 18498","count:   1<br />deposition_date: 18499","count:   3<br />deposition_date: 18500","count:   5<br />deposition_date: 18501","count:   4<br />deposition_date: 18503","count:   1<br />deposition_date: 18504","count:   3<br />deposition_date: 18505","count:  11<br />deposition_date: 18506","count:   6<br />deposition_date: 18507","count:   1<br />deposition_date: 18508","count:   4<br />deposition_date: 18509","count:   1<br />deposition_date: 18512","count:   1<br />deposition_date: 18513","count:  16<br />deposition_date: 18514","count:   4<br />deposition_date: 18516","count:   1<br />deposition_date: 18517","count:   1<br />deposition_date: 18518","count:   5<br />deposition_date: 18519","count:   1<br />deposition_date: 18520","count:  52<br />deposition_date: 18521","count:   2<br />deposition_date: 18522","count:   1<br />deposition_date: 18523","count:   3<br />deposition_date: 18524","count:   3<br />deposition_date: 18527","count:   2<br />deposition_date: 18528","count:   1<br />deposition_date: 18529","count:  15<br />deposition_date: 18532","count: 118<br />deposition_date: 18533","count:   2<br />deposition_date: 18534","count:   3<br />deposition_date: 18535","count:  14<br />deposition_date: 18537","count:   2<br />deposition_date: 18539","count:   2<br />deposition_date: 18541","count:   6<br />deposition_date: 18543","count:   1<br />deposition_date: 18544","count:   9<br />deposition_date: 18545","count:   1<br />deposition_date: 18547","count:   1<br />deposition_date: 18548","count:   2<br />deposition_date: 18549","count:   4<br />deposition_date: 18550","count:   4<br />deposition_date: 18551","count:   6<br />deposition_date: 18553","count:   3<br />deposition_date: 18556","count:   1<br />deposition_date: 18557","count:   2<br />deposition_date: 18558","count:   2<br />deposition_date: 18559","count:   4<br />deposition_date: 18560","count:   1<br />deposition_date: 18561","count:   3<br />deposition_date: 18562","count:   1<br />deposition_date: 18564","count:   2<br />deposition_date: 18565","count:   2<br />deposition_date: 18567","count: 108<br />deposition_date: 18568","count:   4<br />deposition_date: 18569","count:   8<br />deposition_date: 18570","count:   3<br />deposition_date: 18571","count:   2<br />deposition_date: 18572","count:   1<br />deposition_date: 18573","count:   6<br />deposition_date: 18575","count:   1<br />deposition_date: 18576","count:   2<br />deposition_date: 18577","count:   6<br />deposition_date: 18579","count:   2<br />deposition_date: 18580","count:   1<br />deposition_date: 18581","count:   3<br />deposition_date: 18583","count:   2<br />deposition_date: 18584","count:   1<br />deposition_date: 18585","count:   6<br />deposition_date: 18586","count:   2<br />deposition_date: 18587","count:   4<br />deposition_date: 18588","count:   3<br />deposition_date: 18589","count:   2<br />deposition_date: 18594","count:   4<br />deposition_date: 18596","count:   1<br />deposition_date: 18597","count:   2<br />deposition_date: 18599","count:   2<br />deposition_date: 18600","count:   2<br />deposition_date: 18603","count:   1<br />deposition_date: 18604","count:   4<br />deposition_date: 18606","count:   4<br />deposition_date: 18607","count:   1<br />deposition_date: 18608","count:   1<br />deposition_date: 18610","count:   3<br />deposition_date: 18611","count:   5<br />deposition_date: 18612","count:   1<br />deposition_date: 18616","count:   9<br />deposition_date: 18617","count:   1<br />deposition_date: 18618","count:   7<br />deposition_date: 18619","count:   4<br />deposition_date: 18620","count:   4<br />deposition_date: 18627","count:   1<br />deposition_date: 18629","count:   2<br />deposition_date: 18633","count:   1<br />deposition_date: 18634","count:   4<br />deposition_date: 18635","count:   1<br />deposition_date: 18637","count:   2<br />deposition_date: 18638","count:   1<br />deposition_date: 18639","count:   1<br />deposition_date: 18640","count:   1<br />deposition_date: 18641","count:   1<br />deposition_date: 18643","count:  12<br />deposition_date: 18645","count:   2<br />deposition_date: 18647","count:   3<br />deposition_date: 18649","count:   2<br />deposition_date: 18654","count:   1<br />deposition_date: 18655","count:  12<br />deposition_date: 18657","count:   1<br />deposition_date: 18659","count:  10<br />deposition_date: 18660","count:   1<br />deposition_date: 18661","count:   4<br />deposition_date: 18662","count:   3<br />deposition_date: 18663","count:   1<br />deposition_date: 18665","count:   4<br />deposition_date: 18666","count:   1<br />deposition_date: 18667","count:   2<br />deposition_date: 18668","count:   1<br />deposition_date: 18670","count:   2<br />deposition_date: 18673","count:   1<br />deposition_date: 18675","count:   1<br />deposition_date: 18676","count:   1<br />deposition_date: 18677","count:   1<br />deposition_date: 18679","count:   4<br />deposition_date: 18681","count:   1<br />deposition_date: 18682","count:   2<br />deposition_date: 18683","count:   2<br />deposition_date: 18685","count:   2<br />deposition_date: 18686","count:  18<br />deposition_date: 18687","count:   1<br />deposition_date: 18688","count:   1<br />deposition_date: 18689","count:   7<br />deposition_date: 18691","count:  12<br />deposition_date: 18693","count:   5<br />deposition_date: 18694","count:   3<br />deposition_date: 18695","count:  13<br />deposition_date: 18696","count:   1<br />deposition_date: 18697","count:   7<br />deposition_date: 18703","count:   1<br />deposition_date: 18704","count:   2<br />deposition_date: 18708","count:   6<br />deposition_date: 18711","count:   1<br />deposition_date: 18712","count:   1<br />deposition_date: 18713","count:   3<br />deposition_date: 18715","count:   3<br />deposition_date: 18716","count:   1<br />deposition_date: 18717","count:   4<br />deposition_date: 18718","count:   1<br />deposition_date: 18723","count:   1<br />deposition_date: 18725","count:   1<br />deposition_date: 18728","count:   2<br />deposition_date: 18730","count:   4<br />deposition_date: 18731","count:  12<br />deposition_date: 18732","count:   8<br />deposition_date: 18737","count:   1<br />deposition_date: 18739","count:   2<br />deposition_date: 18741","count:   1<br />deposition_date: 18744","count:   1<br />deposition_date: 18746","count:   2<br />deposition_date: 18747","count:   3<br />deposition_date: 18752","count:   1<br />deposition_date: 18755","count:   2<br />deposition_date: 18759","count:  15<br />deposition_date: 18766","count:   5<br />deposition_date: 18772","count:   1<br />deposition_date: 18773","count:   1<br />deposition_date: 18774","count:   1<br />deposition_date: 18778","count:   1<br />deposition_date: 18785","count:   1<br />deposition_date: 18786","count:   2<br />deposition_date: 18792"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(89,89,89,1)","line":{"width":1.88976377952756,"color":"transparent"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":40.1826484018265,"l":43.1050228310502},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Structures of SARS-CoV-2 by deposition date","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[18282.255,18816.745],"tickmode":"array","ticktext":["2020-07","2021-01","2021-07"],"tickvals":[18444,18628,18809],"categoryorder":"array","categoryarray":["2020-07","2021-01","2021-07"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Deposition date","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-5.9,123.9],"tickmode":"array","ticktext":["0","30","60","90","120"],"tickvals":[0,30,60,90,120],"categoryorder":"array","categoryarray":["0","30","60","90","120"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"2623470c7f0d":{"x":{},"type":"bar"}},"cur_data":"2623470c7f0d","visdat":{"2623470c7f0d":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>The two noticeable spikes (68 structures deposited on March 15th and 13 structures deposited on April 7th) come from <a href="https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html">a fragment screening experiment</a> performed at Diamond Light Source. All these structures are the same protein (main protease) in complex with candidate molecules for drug design.</p>
<p><a href="figures/sars-cov2-structures-by-year.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
<div id="sars-cov-2-structures-by-experimental-method" class="section level2">
<h2>SARS-CoV-2 structures by experimental method</h2>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>sars_cov2_structures_method <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> experimental_method, <span class="at">fill =</span> experimental_method)) <span class="sc">+</span></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Experimental method&quot;</span>)) <span class="sc">+</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of SARS-CoV-2 by experimental method&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(sars_cov2_structures_method)</span></code></pre></div>
<div id="htmlwidget-6ea285bef94ad6ced6ab" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-6ea285bef94ad6ced6ab">{"x":{"data":[{"orientation":"v","width":0.9,"base":0,"x":[1],"y":[865],"text":"count: 865<br />experimental_method: X-ray diffraction<br />experimental_method: X-ray diffraction","type":"bar","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"X-ray diffraction","legendgroup":"X-ray diffraction","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[2],"y":[315],"text":"count: 315<br />experimental_method: Electron Microscopy<br />experimental_method: Electron Microscopy","type":"bar","marker":{"autocolorscale":false,"color":"rgba(163,165,0,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Electron Microscopy","legendgroup":"Electron Microscopy","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[3],"y":[4],"text":"count:   4<br />experimental_method: Solution NMR<br />experimental_method: Solution NMR","type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,191,125,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Solution NMR","legendgroup":"Solution NMR","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[4],"y":[3],"text":"count:   3<br />experimental_method: c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")<br />experimental_method: c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","type":"bar","marker":{"autocolorscale":false,"color":"rgba(0,176,246,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","legendgroup":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.9,"base":0,"x":[5],"y":[1],"text":"count:   1<br />experimental_method: Solid-state NMR<br />experimental_method: Solid-state NMR","type":"bar","marker":{"autocolorscale":false,"color":"rgba(231,107,243,1)","line":{"width":1.88976377952756,"color":"transparent"}},"name":"Solid-state NMR","legendgroup":"Solid-state NMR","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.7625570776256,"r":7.30593607305936,"b":25.5707762557078,"l":43.1050228310502},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"title":{"text":"Structures of SARS-CoV-2 by experimental method","font":{"color":"rgba(0,0,0,1)","family":"","size":17.5342465753425},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["X-ray diffraction","Electron Microscopy","Solution NMR","c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","Solid-state NMR"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["X-ray diffraction","Electron Microscopy","Solution NMR","c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","Solid-state NMR"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-43.25,908.25],"tickmode":"array","ticktext":["0","250","500","750"],"tickvals":[0,250,500,750],"categoryorder":"array","categoryarray":["0","250","500","750"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"y":0.913385826771654},"annotations":[{"text":"Experimental method","x":1.02,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"left","yanchor":"bottom","legendTitle":true}],"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"26236e820bbd":{"x":{},"fill":{},"type":"bar"}},"cur_data":"26236e820bbd","visdat":{"26236e820bbd":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/sars-cov2-structures-by-method.svg"><strong>Download figure in SVG format</strong></a></p>
</div>
<div id="dataset" class="section level2">
<h2>Dataset</h2>
<p>The graphs presented above are derived from the following dataset:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Format table for display</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>pdb_table <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">arrange</span>(<span class="fu">desc</span>(citation_year)) <span class="sc">%&gt;%</span> </span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">select</span>(<span class="st">`</span><span class="at">PDB code</span><span class="st">`</span> <span class="ot">=</span> pdb_id,</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>           <span class="st">`</span><span class="at">Citation year</span><span class="st">`</span> <span class="ot">=</span> citation_year,</span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>           <span class="st">`</span><span class="at">Experimental method</span><span class="st">`</span> <span class="ot">=</span> experimental_method,</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>           <span class="at">Title =</span> title)</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>pdb_table</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["PDB code"],"name":[1],"type":["chr"],"align":["left"]},{"label":["Citation year"],"name":[2],"type":["int"],"align":["right"]},{"label":["Experimental method"],"name":[3],"type":["fct"],"align":["left"]},{"label":["Title"],"name":[4],"type":["chr"],"align":["left"]}],"data":[{"1":"7c7p","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with Telaprevir"},{"1":"6yvf","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to AZD6482."},{"1":"6ynq","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to 2-Methyl-1-tetralone."},{"1":"7jlt","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 NSP7-NSP8 complex."},{"1":"7cbt","2":"2021","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 main protease in complex with GC376"},{"1":"7cjf","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with a neutralizing antibody Fab"},{"1":"7ci3","2":"2021","3":"X-ray diffraction","4":"The crystal structure of the SARS-CoV-2 ORF7a ectodomain"},{"1":"7mc5","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 ExoN-nsp10 complex"},{"1":"7mhh","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of Apo/Unliganded SARS-CoV-2 Main Protease (Mpro) at 277 K"},{"1":"7mjg","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain"},{"1":"7lgd","2":"2021","3":"X-ray diffraction","4":"HLA-B*07:02 in complex with SARS-CoV-2 nucleocapsid peptide N105-113"},{"1":"7lyq","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 1-RBD-up conformation"},{"1":"7m6i","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, BG1-24"},{"1":"7mfu","2":"2021","3":"X-ray diffraction","4":"Crystal structure of synthetic nanobody complexes with SARS-CoV-2 receptor binding domain"},{"1":"7lwm","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 1-RBD up conformation"},{"1":"7mho","2":"2021","3":"X-ray diffraction","4":"Ensemble refinement structure of apo/unliganded SARS-CoV-2 main protease (Mpro) at 298 K"},{"1":"7m6f","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, BG1-22"},{"1":"7m8k","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of Brazil (P.1) SARS-CoV-2 spike glycoprotein variant in the prefusion state (1 RBD up)"},{"1":"7mjn","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to human ACE2 ectodomain (focused refinement of RBD and ACE2)"},{"1":"7ld1","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S protein in complex with Receptor Binding Domain antibody DH1047"},{"1":"7lqw","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 2-17 Fab in complex with SARS-CoV-2 S2P spike"},{"1":"7lv2","2":"2021","3":"X-ray diffraction","4":"GSQASS segment from the Nucleoprotein of SARS-CoV-2, residues 179-184"},{"1":"7lwi","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 3-RBD down conformation"},{"1":"7lx5","2":"2021","3":"Electron Microscopy","4":"The SARS-CoV-2 spike protein receptor binding domain bound to neutralizing nanobodies WNb 2 and WNb 10"},{"1":"7m7b","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike:Fab 3D11 complex focused refinement"},{"1":"7m8s","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*02:01 in complex with KLNDLCFTNV, an 10-mer epitope from SARS-CoV-2 Spike (S386-395)"},{"1":"7nta","2":"2021","3":"Electron Microscopy","4":"Trimeric SARS-CoV-2 spike ectodomain in complex with biliverdin (one RBD erect)"},{"1":"7nx6","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7jkv","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease in complex with an inhibitor GRL-2420"},{"1":"7llf","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-83"},{"1":"7lqv","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 4-8 Fab in complex with SARS-CoV-2 S2P spike"},{"1":"7m71","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike:5A6 Fab complex I focused refinement"},{"1":"7ad1","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of a prefusion stabilized SARS-CoV-2 Spike (D614N, R682S, R685G, A892P, A942P and V987P)(One up trimer)"},{"1":"7aha","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Maleate."},{"1":"7lwq","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein(S-GSAS-D614G-delFV) missing the S1 subunit and SD2 subdomain of one protomer"},{"1":"7lyp","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 1-RBD-up conformation"},{"1":"7m6h","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, BG7-20"},{"1":"7nd9","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein (one RBD up) in complex with COVOX-253H55L Fab"},{"1":"7mjl","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to Fab ab1 (focused refinement of RBD and Fab ab1)"},{"1":"7abu","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to RS102895"},{"1":"7adw","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to 2,4'-Dimethylpropiophenone."},{"1":"7lka","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD-targeting antibody COV107-23"},{"1":"7llz","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-69"},{"1":"7nd3","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-40 Fab"},{"1":"7nev","2":"2021","3":"X-ray diffraction","4":"Structure of the hemiacetal complex between the SARS-CoV-2 Main Protease and Leupeptin"},{"1":"7lk9","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD-targeting antibody COV107-23 HC + COVD21-C8 LC"},{"1":"7a1u","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Fusidic Acid."},{"1":"7lo4","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 spike receptor-binding domain with a G485R mutation in complex with human ACE2"},{"1":"7ly2","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2M28 Global Refinement"},{"1":"7akd","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the 47D11 neutralizing antibody Fab fragment"},{"1":"7ndd","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein (one RBD up) in complex with COVOX-159"},{"1":"7ak4","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Tretazicar."},{"1":"7nt9","2":"2021","3":"Electron Microscopy","4":"Trimeric SARS-CoV-2 spike ectodomain in complex with biliverdin (closed conformation)"},{"1":"7nxc","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 P.1 variant Spike glycoprotein in complex with ACE2"},{"1":"7nd4","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-88 Fab"},{"1":"7nxa","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 B.1.351 variant Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7ntc","2":"2021","3":"Electron Microscopy","4":"Trimeric SARS-CoV-2 spike ectodomain bound to P008_056 Fab"},{"1":"7nd6","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-40 Fab"},{"1":"7lss","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab 2-7"},{"1":"7lwv","2":"2021","3":"Electron Microscopy","4":"UK (B.1.1.7) SARS-CoV-2 spike protein variant (S-GSAS-B.1.1.7) in the 1-RBD-up conformation"},{"1":"7lym","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the RBD-down conformation"},{"1":"7luz","2":"2021","3":"X-ray diffraction","4":"GQTVTK segment from the Nucleoprotein of SARS-CoV-2, residues 243-248"},{"1":"7lwt","2":"2021","3":"Electron Microscopy","4":"UK (B.1.1.7) SARS-CoV-2 spike protein variant (S-GSAS-B.1.1.7) in the 1-RBD-up conformation"},{"1":"7neh","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-269 Fab"},{"1":"7nx7","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the K417N mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7a4n","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of a prefusion stabilized SARS-CoV-2 Spike (D614N, R682S, R685G, A892P, A942P and V987P)(S-closed trimer)"},{"1":"7l2d","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 1-87 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7kqe","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein:Fab 3D11 complex"},{"1":"7mjj","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to Fab ab1 (class 1)"},{"1":"7l56","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab 2-43"},{"1":"7mhg","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of Apo/Unliganded SARS-CoV-2 Main Protease (Mpro) at 240 K"},{"1":"7ldj","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain in complex with WNb-2"},{"1":"7lxx","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2L28 Local Refinement"},{"1":"7m6g","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, BG7-15"},{"1":"7m8j","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S-NTD + Fab CM25"},{"1":"7lop","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with antibodies CV05-163 and CR3022"},{"1":"7lwk","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 3-RBD down conformation"},{"1":"7lyn","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 1-RBD-up conformation"},{"1":"7m0j","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 u1S2q All Down RBD State Spike Protein Trimer - asymmetric refinement"},{"1":"7m3i","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike protein receptor binding domain in complex with a neutralizing antibody, CV2-75 Fab"},{"1":"7lwp","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 2-RBD up conformation"},{"1":"7ly0","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2M28 Local Refinement"},{"1":"7m8t","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*11:01 in complex with NSASFSTFK, an 9-mer epitope from SARS-CoV-2 spike (S370-378)"},{"1":"7mhq","2":"2021","3":"X-ray diffraction","4":"Ensemble refinement structure of apo/unliganded SARS-CoV-2 main protease (Mpro) at 310 K"},{"1":"7lwo","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 1-RBD up conformation"},{"1":"7lxw","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2X333 Local Refinement"},{"1":"7mhk","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of Apo/Unliganded SARS-CoV-2 Main Protease (Mpro) at 310 K"},{"1":"7nda","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein (all RBD down) in complex with COVOX-253H55L Fab"},{"1":"7ndb","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-253H165L Fab"},{"1":"7lcn","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S protein in complex with N-terminal domain antibody DH1050.1"},{"1":"7lww","2":"2021","3":"Electron Microscopy","4":"Triple mutant (K417N-E484K-N501Y) SARS-CoV-2 spike protein in the 1-RBD-up conformation (S-GSAS-D614G-K417N-E484K-N501Y)"},{"1":"7lyo","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 1-RBD-up conformation"},{"1":"7krr","2":"2021","3":"Electron Microscopy","4":"Structural impact on SARS-CoV-2 spike protein by D614G substitution"},{"1":"7l2f","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 5-24 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7lwu","2":"2021","3":"Electron Microscopy","4":"UK (B.1.1.7) SARS-CoV-2 spike protein variant (S-GSAS-B.1.1.7) in the 1-RBD-up conformation"},{"1":"7lyk","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 2-RBD-up conformation"},{"1":"7l58","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab H4"},{"1":"7l09","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 2P S ectodomain bound domain-swapped antibody 2G12 from masked 3D refinement"},{"1":"7ans","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Adrafinil."},{"1":"7aph","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Tofogliflozin."},{"1":"7aws","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to TH-302."},{"1":"7m6e","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, BG10-19"},{"1":"7mc6","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 ExoN-nsp10 complex containing Mg2+ ion"},{"1":"7mhi","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of Apo/Unliganded SARS-CoV-2 Main Protease (Mpro) at 298 K"},{"1":"7l2e","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 4-18 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7nxb","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 P.1 variant Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7mjk","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to Fab ab1 (class 2)"},{"1":"7kqw","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain (C2 crystal form, methylated)"},{"1":"7m7w","2":"2021","3":"X-ray diffraction","4":"Antibodies to the SARS-CoV-2 receptor-binding domain that maximize breadth and resistance to viral escape"},{"1":"7mhm","2":"2021","3":"X-ray diffraction","4":"Ensemble refinement structure of apo/unliganded SARS-CoV-2 main protease (Mpro) at 240 K"},{"1":"7bem","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-269 scFv"},{"1":"7czq","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P2B-1A10"},{"1":"7czz","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-2F11_3B"},{"1":"7diy","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nsp10 bound to nsp14-exoribonuclease domain"},{"1":"7ks9","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of prefusion SARS-CoV-2 spike glycoprotein in complex with 910-30 Fab"},{"1":"7d7l","2":"2021","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 papain-like protease in complex with YM155"},{"1":"7djr","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (no ligand)"},{"1":"7l5b","2":"2021","3":"X-ray diffraction","4":"Crystallographic structure of neutralizing antibody 2-15 in complex with SARS-CoV-2 spike receptor-binding Domain (RBD)."},{"1":"7b83","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to pyrithione zinc"},{"1":"7bep","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in a ternary complex with COVOX-384 and S309 Fabs"},{"1":"7deu","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with a neutralizing antibody scFv"},{"1":"7d7k","2":"2021","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 papain-like protease in apo form"},{"1":"7avd","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to SEN1269 ligand"},{"1":"7krn","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 backtracked complex bound to nsp13 helicase - nsp13(1)-BTC"},{"1":"7baj","2":"2021","3":"X-ray diffraction","4":"Crystal structure of ligand-free SARS-CoV-2 main protease"},{"1":"7deo","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with a neutralizing antibody scFv"},{"1":"7aqe","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to UNC-2327"},{"1":"7det","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with a neutralizing antibody scFv"},{"1":"7jt0","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with MAC5576"},{"1":"7ben","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in a ternary complex with COVOX-253 and COVOX-75 Fabs"},{"1":"7l2c","2":"2021","3":"X-ray diffraction","4":"Crystallographic structure of neutralizing antibody 2-51 in complex with SARS-CoV-2 spike N-terminal domain (NTD)"},{"1":"7af0","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Ipidacrine."},{"1":"7aga","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to AT7519"},{"1":"7nd7","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-316 Fab"},{"1":"7nx9","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the N501Y mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7nd5","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-150 Fab"},{"1":"7kn6","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain complexed with nanobody VHH V and antibody Fab CC12.3"},{"1":"7kxb","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Nsp3 Macrodomain complex with PARG329"},{"1":"7b3b","2":"2021","3":"Electron Microscopy","4":"Structure of elongating SARS-CoV-2 RNA-dependent RNA polymerase with Remdesivir at position -3 (structure 1)"},{"1":"7krp","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 backtracked complex complex bound to nsp13 helicase - BTC (local refinement)"},{"1":"7kqo","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain (P43 crystal form)"},{"1":"7m6d","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 RBD in complex with neutralizing antibodies BG4-25 and CR3022"},{"1":"7lbr","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-89"},{"1":"7klw","2":"2021","3":"X-ray diffraction","4":"Crystal structure of synthetic nanobody complexes with SARS-CoV-2 receptor binding domain"},{"1":"7kqp","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose (P43 crystal form)"},{"1":"7kqb","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein:Fab 5A6 complex I"},{"1":"7kj4","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with three ACE2 Bound"},{"1":"7lbs","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-24"},{"1":"7mhp","2":"2021","3":"X-ray diffraction","4":"Ensemble refinement structure of apo/unliganded SARS-CoV-2 main protease (Mpro) at 298 K at high humidity"},{"1":"7mhl","2":"2021","3":"X-ray diffraction","4":"Ensemble refinement structure of apo/unliganded SARS-CoV-2 main protease (Mpro) at 100 K"},{"1":"7mji","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to VH ab8 (focused refinement of RBD and VH ab8)"},{"1":"7lws","2":"2021","3":"Electron Microscopy","4":"UK (B.1.1.7) SARS-CoV-2 S-GSAS-D614G variant spike protein in the 3-RBD-down conformation"},{"1":"7mhj","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of Apo/Unliganded SARS-CoV-2 Main Protease (Mpro) at 298 K and High Humidity"},{"1":"7mjh","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to VH ab8"},{"1":"7nd8","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-384 Fab"},{"1":"7lwn","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 1-RBD up conformation"},{"1":"7ly3","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 S NTD bound to S2M28 Fab"},{"1":"7cr5","2":"2021","3":"X-ray diffraction","4":"Complex structure of a human monoclonal antibody with SARS-CoV-2 nucleocapsid protein NTD"},{"1":"7com","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with Boceprevir (space group P212121)"},{"1":"7bei","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-150 Fab"},{"1":"7axm","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Pelitinib"},{"1":"7b3c","2":"2021","3":"Electron Microscopy","4":"Structure of elongating SARS-CoV-2 RNA-dependent RNA polymerase with Remdesivir at position -4 (structure 2)"},{"1":"7czx","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B9"},{"1":"7l06","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 2P S ectodomain bound to two copies of domain-swapped antibody 2G12"},{"1":"7lc8","2":"2021","3":"Solution NMR","4":"SARS-CoV-2 spike Protein TM domain"},{"1":"7bnm","2":"2021","3":"Electron Microscopy","4":"Closed conformation of D614G SARS-CoV-2 spike protein"},{"1":"7d0b","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-3C12_1B"},{"1":"7b3d","2":"2021","3":"Electron Microscopy","4":"Structure of elongating SARS-CoV-2 RNA-dependent RNA polymerase with AMP at position -4 (structure 3)"},{"1":"7czr","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B8_2B"},{"1":"7dcd","2":"2021","3":"X-ray diffraction","4":"Nonstructural protein 7 and 8 complex of SARS-CoV-2"},{"1":"7d0c","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-3A1"},{"1":"7kj5","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein, prefusion with one RBD up conformation"},{"1":"7l57","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab 2-15"},{"1":"7mkm","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike RBD in complex with neutralizing Fab SARS2-38 (local refinement)"},{"1":"7m53","2":"2021","3":"X-ray diffraction","4":"B6 Fab fragment bound to the SARS-CoV/SARS-CoV-2 spike stem helix peptide"},{"1":"7los","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-65"},{"1":"7neg","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the N501Y mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-269 Fab"},{"1":"7nx8","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the K417T mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7lb7","2":"2021","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Joint X-ray/neutron structure of SARS-CoV-2 main protease (3CL Mpro) in complex with Telaprevir"},{"1":"7awu","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to LSN2463359."},{"1":"7beh","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-316 Fab"},{"1":"7ndc","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein (all RBD down) in complex with COVOX-159"},{"1":"7lux","2":"2021","3":"X-ray diffraction","4":"AALALL segment from the Nucleoprotein of SARS-CoV-2, residues 217-222, crystal form 2"},{"1":"7m8u","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-B*35:01 in complex with IPFAMQMAY, an 9-mer epitope from SARS-CoV-2 spike (S896-904)"},{"1":"7l0d","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML188"},{"1":"7kr1","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain (C2 crystal form, 310 K)"},{"1":"7l1f","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RdRp in complex with 4 Remdesivir monophosphate"},{"1":"7mf1","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody 47D1"},{"1":"7mmo","2":"2021","3":"X-ray diffraction","4":"LY-CoV1404 neutralizing antibody against SARS-CoV-2"},{"1":"7bak","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease treated with ebselen"},{"1":"7laa","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S protein in complex with Receptor Binding Domain antibody DH1041"},{"1":"7beo","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in a ternary complex with COVOX-253H55L and COVOX-75 Fabs"},{"1":"7bnn","2":"2021","3":"Electron Microscopy","4":"Open conformation of D614G SARS-CoV-2 spike with 1 Erect RBD"},{"1":"7d0d","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-3C12_2B"},{"1":"7mhf","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of Apo/Unliganded SARS-CoV-2 Main Protease (Mpro) at 100 K"},{"1":"7mkl","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing Fab SARS2-38 (three down conformation)"},{"1":"7l0n","2":"2021","3":"X-ray diffraction","4":"Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity"},{"1":"7mhn","2":"2021","3":"X-ray diffraction","4":"Ensemble refinement structure of apo/unliganded SARS-CoV-2 main protease (Mpro) at 277 K"},{"1":"7mjm","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to human ACE2 ectodomain"},{"1":"7aku","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Calpeptin."},{"1":"7lwj","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 3-RBD down conformation"},{"1":"7lxy","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2X333 Global Refinement"},{"1":"7amj","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to PD 168568."},{"1":"7lm8","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with two cross-neutralizing antibodies CV38-142 and COVA1-16 Fabs isolated from COVID-19 patients"},{"1":"7ls9","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of neutralizing antibody 1-57 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7lwl","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 3-RBD down conformation"},{"1":"7lyl","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the RBD-down conformation"},{"1":"7d00","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with FabP5A-1B8"},{"1":"7aol","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Climbazole"},{"1":"7bej","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-158 Fab (crystal form 1)"},{"1":"7d3i","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with MI-23"},{"1":"7bel","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in a ternary complex with COVOX-88 and COVOX-45 Fabs"},{"1":"7ljr","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Protein Trimer bound to DH1043 fab"},{"1":"7aww","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Clonidine"},{"1":"7czw","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-2G7"},{"1":"7l02","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 2P S ectodomain bound to one copy of domain-swapped antibody 2G12"},{"1":"7o81","2":"2021","3":"Electron Microscopy","4":"Rabbit 80S ribosome colliding in another ribosome stalled by the SARS-CoV-2 pseudoknot"},{"1":"7kdg","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS)"},{"1":"7o7y","2":"2021","3":"Electron Microscopy","4":"Rabbit 80S ribosome stalled close to the mutated SARS-CoV-2 slippery site by a pseudoknot (high resolution)"},{"1":"7czu","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B6_2B"},{"1":"7o7z","2":"2021","3":"Electron Microscopy","4":"Rabbit 80S ribosome stalled close to the mutated SARS-CoV-2 slippery site by a pseudoknot (classified for pseudoknot)"},{"1":"7d6h","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 papain-like protease (PLPro) C112S mutant"},{"1":"7dx9","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 3 (3 up RBD and 2 PD bound)"},{"1":"7kdj","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD-up Spike Protein Trimer fully cleaved by furin without the P986-P987 stabilizing mutations (S-RRAR-D614G)"},{"1":"7bal","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease treated with ebselen derivative of MR6-31-2"},{"1":"7o80","2":"2021","3":"Electron Microscopy","4":"Rabbit 80S ribosome in complex with eRF1 and ABCE1 stalled at the STOP codon in the mutated SARS-CoV-2 slippery site"},{"1":"7ke6","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub-classification)"},{"1":"7e8f","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 NTD in complex with N9 Fab"},{"1":"7kea","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD-up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub classification)"},{"1":"7kgr","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201in complex with SARS-CoV-2 N159-167"},{"1":"7kgp","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201 in complex with SARS-CoV-2 N316-324"},{"1":"7czt","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-2G9"},{"1":"7e88","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-515 Fab"},{"1":"7k0r","2":"2021","3":"Electron Microscopy","4":"Nucleotide bound SARS-CoV-2 Nsp15"},{"1":"7jtl","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 ORF8 accessory protein"},{"1":"7kdl","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G)"},{"1":"7kgq","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201in complex with SARS-CoV-2 N222-230"},{"1":"7ke8","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub-classification)"},{"1":"7kro","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 backtracked complex complex bound to nsp13 helicase - nsp13(2)-BTC"},{"1":"7dx7","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 1 (1 up RBD and 1 PD bound)"},{"1":"7k3n","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of NSP1 from SARS-CoV-2"},{"1":"7kj3","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with two ACE2 Bound"},{"1":"7k7p","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 nonstuctural protein 1"},{"1":"7dmu","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike receptor-binding domain complexed with high affinity ACE2 mutant 3N39"},{"1":"7e23","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the CA521 neutralizing antibody Fab (focused refinement on Fab-RBD)"},{"1":"7jt7","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with compound 4"},{"1":"7dx1","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 D614G mutant"},{"1":"7dx8","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 2 (2 up RBD and 2 PD bound)"},{"1":"7dwy","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in the locked conformation"},{"1":"7kgo","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201in complex with SARS-CoV-2 N351-359"},{"1":"7ke7","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3-RBD-down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G Sub-Classification)"},{"1":"7czp","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P2B-1A1"},{"1":"7e7b","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 furin site mutant S-Trimer from a subunit vaccine candidate"},{"1":"7dx6","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 3 (2 up RBD and 1 PD bound)"},{"1":"7keb","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1RBD up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub-classification)"},{"1":"7d03","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with FabP5A-2G7"},{"1":"7lab","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S protein in complex with N-terminal domain antibody DH1052"},{"1":"7czv","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B6_3B"},{"1":"7e7d","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 wild-type S-Trimer from a subunit vaccine candidate"},{"1":"7kdh","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RBD up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS)"},{"1":"7ke4","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G Sub-class)"},{"1":"7jsu","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with GC376"},{"1":"7ay7","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Isofloxythepin"},{"1":"7kdk","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G)"},{"1":"7dhg","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Orf9b complex with human TOM70"},{"1":"7e7y","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-623 Fab"},{"1":"7bek","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-158 Fab (crystal form 2)"},{"1":"7czy","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-2F11_2B"},{"1":"7dx0","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2"},{"1":"7dx2","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2 D614G mutant"},{"1":"7bno","2":"2021","3":"Electron Microscopy","4":"Open conformation of D614G SARS-CoV-2 spike with 2 Erect RBDs"},{"1":"7kec","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD-up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G Sub-Classification)"},{"1":"7kdi","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer fully cleaved by furin without the P986-P987 stabilizing mutations (S-RRAR-D614G)"},{"1":"7e7x","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Spike Protein N terminal domain in Complex with N11 Fab"},{"1":"7dx5","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 2 (1 up RBD and 1 PD bound)"},{"1":"7ar6","2":"2021","3":"X-ray diffraction","4":"Structure of apo SARS-CoV-2 Main Protease with large beta angle, space group C2."},{"1":"7kgk","2":"2021","3":"X-ray diffraction","4":"Crystal structure of synthetic nanobody complexes with SARS-CoV-2 receptor binding domain"},{"1":"7jw8","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with compound 4 in space group P1"},{"1":"7kr0","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain (C2 crystal form, 100 K)"},{"1":"7kgj","2":"2021","3":"X-ray diffraction","4":"Crystal structure of synthetic nanobody complexes with SARS-CoV-2 receptor binding domain"},{"1":"7ke9","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD-up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub-classification)"},{"1":"7dwz","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in the active conformation"},{"1":"7dx3","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 1 (1 up RBD and no PD bound)"},{"1":"7ksg","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with nanobodies E"},{"1":"7krs","2":"2021","3":"Electron Microscopy","4":"Structural impact on SARS-CoV-2 spike protein by D614G substitution"},{"1":"7arf","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to thioglucose."},{"1":"7awr","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Tegafur"},{"1":"7axo","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to AR-42."},{"1":"7b62","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein N-terminal domain in complex with biliverdin"},{"1":"7ar5","2":"2021","3":"X-ray diffraction","4":"Structure of apo SARS-CoV-2 Main Protease with small beta angle, space group C2."},{"1":"7lxz","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2L28 Global Refinement"},{"1":"7aqj","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Triglycidyl isocyanurate."},{"1":"7e86","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-508 Fab"},{"1":"7dk1","2":"2021","3":"X-ray diffraction","4":"Crystal structure of Zinc bound SARS-CoV-2 main protease"},{"1":"7ax6","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Glutathione isopropyl ester"},{"1":"7jst","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in apo form"},{"1":"7czs","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B8_3B"},{"1":"7e35","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 papain-like protease (PLPro) C112S mutant bound to compound S43"},{"1":"7e8c","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S-6P in complex with 9 Fabs"},{"1":"7kj2","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with one ACE2 Bound"},{"1":"7kn7","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain complexed with nanobody VHH W and antibody Fab CC12.3"},{"1":"7aqi","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Ifenprodil"},{"1":"7ap6","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to MUT056399."},{"1":"7kn5","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain complexed with nanobodies VHH E and U"},{"1":"7krq","2":"2021","3":"Electron Microscopy","4":"Structural impact on SARS-CoV-2 spike protein by D614G substitution"},{"1":"5rek","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102327"},{"1":"5rgk","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1310876699 (Mpro-x0426)"},{"1":"7c6u","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 complexed with GC376"},{"1":"7che","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with BD-236 Fab and BD-368-2 Fab"},{"1":"7cmd","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 PLpro with GRL0617"},{"1":"5rf9","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z217038356"},{"1":"5rf7","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z316425948_minor"},{"1":"5rgi","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z369936976 (Mpro-x0397)"},{"1":"5rfe","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z509756472"},{"1":"5rfo","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102972"},{"1":"5rem","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0103016"},{"1":"7cah","2":"2020","3":"Electron Microscopy","4":"The interface of H014 Fab binds to SARS-CoV-2 S"},{"1":"5rfm","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102539"},{"1":"6wrz","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of Nsp16-Nsp10 Heterodimer from SARS-CoV-2 with 7-methyl-GpppA and S-adenosyl-L-homocysteine in the Active Site and Sulfates in the mRNA Binding Groove."},{"1":"6wzq","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Nucleocapsid dimerization domain, P21 form"},{"1":"6xlu","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 4.0"},{"1":"5rff","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102704"},{"1":"7cb7","2":"2020","3":"X-ray diffraction","4":"1.7A resolution structure of SARS-CoV-2 main protease (Mpro) in complex with broad-spectrum coronavirus protease inhibitor GC376"},{"1":"5rfk","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102575"},{"1":"5rfi","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102353"},{"1":"6w4h","2":"2020","3":"X-ray diffraction","4":"1.80 Angstrom Resolution Crystal Structure of NSP16 - NSP10 Complex from SARS-CoV-2"},{"1":"5rfa","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2643472210"},{"1":"5rgr","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z328695024 (Mpro-x1101)"},{"1":"7c8d","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of cat ACE2 and SARS-CoV-2 RBD"},{"1":"7cac","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer with one RBD in the open state and complexed with one H014 Fab."},{"1":"5r80","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z18197050"},{"1":"5rf0","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with POB0073"},{"1":"5rf3","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1741970824"},{"1":"5ret","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102269"},{"1":"6y2f","2":"2020","3":"X-ray diffraction","4":"Crystal structure (monoclinic form) of the complex resulting from the reaction between SARS-CoV-2 (2019-nCoV) main protease and tert-butyl (1-((S)-1-(((S)-4-(benzylamino)-3,4-dioxo-1-((S)-2-oxopyrrolidin-3-yl)butan-2-yl)amino)-3-cyclopropyl-1-oxopropan-2-yl)-2-oxo-1,2-dihydropyridin-3-yl)carbamate (alpha-ketoamide 13b)"},{"1":"6wtk","2":"2020","3":"X-ray diffraction","4":"Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication"},{"1":"5res","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102281"},{"1":"5rgh","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1619978933 (Mpro-x0395)"},{"1":"5rf6","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1348371854"},{"1":"5rf5","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z3241250482"},{"1":"5ref","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z24758179"},{"1":"5reh","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z111507846"},{"1":"5rfv","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102306"},{"1":"5rhf","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PG-COV-34 (Mpro-x2754)"},{"1":"5rep","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102201"},{"1":"5rg2","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with NCL-00025058"},{"1":"5rf2","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1741969146"},{"1":"5rfb","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1271660837"},{"1":"5reb","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434899"},{"1":"5rfs","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102739"},{"1":"5rgo","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102248 (Mpro-x0736)"},{"1":"5rf4","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1741982125"},{"1":"5rg0","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102535"},{"1":"6w75","2":"2020","3":"X-ray diffraction","4":"1.95 Angstrom Resolution Crystal Structure of NSP10 - NSP16 Complex from SARS-CoV-2"},{"1":"6wks","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 nsp16/nsp10 in complex with RNA cap analogue (m7GpppA) and S-adenosylmethionine"},{"1":"5ree","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2217052426"},{"1":"5rgq","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1849009686 (Mpro-x1086)"},{"1":"6wq3","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of Nsp16-Nsp10 Heterodimer from SARS-CoV-2 in Complex with 7-methyl-GpppA and S-adenosyl-L-homocysteine."},{"1":"6wps","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the S309 neutralizing antibody Fab fragment"},{"1":"5rfg","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102372"},{"1":"6m3m","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nucleocapsid protein N-terminal RNA binding domain"},{"1":"6m0j","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain bound with ACE2"},{"1":"6m2q","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CL pro) apo structure (space group C21)"},{"1":"6m2n","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CL pro) in complex with a novel inhibitor"},{"1":"6yhu","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the nsp7-nsp8 complex of SARS-CoV-2"},{"1":"6yi3","2":"2020","3":"Solution NMR","4":"The N-terminal RNA-binding domain of the SARS-CoV-2 nucleocapsid phosphoprotein"},{"1":"6ym0","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain in complex with CR3022 Fab (crystal form 1)"},{"1":"6yor","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike S1 protein in complex with CR3022 Fab"},{"1":"6zox","2":"2020","3":"Electron Microscopy","4":"Structure of Disulphide-stabilized SARS-CoV-2 Spike Protein Trimer (x2 disulphide-bond mutant, G413C, V987C, single Arg S1/S2 cleavage site)"},{"1":"6zn5","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to a pre-40S-like ribosome complex - state 2"},{"1":"6zoj","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2-Nsp1-40S complex, composite map"},{"1":"6z5t","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 Macrodomain in complex with ADP-ribose"},{"1":"6zlw","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human 40S ribosomal subunit"},{"1":"6vw1","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 chimeric receptor-binding domain complexed with its receptor human ACE2"},{"1":"6yun","2":"2020","3":"X-ray diffraction","4":"1.45 Angstrom Resolution Crystal Structure of C-terminal Dimerization Domain of Nucleocapsid Phosphoprotein from SARS-CoV-2"},{"1":"6zok","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2-Nsp1-40S complex, focused on body"},{"1":"6zmt","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to a pre-40S-like ribosome complex"},{"1":"6zh9","2":"2020","3":"X-ray diffraction","4":"Ternary complex CR3022 H11-H4 and RBD (SARS-CoV-2)"},{"1":"6zol","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2-Nsp1-40S complex, focused on head"},{"1":"6zp4","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to a human 43S preinitiation ribosome complex - state 2"},{"1":"6yla","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain in complex with CR3022 Fab"},{"1":"6zoy","2":"2020","3":"Electron Microscopy","4":"Structure of Disulphide-stabilized SARS-CoV-2 Spike Protein Trimer (x1 disulphide-bond mutant, S383C, D985C, K986P, V987P, single Arg S1/S2 cleavage site) in Closed State"},{"1":"6yz7","2":"2020","3":"X-ray diffraction","4":"H11-D4, SARS-CoV-2 RBD, CR3022 ternary complex"},{"1":"6zcz","2":"2020","3":"X-ray diffraction","4":"Crystal structure of receptor binding domain of SARS-CoV-2 Spike glycoprotein in ternary complex with EY6A Fab and a nanobody."},{"1":"6yz1","2":"2020","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 nsp10-nsp16 methyltransferase complex with Sinefungin"},{"1":"6zp1","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein Trimer (K986P, V987P, single Arg S1/S2 cleavage site) in Closed State"},{"1":"7c2l","2":"2020","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with 4A8"},{"1":"7cam","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 main protease (Mpro) apo structure (space group P212121)"},{"1":"7bw4","2":"2020","3":"Electron Microscopy","4":"Structure of the RNA-dependent RNA polymerase from SARS-CoV-2"},{"1":"6xey","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab 2-4"},{"1":"6xcm","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the C105 neutralizing antibody Fab fragment (state 1)"},{"1":"6xrz","2":"2020","3":"Electron Microscopy","4":"The 28-kDa Frameshift Stimulation Element from the SARS-CoV-2 RNA Genome"},{"1":"6xhm","2":"2020","3":"X-ray diffraction","4":"Covalent complex of SARS-CoV-2 main protease with N-[(2S)-1-({(2S,3S)-3,4-dihydroxy-1-[(3S)-2-oxopyrrolidin-3-yl]butan-2-yl}amino)-4-methyl-1-oxopentan-2-yl]-4-methoxy-1H-indole-2-carboxamide"},{"1":"6xm0","2":"2020","3":"Electron Microscopy","4":"Consensus structure of SARS-CoV-2 spike at pH 5.5"},{"1":"6wtt","2":"2020","3":"X-ray diffraction","4":"Crystals Structure of the SARS-CoV-2 (COVID-19) main protease with inhibitor GC-376"},{"1":"6x2b","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 u1S2q 2-RBD Up Spike Protein Trimer"},{"1":"6xca","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C105"},{"1":"6xdc","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 ORF3a"},{"1":"7ch5","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-629 Fab"},{"1":"7chh","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with BD-368-2 Fabs"},{"1":"7c01","2":"2020","3":"X-ray diffraction","4":"Molecular basis for a potent human neutralizing antibody targeting SARS-CoV-2 RBD"},{"1":"7chf","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with BD-604 Fab and BD-368-2 Fab"},{"1":"7cai","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer with two RBDs in the open state and complexed with two H014 Fab"},{"1":"7c6s","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease complexed with Boceprevir"},{"1":"6xc4","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody CC12.3"},{"1":"7cn9","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike ectodomain"},{"1":"7c33","2":"2020","3":"X-ray diffraction","4":"Macro domain of SARS-CoV-2 in complex with ADP-ribose"},{"1":"6xbi","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAW248"},{"1":"7ch4","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-604 Fab"},{"1":"7cjd","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 PLpro C111S mutant"},{"1":"6y2g","2":"2020","3":"X-ray diffraction","4":"Crystal structure (orthorhombic form) of the complex resulting from the reaction between SARS-CoV-2 (2019-nCoV) main protease and tert-butyl (1-((S)-1-(((S)-4-(benzylamino)-3,4-dioxo-1-((S)-2-oxopyrrolidin-3-yl)butan-2-yl)amino)-3-cyclopropyl-1-oxopropan-2-yl)-2-oxo-1,2-dihydropyridin-3-yl)carbamate (alpha-ketoamide 13b)"},{"1":"6xqs","2":"2020","3":"X-ray diffraction","4":"Room-temperature X-ray Crystal structure of SARS-CoV-2 main protease in complex with Telaprevir"},{"1":"6wtj","2":"2020","3":"X-ray diffraction","4":"Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication"},{"1":"6xb0","2":"2020","3":"X-ray diffraction","4":"Room temperature X-ray crystallography reveals catalytic cysteine in the SARS-CoV-2 3CL Mpro is highly reactive: Insights for enzyme mechanism and drug design"},{"1":"6xkp","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody CV07-270"},{"1":"6xs6","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike D614G variant, minus RBD"},{"1":"6xm5","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 5.5, all RBDs down"},{"1":"6x2c","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 u1S2q All Down RBD State Spike Protein Trimer"},{"1":"6x2a","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 u1S2q 1-RBD Up Spike Protein Trimer"},{"1":"6x79","2":"2020","3":"Electron Microscopy","4":"Prefusion SARS-CoV-2 S ectodomain trimer covalently stabilized in the closed conformation"},{"1":"6xb2","2":"2020","3":"X-ray diffraction","4":"Room temperature X-ray crystallography reveals catalytic cysteine in the SARS-CoV-2 3CL Mpro is highly reactive: Insights for enzyme mechanism and drug design"},{"1":"6xhu","2":"2020","3":"X-ray diffraction","4":"Room temperature X-ray crystallography reveals oxidation and reactivity of cysteine residues in SARS-CoV-2 3CL Mpro: Insights for enzyme mechanism and drug design"},{"1":"6x29","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 rS2d Down State Spike Protein Trimer"},{"1":"6xr8","2":"2020","3":"Electron Microscopy","4":"Distinct conformational states of SARS-CoV-2 spike protein"},{"1":"6xbh","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAW247"},{"1":"6xez","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(2)-RTC"},{"1":"6xm4","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 5.5, single RBD up, conformation 2"},{"1":"5reg","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1545313172"},{"1":"5rfy","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102974"},{"1":"5reu","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102395"},{"1":"7chc","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-629 Fab and BD-368-2 Fab"},{"1":"5red","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434865"},{"1":"5re7","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z30932204"},{"1":"5rg1","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with NCL-00024905"},{"1":"5rhe","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PG-COV-42 (Mpro-x2052)"},{"1":"5rey","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102911"},{"1":"5rgm","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102142 (Mpro-x0708)"},{"1":"5rec","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1587220559"},{"1":"5rfw","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102243"},{"1":"6vyb","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike ectodomain structure (open state)"},{"1":"7c2i","2":"2020","3":"X-ray diffraction","4":"Crystal structure of nsp16-nsp10 heterodimer from SARS-CoV-2 in complex with SAM (with additional SAM during crystallization)"},{"1":"5rev","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0103072"},{"1":"5rgp","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102628 (Mpro-x0771)"},{"1":"5rfh","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102277"},{"1":"6wtm","2":"2020","3":"X-ray diffraction","4":"Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication"},{"1":"6xc3","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with antibodies CC12.1 and CR3022"},{"1":"6xc2","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody CC12.1"},{"1":"6xe1","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike protein receptor binding domain in complex with a potent neutralizing antibody, CV30 Fab"},{"1":"6xmk","2":"2020","3":"X-ray diffraction","4":"1.70 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 7j"},{"1":"6xfn","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with UAW243"},{"1":"6xch","2":"2020","3":"X-ray diffraction","4":"Room-temperature X-ray Crystal structure of SARS-CoV-2 main protease in complex with Leupeptin"},{"1":"7c2j","2":"2020","3":"X-ray diffraction","4":"Crystal structure of nsp16-nsp10 heterodimer from SARS-CoV-2 in complex with SAM (without additional SAM during crystallization)"},{"1":"7cak","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer with three RBD in the open state and complexed with three H014 Fab"},{"1":"7btf","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 RNA-dependent RNA polymerase in complex with cofactors in reduced condition"},{"1":"7bwj","2":"2020","3":"X-ray diffraction","4":"crystal structure of SARS-CoV-2 antibody with RBD"},{"1":"7c22","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the C-terminal domain of SARS-CoV-2 nucleocapsid protein"},{"1":"7chb","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with BD-236 Fab"},{"1":"6xkq","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody CV07-250"},{"1":"6x6p","2":"2020","3":"Electron Microscopy","4":"Characterization of the SARS-CoV-2 S Protein: Biophysical, Biochemical, Structural, and Antigenic Analysis"},{"1":"6xa4","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with UAW241"},{"1":"5re8","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2737076969"},{"1":"5rfp","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102190"},{"1":"5rer","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102615"},{"1":"5rgj","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1401276297 (Mpro-x0425)"},{"1":"5rew","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102275"},{"1":"5rgl","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102962 (Mpro-x0705)"},{"1":"6w9q","2":"2020","3":"X-ray diffraction","4":"Peptide-bound SARS-CoV-2 Nsp9 RNA-replicase"},{"1":"5rfz","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102274"},{"1":"5rel","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102340"},{"1":"5rgg","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434890 (Mpro-x0165)"},{"1":"6wcf","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of ADP ribose phosphatase of NSP3 from SARS-CoV-2 in complex with MES"},{"1":"6woj","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 macrodomain (NSP3) in complex with ADP-ribose"},{"1":"6wjt","2":"2020","3":"X-ray diffraction","4":"2.0 Angstrom Resolution Crystal Structure of Nsp16-Nsp10 Heterodimer from SARS-CoV-2 in Complex with S-Adenosyl-L-Homocysteine"},{"1":"5re5","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z33545544"},{"1":"5rfr","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102169"},{"1":"5rhc","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Cov_HetLib053 (Mpro-x2119)"},{"1":"7cab","2":"2020","3":"Electron Microscopy","4":"Structural basis for neutralization of SARS-CoV-2 and SARS-CoV by a potent therapeutic antibody"},{"1":"5re6","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z54571979"},{"1":"5rft","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102432"},{"1":"5rhb","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Cov_HetLib030 (Mpro-x2097)"},{"1":"5rfd","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z126932614"},{"1":"5rgs","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1259086950 (Mpro-x1163)"},{"1":"5rf1","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with NCL-00023830"},{"1":"6vxx","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein (closed state)"},{"1":"6wen","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of ADP ribose phosphatase of NSP3 from SARS-CoV-2 in the apo form"},{"1":"5reo","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102578"},{"1":"5r83","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z44592329"},{"1":"5rej","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102241"},{"1":"5rfn","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102868"},{"1":"6wqf","2":"2020","3":"X-ray diffraction","4":"Structural Plasticity of the SARS-CoV-2 3CL Mpro Active Site Cavity Revealed by Room Temperature X-ray Crystallography"},{"1":"5rfc","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z979145504"},{"1":"5rea","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z31432226"},{"1":"5rfx","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102254"},{"1":"5ren","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102425"},{"1":"5rg3","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with NCL-00025412"},{"1":"5rfj","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0103067"},{"1":"5rez","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with POB0129"},{"1":"5rei","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434856"},{"1":"5rgn","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102759 (Mpro-x0731)"},{"1":"5re4","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1129283193"},{"1":"6w41","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with human antibody CR3022"},{"1":"5re9","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434836"},{"1":"5rfq","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102179"},{"1":"6wkq","2":"2020","3":"X-ray diffraction","4":"1.98 Angstrom Resolution Crystal Structure of NSP16-NSP10 Heterodimer from SARS-CoV-2 in Complex with Sinefungin"},{"1":"6wpt","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the S309 neutralizing antibody Fab fragment (open state)"},{"1":"6wey","2":"2020","3":"X-ray diffraction","4":"High-resolution structure of the SARS-CoV-2 NSP3 Macro X domain"},{"1":"6xb1","2":"2020","3":"X-ray diffraction","4":"Room temperature X-ray crystallography reveals catalytic cysteine in the SARS-CoV-2 3CL Mpro is highly reactive: Insights for enzyme mechanism and drug design"},{"1":"5rfl","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102389"},{"1":"6xkl","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 HexaPro S One RBD up"},{"1":"6xm3","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 5.5, single RBD up, conformation 1"},{"1":"5rfu","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102121"},{"1":"5rex","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102287"},{"1":"5r7z","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1220452176"},{"1":"5rhd","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with SF013 (Mpro-x2193)"},{"1":"6xc7","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with antibodies CC12.3 and CR3022"},{"1":"6xf5","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of a biotinylated SARS-CoV-2 spike probe in the prefusion state (RBDs down)"},{"1":"6zge","2":"2020","3":"Electron Microscopy","4":"Uncleavable Spike Protein of SARS-CoV-2 in Closed Conformation"},{"1":"5rf8","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z271004858"},{"1":"7kml","2":"2020","3":"Electron Microscopy","4":"cryo-EM structure of SARS-CoV-2 spike in complex with Fab 15033-7, three RBDs bound"},{"1":"6xdg","2":"2020","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 receptor binding domain with the Fab fragments of two neutralizing antibodies"},{"1":"6wzo","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Nucleocapsid dimerization domain, P1 form"},{"1":"6xcn","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the C105 neutralizing antibody Fab fragment (state 2)"},{"1":"6xra","2":"2020","3":"Electron Microscopy","4":"Distinct conformational states of SARS-CoV-2 spike protein"},{"1":"6xqt","2":"2020","3":"X-ray diffraction","4":"Room-temperature X-ray Crystal structure of SARS-CoV-2 main protease in complex with Narlaprevir"},{"1":"6wvn","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of Nsp16-Nsp10 from SARS-CoV-2 in Complex with 7-methyl-GpppA and S-Adenosylmethionine."},{"1":"6xqu","2":"2020","3":"X-ray diffraction","4":"Room-temperature X-ray Crystal structure of SARS-CoV-2 main protease in complex with Boceprevir"},{"1":"6xbg","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAW246"},{"1":"6xf6","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of a biotinylated SARS-CoV-2 spike probe in the prefusion state (1 RBD up)"},{"1":"6y2e","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the free enzyme of the SARS-CoV-2 (2019-nCoV) main protease"},{"1":"6wxd","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 Nsp9 RNA-replicase"},{"1":"6zer","2":"2020","3":"X-ray diffraction","4":"Crystal structure of receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with EY6A Fab"},{"1":"6zon","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to a human 43S preinitiation ribosome complex - state 1"},{"1":"6zme","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human CCDC124-80S-eERF1 ribosome complex"},{"1":"7act","2":"2020","3":"Solution NMR","4":"The SARS-CoV-2 nucleocapsid phosphoprotein N-terminal domain in complex with 10mer ssRNA"},{"1":"6zp0","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein Trimer (single Arg S1/S2 cleavage site) in Closed State"},{"1":"7acs","2":"2020","3":"Solution NMR","4":"The SARS-CoV-2 nucleocapsid phosphoprotein N-terminal domain in complex with 7mer dsRNA"},{"1":"6z6i","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 Macrodomain in complex with ADP-HPD"},{"1":"6zdh","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike glycoprotein in complex with a neutralizing antibody EY6A Fab"},{"1":"7jmx","2":"2020","3":"X-ray diffraction","4":"Crystal structure of a SARS-CoV-2 cross-neutralizing antibody COVA1-16 Fab"},{"1":"6zp2","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein Trimer (K986P, V987P, single Arg S1/S2 cleavage site) in Locked State"},{"1":"6yyt","2":"2020","3":"Electron Microscopy","4":"Structure of replicating SARS-CoV-2 polymerase"},{"1":"7a96","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 1 ACE2 Bound and 1 RBD Erect in Anticlockwise Direction"},{"1":"7jmw","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with cross-neutralizing antibody COVA1-16 Fab"},{"1":"7a5s","2":"2020","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 spike and CR3022 Fab (Homogeneous Refinement)"},{"1":"7jjj","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 3Q-2P full-length dimers of spike trimers"},{"1":"7a94","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 1 ACE2 Bound"},{"1":"7a25","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike protein bound to neutralizing sybodies (Sb23)"},{"1":"6zm7","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human CCDC124-80S-EBP1 ribosome complex"},{"1":"6zmo","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human LYAR-80S-eEF1a ribosome complex"},{"1":"7a98","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 3 ACE2 Bound"},{"1":"7klh","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 RBD in complex with Fab 15033-7"},{"1":"7kjr","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 ORF3a"},{"1":"7kmb","2":"2020","3":"Electron Microscopy","4":"ACE2-RBD Focused Refinement Using Symmetry Expansion of Applied C3 for Triple ACE2-bound SARS-CoV-2 Trimer Spike at pH 7.4"},{"1":"7jmp","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody COVA2-39"},{"1":"7b3o","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with STE90-C11 Fab"},{"1":"7knh","2":"2020","3":"Electron Microscopy","4":"Cryo-EM Structure of Double ACE2-Bound SARS-CoV-2 Trimer Spike at pH 5.5"},{"1":"7kmk","2":"2020","3":"Electron Microscopy","4":"cryo-EM structure of SARS-CoV-2 spike in complex with Fab 15033-7, two RBDs bound"},{"1":"7klg","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 RBD in complex with Fab 15033"},{"1":"7knb","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of single ACE2-bound SARS-CoV-2 trimer spike at pH 7.4"},{"1":"7jpy","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in its apo-form"},{"1":"7cyq","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of an extended SARS-CoV-2 replication and transcription complex reveals an intermediate state in cap synthesis"},{"1":"7a29","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike protein bound to neutralizing sybodies (Sb23) 2-up conformation"},{"1":"7df3","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer, S-closed"},{"1":"7a97","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 2 ACE2 Bound"},{"1":"7jp1","2":"2020","3":"X-ray diffraction","4":"Structure of wild-type substrate free SARS-CoV-2 Mpro."},{"1":"7ct5","2":"2020","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with T-ACE2"},{"1":"7cxn","2":"2020","3":"Electron Microscopy","4":"Architecture of a SARS-CoV-2 mini replication and transcription complex"},{"1":"7cwu","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike proteins trimer in complex with P17 and FC05 Fabs cocktail"},{"1":"7kfw","2":"2020","3":"X-ray diffraction","4":"Structural basis for a germline-biased antibody response to SARS-CoV-2 (RBD:C1A-B3 Fab)"},{"1":"6zxn","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike protein bound to neutralizing nanobodies (Ty1)"},{"1":"7a5r","2":"2020","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 spike and CR3022 Fab (Non-Uniform Refinement)"},{"1":"7a95","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 1 ACE2 Bound and 1 RBD Erect in Clockwise Direction"},{"1":"7jjc","2":"2020","3":"X-ray diffraction","4":"Crystal structure of neuropilin-1 b1 domain in complex with SARS-CoV-2 S1 C-end rule (CendR) peptide"},{"1":"7a92","2":"2020","3":"Electron Microscopy","4":"Dissociated S1 domain of SARS-CoV-2 Spike bound to ACE2 (Unmasked Refinement)"},{"1":"6zwv","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike Proteins on intact virions: 3 Closed RBDs"},{"1":"6zgi","2":"2020","3":"Electron Microscopy","4":"Furin Cleaved Spike Protein of SARS-CoV-2 in Closed Conformation"},{"1":"6zp7","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in prefusion state (flexibility analysis, 1-up open conformation)"},{"1":"7kmz","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of double ACE2-bound SARS-CoV-2 trimer Spike at pH 7.4"},{"1":"7kne","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of single ACE2-bound SARS-CoV-2 trimer spike at pH 5.5"},{"1":"7kl9","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the ACE2 protein decoy, CTC-445.2 (State 4)"},{"1":"7kkj","2":"2020","3":"X-ray diffraction","4":"Structure of anti-SARS-CoV-2 Spike nanobody mNb6"},{"1":"7kms","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of triple ACE2-bound SARS-CoV-2 trimer spike at pH 7.4"},{"1":"7cz4","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 macro domain in complex with ADP-ribose"},{"1":"7cwl","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein and P17 fab complex with one RBD in close state"},{"1":"6zp5","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in prefusion state (flexibility analysis, 1-up closed conformation)"},{"1":"6zlr","2":"2020","3":"X-ray diffraction","4":"Soaking competent crystal form of the SARS-CoV-2 Receptor Binding Domain (RBD):CR3022 complex."},{"1":"6z97","2":"2020","3":"Electron Microscopy","4":"Structure of the prefusion SARS-CoV-2 spike glycoprotein"},{"1":"6zmi","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human LYAR-80S ribosome complex"},{"1":"6zow","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in prefusion state"},{"1":"7jji","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 3Q-2P full-length prefusion spike trimer (C3 symmetry)"},{"1":"7jmo","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody COVA2-04"},{"1":"7a93","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 2 RBDs Erect"},{"1":"7a91","2":"2020","3":"Electron Microscopy","4":"Dissociated S1 domain of SARS-CoV-2 Spike bound to ACE2 (Non-Uniform Refinement)"},{"1":"7kni","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of Triple ACE2-bound SARS-CoV-2 Trimer Spike at pH 5.5"},{"1":"7cws","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Proteins Trimer in Complex with FC05 and H014 Fabs Cocktail"},{"1":"7cwm","2":"2020","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 spike protein and Fab P17 with one RBD in open state and two RBD in closed state"},{"1":"7kkl","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing nanobody mNb6"},{"1":"7k8p","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C110"},{"1":"7jqb","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 and rabbit 40S ribosome complex"},{"1":"7k8u","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, C104"},{"1":"7d1m","2":"2020","3":"X-ray diffraction","4":"CRYSTAL STRUCTURE OF THE SARS-CoV-2 MAIN PROTEASE COMPLEXED WITH GC376"},{"1":"7k4n","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2E12 neutralizing antibody Fab fragment"},{"1":"7jzn","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with LCB3 (2RBDs open)"},{"1":"7kfy","2":"2020","3":"X-ray diffraction","4":"Structural basis for a germline-biased antibody response to SARS-CoV-2 (RBD:C1A-F10 Fab)"},{"1":"7jwy","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 4.5"},{"1":"7k8o","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C002"},{"1":"7dk3","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer, S-open"},{"1":"7jun","2":"2020","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Joint neutron/X-ray structure of SARS-CoV-2 3CL Mpro at room temperature"},{"1":"7cwt","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with hb27 and fc05 Fab cocktail"},{"1":"7k8r","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C135"},{"1":"7jv4","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2H13 neutralizing antibody (one RBD open)"},{"1":"7jqc","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1, CrPV IRES and rabbit 40S ribosome complex"},{"1":"7jxe","2":"2020","3":"X-ray diffraction","4":"Mapping neutralizing and immunodominant sites on the SARS-CoV-2 spike receptor-binding domain by structure-guided high-resolution serology"},{"1":"7k8v","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C110"},{"1":"7k90","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, C144"},{"1":"7jx3","2":"2020","3":"X-ray diffraction","4":"Mapping neutralizing and immunodominant sites on the SARS-CoV-2 spike receptor-binding domain by structure-guided high-resolution serology"},{"1":"7kfv","2":"2020","3":"X-ray diffraction","4":"Structural basis for a germline-biased antibody response to SARS-CoV-2 (RBD:C1A-B12 Fab)"},{"1":"7k8w","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C119"},{"1":"7jvb","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 spike receptor-binding domain (RBD) with nanobody Nb20"},{"1":"7cwn","2":"2020","3":"Electron Microscopy","4":"P17-H014 Fab cocktail in complex with SARS-CoV-2 spike protein"},{"1":"7kfx","2":"2020","3":"X-ray diffraction","4":"Structural basis for a germline-biased antibody response to SARS-CoV-2 (RBD:C1A-C2 Fab)"},{"1":"7jq0","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI3"},{"1":"7k8x","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C121 (State 1)"},{"1":"7k9z","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with the Fab fragments of neutralizing antibodies 298 and 52"},{"1":"7jzu","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with LCB1 (local refinement of the RBD and LCB1)"},{"1":"7jva","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2A4 neutralizing antibody Fab fragment (local refinement of the receptor-binding domain and Fab variable domains)"},{"1":"7jpz","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI1"},{"1":"7jxd","2":"2020","3":"X-ray diffraction","4":"Mapping neutralizing and immunodominant sites on the SARS-CoV-2 spike receptor-binding domain by structure-guided high-resolution serology"},{"1":"7k8t","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, C002 (State 2)"},{"1":"7k8q","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C121"},{"1":"7jzm","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with LCB3 (local refinement of the RBD and LCB3)"},{"1":"7k3q","2":"2020","3":"X-ray diffraction","4":"An ultra-potent human neutralizing antibody locks the SARS-CoV-2 spike in the closed conformation"},{"1":"7k45","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2E12 neutralizing antibody Fab fragment (local refinement of the RBD and Fab variable domains)"},{"1":"7k43","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2M11 neutralizing antibody Fab fragment"},{"1":"7jq5","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI8"},{"1":"7jq3","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI6"},{"1":"7jzl","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with LCB1 (2RBDs open)"},{"1":"7k8z","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C135"},{"1":"7k8s","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C002 (state 1)"},{"1":"7jq4","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI7"},{"1":"7k8m","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 receptor binding domain in complex with the human neutralizing antibody Fab fragment, C102"},{"1":"7khp","2":"2020","3":"X-ray diffraction","4":"Acyl-enzyme intermediate structure of SARS-CoV-2 Mpro in complex with its C-terminal autoprocessing sequence."},{"1":"7jw0","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S304 neutralizing antibody Fab fragment"},{"1":"7de1","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nucleocapsid protein C-terminal RNA binding domain"},{"1":"7jq1","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI4"},{"1":"7jq2","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI5"},{"1":"7df4","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S-ACE2 complex"},{"1":"7joy","2":"2020","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with its C-terminal autoprocessing sequence."},{"1":"7kkk","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing nanobody Nb6"},{"1":"6z72","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 Macrodomain in complex with ADP-HPM"},{"1":"6zco","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of C-terminal Dimerization Domain of Nucleocapsid Phosphoprotein from SARS-CoV-2, crystal form II"},{"1":"6zgg","2":"2020","3":"Electron Microscopy","4":"Furin Cleaved Spike Protein of SARS-CoV-2 with One RBD Erect"},{"1":"6zoz","2":"2020","3":"Electron Microscopy","4":"Structure of Disulphide-stabilized SARS-CoV-2 Spike Protein Trimer (x1 disulphide-bond mutant, S383C, D985C, K986P, V987P, single Arg S1/S2 cleavage site) in Locked State"},{"1":"7jv2","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2H13 neutralizing antibody Fab fragment (local refinement of the receptor-binding motif and Fab variable domains)"},{"1":"7k8y","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C121 (State 2)"},{"1":"7jv6","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2H13 neutralizing antibody (closed conformation)"},{"1":"7k8n","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C102"},{"1":"7jxc","2":"2020","3":"X-ray diffraction","4":"Mapping neutralizing and immunodominant sites on the SARS-CoV-2 spike receptor-binding domain by structure-guided high-resolution serology"},{"1":"7jvc","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2A4 neutralizing antibody Fab fragment"},{"1":"7k3g","2":"2020","3":"Solid-state NMR","4":"SARS-CoV-2 Envelope Protein Transmembrane Domain: Pentameric Structure Determined by Solid-State NMR"},{"1":"6zgh","2":"2020","3":"Electron Microscopy","4":"Furin Cleaved Spike Protein of SARS-CoV-2 in Intermediate Conformation"},{"1":"7cxm","2":"2020","3":"Electron Microscopy","4":"Architecture of a SARS-CoV-2 mini replication and transcription complex"},{"1":"7cwo","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein RBD and P17 fab complex"},{"1":"7c8t","2":"NA","3":"X-ray diffraction","4":"Complex Structure of SARS-CoV-2 3CL Protease with TG-0205221"},{"1":"7cho","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P5A-1D2 with RBD"},{"1":"5rh1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2010253653 (Mpro-x2643)"},{"1":"5rl1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-27 (Mpro-x3113)"},{"1":"5rh8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4444621965 (Mpro-x2764)"},{"1":"5rmc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z24758179"},{"1":"5rm3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1745658474"},{"1":"5rmh","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1101755952"},{"1":"5rl7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z364321922"},{"1":"5rgw","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4444621910 (Mpro-x2569)"},{"1":"5rl5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-30 (Mpro-x3359)"},{"1":"5rh9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4438424255 (Mpro-x2776)"},{"1":"5rm5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z373768900"},{"1":"5rh2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1129289650 (Mpro-x2646)"},{"1":"5rlo","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1454310449"},{"1":"5rmm","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with POB0066"},{"1":"5rgv","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4444622066 (Mpro-x2563)"},{"1":"5rm2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1741964527"},{"1":"6w37","2":"NA","3":"X-ray diffraction","4":"STRUCTURE OF THE SARS-CoV-2 ORF7A ENCODED ACCESSORY PROTEIN"},{"1":"5rlm","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1650168321"},{"1":"5rl0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-2 (Mpro-x3110)"},{"1":"5rmj","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z68299550"},{"1":"7cdi","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P2C-1F11 with RBD"},{"1":"5rgu","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4444622180 (Mpro-x2562)"},{"1":"5rm1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z426041412"},{"1":"5rgy","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1535580916 (Mpro-x2581)"},{"1":"5rlv","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2467208649"},{"1":"5rln","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z364328788"},{"1":"5rmg","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z285675722"},{"1":"5rlu","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z744754722"},{"1":"5rl9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1703168683"},{"1":"5rma","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z321318226"},{"1":"5rlq","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z285782452"},{"1":"5rlz","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2293643386"},{"1":"5rh4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1530425063 (Mpro-x2659)"},{"1":"5rml","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z85956652"},{"1":"5rly","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2027049478"},{"1":"5rl2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-26 (Mpro-x3115)"},{"1":"5rll","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z425387594"},{"1":"6wqd","2":"NA","3":"X-ray diffraction","4":"The 1.95 A Crystal Structure of the Co-factor Complex of NSP7 and the C-terminal Domain of NSP8 from SARS-CoV-2"},{"1":"5rm4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1639162606"},{"1":"6wji","2":"NA","3":"X-ray diffraction","4":"2.05 Angstrom Resolution Crystal Structure of C-terminal Dimerization Domain of Nucleocapsid Phosphoprotein from SARS-CoV-2"},{"1":"6w7y","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV and SARS-CoV-2 reactive human antibody CR3022"},{"1":"5rh5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4439011520 (Mpro-x2694)"},{"1":"6wc1","2":"NA","3":"X-ray diffraction","4":"Peptide-bound SARS-CoV-2 Nsp9 RNA-replicase"},{"1":"6ywl","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 (Covid-19) NSP3 macrodomain in complex with ADP-ribose"},{"1":"6z2e","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro in complex with the activity-based probe, biotin-PEG(4)-Abu-Tle-Leu-Gln-vinylsulfone"},{"1":"6zsl","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 helicase at 1.94 Angstrom resolution"},{"1":"6ywm","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 (Covid-19) NSP3 macrodomain in complex with MES"},{"1":"6yb7","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease with unliganded active site (2019-nCoV, coronavirus disease 2019, COVID-19)."},{"1":"6ywk","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 (Covid-19) NSP3 macrodomain in complex with HEPES"},{"1":"6zbp","2":"NA","3":"X-ray diffraction","4":"H11-H4 complex with SARS-CoV-2"},{"1":"7ej4","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with a neutralizing antibody RBD-chAb-25"},{"1":"7jib","2":"NA","3":"X-ray diffraction","4":"Room Temperature Crystal Structure of Nsp10/Nsp16 from SARS-CoV-2 with Substrates and Products of 2'-O-methylation of the Cap-1"},{"1":"7jhe","2":"NA","3":"X-ray diffraction","4":"Room Temperature Structure of SARS-CoV-2 Nsp10/Nsp16 Methyltransferase in a Complex with 2'-O-methylated m7GpppA Cap-1 and SAH Determined by Fixed-Target Serial Crystallography"},{"1":"6z2m","2":"NA","3":"X-ray diffraction","4":"H11-D4, SARS-CoV-2 RBD, CR3022 ternary complex"},{"1":"6y84","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease with unliganded active site (2019-nCoV, coronavirus disease 2019, COVID-19)"},{"1":"6xkm","2":"NA","3":"X-ray diffraction","4":"Room Temperature Structure of SARS-CoV-2 NSP10/NSP16 Methyltransferase in a Complex with SAM Determined by Fixed-Target Serial Crystallography"},{"1":"7c8b","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with Z-VAD(OMe)-FMK"},{"1":"6xr3","2":"NA","3":"X-ray diffraction","4":"X-ray Structure of SARS-CoV-2 main protease bound to GRL-024-20 at 1.45 A"},{"1":"7can","2":"NA","3":"X-ray diffraction","4":"Structure of sybody MR17-K99Y in complex with the SARS-CoV-2 S Receptor-binding domain (RBD)"},{"1":"7c8r","2":"NA","3":"X-ray diffraction","4":"Complex Structure of SARS-CoV-2 3CL Protease with TG-0203770"},{"1":"7chp","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P5A-3C8 with RBD"},{"1":"7chs","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P22A-1D1 with RBD"},{"1":"6xkf","2":"NA","3":"X-ray diffraction","4":"The crystal structure of 3CL MainPro of SARS-CoV-2 with oxidized Cys145 (Sulfenic acid cysteine)."},{"1":"6xqb","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 RdRp/RNA complex"},{"1":"5rgx","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1344037997 (Mpro-x2572)"},{"1":"5rlh","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2856434778"},{"1":"5rh6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4439011588 (Mpro-x2703)"},{"1":"5rlt","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z53116498"},{"1":"5rlc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z56923284"},{"1":"5rlb","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z216450634"},{"1":"5rli","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z45617795"},{"1":"5rm9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2856434942"},{"1":"7cdj","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P2C-1A3 with RBD"},{"1":"5rls","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z59181945"},{"1":"6xdh","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of NendoU (Uridylate-specific endoribonuclease, nsp15) from Betacoronavirus SARS-CoV-2"},{"1":"7c8w","2":"NA","3":"X-ray diffraction","4":"Structure of sybody MR17 in complex with the SARS-CoV-2 S receptor-binding domain (RBD)"},{"1":"7c8v","2":"NA","3":"X-ray diffraction","4":"Structure of sybody SR4 in complex with the SARS-CoV-2 S Receptor Binding domain (RBD)"},{"1":"5rme","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z26333434"},{"1":"5rlw","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z45705015"},{"1":"5rm7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z69118333"},{"1":"5r8t","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 main protease screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries"},{"1":"5rl8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z53825177"},{"1":"5rmd","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z57614330"},{"1":"5rm6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z396380540"},{"1":"5rld","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z19735981"},{"1":"5rmf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z54226006"},{"1":"5rl4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-3 (Mpro-x3124)"},{"1":"5rmi","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z53860899"},{"1":"5rlj","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1407673036"},{"1":"5rgt","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4439011607 (Mpro-x2540)"},{"1":"5rgz","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1343543528 (Mpro-x2600)"},{"1":"5rlk","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1509882419"},{"1":"5rle","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1429867185"},{"1":"5rlf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z235341991"},{"1":"5rha","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z147647874 (Mpro-x2779)"},{"1":"5rmb","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2856434920"},{"1":"7c53","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 HR1 motif in complex with pan-CoVs inhibitor EK1"},{"1":"5rh3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1264525706 (Mpro-x2649)"},{"1":"5rlp","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z166605480"},{"1":"5rl6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z198195770"},{"1":"5rmk","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1273312153"},{"1":"5rlg","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z19739650"},{"1":"5rlr","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z822382694"},{"1":"5rm8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1614545742"},{"1":"6wnp","2":"NA","3":"X-ray diffraction","4":"X-ray Structure of SARS-CoV-2 main protease bound to Boceprevir at 1.45 A"},{"1":"5rh7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4439011584 (Mpro-x2705)"},{"1":"5rm0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1492796719"},{"1":"5rh0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1286870272 (Mpro-x2608)"},{"1":"5rl3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-39 (Mpro-x3117)"},{"1":"5rsd","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000331945"},{"1":"5rtl","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388056"},{"1":"5rus","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388081"},{"1":"5s2i","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z57299529"},{"1":"5s3y","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0012"},{"1":"6xoa","2":"NA","3":"X-ray diffraction","4":"The crystal structure of 3CL MainPro of SARS-CoV-2 with C145S mutation"},{"1":"5s29","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z199959602"},{"1":"5rsq","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000158490"},{"1":"5rtn","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000013514509"},{"1":"5rv3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000057162"},{"1":"5s30","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z65532537"},{"1":"7lkr","2":"NA","3":"X-ray diffraction","4":"1.65 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 2a"},{"1":"7lzv","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 19b"},{"1":"5rt2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000008652361"},{"1":"5rti","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000004219237"},{"1":"5ruj","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000404314"},{"1":"5rvu","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002506130"},{"1":"5s2n","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1787627869"},{"1":"5s3x","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0136"},{"1":"5sac","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z59181945"},{"1":"7lzt","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 8b"},{"1":"7ej5","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with a neutralizing antibody RBD-chAb-45"},{"1":"7ali","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the main protease (3CLpro/Mpro) of SARS-CoV-2 at 1.65A resolution (spacegroup P2(1))."},{"1":"7ldx","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - F2X Entry Library E06 fragment"},{"1":"7lkt","2":"NA","3":"X-ray diffraction","4":"1.50 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 2k"},{"1":"7lw4","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 nsp16/nsp10 complex in presence of S-adenosyl-L-homocysteine (SAH)"},{"1":"7lyi","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAWJ9-36-3"},{"1":"7lzw","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 20b (deuterated analog of 19b)"},{"1":"7mbg","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease in orthorhombic space group"},{"1":"7n0g","2":"NA","3":"Electron Microscopy","4":"CryoEm structure of SARS-CoV-2 spike protein (S-6P, 1-up) in complex with sybodies (Sb45)"},{"1":"7nf5","2":"NA","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in spacegroup C2."},{"1":"7lfp","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - F2X Entry Library G05 fragment"},{"1":"7lku","2":"NA","3":"X-ray diffraction","4":"1.65 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 3b (deuterated analog of inhibitor 2a)"},{"1":"7exm","2":"NA","3":"X-ray diffraction","4":"The N-terminal crystal structure of SARS-CoV-2 NSP2"},{"1":"5rt7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000015442276"},{"1":"5ru2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000331715"},{"1":"5rvb","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000014419577"},{"1":"7n5z","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease C145S mutant"},{"1":"7eq4","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of the N-terminus of Nonstructural protein 1 from SARS-CoV-2"},{"1":"7n0i","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 N protein C-terminal domain bound to single-domain antibody E2"},{"1":"7n8c","2":"NA","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Joint X-ray/neutron structure of SARS-CoV-2 main protease (Mpro) in complex with Mcule5948770040"},{"1":"7n0s","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 N protein RNA-binding domain bound to single-domain antibody B6"},{"1":"7lfe","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - F2X Entry Library E03 fragment"},{"1":"7alh","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the main protease (3CLpro/Mpro) of SARS-CoV-2 at 1.65A resolution (spacegroup C2)."},{"1":"7n06","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp15 endoribonuclease post-cleavage state"},{"1":"7ng3","2":"NA","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in spacegroup P1."},{"1":"7ltj","2":"NA","3":"X-ray diffraction","4":"Room-temperature X-ray structure of SARS-CoV-2 main protease (3CL Mpro) in complex with a non-covalent inhibitor Mcule-5948770040"},{"1":"7lzx","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 1c"},{"1":"6z43","2":"NA","3":"Electron Microscopy","4":"Cryo-EM Structure of SARS-CoV-2 Spike : H11-D4 Nanobody Complex"},{"1":"7lyj","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 frameshifting pseudoknot RNA"},{"1":"7jme","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 NSP3 Macro X domain in complex with cyclic AMP"},{"1":"7nbs","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (Mpro) in complex with the HCV NS3/4A inhibitor telaprevir"},{"1":"7mrr","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in Complex with Covalent Inhibitor Leupeptin"},{"1":"5rss","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000006691828"},{"1":"5rty","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000157088"},{"1":"5rv1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000251609"},{"1":"5s20","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with PB1827975385"},{"1":"5s37","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1954800564"},{"1":"5s72","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with BBL029427"},{"1":"5rsk","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000901381520_N3"},{"1":"5rtp","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001679336"},{"1":"5ruw","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000045014941"},{"1":"5s1y","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with STK346965"},{"1":"5s3k","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z219104216"},{"1":"5rsg","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000263392672"},{"1":"5rtd","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000157108"},{"1":"5ruq","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000032199226"},{"1":"5s36","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434938"},{"1":"7mkb","2":"NA","3":"X-ray diffraction","4":"Human leukocyte antigen A*0201 in complex with SARS-CoV-2 epitope YLQPRTFLL"},{"1":"5rt5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000058111"},{"1":"5ru0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388514"},{"1":"5rve","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000736709772"},{"1":"5s1a","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-43406"},{"1":"5s2u","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z85956652"},{"1":"5s41","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00023825"},{"1":"5saf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with EN300-321461"},{"1":"7lzy","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 3c"},{"1":"5rt1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000039810"},{"1":"5ru4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001688638"},{"1":"5rvj","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001612349"},{"1":"5s2h","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434920"},{"1":"5s3h","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434892"},{"1":"5s70","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with EN300-181428"},{"1":"7n33","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp15 endoribonuclease pre-cleavage state"},{"1":"5rsb","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001674697"},{"1":"5run","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000194295"},{"1":"5s3n","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z287484230"},{"1":"5s1i","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-301084"},{"1":"5rts","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000159004"},{"1":"5s27","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1262398530"},{"1":"5s3r","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0014"},{"1":"5rt4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000051581"},{"1":"5rug","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000038389"},{"1":"5rvo","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000013514509"},{"1":"7lke","2":"NA","3":"X-ray diffraction","4":"X-ray crystal structure of the SARS-CoV-2 main protease in space group C2"},{"1":"7me0","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 NSP15 NendoU at pH 6.0"},{"1":"5rvn","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332748"},{"1":"5s2g","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z321318226"},{"1":"5s44","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00024890"},{"1":"5rs9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000007636250"},{"1":"5rtm","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000002005"},{"1":"5rur","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000017744334"},{"1":"5s1u","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-52144"},{"1":"5s38","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1745658474"},{"1":"5s4a","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z955123498"},{"1":"5sai","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z1424343998"},{"1":"7lkx","2":"NA","3":"X-ray diffraction","4":"1.60 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 3e"},{"1":"7m02","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 17c"},{"1":"7kxk","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with Fab 15033-7, 2-\"up\"-1-\"down\" conformation"},{"1":"7lkw","2":"NA","3":"X-ray diffraction","4":"1.65 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 3d (deuterated analog of inhibitor 3c)"},{"1":"7m00","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 13c"},{"1":"5rtg","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000395673"},{"1":"5s26","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z605596346"},{"1":"7lbn","2":"NA","3":"X-ray diffraction","4":"X-ray crystal structure of the SARS-CoV-2 main protease with Calpain I Inhibitor"},{"1":"5rsi","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000374420934"},{"1":"5rtc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000006490906"},{"1":"5ruk","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000161692"},{"1":"5rvs","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000159004"},{"1":"5s18","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-321461"},{"1":"5s35","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z68404778"},{"1":"5s4b","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z3219959731"},{"1":"5rt3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000039281982"},{"1":"5s1m","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with STK497968"},{"1":"5s32","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z26781943"},{"1":"5s47","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z940713508"},{"1":"5sa7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z1673618163"},{"1":"5s46","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z57131035"},{"1":"5sa4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z239136710"},{"1":"7kph","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease in mature form"},{"1":"7kza","2":"NA","3":"X-ray diffraction","4":"Potent SARS-CoV-2 binding and neutralization through maturation of iconic SARS-CoV-1antibodies"},{"1":"7l6t","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer in Complex with (m7GpppA2m)pUpUpApApA (Cap-1), S-Adenosyl-L-homocysteine (SAH) and two Magnesium (Mg) ions."},{"1":"7ofu","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like protease PLpro in complex with 3, 4-Dihydroxybenzoic acid, methyl ester"},{"1":"5s71","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with FUZS-5"},{"1":"5sa8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z68299550"},{"1":"5rsw","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000337835"},{"1":"5ru9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000165882"},{"1":"5rvh","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000265642"},{"1":"5s1o","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with STL414928"},{"1":"5s3b","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1741966151"},{"1":"5s4h","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF048"},{"1":"5rso","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000000226"},{"1":"5rtt","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000873830"},{"1":"5rux","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002020050"},{"1":"5s2v","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1186029914"},{"1":"5s4g","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF005"},{"1":"5s24","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-697611"},{"1":"5s3a","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1562205518"},{"1":"5s4i","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF051"},{"1":"7cyh","2":"NA","3":"Electron Microscopy","4":"Binding interface of SARS-CoV-2 RBD and its neutralizing antibody HB27"},{"1":"5ruo","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001683100"},{"1":"5rvv","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000020269197"},{"1":"5s2s","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434894"},{"1":"5s42","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00023833"},{"1":"5s74","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 Nsp3 macrodomain"},{"1":"7kyu","2":"NA","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 Main Protease with the formation of Cys145-1H-indole-5-carboxylate"},{"1":"7be7","2":"NA","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"7k40","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in Complex with Covalent Inhibitor Boceprevir at 1.35 A Resolution"},{"1":"7kx5","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with noncovalent inhibitor Jun8-76-3A"},{"1":"7jr3","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease crystallized under reducing conditions"},{"1":"7cwc","2":"NA","3":"X-ray diffraction","4":"Ambient-Temperature Serial Femtosecond X-ray Crystal structure of SARS-CoV-2 Main Protease at 2.1 A Resolution (P212121)"},{"1":"7cut","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with Z-VAD-FMK"},{"1":"5s2e","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1152242726"},{"1":"5s3t","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0128"},{"1":"7n0r","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 N protein RNA-binding domain bound to single-domain antibody C2"},{"1":"7nfv","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like protease PLpro"},{"1":"7n6n","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease C145S mutant in complex with N and C-terminal residues"},{"1":"5s2b","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z373769142"},{"1":"5s3w","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0135"},{"1":"5s2y","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z19727416"},{"1":"5s49","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z56866006"},{"1":"7bfb","2":"NA","3":"X-ray diffraction","4":"Crystal structure of ebselen covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"7jfq","2":"NA","3":"X-ray diffraction","4":"The crystal structure of 3CL MainPro of SARS-CoV-2 with de-oxidized C145"},{"1":"7mng","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in Complex with Covalent Inhibitor VBY-825 (Partial Occupancy)"},{"1":"7msx","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp2"},{"1":"5rsj","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000089254160_N3"},{"1":"5rtj","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332752"},{"1":"5rul","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000163774"},{"1":"5rvf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000082473428_N3"},{"1":"5s2z","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z126932614"},{"1":"5s4e","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2301685688"},{"1":"5rse","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000336438345"},{"1":"5rto","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388302"},{"1":"5ruu","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000438614"},{"1":"7ofs","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like protease PLpro in complex with 4-(2-hydroxyethyl)phenol"},{"1":"5s2j","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z509756472"},{"1":"5s45","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00024773"},{"1":"5saa","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z319891284"},{"1":"5rvt","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002582714"},{"1":"5s2d","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z369936976"},{"1":"5s3p","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1238477790"},{"1":"5sae","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z3219959731"},{"1":"5s28","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z409974522"},{"1":"5s3j","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1324853681"},{"1":"7oft","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like protease PLpro in complex with p-hydroxybenzaldehyde"},{"1":"7kxj","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with Fab 15033-7, 3-\"up\", asymmetric"},{"1":"7kmg","2":"NA","3":"X-ray diffraction","4":"LY-CoV555 neutralizing antibody against SARS-CoV-2"},{"1":"7ng6","2":"NA","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in spacegroup P1 in absence of DTT."},{"1":"7nio","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 helicase APO form"},{"1":"7lw3","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 nsp16/nsp10 complex in presence of Cap-1 analog (m7GpppAmU) and SAH"},{"1":"5rsn","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000064576"},{"1":"5ru3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000161696"},{"1":"5rv2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000311783"},{"1":"5s1w","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z838838708"},{"1":"5s2t","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z26781964"},{"1":"5s48","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1741982125"},{"1":"5sa6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z2856434783"},{"1":"7mgs","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in complex with N-terminal autoprocessing substrate"},{"1":"5s2w","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1407672867"},{"1":"5s4j","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF054"},{"1":"7koa","2":"NA","3":"X-ray diffraction","4":"Room Temperature Structure of SARS-CoV-2 Nsp10/16 Methyltransferase in a Complex with Cap-0 and SAM Determined by Pink-Beam Serial Crystallography"},{"1":"7kvr","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - FMAX Library E09 fragment"},{"1":"5rsm","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000001099"},{"1":"5rtv","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001698894"},{"1":"5rv5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000008578948"},{"1":"5s2p","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z927746322"},{"1":"5s4f","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF003"},{"1":"7kzb","2":"NA","3":"X-ray diffraction","4":"Potent SARS-CoV-2 binding and neutralization through maturation of iconic SARS-CoV-1antibodies"},{"1":"5sad","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z425449682"},{"1":"5rsf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000026180281"},{"1":"5rtb","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000006534965"},{"1":"5rue","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000000922"},{"1":"5rv7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000003954002"},{"1":"5s2a","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1263529624"},{"1":"5s3l","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z54628578"},{"1":"5rsr","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000158650"},{"1":"5rtu","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000159056"},{"1":"5rv6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000158540"},{"1":"7kvl","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - FMAX Library E01 fragment"},{"1":"5rsv","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000340465"},{"1":"5ru1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000034687"},{"1":"5rv9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388150"},{"1":"5s2f","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z44592329"},{"1":"5s43","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00024661"},{"1":"7n0h","2":"NA","3":"Electron Microscopy","4":"CryoEM structure of SARS-CoV-2 spike protein (S-6P, 2-up) in complex with sybodies (Sb45)"},{"1":"5s2c","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z45612755"},{"1":"5s3s","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0103"},{"1":"7lks","2":"NA","3":"X-ray diffraction","4":"1.70 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 2f"},{"1":"5rtk","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000164504"},{"1":"5ruh","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000123600"},{"1":"7nby","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SU3327 (halicin) covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"7nng","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 helicase in complex with Z2327226104"},{"1":"7m03","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 18c"},{"1":"5rst","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332673"},{"1":"5rtw","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000164777"},{"1":"5ruv","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000019015194"},{"1":"5rv4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000039224"},{"1":"5rsx","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388262"},{"1":"5ru8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000154817"},{"1":"5rvd","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000263980802"},{"1":"5s1e","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with AB-601_30915014"},{"1":"5s3z","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0140"},{"1":"7msw","2":"NA","3":"Electron Microscopy","4":"Full length SARS-CoV-2 Nsp2"},{"1":"5rsh","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000274438208"},{"1":"5s3f","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z57446103"},{"1":"7lgo","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the nucleic acid binding domain (NAB) of Nsp3 from SARS-CoV-2"},{"1":"7nbr","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (Mpro) in complex with the HCV NS3/4A inhibitor boceprevir"},{"1":"7d30","2":"NA","3":"X-ray diffraction","4":"Structure of sybody MR17-SR31 fusion in complex with the SARS-CoV-2 S Receptor Binding domain (RBD)"},{"1":"7ctt","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of Favipiravir bound to replicating polymerase complex of SARS-CoV-2 in the pre-catalytic state."},{"1":"7cx9","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with INZ-1"},{"1":"7dk0","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with MW05 Fab"},{"1":"7e3j","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD binding to dog ACE2"},{"1":"7d2z","2":"NA","3":"X-ray diffraction","4":"Structure of sybody SR31 in complex with the SARS-CoV-2 S Receptor Binding domain (RBD)"},{"1":"7d47","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Papain-like protease C111S"},{"1":"5rtx","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000090873"},{"1":"5rs7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000034618676"},{"1":"5rt8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000161908"},{"1":"5rud","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000008615114"},{"1":"5rvk","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002977810"},{"1":"5ruc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000005878"},{"1":"5rvi","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000084843283"},{"1":"5rsc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000003888754"},{"1":"5rth","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000156863"},{"1":"5ruf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000016989831"},{"1":"5rvr","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000016052862"},{"1":"5rs8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000001601"},{"1":"5rtf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002047514"},{"1":"5rum","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000008861082"},{"1":"5rvp","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000154817"},{"1":"7lkd","2":"NA","3":"X-ray diffraction","4":"X-ray crystal structure of the SARS-CoV-2 main protease in space group P21."},{"1":"5s3g","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z384468096"},{"1":"7ntk","2":"NA","3":"X-ray diffraction","4":"PALS1 PDZ1 domain with SARS-CoV-2_E PBM complex"},{"1":"7n89","2":"NA","3":"X-ray diffraction","4":"Room-temperature X-ray structure of SARS-CoV-2 main protease C145A mutant in complex with substrate Ac-SAVLQSGF-CONH2"},{"1":"7nn0","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 helicase in complex with AMP-PNP"},{"1":"7kmh","2":"NA","3":"X-ray diffraction","4":"LY-CoV488 neutralizing antibody against SARS-CoV-2"},{"1":"5sag","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with EN300-1605072"},{"1":"7lzu","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 12b"},{"1":"7m04","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 21c"},{"1":"7m4r","2":"NA","3":"Electron Microscopy","4":"Structural basis for SARS-CoV-2 envelope protein in recognition of human cell junction protein PALS1"},{"1":"7mgr","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in complex with nsp8/9 substrate peptide"},{"1":"7mky","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 frameshifting pseudoknot RNA"},{"1":"5rsz","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000004218283"},{"1":"5rtz","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000404062"},{"1":"5rv0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000039994"},{"1":"5s1k","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-105873"},{"1":"5s34","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434941"},{"1":"5s39","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z165170770"},{"1":"5sah","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with EN300-100112"},{"1":"7m01","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 14c"},{"1":"5s2q","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z26781952"},{"1":"5s40","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00023824"},{"1":"5s6z","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with PB2255187532"},{"1":"5sab","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z31504642"},{"1":"7kmi","2":"NA","3":"X-ray diffraction","4":"LY-CoV481 neutralizing antibody against SARS-CoV-2"},{"1":"7kzc","2":"NA","3":"X-ray diffraction","4":"Potent SARS-CoV-2 binding and neutralization through maturation of iconic SARS-CoV-1antibodies"},{"1":"5rt9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388280"},{"1":"5rup","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000004976927"},{"1":"5s2r","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z57292369"},{"1":"5s3u","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0041"},{"1":"7l5d","2":"NA","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 Main Protease in complex with demethylated analog of masitinib"},{"1":"5rsu","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000002055"},{"1":"5rvc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000933940912"},{"1":"5s2m","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z56827661"},{"1":"5s3o","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z102768020"},{"1":"5s2k","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z445856640"},{"1":"5s3v","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0120"},{"1":"7l4z","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike RBD in complex with cyclic peptide"},{"1":"5rta","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332540"},{"1":"5rui","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332651"},{"1":"5rsp","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002560357"},{"1":"5rtq","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000019015078"},{"1":"5ruz","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000019685960"},{"1":"5s22","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z145120524"},{"1":"5s3c","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434937"},{"1":"7lkv","2":"NA","3":"X-ray diffraction","4":"1.55 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 3c"},{"1":"7lzz","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 5c"},{"1":"7lyh","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAWJ9-36-1"},{"1":"7bf4","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 macrodomain in complex with GMP"},{"1":"7bh9","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 RBD-62 in complex with ACE2 peptidase domain"},{"1":"7kgt","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201 in complex with SARS-CoV-2 N226-234"},{"1":"5rt0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002582714"},{"1":"5rtr","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000018169763"},{"1":"5ruy","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000013517187"},{"1":"7kgs","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201 in complex with SARS-CoV-2 N138-146"},{"1":"5s31","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1741959530"},{"1":"7cuu","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with MG132"},{"1":"7b3e","2":"NA","3":"X-ray diffraction","4":"Crystal structure of myricetin covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2"},{"1":"5ru7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000003591110"},{"1":"5rv8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000039575"},{"1":"5s2x","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1139246057"},{"1":"5s4d","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1741982441"},{"1":"5s73","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 Nsp3 macrodomain"},{"1":"5sa9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z2697514548"},{"1":"5s2o","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z645232558"},{"1":"5s3i","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z50145861"},{"1":"7keh","2":"NA","3":"X-ray diffraction","4":"Crystal structure from SARS-CoV-2 NendoU NSP15"},{"1":"7bf6","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 macrodomain in complex with remdesivir metabolite GS-441524"},{"1":"7dpm","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with MW06 Fab"},{"1":"7jz0","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer in Complex with (m7GpppA2m)pUpUpApApA (Cap-1) and S-Adenosyl-L-homocysteine (SAH)."},{"1":"7l6r","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer in Complex with (m7GpppA2m)pUpUpApApA (Cap-1), S-Adenosyl-L-homocysteine (SAH) and Manganese (Mn)."},{"1":"7k0f","2":"NA","3":"X-ray diffraction","4":"1.65 A resolution structure of SARS-CoV-2 3CL protease in complex with a deuterated GC376 alpha-ketoamide analog (compound 5)"},{"1":"5rvl","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000149580"},{"1":"7eam","2":"NA","3":"X-ray diffraction","4":"immune complex of SARS-CoV-2 RBD and cross-neutralizing antibody 7D6"},{"1":"7jx6","2":"NA","3":"X-ray diffraction","4":"STRUCTURE OF THE SARS-CoV-2 ORF8 ENCODED ACCESSORY PROTEIN"},{"1":"5s4k","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with FMOOA000509a"},{"1":"7eb4","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, two RBD-up conformation 1"},{"1":"7k3t","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in the Apo Form at 1.2 A Resolution"},{"1":"7cyp","2":"NA","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 spike trimer with its neutralizing antibody HB27"},{"1":"7ju7","2":"NA","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 Main Protease in complex with masitinib"},{"1":"7k6d","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease Co-Crystal Structure with Telaprevir Determined from Crystals Grown with 40 nL Acoustically Ejected Mpro Droplets at 1.48 A Resolution (Cryo-protected)"},{"1":"7jpe","2":"NA","3":"X-ray diffraction","4":"Room Temperature Structure of SARS-CoV-2 Nsp10/Nsp16 Methyltransferase in a Complex with m7GpppA Cap-0 and SAM Determined by Fixed-Target Serial Crystallography"},{"1":"5ru5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000098208711"},{"1":"7kf4","2":"NA","3":"X-ray diffraction","4":"Crystal structure from SARS-CoV-2 NendoU NSP15"},{"1":"7jrn","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the wild type SARS-CoV-2 papain-like protease (PLPro) with inhibitor GRL0617"},{"1":"7kag","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the ubiquitin-like domain 1 (Ubl1) of Nsp3 from SARS-CoV-2"},{"1":"7e5y","2":"NA","3":"X-ray diffraction","4":"Molecular basis for neutralizing antibody 2B11 targeting SARS-CoV-2 RBD"},{"1":"5rua","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000033986325"},{"1":"5rte","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000013283576"},{"1":"7eb0","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, one RBD-up conformation 2"},{"1":"7dpv","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with 7-O-methyl-dihydromyricetin"},{"1":"7e8m","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P2C-1F11 with mutated RBD"},{"1":"5s1g","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-108952"},{"1":"5s3d","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z30820160"},{"1":"7ean","2":"NA","3":"X-ray diffraction","4":"immune complex of SARS-CoV-2 RBD and cross-neutralizing antibody 6D6"},{"1":"7kfi","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - apo structure"},{"1":"5rsl","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000365052868"},{"1":"5s3q","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0013"},{"1":"5rvg","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000400552187_N3"},{"1":"5s2l","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2234920345"},{"1":"7d4g","2":"NA","3":"Electron Microscopy","4":"A proof of concept for neutralizing antibody-guided vaccine design against SARS-CoV-2"},{"1":"7eb5","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, two RBD-up conformation 2"},{"1":"5rob","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 helicase"},{"1":"5rsy","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000004787230"},{"1":"6yz5","2":"NA","3":"X-ray diffraction","4":"H11-D4 complex with SARS-CoV-2 RBD"},{"1":"6z4u","2":"NA","3":"X-ray diffraction","4":"X-ray Crystallographic Structure of Orf9b from SARS-CoV-2"},{"1":"5s6y","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z56900771"},{"1":"7bf3","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 macrodomain in complex with adenosine"},{"1":"7bb2","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the main protease (3CLpro/Mpro) of SARS-CoV-2 at 1.6A resolution (spacegroup P2(1)2(1)2(1))"},{"1":"7djz","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with MW01 Fab"},{"1":"7jyy","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer in Complex with (m7GpppA)pUpUpApApA (Cap-0) and S-Adenosylmethionine (SAM)."},{"1":"7jyc","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in Complex with Covalent Inhibitor Narlaprevir"},{"1":"7dpp","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with myricetin"},{"1":"7eb3","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, one RBD-up conformation 3"},{"1":"7k6e","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease Co-Crystal Structure with Telaprevir Determined from Crystals Grown with 40 nL Acoustically Ejected Mpro Droplets at 1.63 A Resolution (Direct Vitrification)"},{"1":"7jr4","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease with alternative conformation of the active site promoted by methylene-bridged cysteine and lysine residues"},{"1":"7eaz","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, one RBD-up conformation 1"},{"1":"7bgp","2":"NA","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in absence of DTT."},{"1":"7cwb","2":"NA","3":"X-ray diffraction","4":"Ambient-Temperature Serial Femtosecond X-ray Crystal structure of SARS-CoV-2 Main Protease at 1.9 A Resolution (C121)"},{"1":"7dpu","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with 7-O-methyl-myricetin"},{"1":"5rva","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000016343276"},{"1":"5s1c","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z3034471507"},{"1":"5s3m","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z45656995"},{"1":"5rt6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000156509"},{"1":"5ru6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001442764"},{"1":"5rvq","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002508153"},{"1":"5s1q","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-17035"},{"1":"5s3e","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z274553586"},{"1":"5s6x","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z2889976755"},{"1":"5sa5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z1530301542"},{"1":"7bf5","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 macrodomain in complex with ADP-ribose-phosphate (ADP-ribose-2'-phosphate, ADPRP)"},{"1":"5rut","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000161958"},{"1":"5rvm","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000157088"},{"1":"5s1s","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1613477500"},{"1":"5s33","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z906021418"},{"1":"5s4c","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1954800348"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p><a href="datasets/sars-cov2-structures.json"><strong>Download raw dataset in JSON format</strong></a></p>
</div>

<div id="rmd-source-code">LS0tCnRpdGxlOiAiU0FSUy1Db1YtMiBzdHJ1Y3R1cmVzIgotLS0KCmBgYHtyIHNldHVwLCBpbmNsdWRlPUZBTFNFfQprbml0cjo6b3B0c19jaHVuayRzZXQobWVzc2FnZSA9IEZBTFNFLCB3YXJuaW5nID0gRkFMU0UpCmBgYAoKKipMYXN0IHVwZGF0ZWQgb24gYHIgZGF0ZSgpYC4qKgoKIyMgSG93IHRvIHJlLXVzZSB0aGlzIHdvcmsKCklmIHlvdSB1c2UgdGhlc2UgZmlndXJlcyBpbiB5b3VyIG93biB3b3JrLCBwbGVhc2UgY2l0ZSB0aGlzIHdlYnNpdGU6CjxodHRwczovL2RvaS5vcmcvMTAuNTI4MS96ZW5vZG8uMzQ3MDExOT4KCmBgYHtyIExvYWQgcGFja2FnZXMsIGRvd25sb2FkIGRhdGEgYW5kIHByZXBhcmUgZGF0YXNldH0KIyBMb2FkIHJlcXVpcmVkIHBhY2thZ2VzCmxpYnJhcnkobWFncml0dHIpCmxpYnJhcnkoZHBseXIpCmxpYnJhcnkoanNvbmxpdGUpCmxpYnJhcnkoZm9yY2F0cykKbGlicmFyeShnZ3Bsb3QyKQpsaWJyYXJ5KHBsb3RseSkKbGlicmFyeShoZXJlKQpsaWJyYXJ5KGx1YnJpZGF0ZSkKCiMgV2UgbmVlZCB0d28gcXVlcmllczogb25lIGZvciB1cHBlcmNhc2UgdGl0bGVzLCB0aGUgb3RoZXIgZm9yIGxvd2VyY2FzZSBvbmVzCnBkYl9xdWVyeSA8LSAnaHR0cHM6Ly93d3cuZWJpLmFjLnVrL3BkYmUvc2VhcmNoL3BkYi9zZWxlY3Q/cT10aXRsZToqU0FSUy1Db1YtMiolMjBBTkQlMjBzdGF0dXM6UkVMJmZsPXBkYl9pZCxkZXBvc2l0aW9uX2RhdGUsY2l0YXRpb25feWVhcix0aXRsZSxleHBlcmltZW50YWxfbWV0aG9kLHJlc29sdXRpb24sb3JnYW5pc21fc2NpZW50aWZpY19uYW1lLG1vbGVjdWxlX25hbWUsbW9sZWN1bGVfdHlwZSZyb3dzPTEwMDAwMDAmd3Q9anNvbicKCiMgVGhpcyBpcyBhIGhlbHBlciBwaXBlbGluZSB0byBleHRyYWN0IGRhdGEKZGlnX3VwX2RhdGEgPC0gLiAlPiUKICAgIC4kcmVzcG9uc2UgJT4lCiAgICAuJGRvY3MgJT4lCiAgICBhc190aWJibGUoKQoKIyBRdWVyeSB0aGUgUERCIGFuZCBjbGVhbiB1cCBkYXRhCnBkYl9kYXRhIDwtIHBkYl9xdWVyeSAlPiUgCiAgICBmcm9tSlNPTigpICU+JSAKICAgIGRpZ191cF9kYXRhKCkgJT4lIAogICAgYmluZF9yb3dzKCkgJT4lIAogICAgbXV0YXRlKAogICAgICBleHBlcmltZW50YWxfbWV0aG9kICAgPSBhc19mYWN0b3IoYXMuY2hhcmFjdGVyKGV4cGVyaW1lbnRhbF9tZXRob2QpKSwKICAgICAgZGVwb3NpdGlvbl9kYXRlICAgICAgID0gYXNfZGF0ZSh5bWRfaG1zKGRlcG9zaXRpb25fZGF0ZSkpLAogICAgICBjaXRhdGlvbl95ZWFyICAgICAgICAgPSBhcy5pbnRlZ2VyKGNpdGF0aW9uX3llYXIpLAogICAgICBtb2xlY3VsZV9uYW1lICAgICAgICAgPSBhcy5jaGFyYWN0ZXIobW9sZWN1bGVfbmFtZSksCiAgICAgIG1vbGVjdWxlX3R5cGUgICAgICAgICA9IGFzX2ZhY3Rvcihhcy5jaGFyYWN0ZXIobW9sZWN1bGVfdHlwZSkpCiAgICAgICkgJT4lIAogICAgZGlzdGluY3QocGRiX2lkLCAua2VlcF9hbGwgPSBUUlVFKQpgYGAKCiMjIEFib3V0IHRoaXMgc2VjdGlvbgoKV2l0aCB0aGUgb25nb2luZyBbQ09WSUQtMTkKcGFuZGVtaWNdKGh0dHBzOi8vZW4ud2lraXBlZGlhLm9yZy93aWtpL0NPVklELTE5X3BhbmRlbWljKSwgbmV3IHN0cnVjdHVyZXMgb2YKcHJvdGVpbnMgZnJvbQpbU0FSUy1Db1YtMl0oaHR0cHM6Ly9lbi53aWtpcGVkaWEub3JnL3dpa2kvU2V2ZXJlX2FjdXRlX3Jlc3BpcmF0b3J5X3N5bmRyb21lX2Nvcm9uYXZpcnVzXzIpIGFyZSBkZXBvc2l0ZWQgdG8gdGhlIFBEQiBhdCBhIGZhc3QgcGFjZS4gVGhlIHZpc3VhbGl6YXRpb25zIHByZXNlbnRlZCBoZXJlCndvbid0IGhlbHAgY3VyYiB0aGUgcGFuZGVtaWMgKGFuZCBhcmUgbm90IG1lZGljYWwgYWR2aWNlLCBvYnZpb3VzbHkpLCBidXQgSSBob3BlCnRoZXkgd2lsbCBtYWtlIGN1cmlvdXMgYW5kIGhlbHBsZXNzIG1pbmRzIChsaWtlIG15c2VsZikgZ2V0IGEgZmVlbCBmb3IgdGhlCnJlc2VhcmNoIGVmZm9ydCBjdXJyZW50bHkgYmVpbmcgZGVwbG95ZWQgYWdhaW5zdCBpdC4KClRoZSBmb2xsb3dpbmcgcmVzb3VyY2VzIGFyZSBhIGxvdCBtb3JlIHJlbGV2YW50IGFuZCBhY3Rpb25hYmxlOgoKLSBbQ29yb25hdmlydXMgU3RydWN0dXJhbCBUYXNrIEZvcmNlXShodHRwczovL2luc2lkZWNvcm9uYS5uZXQvKQotIFtQREJlIENPVklELTE5IERhdGEgUG9ydGFsXShodHRwczovL3d3dy5lYmkuYWMudWsvcGRiZS9jb3ZpZC0xOSkKLSBbQ3Jvd2RmaWdodCBDT1ZJRC0xOV0oaHR0cHM6Ly9jcm93ZGZpZ2h0Y292aWQxOS5vcmcvKQoKIyMgU0FSUy1Db1YtMiBzdHJ1Y3R1cmVzIGJ5IGRlcG9zaXRpb24gZGF0ZQoKQWxsIGZpZ3VyZXMgYXJlIGludGVyYWN0aXZlICh5b3UgY2FuIHpvb20gaW4sIGFuZCBob3ZlcmluZyBvdmVyIGVsZW1lbnRzIHdpbGwKc2hvdyBtb3JlIGluZm9ybWF0aW9uKS4KCmBgYHtyIFNBUlMtQ29WLTIgc3RydWN0dXJlcyBieSB5ZWFyfQpzYXJzX2NvdjJfc3RydWN0dXJlc195ZWFyIDwtIHBkYl9kYXRhICU+JSAKICAgIGdncGxvdCgpICsKICAgIGdlb21fYmFyKG1hcHBpbmcgPSBhZXMoeCA9IGRlcG9zaXRpb25fZGF0ZSkpICsKICAgIGdndGl0bGUoIlN0cnVjdHVyZXMgb2YgU0FSUy1Db1YtMiBieSBkZXBvc2l0aW9uIGRhdGUiKSArCiAgICB4bGFiKCJEZXBvc2l0aW9uIGRhdGUiKSArCiAgICB5bGFiKCJOdW1iZXIgb2YgUERCIGVudHJpZXMiKSArCiAgICB0aGVtZV9idygpCmdncGxvdGx5KHNhcnNfY292Ml9zdHJ1Y3R1cmVzX3llYXIpCmBgYAoKVGhlIHR3byBub3RpY2VhYmxlIHNwaWtlcyAoNjggc3RydWN0dXJlcyBkZXBvc2l0ZWQgb24gTWFyY2ggMTV0aCBhbmQgMTMKc3RydWN0dXJlcyBkZXBvc2l0ZWQgb24gQXByaWwgN3RoKSBjb21lIGZyb20gW2EgZnJhZ21lbnQgc2NyZWVuaW5nCmV4cGVyaW1lbnRdKGh0dHBzOi8vd3d3LmRpYW1vbmQuYWMudWsvY292aWQtMTkvZm9yLXNjaWVudGlzdHMvTWFpbi1wcm90ZWFzZS1zdHJ1Y3R1cmUtYW5kLVhDaGVtLmh0bWwpCnBlcmZvcm1lZCBhdCBEaWFtb25kIExpZ2h0IFNvdXJjZS4gQWxsIHRoZXNlIHN0cnVjdHVyZXMgYXJlIHRoZSBzYW1lIHByb3RlaW4KKG1haW4gcHJvdGVhc2UpIGluIGNvbXBsZXggd2l0aCBjYW5kaWRhdGUgbW9sZWN1bGVzIGZvciBkcnVnIGRlc2lnbi4KCmBgYHtyIFNhdmUgU1ZHIGZpbGUgb2YgeWVhciBncmFwaCwgaW5jbHVkZT1GQUxTRX0KIyBTYXZlIGZpZ3VyZSBmb3IgZG93bmxvYWQKaWYgKCFkaXIuZXhpc3RzKGhlcmUoImZpZ3VyZXMiKSkpIHsKICAgIGRpci5jcmVhdGUoaGVyZSgiZmlndXJlcyIpKQp9Cmdnc2F2ZShmaWxlbmFtZSA9ICJzYXJzLWNvdjItc3RydWN0dXJlcy1ieS15ZWFyLnN2ZyIsCiAgICAgICBwbG90ID0gc2Fyc19jb3YyX3N0cnVjdHVyZXNfeWVhciwKICAgICAgIGRldmljZSA9ICJzdmciLAogICAgICAgcGF0aCA9IGhlcmUoImZpZ3VyZXMiKSkKYGBgCgpbKipEb3dubG9hZCBmaWd1cmUgaW4gU1ZHIGZvcm1hdCoqXShmaWd1cmVzL3NhcnMtY292Mi1zdHJ1Y3R1cmVzLWJ5LXllYXIuc3ZnKQoKIyMgU0FSUy1Db1YtMiBzdHJ1Y3R1cmVzIGJ5IGV4cGVyaW1lbnRhbCBtZXRob2QKCmBgYHtyIFNBUlMtQ29WLTIgc3RydWN0dXJlcyBieSBleHBlcmltZW50YWwgbWV0aG9kfQpzYXJzX2NvdjJfc3RydWN0dXJlc19tZXRob2QgPC0gcGRiX2RhdGEgJT4lIAogICAgZ2dwbG90KCkgKwogICAgZ2VvbV9iYXIobWFwcGluZyA9IGFlcyh4ID0gZXhwZXJpbWVudGFsX21ldGhvZCwgZmlsbCA9IGV4cGVyaW1lbnRhbF9tZXRob2QpKSArCiAgICBndWlkZXMoZmlsbCA9IGd1aWRlX2xlZ2VuZCh0aXRsZSA9ICJFeHBlcmltZW50YWwgbWV0aG9kIikpICsKICAgIGdndGl0bGUoIlN0cnVjdHVyZXMgb2YgU0FSUy1Db1YtMiBieSBleHBlcmltZW50YWwgbWV0aG9kIikgKwogICAgeGxhYigiIikgKwogICAgeWxhYigiTnVtYmVyIG9mIFBEQiBlbnRyaWVzIikgKwogICAgdGhlbWVfYncoKQpnZ3Bsb3RseShzYXJzX2NvdjJfc3RydWN0dXJlc19tZXRob2QpCmBgYAoKYGBge3IgU2F2ZSBTVkcgZmlsZSBvZiBtZXRob2QgZ3JhcGgsIGluY2x1ZGU9RkFMU0V9CiMgU2F2ZSBmaWd1cmUgZm9yIGRvd25sb2FkCmlmICghZGlyLmV4aXN0cyhoZXJlKCJmaWd1cmVzIikpKSB7CiAgICBkaXIuY3JlYXRlKGhlcmUoImZpZ3VyZXMiKSkKfQpnZ3NhdmUoZmlsZW5hbWUgPSAic2Fycy1jb3YyLXN0cnVjdHVyZXMtYnktbWV0aG9kLnN2ZyIsCiAgICAgICBwbG90ID0gc2Fyc19jb3YyX3N0cnVjdHVyZXNfbWV0aG9kLAogICAgICAgZGV2aWNlID0gInN2ZyIsCiAgICAgICBwYXRoID0gaGVyZSgiZmlndXJlcyIpKQpgYGAKClsqKkRvd25sb2FkIGZpZ3VyZSBpbiBTVkcgZm9ybWF0KipdKGZpZ3VyZXMvc2Fycy1jb3YyLXN0cnVjdHVyZXMtYnktbWV0aG9kLnN2ZykKCgojIyBEYXRhc2V0CgpUaGUgZ3JhcGhzIHByZXNlbnRlZCBhYm92ZSBhcmUgZGVyaXZlZCBmcm9tIHRoZSBmb2xsb3dpbmcgZGF0YXNldDoKCmBgYHtyIEVudGlyZSBkYXRhc2V0fQojIEZvcm1hdCB0YWJsZSBmb3IgZGlzcGxheQpwZGJfdGFibGUgPC0gcGRiX2RhdGEgJT4lIAogICAgYXJyYW5nZShkZXNjKGNpdGF0aW9uX3llYXIpKSAlPiUgCiAgICBzZWxlY3QoYFBEQiBjb2RlYCA9IHBkYl9pZCwKICAgICAgICAgICBgQ2l0YXRpb24geWVhcmAgPSBjaXRhdGlvbl95ZWFyLAogICAgICAgICAgIGBFeHBlcmltZW50YWwgbWV0aG9kYCA9IGV4cGVyaW1lbnRhbF9tZXRob2QsCiAgICAgICAgICAgVGl0bGUgPSB0aXRsZSkKcGRiX3RhYmxlCmBgYAoKYGBge3IgU2F2ZSBkYXRhc2V0IGFzIEpTT04sIGluY2x1ZGU9RkFMU0V9CiMgU2F2ZSBkYXRhc2V0IGZvciBkb3dubG9hZAppZiAoIWRpci5leGlzdHMoaGVyZSgiZGF0YXNldHMiKSkpIHsKICAgIGRpci5jcmVhdGUoaGVyZSgiZGF0YXNldHMiKSkKfQoKd3JpdGVfanNvbihwZGJfZGF0YSwgaGVyZSgiZGF0YXNldHMiLCAic2Fycy1jb3YyLXN0cnVjdHVyZXMuanNvbiIpKQpgYGAKClsqKkRvd25sb2FkIHJhdyBkYXRhc2V0IGluIEpTT04gZm9ybWF0KipdKGRhdGFzZXRzL3NhcnMtY292Mi1zdHJ1Y3R1cmVzLmpzb24pCg==</div>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeSourceEmbed("sars-cov-2.Rmd");
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
