<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>SARS-CoV-2 structures</title>

<script src="site_libs/header-attrs-2.25/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/simplex.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<script src="site_libs/navigation-1.1/sourceembed.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<script src="site_libs/plotly-binding-4.10.3/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-2.11.1/plotly-latest.min.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>


<style type="text/css">
#rmd-source-code {
  display: none;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Insights from the PDB</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="nucleosome-structures.html">Nucleosome structures</a>
</li>
<li>
  <a href="sars-cov-2.html">SARS-CoV-2</a>
</li>
<li>
  <a href="dna-length-in-protein-dna-complexes.html">DNA length in protein-DNA complexes</a>
</li>
<li>
  <a href="free-dna.html">Free DNA</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/Guillawme/insights-from-the-pdb">GitHub repository</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
<li role="separator" class="divider"></li>
<li><a id="rmd-download-source" href="#">Download Rmd</a></li>
</ul>
</div>



<h1 class="title toc-ignore">SARS-CoV-2 structures</h1>

</div>


<p><strong>Last updated on Sun Feb 11 22:56:11 2024.</strong></p>
<div id="how-to-re-use-this-work" class="section level2">
<h2>How to re-use this work</h2>
<p>If you use these figures in your own work, please cite this website:
<a href="https://doi.org/10.5281/zenodo.3470119"
class="uri">https://doi.org/10.5281/zenodo.3470119</a></p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="co"># Load required packages</span></span>
<span id="cb1-2"><a href="#cb1-2" tabindex="-1"></a><span class="fu">library</span>(magrittr)</span>
<span id="cb1-3"><a href="#cb1-3" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb1-4"><a href="#cb1-4" tabindex="-1"></a><span class="fu">library</span>(jsonlite)</span>
<span id="cb1-5"><a href="#cb1-5" tabindex="-1"></a><span class="fu">library</span>(forcats)</span>
<span id="cb1-6"><a href="#cb1-6" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb1-7"><a href="#cb1-7" tabindex="-1"></a><span class="fu">library</span>(plotly)</span>
<span id="cb1-8"><a href="#cb1-8" tabindex="-1"></a><span class="fu">library</span>(here)</span>
<span id="cb1-9"><a href="#cb1-9" tabindex="-1"></a><span class="fu">library</span>(lubridate)</span>
<span id="cb1-10"><a href="#cb1-10" tabindex="-1"></a></span>
<span id="cb1-11"><a href="#cb1-11" tabindex="-1"></a><span class="co"># We need two queries: one for uppercase titles, the other for lowercase ones</span></span>
<span id="cb1-12"><a href="#cb1-12" tabindex="-1"></a>pdb_query <span class="ot">&lt;-</span> <span class="st">&#39;https://www.ebi.ac.uk/pdbe/search/pdb/select?q=title:*SARS-CoV-2*%20AND%20status:REL&amp;fl=pdb_id,deposition_date,citation_year,title,experimental_method,resolution,organism_scientific_name,molecule_name,molecule_type&amp;rows=1000000&amp;wt=json&#39;</span></span>
<span id="cb1-13"><a href="#cb1-13" tabindex="-1"></a></span>
<span id="cb1-14"><a href="#cb1-14" tabindex="-1"></a><span class="co"># This is a helper pipeline to extract data</span></span>
<span id="cb1-15"><a href="#cb1-15" tabindex="-1"></a>dig_up_data <span class="ot">&lt;-</span> . <span class="sc">%&gt;%</span></span>
<span id="cb1-16"><a href="#cb1-16" tabindex="-1"></a>    .<span class="sc">$</span>response <span class="sc">%&gt;%</span></span>
<span id="cb1-17"><a href="#cb1-17" tabindex="-1"></a>    .<span class="sc">$</span>docs <span class="sc">%&gt;%</span></span>
<span id="cb1-18"><a href="#cb1-18" tabindex="-1"></a>    <span class="fu">as_tibble</span>()</span>
<span id="cb1-19"><a href="#cb1-19" tabindex="-1"></a></span>
<span id="cb1-20"><a href="#cb1-20" tabindex="-1"></a><span class="co"># Query the PDB and clean up data</span></span>
<span id="cb1-21"><a href="#cb1-21" tabindex="-1"></a>pdb_data <span class="ot">&lt;-</span> pdb_query <span class="sc">%&gt;%</span> </span>
<span id="cb1-22"><a href="#cb1-22" tabindex="-1"></a>    <span class="fu">fromJSON</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb1-23"><a href="#cb1-23" tabindex="-1"></a>    <span class="fu">dig_up_data</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb1-24"><a href="#cb1-24" tabindex="-1"></a>    <span class="fu">bind_rows</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb1-25"><a href="#cb1-25" tabindex="-1"></a>    <span class="fu">mutate</span>(</span>
<span id="cb1-26"><a href="#cb1-26" tabindex="-1"></a>      <span class="at">experimental_method   =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(experimental_method)),</span>
<span id="cb1-27"><a href="#cb1-27" tabindex="-1"></a>      <span class="at">deposition_date       =</span> <span class="fu">as_date</span>(<span class="fu">ymd_hms</span>(deposition_date)),</span>
<span id="cb1-28"><a href="#cb1-28" tabindex="-1"></a>      <span class="at">citation_year         =</span> <span class="fu">as.integer</span>(citation_year),</span>
<span id="cb1-29"><a href="#cb1-29" tabindex="-1"></a>      <span class="at">molecule_name         =</span> <span class="fu">as.character</span>(molecule_name),</span>
<span id="cb1-30"><a href="#cb1-30" tabindex="-1"></a>      <span class="at">molecule_type         =</span> <span class="fu">as_factor</span>(<span class="fu">as.character</span>(molecule_type))</span>
<span id="cb1-31"><a href="#cb1-31" tabindex="-1"></a>      ) <span class="sc">%&gt;%</span> </span>
<span id="cb1-32"><a href="#cb1-32" tabindex="-1"></a>    <span class="fu">distinct</span>(pdb_id, <span class="at">.keep_all =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</div>
<div id="about-this-section" class="section level2">
<h2>About this section</h2>
<p>With the ongoing <a
href="https://en.wikipedia.org/wiki/COVID-19_pandemic">COVID-19
pandemic</a>, new structures of proteins from <a
href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
are deposited to the PDB at a fast pace. The visualizations presented
here wonâ€™t help curb the pandemic (and are not medical advice,
obviously), but I hope they will make curious and helpless minds (like
myself) get a feel for the research effort currently being deployed
against it.</p>
<p>The following resources are a lot more relevant and actionable:</p>
<ul>
<li><a href="https://insidecorona.net/">Coronavirus Structural Task
Force</a></li>
<li><a href="https://www.ebi.ac.uk/pdbe/covid-19">PDBe COVID-19 Data
Portal</a></li>
<li><a href="https://crowdfightcovid19.org/">Crowdfight
COVID-19</a></li>
</ul>
</div>
<div id="sars-cov-2-structures-by-deposition-date"
class="section level2">
<h2>SARS-CoV-2 structures by deposition date</h2>
<p>All figures are interactive (you can zoom in, and hovering over
elements will show more information).</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" tabindex="-1"></a>sars_cov2_structures_year <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb2-2"><a href="#cb2-2" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb2-3"><a href="#cb2-3" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> deposition_date)) <span class="sc">+</span></span>
<span id="cb2-4"><a href="#cb2-4" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of SARS-CoV-2 by deposition date&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-5"><a href="#cb2-5" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;Deposition date&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-6"><a href="#cb2-6" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb2-7"><a href="#cb2-7" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb2-8"><a href="#cb2-8" tabindex="-1"></a><span class="fu">ggplotly</span>(sars_cov2_structures_year)</span></code></pre></div>
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-77858f703a3f4a583015" style="width:672px;height:480px;"></div>
<script type="application/json" data-for="htmlwidget-77858f703a3f4a583015">{"x":{"data":[{"orientation":"v","width":[0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519,0.90000000000145519],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[18307,18310,18313,18317,18320,18324,18325,18330,18331,18336,18337,18339,18340,18344,18349,18351,18352,18353,18354,18355,18358,18359,18365,18366,18367,18368,18375,18376,18379,18380,18381,18382,18384,18385,18388,18389,18390,18392,18393,18396,18397,18398,18399,18401,18402,18404,18407,18408,18409,18410,18411,18416,18418,18419,18421,18422,18423,18424,18425,18426,18427,18428,18429,18431,18432,18434,18439,18442,18443,18444,18445,18446,18448,18449,18450,18451,18452,18453,18454,18457,18458,18460,18463,18466,18468,18469,18470,18471,18472,18473,18474,18476,18477,18478,18479,18481,18484,18485,18486,18488,18490,18491,18492,18493,18494,18495,18498,18499,18500,18501,18503,18504,18505,18506,18507,18508,18509,18512,18513,18514,18516,18517,18518,18519,18520,18521,18522,18523,18524,18527,18528,18529,18532,18533,18534,18535,18537,18539,18541,18543,18544,18545,18547,18548,18549,18550,18551,18553,18556,18557,18558,18559,18560,18561,18562,18563,18564,18565,18567,18568,18569,18570,18571,18572,18573,18575,18576,18577,18579,18580,18581,18583,18584,18585,18586,18587,18588,18589,18593,18594,18596,18597,18599,18600,18603,18604,18605,18606,18607,18608,18610,18611,18612,18616,18617,18618,18619,18620,18624,18625,18627,18629,18631,18633,18634,18635,18637,18638,18639,18640,18641,18642,18643,18645,18647,18649,18653,18654,18655,18657,18659,18660,18661,18662,18663,18665,18666,18667,18668,18670,18671,18673,18675,18676,18677,18678,18679,18680,18681,18682,18683,18684,18685,18686,18687,18688,18689,18690,18691,18693,18694,18695,18696,18697,18701,18702,18703,18704,18705,18708,18709,18711,18712,18713,18714,18715,18716,18717,18718,18722,18723,18725,18728,18730,18731,18732,18733,18737,18739,18741,18744,18745,18746,18747,18752,18755,18759,18760,18762,18765,18766,18767,18771,18772,18774,18775,18776,18777,18778,18779,18780,18781,18782,18783,18785,18786,18788,18789,18790,18792,18794,18795,18796,18799,18800,18801,18802,18803,18804,18806,18807,18808,18809,18810,18813,18814,18815,18816,18819,18820,18821,18822,18823,18827,18829,18830,18831,18834,18835,18836,18837,18838,18841,18842,18843,18844,18845,18846,18847,18852,18853,18855,18858,18860,18861,18864,18865,18866,18869,18870,18877,18878,18879,18881,18884,18885,18887,18888,18890,18891,18892,18895,18896,18899,18901,18902,18903,18905,18908,18910,18912,18913,18914,18917,18918,18922,18927,18928,18929,18932,18933,18934,18938,18939,18943,18946,18947,18948,18949,18951,18952,18954,18955,18956,18958,18960,18963,18964,18968,18969,18970,18971,18972,18974,18975,18976,18977,18978,18980,18981,18982,18983,18984,18989,18990,18991,18993,18995,18997,18998,18999,19000,19001,19002,19003,19004,19005,19006,19009,19010,19011,19012,19013,19014,19015,19016,19017,19018,19019,19020,19021,19025,19026,19027,19028,19029,19030,19031,19032,19033,19036,19037,19038,19039,19040,19041,19044,19045,19046,19048,19051,19053,19054,19055,19057,19058,19059,19060,19061,19062,19064,19065,19066,19068,19069,19073,19074,19075,19077,19078,19079,19081,19082,19083,19085,19087,19088,19089,19093,19094,19095,19096,19097,19100,19102,19103,19104,19108,19109,19110,19111,19112,19113,19114,19115,19118,19120,19123,19124,19125,19127,19130,19131,19134,19135,19136,19137,19138,19139,19142,19143,19144,19145,19147,19148,19149,19151,19152,19154,19156,19157,19159,19160,19161,19163,19164,19165,19166,19167,19168,19169,19170,19171,19172,19173,19174,19175,19177,19178,19179,19180,19181,19184,19185,19186,19187,19189,19191,19192,19193,19194,19198,19199,19200,19203,19205,19206,19207,19210,19211,19212,19214,19215,19217,19218,19219,19222,19223,19226,19229,19234,19238,19240,19243,19249,19250,19251,19252,19254,19255,19256,19257,19259,19260,19261,19262,19263,19265,19268,19269,19271,19272,19273,19274,19276,19277,19278,19279,19282,19283,19285,19286,19289,19291,19292,19294,19297,19298,19299,19303,19304,19306,19307,19309,19310,19311,19313,19315,19320,19321,19322,19328,19330,19331,19333,19336,19337,19338,19339,19340,19341,19342,19346,19347,19349,19354,19359,19366,19368,19369,19373,19374,19375,19376,19377,19378,19380,19381,19382,19404,19405,19407,19408,19409,19411,19412,19419,19423,19424,19429,19432,19439,19442,19444,19447,19458,19460,19465,19466,19467,19468,19472,19473,19480,19481,19488,19491,19500,19513,19514,19516,19520,19522,19534,19536,19541,19551,19555,19559,19571,19573,19574,19580,19584,19585,19588,19594,19600,19602,19605,19606,19620,19633,19634,19636,19637,19638,19644,19662,19668,19675,19694,19705],"y":[3,1,1,2,2,5,1,2,1,68,1,4,1,1,1,1,2,1,1,1,1,14,2,3,2,2,1,1,3,4,4,1,1,3,5,3,1,1,1,2,19,5,1,1,4,3,1,1,1,2,4,6,4,2,14,2,4,2,1,1,3,3,1,4,4,1,5,5,1,3,2,2,7,5,6,9,1,6,2,1,2,1,1,1,1,2,1,1,1,3,1,2,2,1,7,3,10,2,2,1,5,1,2,1,9,2,3,1,3,5,4,1,3,11,6,1,6,1,1,16,4,1,1,5,1,52,4,1,3,3,2,1,15,118,7,4,14,2,2,6,1,9,1,1,2,4,4,6,3,1,2,2,4,1,3,1,1,2,2,110,4,10,3,2,1,6,1,7,6,2,2,4,2,2,7,2,5,3,2,2,4,1,2,2,3,1,1,4,4,1,1,3,5,1,9,1,7,4,2,1,6,1,1,2,1,4,1,6,2,2,2,5,1,12,2,4,1,3,1,12,1,10,1,5,6,1,6,4,2,2,1,2,3,1,2,2,1,1,4,1,2,1,2,2,18,1,1,4,9,12,5,3,16,2,1,6,8,1,1,3,1,8,1,1,1,3,3,18,7,6,4,1,1,2,4,12,4,8,1,2,1,3,4,2,3,1,3,3,6,1,16,2,9,9,7,7,3,1,1,6,1,1,7,1,6,1,2,4,1,4,1,5,1,3,2,2,4,1,4,4,2,1,2,1,3,8,1,6,2,7,1,9,2,5,2,2,1,2,2,5,4,1,1,1,4,1,1,2,1,1,1,6,18,22,14,1,1,6,4,2,2,2,1,3,4,10,1,2,3,2,12,9,2,17,1,2,3,1,4,4,3,2,5,1,3,3,4,2,8,1,2,2,1,1,20,1,1,1,1,4,2,1,23,3,2,1,1,3,1,3,18,1,1,1,4,2,2,5,4,13,10,2,17,1,4,5,2,3,4,11,2,7,3,2,2,1,4,16,1,12,3,7,7,6,9,13,16,10,2,1,3,13,5,9,1,2,21,4,2,5,3,1,3,2,2,2,1,1,6,4,1,1,59,1,2,3,3,3,3,2,3,3,11,1,2,3,3,5,6,2,2,2,3,14,2,1,1,1,1,5,2,4,3,3,3,1,2,1,2,12,3,1,9,2,2,2,10,1,2,4,3,8,2,2,9,5,2,2,3,1,4,2,7,4,2,1,2,153,1,6,1,2,4,3,2,5,4,1,2,1,3,1,2,16,3,4,4,3,11,1,2,31,2,2,1,3,1,1,1,1,12,2,4,3,2,2,1,4,8,2,13,1,1,5,2,1,3,13,10,4,1,2,5,3,6,1,6,6,1,3,2,1,2,2,4,3,3,3,1,1,3,2,5,4,2,5,4,2,3,1,16,6,1,3,12,3,13,2,2,3,8,3,3,4,2,3,5,1,3,2,1,1,1,1,5,2,1,4,3,1,9,3,2,2,1,1,1,1,1,1,1,1,1,1,1,1,5,2,7,4,1,13,2,4,1,5,1,1,8,5,2,1,4,5,1,1,1,2,1,2,2,4,2,1,3,2,1,2,3,1,1,1,2,1,1,5,2,3,2,1,8,1,466,1,1,1,3,1,1,1,1,1,2,1,1,2,2,1,10,1,1,1,1],"text":["count:   3<br />deposition_date: 18307","count:   1<br />deposition_date: 18310","count:   1<br />deposition_date: 18313","count:   2<br />deposition_date: 18317","count:   2<br />deposition_date: 18320","count:   5<br />deposition_date: 18324","count:   1<br />deposition_date: 18325","count:   2<br />deposition_date: 18330","count:   1<br />deposition_date: 18331","count:  68<br />deposition_date: 18336","count:   1<br />deposition_date: 18337","count:   4<br />deposition_date: 18339","count:   1<br />deposition_date: 18340","count:   1<br />deposition_date: 18344","count:   1<br />deposition_date: 18349","count:   1<br />deposition_date: 18351","count:   2<br />deposition_date: 18352","count:   1<br />deposition_date: 18353","count:   1<br />deposition_date: 18354","count:   1<br />deposition_date: 18355","count:   1<br />deposition_date: 18358","count:  14<br />deposition_date: 18359","count:   2<br />deposition_date: 18365","count:   3<br />deposition_date: 18366","count:   2<br />deposition_date: 18367","count:   2<br />deposition_date: 18368","count:   1<br />deposition_date: 18375","count:   1<br />deposition_date: 18376","count:   3<br />deposition_date: 18379","count:   4<br />deposition_date: 18380","count:   4<br />deposition_date: 18381","count:   1<br />deposition_date: 18382","count:   1<br />deposition_date: 18384","count:   3<br />deposition_date: 18385","count:   5<br />deposition_date: 18388","count:   3<br />deposition_date: 18389","count:   1<br />deposition_date: 18390","count:   1<br />deposition_date: 18392","count:   1<br />deposition_date: 18393","count:   2<br />deposition_date: 18396","count:  19<br />deposition_date: 18397","count:   5<br />deposition_date: 18398","count:   1<br />deposition_date: 18399","count:   1<br />deposition_date: 18401","count:   4<br />deposition_date: 18402","count:   3<br />deposition_date: 18404","count:   1<br />deposition_date: 18407","count:   1<br />deposition_date: 18408","count:   1<br />deposition_date: 18409","count:   2<br />deposition_date: 18410","count:   4<br />deposition_date: 18411","count:   6<br />deposition_date: 18416","count:   4<br />deposition_date: 18418","count:   2<br />deposition_date: 18419","count:  14<br />deposition_date: 18421","count:   2<br />deposition_date: 18422","count:   4<br />deposition_date: 18423","count:   2<br />deposition_date: 18424","count:   1<br />deposition_date: 18425","count:   1<br />deposition_date: 18426","count:   3<br />deposition_date: 18427","count:   3<br />deposition_date: 18428","count:   1<br />deposition_date: 18429","count:   4<br />deposition_date: 18431","count:   4<br />deposition_date: 18432","count:   1<br />deposition_date: 18434","count:   5<br />deposition_date: 18439","count:   5<br />deposition_date: 18442","count:   1<br />deposition_date: 18443","count:   3<br />deposition_date: 18444","count:   2<br />deposition_date: 18445","count:   2<br />deposition_date: 18446","count:   7<br />deposition_date: 18448","count:   5<br />deposition_date: 18449","count:   6<br />deposition_date: 18450","count:   9<br />deposition_date: 18451","count:   1<br />deposition_date: 18452","count:   6<br />deposition_date: 18453","count:   2<br />deposition_date: 18454","count:   1<br />deposition_date: 18457","count:   2<br />deposition_date: 18458","count:   1<br />deposition_date: 18460","count:   1<br />deposition_date: 18463","count:   1<br />deposition_date: 18466","count:   1<br />deposition_date: 18468","count:   2<br />deposition_date: 18469","count:   1<br />deposition_date: 18470","count:   1<br />deposition_date: 18471","count:   1<br />deposition_date: 18472","count:   3<br />deposition_date: 18473","count:   1<br />deposition_date: 18474","count:   2<br />deposition_date: 18476","count:   2<br />deposition_date: 18477","count:   1<br />deposition_date: 18478","count:   7<br />deposition_date: 18479","count:   3<br />deposition_date: 18481","count:  10<br />deposition_date: 18484","count:   2<br />deposition_date: 18485","count:   2<br />deposition_date: 18486","count:   1<br />deposition_date: 18488","count:   5<br />deposition_date: 18490","count:   1<br />deposition_date: 18491","count:   2<br />deposition_date: 18492","count:   1<br />deposition_date: 18493","count:   9<br />deposition_date: 18494","count:   2<br />deposition_date: 18495","count:   3<br />deposition_date: 18498","count:   1<br />deposition_date: 18499","count:   3<br />deposition_date: 18500","count:   5<br />deposition_date: 18501","count:   4<br />deposition_date: 18503","count:   1<br />deposition_date: 18504","count:   3<br />deposition_date: 18505","count:  11<br />deposition_date: 18506","count:   6<br />deposition_date: 18507","count:   1<br />deposition_date: 18508","count:   6<br />deposition_date: 18509","count:   1<br />deposition_date: 18512","count:   1<br />deposition_date: 18513","count:  16<br />deposition_date: 18514","count:   4<br />deposition_date: 18516","count:   1<br />deposition_date: 18517","count:   1<br />deposition_date: 18518","count:   5<br />deposition_date: 18519","count:   1<br />deposition_date: 18520","count:  52<br />deposition_date: 18521","count:   4<br />deposition_date: 18522","count:   1<br />deposition_date: 18523","count:   3<br />deposition_date: 18524","count:   3<br />deposition_date: 18527","count:   2<br />deposition_date: 18528","count:   1<br />deposition_date: 18529","count:  15<br />deposition_date: 18532","count: 118<br />deposition_date: 18533","count:   7<br />deposition_date: 18534","count:   4<br />deposition_date: 18535","count:  14<br />deposition_date: 18537","count:   2<br />deposition_date: 18539","count:   2<br />deposition_date: 18541","count:   6<br />deposition_date: 18543","count:   1<br />deposition_date: 18544","count:   9<br />deposition_date: 18545","count:   1<br />deposition_date: 18547","count:   1<br />deposition_date: 18548","count:   2<br />deposition_date: 18549","count:   4<br />deposition_date: 18550","count:   4<br />deposition_date: 18551","count:   6<br />deposition_date: 18553","count:   3<br />deposition_date: 18556","count:   1<br />deposition_date: 18557","count:   2<br />deposition_date: 18558","count:   2<br />deposition_date: 18559","count:   4<br />deposition_date: 18560","count:   1<br />deposition_date: 18561","count:   3<br />deposition_date: 18562","count:   1<br />deposition_date: 18563","count:   1<br />deposition_date: 18564","count:   2<br />deposition_date: 18565","count:   2<br />deposition_date: 18567","count: 110<br />deposition_date: 18568","count:   4<br />deposition_date: 18569","count:  10<br />deposition_date: 18570","count:   3<br />deposition_date: 18571","count:   2<br />deposition_date: 18572","count:   1<br />deposition_date: 18573","count:   6<br />deposition_date: 18575","count:   1<br />deposition_date: 18576","count:   7<br />deposition_date: 18577","count:   6<br />deposition_date: 18579","count:   2<br />deposition_date: 18580","count:   2<br />deposition_date: 18581","count:   4<br />deposition_date: 18583","count:   2<br />deposition_date: 18584","count:   2<br />deposition_date: 18585","count:   7<br />deposition_date: 18586","count:   2<br />deposition_date: 18587","count:   5<br />deposition_date: 18588","count:   3<br />deposition_date: 18589","count:   2<br />deposition_date: 18593","count:   2<br />deposition_date: 18594","count:   4<br />deposition_date: 18596","count:   1<br />deposition_date: 18597","count:   2<br />deposition_date: 18599","count:   2<br />deposition_date: 18600","count:   3<br />deposition_date: 18603","count:   1<br />deposition_date: 18604","count:   1<br />deposition_date: 18605","count:   4<br />deposition_date: 18606","count:   4<br />deposition_date: 18607","count:   1<br />deposition_date: 18608","count:   1<br />deposition_date: 18610","count:   3<br />deposition_date: 18611","count:   5<br />deposition_date: 18612","count:   1<br />deposition_date: 18616","count:   9<br />deposition_date: 18617","count:   1<br />deposition_date: 18618","count:   7<br />deposition_date: 18619","count:   4<br />deposition_date: 18620","count:   2<br />deposition_date: 18624","count:   1<br />deposition_date: 18625","count:   6<br />deposition_date: 18627","count:   1<br />deposition_date: 18629","count:   1<br />deposition_date: 18631","count:   2<br />deposition_date: 18633","count:   1<br />deposition_date: 18634","count:   4<br />deposition_date: 18635","count:   1<br />deposition_date: 18637","count:   6<br />deposition_date: 18638","count:   2<br />deposition_date: 18639","count:   2<br />deposition_date: 18640","count:   2<br />deposition_date: 18641","count:   5<br />deposition_date: 18642","count:   1<br />deposition_date: 18643","count:  12<br />deposition_date: 18645","count:   2<br />deposition_date: 18647","count:   4<br />deposition_date: 18649","count:   1<br />deposition_date: 18653","count:   3<br />deposition_date: 18654","count:   1<br />deposition_date: 18655","count:  12<br />deposition_date: 18657","count:   1<br />deposition_date: 18659","count:  10<br />deposition_date: 18660","count:   1<br />deposition_date: 18661","count:   5<br />deposition_date: 18662","count:   6<br />deposition_date: 18663","count:   1<br />deposition_date: 18665","count:   6<br />deposition_date: 18666","count:   4<br />deposition_date: 18667","count:   2<br />deposition_date: 18668","count:   2<br />deposition_date: 18670","count:   1<br />deposition_date: 18671","count:   2<br />deposition_date: 18673","count:   3<br />deposition_date: 18675","count:   1<br />deposition_date: 18676","count:   2<br />deposition_date: 18677","count:   2<br />deposition_date: 18678","count:   1<br />deposition_date: 18679","count:   1<br />deposition_date: 18680","count:   4<br />deposition_date: 18681","count:   1<br />deposition_date: 18682","count:   2<br />deposition_date: 18683","count:   1<br />deposition_date: 18684","count:   2<br />deposition_date: 18685","count:   2<br />deposition_date: 18686","count:  18<br />deposition_date: 18687","count:   1<br />deposition_date: 18688","count:   1<br />deposition_date: 18689","count:   4<br />deposition_date: 18690","count:   9<br />deposition_date: 18691","count:  12<br />deposition_date: 18693","count:   5<br />deposition_date: 18694","count:   3<br />deposition_date: 18695","count:  16<br />deposition_date: 18696","count:   2<br />deposition_date: 18697","count:   1<br />deposition_date: 18701","count:   6<br />deposition_date: 18702","count:   8<br />deposition_date: 18703","count:   1<br />deposition_date: 18704","count:   1<br />deposition_date: 18705","count:   3<br />deposition_date: 18708","count:   1<br />deposition_date: 18709","count:   8<br />deposition_date: 18711","count:   1<br />deposition_date: 18712","count:   1<br />deposition_date: 18713","count:   1<br />deposition_date: 18714","count:   3<br />deposition_date: 18715","count:   3<br />deposition_date: 18716","count:  18<br />deposition_date: 18717","count:   7<br />deposition_date: 18718","count:   6<br />deposition_date: 18722","count:   4<br />deposition_date: 18723","count:   1<br />deposition_date: 18725","count:   1<br />deposition_date: 18728","count:   2<br />deposition_date: 18730","count:   4<br />deposition_date: 18731","count:  12<br />deposition_date: 18732","count:   4<br />deposition_date: 18733","count:   8<br />deposition_date: 18737","count:   1<br />deposition_date: 18739","count:   2<br />deposition_date: 18741","count:   1<br />deposition_date: 18744","count:   3<br />deposition_date: 18745","count:   4<br />deposition_date: 18746","count:   2<br />deposition_date: 18747","count:   3<br />deposition_date: 18752","count:   1<br />deposition_date: 18755","count:   3<br />deposition_date: 18759","count:   3<br />deposition_date: 18760","count:   6<br />deposition_date: 18762","count:   1<br />deposition_date: 18765","count:  16<br />deposition_date: 18766","count:   2<br />deposition_date: 18767","count:   9<br />deposition_date: 18771","count:   9<br />deposition_date: 18772","count:   7<br />deposition_date: 18774","count:   7<br />deposition_date: 18775","count:   3<br />deposition_date: 18776","count:   1<br />deposition_date: 18777","count:   1<br />deposition_date: 18778","count:   6<br />deposition_date: 18779","count:   1<br />deposition_date: 18780","count:   1<br />deposition_date: 18781","count:   7<br />deposition_date: 18782","count:   1<br />deposition_date: 18783","count:   6<br />deposition_date: 18785","count:   1<br />deposition_date: 18786","count:   2<br />deposition_date: 18788","count:   4<br />deposition_date: 18789","count:   1<br />deposition_date: 18790","count:   4<br />deposition_date: 18792","count:   1<br />deposition_date: 18794","count:   5<br />deposition_date: 18795","count:   1<br />deposition_date: 18796","count:   3<br />deposition_date: 18799","count:   2<br />deposition_date: 18800","count:   2<br />deposition_date: 18801","count:   4<br />deposition_date: 18802","count:   1<br />deposition_date: 18803","count:   4<br />deposition_date: 18804","count:   4<br />deposition_date: 18806","count:   2<br />deposition_date: 18807","count:   1<br />deposition_date: 18808","count:   2<br />deposition_date: 18809","count:   1<br />deposition_date: 18810","count:   3<br />deposition_date: 18813","count:   8<br />deposition_date: 18814","count:   1<br />deposition_date: 18815","count:   6<br />deposition_date: 18816","count:   2<br />deposition_date: 18819","count:   7<br />deposition_date: 18820","count:   1<br />deposition_date: 18821","count:   9<br />deposition_date: 18822","count:   2<br />deposition_date: 18823","count:   5<br />deposition_date: 18827","count:   2<br />deposition_date: 18829","count:   2<br />deposition_date: 18830","count:   1<br />deposition_date: 18831","count:   2<br />deposition_date: 18834","count:   2<br />deposition_date: 18835","count:   5<br />deposition_date: 18836","count:   4<br />deposition_date: 18837","count:   1<br />deposition_date: 18838","count:   1<br />deposition_date: 18841","count:   1<br />deposition_date: 18842","count:   4<br />deposition_date: 18843","count:   1<br />deposition_date: 18844","count:   1<br />deposition_date: 18845","count:   2<br />deposition_date: 18846","count:   1<br />deposition_date: 18847","count:   1<br />deposition_date: 18852","count:   1<br />deposition_date: 18853","count:   6<br />deposition_date: 18855","count:  18<br />deposition_date: 18858","count:  22<br />deposition_date: 18860","count:  14<br />deposition_date: 18861","count:   1<br />deposition_date: 18864","count:   1<br />deposition_date: 18865","count:   6<br />deposition_date: 18866","count:   4<br />deposition_date: 18869","count:   2<br />deposition_date: 18870","count:   2<br />deposition_date: 18877","count:   2<br />deposition_date: 18878","count:   1<br />deposition_date: 18879","count:   3<br />deposition_date: 18881","count:   4<br />deposition_date: 18884","count:  10<br />deposition_date: 18885","count:   1<br />deposition_date: 18887","count:   2<br />deposition_date: 18888","count:   3<br />deposition_date: 18890","count:   2<br />deposition_date: 18891","count:  12<br />deposition_date: 18892","count:   9<br />deposition_date: 18895","count:   2<br />deposition_date: 18896","count:  17<br />deposition_date: 18899","count:   1<br />deposition_date: 18901","count:   2<br />deposition_date: 18902","count:   3<br />deposition_date: 18903","count:   1<br />deposition_date: 18905","count:   4<br />deposition_date: 18908","count:   4<br />deposition_date: 18910","count:   3<br />deposition_date: 18912","count:   2<br />deposition_date: 18913","count:   5<br />deposition_date: 18914","count:   1<br />deposition_date: 18917","count:   3<br />deposition_date: 18918","count:   3<br />deposition_date: 18922","count:   4<br />deposition_date: 18927","count:   2<br />deposition_date: 18928","count:   8<br />deposition_date: 18929","count:   1<br />deposition_date: 18932","count:   2<br />deposition_date: 18933","count:   2<br />deposition_date: 18934","count:   1<br />deposition_date: 18938","count:   1<br />deposition_date: 18939","count:  20<br />deposition_date: 18943","count:   1<br />deposition_date: 18946","count:   1<br />deposition_date: 18947","count:   1<br />deposition_date: 18948","count:   1<br />deposition_date: 18949","count:   4<br />deposition_date: 18951","count:   2<br />deposition_date: 18952","count:   1<br />deposition_date: 18954","count:  23<br />deposition_date: 18955","count:   3<br />deposition_date: 18956","count:   2<br />deposition_date: 18958","count:   1<br />deposition_date: 18960","count:   1<br />deposition_date: 18963","count:   3<br />deposition_date: 18964","count:   1<br />deposition_date: 18968","count:   3<br />deposition_date: 18969","count:  18<br />deposition_date: 18970","count:   1<br />deposition_date: 18971","count:   1<br />deposition_date: 18972","count:   1<br />deposition_date: 18974","count:   4<br />deposition_date: 18975","count:   2<br />deposition_date: 18976","count:   2<br />deposition_date: 18977","count:   5<br />deposition_date: 18978","count:   4<br />deposition_date: 18980","count:  13<br />deposition_date: 18981","count:  10<br />deposition_date: 18982","count:   2<br />deposition_date: 18983","count:  17<br />deposition_date: 18984","count:   1<br />deposition_date: 18989","count:   4<br />deposition_date: 18990","count:   5<br />deposition_date: 18991","count:   2<br />deposition_date: 18993","count:   3<br />deposition_date: 18995","count:   4<br />deposition_date: 18997","count:  11<br />deposition_date: 18998","count:   2<br />deposition_date: 18999","count:   7<br />deposition_date: 19000","count:   3<br />deposition_date: 19001","count:   2<br />deposition_date: 19002","count:   2<br />deposition_date: 19003","count:   1<br />deposition_date: 19004","count:   4<br />deposition_date: 19005","count:  16<br />deposition_date: 19006","count:   1<br />deposition_date: 19009","count:  12<br />deposition_date: 19010","count:   3<br />deposition_date: 19011","count:   7<br />deposition_date: 19012","count:   7<br />deposition_date: 19013","count:   6<br />deposition_date: 19014","count:   9<br />deposition_date: 19015","count:  13<br />deposition_date: 19016","count:  16<br />deposition_date: 19017","count:  10<br />deposition_date: 19018","count:   2<br />deposition_date: 19019","count:   1<br />deposition_date: 19020","count:   3<br />deposition_date: 19021","count:  13<br />deposition_date: 19025","count:   5<br />deposition_date: 19026","count:   9<br />deposition_date: 19027","count:   1<br />deposition_date: 19028","count:   2<br />deposition_date: 19029","count:  21<br />deposition_date: 19030","count:   4<br />deposition_date: 19031","count:   2<br />deposition_date: 19032","count:   5<br />deposition_date: 19033","count:   3<br />deposition_date: 19036","count:   1<br />deposition_date: 19037","count:   3<br />deposition_date: 19038","count:   2<br />deposition_date: 19039","count:   2<br />deposition_date: 19040","count:   2<br />deposition_date: 19041","count:   1<br />deposition_date: 19044","count:   1<br />deposition_date: 19045","count:   6<br />deposition_date: 19046","count:   4<br />deposition_date: 19048","count:   1<br />deposition_date: 19051","count:   1<br />deposition_date: 19053","count:  59<br />deposition_date: 19054","count:   1<br />deposition_date: 19055","count:   2<br />deposition_date: 19057","count:   3<br />deposition_date: 19058","count:   3<br />deposition_date: 19059","count:   3<br />deposition_date: 19060","count:   3<br />deposition_date: 19061","count:   2<br />deposition_date: 19062","count:   3<br />deposition_date: 19064","count:   3<br />deposition_date: 19065","count:  11<br />deposition_date: 19066","count:   1<br />deposition_date: 19068","count:   2<br />deposition_date: 19069","count:   3<br />deposition_date: 19073","count:   3<br />deposition_date: 19074","count:   5<br />deposition_date: 19075","count:   6<br />deposition_date: 19077","count:   2<br />deposition_date: 19078","count:   2<br />deposition_date: 19079","count:   2<br />deposition_date: 19081","count:   3<br />deposition_date: 19082","count:  14<br />deposition_date: 19083","count:   2<br />deposition_date: 19085","count:   1<br />deposition_date: 19087","count:   1<br />deposition_date: 19088","count:   1<br />deposition_date: 19089","count:   1<br />deposition_date: 19093","count:   5<br />deposition_date: 19094","count:   2<br />deposition_date: 19095","count:   4<br />deposition_date: 19096","count:   3<br />deposition_date: 19097","count:   3<br />deposition_date: 19100","count:   3<br />deposition_date: 19102","count:   1<br />deposition_date: 19103","count:   2<br />deposition_date: 19104","count:   1<br />deposition_date: 19108","count:   2<br />deposition_date: 19109","count:  12<br />deposition_date: 19110","count:   3<br />deposition_date: 19111","count:   1<br />deposition_date: 19112","count:   9<br />deposition_date: 19113","count:   2<br />deposition_date: 19114","count:   2<br />deposition_date: 19115","count:   2<br />deposition_date: 19118","count:  10<br />deposition_date: 19120","count:   1<br />deposition_date: 19123","count:   2<br />deposition_date: 19124","count:   4<br />deposition_date: 19125","count:   3<br />deposition_date: 19127","count:   8<br />deposition_date: 19130","count:   2<br />deposition_date: 19131","count:   2<br />deposition_date: 19134","count:   9<br />deposition_date: 19135","count:   5<br />deposition_date: 19136","count:   2<br />deposition_date: 19137","count:   2<br />deposition_date: 19138","count:   3<br />deposition_date: 19139","count:   1<br />deposition_date: 19142","count:   4<br />deposition_date: 19143","count:   2<br />deposition_date: 19144","count:   7<br />deposition_date: 19145","count:   4<br />deposition_date: 19147","count:   2<br />deposition_date: 19148","count:   1<br />deposition_date: 19149","count:   2<br />deposition_date: 19151","count: 153<br />deposition_date: 19152","count:   1<br />deposition_date: 19154","count:   6<br />deposition_date: 19156","count:   1<br />deposition_date: 19157","count:   2<br />deposition_date: 19159","count:   4<br />deposition_date: 19160","count:   3<br />deposition_date: 19161","count:   2<br />deposition_date: 19163","count:   5<br />deposition_date: 19164","count:   4<br />deposition_date: 19165","count:   1<br />deposition_date: 19166","count:   2<br />deposition_date: 19167","count:   1<br />deposition_date: 19168","count:   3<br />deposition_date: 19169","count:   1<br />deposition_date: 19170","count:   2<br />deposition_date: 19171","count:  16<br />deposition_date: 19172","count:   3<br />deposition_date: 19173","count:   4<br />deposition_date: 19174","count:   4<br />deposition_date: 19175","count:   3<br />deposition_date: 19177","count:  11<br />deposition_date: 19178","count:   1<br />deposition_date: 19179","count:   2<br />deposition_date: 19180","count:  31<br />deposition_date: 19181","count:   2<br />deposition_date: 19184","count:   2<br />deposition_date: 19185","count:   1<br />deposition_date: 19186","count:   3<br />deposition_date: 19187","count:   1<br />deposition_date: 19189","count:   1<br />deposition_date: 19191","count:   1<br />deposition_date: 19192","count:   1<br />deposition_date: 19193","count:  12<br />deposition_date: 19194","count:   2<br />deposition_date: 19198","count:   4<br />deposition_date: 19199","count:   3<br />deposition_date: 19200","count:   2<br />deposition_date: 19203","count:   2<br />deposition_date: 19205","count:   1<br />deposition_date: 19206","count:   4<br />deposition_date: 19207","count:   8<br />deposition_date: 19210","count:   2<br />deposition_date: 19211","count:  13<br />deposition_date: 19212","count:   1<br />deposition_date: 19214","count:   1<br />deposition_date: 19215","count:   5<br />deposition_date: 19217","count:   2<br />deposition_date: 19218","count:   1<br />deposition_date: 19219","count:   3<br />deposition_date: 19222","count:  13<br />deposition_date: 19223","count:  10<br />deposition_date: 19226","count:   4<br />deposition_date: 19229","count:   1<br />deposition_date: 19234","count:   2<br />deposition_date: 19238","count:   5<br />deposition_date: 19240","count:   3<br />deposition_date: 19243","count:   6<br />deposition_date: 19249","count:   1<br />deposition_date: 19250","count:   6<br />deposition_date: 19251","count:   6<br />deposition_date: 19252","count:   1<br />deposition_date: 19254","count:   3<br />deposition_date: 19255","count:   2<br />deposition_date: 19256","count:   1<br />deposition_date: 19257","count:   2<br />deposition_date: 19259","count:   2<br />deposition_date: 19260","count:   4<br />deposition_date: 19261","count:   3<br />deposition_date: 19262","count:   3<br />deposition_date: 19263","count:   3<br />deposition_date: 19265","count:   1<br />deposition_date: 19268","count:   1<br />deposition_date: 19269","count:   3<br />deposition_date: 19271","count:   2<br />deposition_date: 19272","count:   5<br />deposition_date: 19273","count:   4<br />deposition_date: 19274","count:   2<br />deposition_date: 19276","count:   5<br />deposition_date: 19277","count:   4<br />deposition_date: 19278","count:   2<br />deposition_date: 19279","count:   3<br />deposition_date: 19282","count:   1<br />deposition_date: 19283","count:  16<br />deposition_date: 19285","count:   6<br />deposition_date: 19286","count:   1<br />deposition_date: 19289","count:   3<br />deposition_date: 19291","count:  12<br />deposition_date: 19292","count:   3<br />deposition_date: 19294","count:  13<br />deposition_date: 19297","count:   2<br />deposition_date: 19298","count:   2<br />deposition_date: 19299","count:   3<br />deposition_date: 19303","count:   8<br />deposition_date: 19304","count:   3<br />deposition_date: 19306","count:   3<br />deposition_date: 19307","count:   4<br />deposition_date: 19309","count:   2<br />deposition_date: 19310","count:   3<br />deposition_date: 19311","count:   5<br />deposition_date: 19313","count:   1<br />deposition_date: 19315","count:   3<br />deposition_date: 19320","count:   2<br />deposition_date: 19321","count:   1<br />deposition_date: 19322","count:   1<br />deposition_date: 19328","count:   1<br />deposition_date: 19330","count:   1<br />deposition_date: 19331","count:   5<br />deposition_date: 19333","count:   2<br />deposition_date: 19336","count:   1<br />deposition_date: 19337","count:   4<br />deposition_date: 19338","count:   3<br />deposition_date: 19339","count:   1<br />deposition_date: 19340","count:   9<br />deposition_date: 19341","count:   3<br />deposition_date: 19342","count:   2<br />deposition_date: 19346","count:   2<br />deposition_date: 19347","count:   1<br />deposition_date: 19349","count:   1<br />deposition_date: 19354","count:   1<br />deposition_date: 19359","count:   1<br />deposition_date: 19366","count:   1<br />deposition_date: 19368","count:   1<br />deposition_date: 19369","count:   1<br />deposition_date: 19373","count:   1<br />deposition_date: 19374","count:   1<br />deposition_date: 19375","count:   1<br />deposition_date: 19376","count:   1<br />deposition_date: 19377","count:   1<br />deposition_date: 19378","count:   5<br />deposition_date: 19380","count:   2<br />deposition_date: 19381","count:   7<br />deposition_date: 19382","count:   4<br />deposition_date: 19404","count:   1<br />deposition_date: 19405","count:  13<br />deposition_date: 19407","count:   2<br />deposition_date: 19408","count:   4<br />deposition_date: 19409","count:   1<br />deposition_date: 19411","count:   5<br />deposition_date: 19412","count:   1<br />deposition_date: 19419","count:   1<br />deposition_date: 19423","count:   8<br />deposition_date: 19424","count:   5<br />deposition_date: 19429","count:   2<br />deposition_date: 19432","count:   1<br />deposition_date: 19439","count:   4<br />deposition_date: 19442","count:   5<br />deposition_date: 19444","count:   1<br />deposition_date: 19447","count:   1<br />deposition_date: 19458","count:   1<br />deposition_date: 19460","count:   2<br />deposition_date: 19465","count:   1<br />deposition_date: 19466","count:   2<br />deposition_date: 19467","count:   2<br />deposition_date: 19468","count:   4<br />deposition_date: 19472","count:   2<br />deposition_date: 19473","count:   1<br />deposition_date: 19480","count:   3<br />deposition_date: 19481","count:   2<br />deposition_date: 19488","count:   1<br />deposition_date: 19491","count:   2<br />deposition_date: 19500","count:   3<br />deposition_date: 19513","count:   1<br />deposition_date: 19514","count:   1<br />deposition_date: 19516","count:   1<br />deposition_date: 19520","count:   2<br />deposition_date: 19522","count:   1<br />deposition_date: 19534","count:   1<br />deposition_date: 19536","count:   5<br />deposition_date: 19541","count:   2<br />deposition_date: 19551","count:   3<br />deposition_date: 19555","count:   2<br />deposition_date: 19559","count:   1<br />deposition_date: 19571","count:   8<br />deposition_date: 19573","count:   1<br />deposition_date: 19574","count: 466<br />deposition_date: 19580","count:   1<br />deposition_date: 19584","count:   1<br />deposition_date: 19585","count:   1<br />deposition_date: 19588","count:   3<br />deposition_date: 19594","count:   1<br />deposition_date: 19600","count:   1<br />deposition_date: 19602","count:   1<br />deposition_date: 19605","count:   1<br />deposition_date: 19606","count:   1<br />deposition_date: 19620","count:   2<br />deposition_date: 19633","count:   1<br />deposition_date: 19634","count:   1<br />deposition_date: 19636","count:   2<br />deposition_date: 19637","count:   2<br />deposition_date: 19638","count:   1<br />deposition_date: 19644","count:  10<br />deposition_date: 19662","count:   1<br />deposition_date: 19668","count:   1<br />deposition_date: 19675","count:   1<br />deposition_date: 19694","count:   1<br />deposition_date: 19705"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(89,89,89,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.762557077625573,"r":7.3059360730593621,"b":40.182648401826498,"l":43.105022831050235},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"title":{"text":"Structures of SARS-CoV-2 by deposition date","font":{"color":"rgba(0,0,0,1)","family":"","size":17.534246575342465},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[18236.605,19775.395],"tickmode":"array","ticktext":["2020","2021","2022","2023","2024"],"tickvals":[18262,18628,18993,19358,19723],"categoryorder":"array","categoryarray":["2020","2021","2022","2023","2024"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":{"text":"Deposition date","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-23.300000000000001,489.30000000000001],"tickmode":"array","ticktext":["0","100","200","300","400"],"tickvals":[0,100,200,300,400],"categoryorder":"array","categoryarray":["0","100","200","300","400"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176002,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"f05b18ca74e1":{"x":{},"type":"bar"}},"cur_data":"f05b18ca74e1","visdat":{"f05b18ca74e1":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>Noticeable very large numbers of depositions on a same day (for
example, 68 structures deposited on March 15th and 14 structures
deposited on April 7th) come from large-scale fragment screening
experiments performed at Diamond Light Source:</p>
<ul>
<li><a
href="https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html">main
protease</a></li>
<li><a
href="https://www.diamond.ac.uk/covid-19/for-scientists/NSP3-macrodomain-structure-and-XChem.html">macro
domain</a></li>
<li><a
href="https://www.diamond.ac.uk/covid-19/for-scientists/NSP13-Helicase-Structure-and-XChem.html">helicase</a></li>
</ul>
<p><a href="figures/sars-cov2-structures-by-year.svg"><strong>Download
figure in SVG format</strong></a></p>
</div>
<div id="sars-cov-2-structures-by-experimental-method"
class="section level2">
<h2>SARS-CoV-2 structures by experimental method</h2>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" tabindex="-1"></a>sars_cov2_structures_method <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb3-2"><a href="#cb3-2" tabindex="-1"></a>    <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb3-3"><a href="#cb3-3" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> experimental_method, <span class="at">fill =</span> experimental_method)) <span class="sc">+</span></span>
<span id="cb3-4"><a href="#cb3-4" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">title =</span> <span class="st">&quot;Experimental method&quot;</span>)) <span class="sc">+</span></span>
<span id="cb3-5"><a href="#cb3-5" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">&quot;Structures of SARS-CoV-2 by experimental method&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-6"><a href="#cb3-6" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">&quot;&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-7"><a href="#cb3-7" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">&quot;Number of PDB entries&quot;</span>) <span class="sc">+</span></span>
<span id="cb3-8"><a href="#cb3-8" tabindex="-1"></a>    <span class="fu">theme_bw</span>()</span>
<span id="cb3-9"><a href="#cb3-9" tabindex="-1"></a><span class="fu">ggplotly</span>(sars_cov2_structures_method)</span></code></pre></div>
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-d9bf1e752ea61fa506eb" style="width:672px;height:480px;"></div>
<script type="application/json" data-for="htmlwidget-d9bf1e752ea61fa506eb">{"x":{"data":[{"orientation":"v","width":0.89999999999999991,"base":0,"x":[1],"y":[2447],"text":"count: 2447<br />experimental_method: X-ray diffraction<br />experimental_method: X-ray diffraction","type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(248,118,109,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"X-ray diffraction","legendgroup":"X-ray diffraction","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.90000000000000013,"base":0,"x":[2],"y":[1126],"text":"count: 1126<br />experimental_method: Electron Microscopy<br />experimental_method: Electron Microscopy","type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(163,165,0,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Electron Microscopy","legendgroup":"Electron Microscopy","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.90000000000000036,"base":0,"x":[3],"y":[4],"text":"count:    4<br />experimental_method: Solid-state NMR<br />experimental_method: Solid-state NMR","type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(0,191,125,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Solid-state NMR","legendgroup":"Solid-state NMR","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.90000000000000036,"base":0,"x":[4],"y":[14],"text":"count:   14<br />experimental_method: Solution NMR<br />experimental_method: Solution NMR","type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(0,176,246,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"Solution NMR","legendgroup":"Solution NMR","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"orientation":"v","width":0.90000000000000036,"base":0,"x":[5],"y":[7],"text":"count:    7<br />experimental_method: c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")<br />experimental_method: c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(231,107,243,1)","line":{"width":1.8897637795275593,"color":"transparent"}},"name":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","legendgroup":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.762557077625573,"r":7.3059360730593621,"b":25.57077625570777,"l":48.949771689497723},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724},"title":{"text":"Structures of SARS-CoV-2 by experimental method","font":{"color":"rgba(0,0,0,1)","family":"","size":17.534246575342465},"x":0,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,5.5999999999999996],"tickmode":"array","ticktext":["X-ray diffraction","Electron Microscopy","Solid-state NMR","Solution NMR","c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["X-ray diffraction","Electron Microscopy","Solid-state NMR","Solution NMR","c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"y","title":{"text":"","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-122.35000000000001,2569.3499999999999],"tickmode":"array","ticktext":["0","500","1000","1500","2000","2500"],"tickvals":[0,500,999.99999999999989,1500,2000,2500],"categoryorder":"array","categoryarray":["0","500","1000","1500","2000","2500"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.68949771689498},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176002,"zeroline":false,"anchor":"x","title":{"text":"Number of PDB entries","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176002,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.68949771689498},"title":{"text":"Experimental method","font":{"color":"rgba(0,0,0,1)","family":"","size":14.611872146118724}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"f05ba26065":{"x":{},"fill":{},"type":"bar"}},"cur_data":"f05ba26065","visdat":{"f05ba26065":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p><a href="figures/sars-cov2-structures-by-method.svg"><strong>Download
figure in SVG format</strong></a></p>
</div>
<div id="dataset" class="section level2">
<h2>Dataset</h2>
<p>The graphs presented above are derived from the following
dataset:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" tabindex="-1"></a><span class="co"># Format table for display</span></span>
<span id="cb4-2"><a href="#cb4-2" tabindex="-1"></a>pdb_table <span class="ot">&lt;-</span> pdb_data <span class="sc">%&gt;%</span> </span>
<span id="cb4-3"><a href="#cb4-3" tabindex="-1"></a>    <span class="fu">arrange</span>(<span class="fu">desc</span>(citation_year)) <span class="sc">%&gt;%</span> </span>
<span id="cb4-4"><a href="#cb4-4" tabindex="-1"></a>    <span class="fu">select</span>(<span class="st">`</span><span class="at">PDB code</span><span class="st">`</span> <span class="ot">=</span> pdb_id,</span>
<span id="cb4-5"><a href="#cb4-5" tabindex="-1"></a>           <span class="st">`</span><span class="at">Citation year</span><span class="st">`</span> <span class="ot">=</span> citation_year,</span>
<span id="cb4-6"><a href="#cb4-6" tabindex="-1"></a>           <span class="st">`</span><span class="at">Experimental method</span><span class="st">`</span> <span class="ot">=</span> experimental_method,</span>
<span id="cb4-7"><a href="#cb4-7" tabindex="-1"></a>           <span class="at">Title =</span> title)</span>
<span id="cb4-8"><a href="#cb4-8" tabindex="-1"></a>pdb_table</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["PDB code"],"name":[1],"type":["chr"],"align":["left"]},{"label":["Citation year"],"name":[2],"type":["int"],"align":["right"]},{"label":["Experimental method"],"name":[3],"type":["fct"],"align":["left"]},{"label":["Title"],"name":[4],"type":["chr"],"align":["left"]}],"data":[{"1":"8tpg","2":"2024","3":"X-ray diffraction","4":"Synthesis, X-Ray Crystallographic and Biological Activities of Covalent, Non-Peptidic Inhibitors of SARS-CoV-2 Main Protease"},{"1":"8tph","2":"2024","3":"X-ray diffraction","4":"Synthesis, X-Ray Crystallographic and Biological Activities of Covalent, Non-Peptidic Inhibitors of SARS-CoV-2 Main Protease"},{"1":"8tpc","2":"2024","3":"X-ray diffraction","4":"Synthesis, X-Ray Crystallographic and Biological Activities of Covalent, Non-Peptidic Inhibitors of SARS-CoV-2 Main Protease"},{"1":"8qq0","2":"2024","3":"Electron Microscopy","4":"SARS-CoV-2 S protein bound to neutralising antibody UZGENT_A3"},{"1":"8c89","2":"2024","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the 17T2 neutralizing antibody Fab fragment (local refinement of RBD and Fab)"},{"1":"8fi9","2":"2024","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody WRAIR-5001"},{"1":"8fhy","2":"2024","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain in complex with neutralizing antibody WRAIR-5021"},{"1":"8tpd","2":"2024","3":"X-ray diffraction","4":"Synthesis, X-Ray Crystallographic and Biological Activities of Covalent, Non-Peptidic Inhibitors of SARS-CoV-2 Main Protease"},{"1":"8tpf","2":"2024","3":"X-ray diffraction","4":"Synthesis, X-Ray Crystallographic and Biological Activities of Covalent, Non-Peptidic Inhibitors of SARS-CoV-2 Main Protease"},{"1":"8tyl","2":"2024","3":"Electron Microscopy","4":"Structural and biochemical rationale for Beta variant protein booster vaccine broad cross-neutralization of SARS-CoV-2"},{"1":"8tpe","2":"2024","3":"X-ray diffraction","4":"Synthesis, X-Ray Crystallographic and Biological Activities of Covalent, Non-Peptidic Inhibitors of SARS-CoV-2 Main Protease"},{"1":"8tpb","2":"2024","3":"X-ray diffraction","4":"Synthesis, X-Ray Crystallographic and Biological Activities of Covalent, Non-Peptidic Inhibitors of SARS-CoV-2 Main Protease"},{"1":"8tpi","2":"2024","3":"X-ray diffraction","4":"Synthesis, X-Ray Crystallographic and Biological Activities of Covalent, Non-Peptidic Inhibitors of SARS-CoV-2 Main Protease"},{"1":"8tyo","2":"2024","3":"Electron Microscopy","4":"Structural and biochemical rationale for Beta variant protein booster vaccine broad cross-neutralization of SARS-CoV-2"},{"1":"8qpr","2":"2024","3":"Electron Microscopy","4":"SARS-CoV-2 S protein bound to human neutralising antibody UZGENT_G5"},{"1":"7zb7","2":"2024","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) variant Y54F at 1.63 A resolution"},{"1":"7zb8","2":"2024","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) variant K61A at 2.48 A resolution"},{"1":"7zb6","2":"2024","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) variant C44S at 2.12 A resolution"},{"1":"5sqf","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL250000548538 - (R) isomer"},{"1":"5sph","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4718398515 - (R,S) isomer"},{"1":"5ssq","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5459166291"},{"1":"5sqy","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5211314110 - (S,S) isomer"},{"1":"5sq1","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC001601221314 - (S) isomer"},{"1":"5sok","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000302059710 - (R) isomer"},{"1":"5saf","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with EN300-321461"},{"1":"5sra","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5372052920 - (R) isomer"},{"1":"5soy","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000222377450"},{"1":"5srp","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5340019182 - (R) isomer"},{"1":"5ss3","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCnu000001eLaQ"},{"1":"5ss0","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCn9000000uj1v"},{"1":"5ssk","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCkk00000cjQyM - (R,S) isomer"},{"1":"5spu","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC001364774273 - (S) isomer"},{"1":"5soq","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000896845531 - (R) isomer"},{"1":"5sac","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z59181945"},{"1":"5srr","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4158218973 - (S,S) isomer"},{"1":"5sqi","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5016127255 - (R,R) and (S,S) isomers"},{"1":"5sr2","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with EN300-36602160"},{"1":"5sop","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC001364194305 - (R) isomer"},{"1":"5srv","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562533 - (R,R) isomer"},{"1":"5sr5","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5265454473 - (R) isomer"},{"1":"5ssc","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCk500000doQ8X"},{"1":"5sr9","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z3562259556 - (R) isomer"},{"1":"5srh","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5265470867 - pyrimido-indole core only"},{"1":"5sq9","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894420 - (R,R) and (S,S) isomers"},{"1":"5spd","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4718398539 - (R,R) and (S,S) isomers"},{"1":"5sa4","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z239136710"},{"1":"5ssi","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCn500000bifGU"},{"1":"5spr","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL250002852032 - (S) isomer"},{"1":"5sol","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000910475722 - (S,R) isomer"},{"1":"5ssa","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINClv000001jcNa - (r,r) isomer"},{"1":"5sry","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5265428218"},{"1":"5spg","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4718398585"},{"1":"5saa","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z319891284"},{"1":"5sqp","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5028367859 - (S) isomer"},{"1":"5spt","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000850008207"},{"1":"5ss8","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCny000002NPIr"},{"1":"5sqq","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with FRESH00014649046"},{"1":"5spz","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL250001448407 - (S) isomer"},{"1":"5ss9","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCow000000AiWv - (R) isomer"},{"1":"5sqx","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5183357278 - (R,R) and (S,S) isomers"},{"1":"5sq7","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z1445235880"},{"1":"5som","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000835985505 - (S) isomer"},{"1":"5sso","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562519 - (R) and (S) isomers"},{"1":"5sqw","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5014193706 - (R,R) and (S,S) isomers"},{"1":"5srx","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562503 - (R) isomer"},{"1":"5spc","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894387 - (R,R) and (S,S) isomers"},{"1":"5sq8","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z1445261766"},{"1":"5sp0","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000681764827"},{"1":"5sr7","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4914649782 - (R,R,S) and (S,S,R) isomers"},{"1":"5sqk","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2479782408 - (R,S) isomer"},{"1":"5sp1","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC001472868186"},{"1":"5ssj","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCno00000broQT"},{"1":"5sqo","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5030903496 - (R) isomer"},{"1":"5sqb","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894390 - (R,R) and (S,S) isomers"},{"1":"5spa","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894417 - (R,R) and (S,S) isomers"},{"1":"5sad","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z425449682"},{"1":"5spv","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL250003774401"},{"1":"5s6y","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z56900771"},{"1":"5sr1","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z1272415642 - (R) isomer"},{"1":"5srb","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562532 - (R) and (S) isomers"},{"1":"5sp6","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4718398580"},{"1":"5srt","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562791 - (R) isomer"},{"1":"5sor","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000110510893"},{"1":"5sqc","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894388 - (R,R) and (S,S) isomers"},{"1":"5spe","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4718398531 - (S,S) isomer"},{"1":"5spi","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4574659604 - (R,R) and (S,S) isomers"},{"1":"5sae","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z3219959731"},{"1":"5spj","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000893101964"},{"1":"5sov","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000893191027 - (S) and (R) isomers"},{"1":"5s6x","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z2889976755"},{"1":"5srj","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5265428226"},{"1":"5ssr","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5459166300"},{"1":"5squ","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL250004627335"},{"1":"5sre","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562530 - (R) isomer"},{"1":"5sr3","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5021669050 - (S,S) isomer"},{"1":"5ss2","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCnt000006kx7L"},{"1":"5sr4","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2479779298 - (R,S) and (S,R) isomers"},{"1":"5srd","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5373433723 - (S) isomer"},{"1":"5srn","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2466029596 - (R) and (S) isomers"},{"1":"5sq5","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894407 - (R,S) and (S,S) isomers"},{"1":"5soi","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000078036511 - (R) and (S) isomers"},{"1":"5srf","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4175156780 - (R) isomer"},{"1":"5sr6","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z3011799020 - (R) isomer"},{"1":"5sqh","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894431- (S,S) isomer"},{"1":"5sp8","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894415 - (S) isomer"},{"1":"5ss5","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCpE000000mAwk - (S) isomer"},{"1":"5srm","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z3860662215 - (R) and (S) isomers"},{"1":"5sq2","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2976440814 - (S) isomer"},{"1":"5sa7","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z1673618163"},{"1":"5src","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562500 - (R,R) and (R,S) isomers"},{"1":"5spm","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with FRESH00002410346"},{"1":"5sql","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2689779890"},{"1":"5spk","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL250003296134 - (R) isomer"},{"1":"5sa8","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z68299550"},{"1":"5ss6","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINClf00000cdzal"},{"1":"5sot","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000292637864 - (R) and (S) isomers"},{"1":"5sqd","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894382 - (R,R) and (S,S) isomers"},{"1":"5sp9","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z3508769536 - (S) isomer"},{"1":"5ssh","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCpx000006Mh4L - (S) isomer"},{"1":"5spx","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL250003958539"},{"1":"5son","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000920153280 - (R) isomer"},{"1":"5soj","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000642067873 - (R) isomer"},{"1":"5s71","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with FUZS-5"},{"1":"5sag","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with EN300-1605072"},{"1":"5ssd","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCm4000007vvRA - (R,S) isomer"},{"1":"5sq6","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894406"},{"1":"5soo","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000897286891 - (R) isomer"},{"1":"5srz","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5281440906 - (R,S) and (S,R) isomers"},{"1":"5sse","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCoD000001aHBe"},{"1":"5sqs","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC001240411747"},{"1":"5s6z","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with PB2255187532"},{"1":"5ssm","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5459166256 - (R,R) and (S,S) isomers"},{"1":"5ssg","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCpv000006Li5M - (R,R) isomer"},{"1":"5sqv","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894399 - (S,S) isomer"},{"1":"5sq3","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5028367848 - (R) isomer"},{"1":"5sow","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000118179920"},{"1":"5sr8","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4914650235 - (S) isomer"},{"1":"5spo","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with FRESH00020289192 - (S) isomer"},{"1":"5sa9","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z2697514548"},{"1":"5ss4","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCns000000RJoU"},{"1":"5sqn","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4914649780 - (S) isomer"},{"1":"5spw","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with FRESH00004674769 - (R,S,R) isomer"},{"1":"5srw","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2364914118 - (S) isomer"},{"1":"5ssn","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894382 - (R,S) isomer"},{"1":"5sqr","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL300016493575 - (R,S) isomer"},{"1":"5ssf","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCmr000000sTGN"},{"1":"5spy","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL300019621104"},{"1":"5sos","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000559260078"},{"1":"5sqz","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z1039058598"},{"1":"5spb","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894404 - (R,R) and (S,S) isomers"},{"1":"5ssl","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCoj00000doMWF"},{"1":"5sro","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562509 - (R) and (S) isomers"},{"1":"5sqj","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5021668601"},{"1":"5spf","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4718398569"},{"1":"5sqe","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894392- (S,S) isomer"},{"1":"5spl","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000611664196 - (S,S) isomer"},{"1":"5sr0","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z3649721459 - (R,S) and (S,R) isomers"},{"1":"5sai","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z1424343998"},{"1":"5sqg","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894430 - (R,R) and (S,S) isomers"},{"1":"5sp4","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4718398572"},{"1":"5sa5","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z1530301542"},{"1":"5srk","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5373433775 - (R) isomer"},{"1":"5sp2","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000579359572 - (R) and (S) isomers"},{"1":"5sq4","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2364980062 - (R) isomer"},{"1":"5sp7","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010903509 - (S,S) isomer"},{"1":"5srq","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z3831836449 - (R) isomer"},{"1":"5sqm","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5028367849 - (S) isomer"},{"1":"5spn","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with FRESH00010608284"},{"1":"5soz","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000827900828"},{"1":"5sab","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z31504642"},{"1":"5spp","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL250002155324"},{"1":"5sru","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5198562523 - (S) isomer"},{"1":"5sri","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5278734565 - pyrimido-indole core only"},{"1":"5ssp","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5459166285 - (R,R) and (S,S) isomers"},{"1":"5sqt","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000833624464 - (R,R) and (S,S) isomers"},{"1":"5ss7","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCnz000004Qo8S"},{"1":"5sah","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with EN300-100112"},{"1":"5sbf","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 NendoU"},{"1":"5sox","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000043461211"},{"1":"5srs","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2614735107 - (R) and (S) isomers"},{"1":"5spq","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with FRESH00014134848 - (R) isomer"},{"1":"5srl","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5352447655 - (R,R) isomer"},{"1":"5s70","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with EN300-181428"},{"1":"5ssb","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCmk000007RhkC"},{"1":"5sq0","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with REAL300007260658 - (S,S) isomer"},{"1":"5sou","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000285507655 - (R) and (S) isomers"},{"1":"5sa6","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with Z2856434783"},{"1":"5sqa","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5010894395 - (R,R) and (S,S) isomers"},{"1":"5sp3","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000450476923 - (S,R) isomer"},{"1":"5srg","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5265428403"},{"1":"5sps","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with FRESH00012962804 - (S) isomer"},{"1":"5s72","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NendoU in complex with BBL029427"},{"1":"5ss1","2":"2023","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINCou000000a2Hm"},{"1":"7n7r","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 NendoU in complex with Z2472938267"},{"1":"7n83","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 NendoU in complex with Z2443429438"},{"1":"7n7w","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 NendoU in complex with CSC000178569"},{"1":"8u1t","2":"2023","3":"Solid-state NMR","4":"SARS-CoV-2 Envelope Protein Transmembrane Domain: Dimeric Structure Determined by Solid-State NMR"},{"1":"8u28","2":"2023","3":"Electron Microscopy","4":"Gaussian mixture model based single particle refinement - SARS (SARS-CoV-2 Spike Proteins on intact virions from EMPIAR-10492)"},{"1":"8enx","2":"2023","3":"X-ray diffraction","4":"Crystal structure of beta'-COPI-WD40 domain Y33A mutant in complex with SARS-CoV-2 clientized spike tail heptapeptide."},{"1":"8el2","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 RBD bound to neutralizing antibody Fab ICO-hu23"},{"1":"8hn7","2":"2023","3":"X-ray diffraction","4":"Crystal structure of monoclonal antibody complexed with SARS-CoV-2 RBD"},{"1":"8gjm","2":"2023","3":"Electron Microscopy","4":"17b10 fab in complex with full-length SARS-CoV-2 Spike G614 trimer"},{"1":"8okn","2":"2023","3":"X-ray diffraction","4":"Crystal structure of F2F-2020198-00X bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"8gb7","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody 20A7"},{"1":"8sqk","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to RNA-nsp9 and GDP-betaS, as a pre-catalytic deRNAylation/mRNA capping intermediate"},{"1":"8sk5","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 neutralizing VHH 7A9 bound to the spike receptor binding domain"},{"1":"8ify","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.4/5 spike protein in complex with white-tailed deer ACE2"},{"1":"8if2","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Omicron BQ.1.1 variant spike protein in complex with its receptor ACE2"},{"1":"8hrd","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Delta variant in complex with IMCAS74 Fab and W14 Fab"},{"1":"8hpf","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 RBD in complex with fab L4.65 and L5.34"},{"1":"8g72","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike/Nb2 complex with 1 RBD up (local refinement at 5.6 A)"},{"1":"8eqt","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Orf3a in plasma membrane-like environment, MSP1D1 nanodisc"},{"1":"8gb5","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody 25F9"},{"1":"8g73","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike/Nb3 complex with 2 RBDs up and 3 Nb3 bound at 2.5 A"},{"1":"8fiv","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor Jun10541R"},{"1":"8fd5","2":"2023","3":"Electron Microscopy","4":"Nucleocapsid monomer structure from SARS-CoV-2"},{"1":"8smi","2":"2023","3":"X-ray diffraction","4":"Crystal structure of antibody WRAIR-2123 in complex with SARS-CoV-2 receptor binding domain"},{"1":"8gfr","2":"2023","3":"X-ray diffraction","4":"Room temperature X-ray structure of truncated SARS-CoV-2 main protease C145A mutant, residues 1-304, in complex with NBH2"},{"1":"8tbe","2":"2023","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro with Pomotrelvir"},{"1":"8gou","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S in complex with TH003 Fab"},{"1":"8gfu","2":"2023","3":"X-ray diffraction","4":"Room temperature X-ray structure of truncated SARS-CoV-2 main protease C145A mutant, residues 1-304, in complex with nirmatrelvir (NMV)"},{"1":"8gia","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 (Covid-19) Nsp3 macrodomain in complex with TFMU-ADPr"},{"1":"8f45","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a phenyl dimethyl sulfane inhibitor (cyclopropyl ketoamide warhead)"},{"1":"8f0h","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike with antibody Fabs 2A10 and 1H2 (Local refinement of the RBD and Fabs 1H2 and 2A10)"},{"1":"8i4s","2":"2023","3":"X-ray diffraction","4":"the complex structure of SARS-CoV-2 Mpro with D8"},{"1":"8hwt","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 RBD complexed with BD-604 and S304 Fab"},{"1":"8p9y","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 S protein S:D614G mutant in 3-down with binding site of an entry inhibitor"},{"1":"8t5p","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 ORF3a peptide in complex with TRAF3 TRAF domain"},{"1":"8i5i","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 delta variant spike receptor-binding domain (RBD) in complex with NCV2SG53 Fab"},{"1":"8i3s","2":"2023","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6P in complex with 7B3 Fab"},{"1":"8hp9","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 S-trimer in complex with fab L4.65 and L5.34"},{"1":"8hpv","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron Prototype S-trimer in complex with fab L4.65 and L5.34"},{"1":"8hn6","2":"2023","3":"X-ray diffraction","4":"Crystal structure of monoclonal antibody complexed with SARS-CoV-2 RBD"},{"1":"8hpu","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.4 RBD in complex with fab L4.65 and L5.34"},{"1":"8i30","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with 32j"},{"1":"8psd","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 XBB 1.0 closed conformation."},{"1":"8sxr","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro with C5a"},{"1":"8sqj","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to RNA-nsp9, as a noncatalytic RNA-nsp9 binding mode"},{"1":"8sq9","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to nsp9 and UMPCPP, as a pre-catalytic NMPylation intermediate"},{"1":"8skh","2":"2023","3":"X-ray diffraction","4":"Co-structure of SARS-CoV-2 (COVID-19 with covalent pyrazoline based inhibitors"},{"1":"8suz","2":"2023","3":"Solid-state NMR","4":"Open State of the SARS-CoV-2 Envelope Protein Transmembrane Domain, Determined by Solid-State NMR"},{"1":"8sg6","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) H163A Mutant Reduced with 20mM TCEP"},{"1":"8hpq","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.4 S-trimer in complex with fab L4.65 and L5.34"},{"1":"8htv","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 3a"},{"1":"8p99","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 S-protein:D614G mutant in 1-up conformation"},{"1":"8gb8","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 BA.2 receptor binding domain in complex with neutralizing antibody 20A7"},{"1":"8g75","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike/Nb4 complex with 2 RBDs up and 3 Nb4 bound"},{"1":"8gfo","2":"2023","3":"X-ray diffraction","4":"Room temperature X-ray structure of truncated SARS-CoV-2 main protease C145A mutant, residues 1-304, in complex with GC373"},{"1":"8okk","2":"2023","3":"X-ray diffraction","4":"Crystal structure of F2F-2020184-00X bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"8h06","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.4/5 RBD in complex with human ACE2 (local refinement)"},{"1":"8h51","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) Double Mutant (T21I and E166V) in Complex with Inhibitor Nirmatrelvir"},{"1":"8h3g","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) E166V Mutant in Complex with Inhibitor Enstrelvir"},{"1":"8cmf","2":"2023","3":"X-ray diffraction","4":"Human Leukocyte Antigen class II allotype DR1 presenting SARS-CoV-2 nsp3 epitope (orf1ab)1350-1364"},{"1":"8c1a","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 NSP3 macrodomain in complex with aztreonam"},{"1":"8d56","2":"2023","3":"Electron Microscopy","4":"One RBD-up state of SARS-CoV-2 BA.2 variant spike protein"},{"1":"8woy","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.4/5 RBD in complex with rabbit ACE2 (local refinement)"},{"1":"8aqv","2":"2023","3":"Electron Microscopy","4":"BA.2.12.1 SARS-CoV-2 Spike bound to mouse ACE2 (local)"},{"1":"8aou","2":"2023","3":"Solution NMR","4":"Solution NMR structure of full-length Nsp1 from SARS-CoV-2."},{"1":"8e25","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease M49I mutant in complex with Nirmatrelvir"},{"1":"8dk8","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8h3k","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) Double Mutant (L50F and E166V) in Complex with Inhibitor Enstrelvir"},{"1":"8gs9","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2 spike RBD in complex bound with VacBB-551"},{"1":"7whd","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the ZB8 neutralizing antibody Fab (2u1d)"},{"1":"8gzz","2":"2023","3":"Electron Microscopy","4":"Local refinement of SARS-CoV-2 Omicron BA.1 Spike glycoprotein in complex with rabbit monoclonal antibody 1H1 Fab"},{"1":"8h82","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (Mpro) Mutant (E166V) in complex with protease inhibitor Nirmatrelvir"},{"1":"8uum","2":"2023","3":"Electron Microscopy","4":"Prototypic SARS-CoV-2 spike (containing K417) in the open conformation"},{"1":"8bcz","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Delta-RBD complexed with Fabs BA.2-36, BA.2-23, EY6A and COVOX-45"},{"1":"8h5p","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) Double Mutant (L50F and E166V) in Complex with Inhibitor Nirmatrelvir"},{"1":"8h3m","2":"2023","3":"Electron Microscopy","4":"Conformation 1 of SARS-CoV-2 Omicron BA.1 Variant Spike protein complexed with MO1 Fab"},{"1":"8hed","2":"2023","3":"Electron Microscopy","4":"Local refinement of the SARS-CoV-2 Spike trimer in complex with RmAb 9H1 Fab"},{"1":"8hc8","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike trimer (6P) in complex with YB13-292 Fab, focused refinement of Fab region"},{"1":"8h5t","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain in complex with neutralizing nanobody Nb-015"},{"1":"8hc4","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 wildtype spike trimer (6P) in complex with 3 YB9-258 Fabs and 3 R1-32 Fabs (3 RBD up)"},{"1":"8h7k","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro Double Mutant (H41A and T21I) in complex with nsp4/5 peptidyl substrate"},{"1":"8h07","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 BA.4 variants S ectodomain trimer in complex with neutralizing antibody 10-5B and 6-2C"},{"1":"8hbk","2":"2023","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 3CL protease in complex with Ensitrelvir"},{"1":"8gry","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 RBD in complex with rat ACE2 (local refinement)"},{"1":"8hht","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with Hit-1"},{"1":"8hc9","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike trimer (6P) in complex with 3 YB13-292 Fabs (3 RBD down)"},{"1":"8h57","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) A193P Mutant in Complex with Inhibitor Nirmatrelvir"},{"1":"8hes","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD and NIV-10 complex"},{"1":"8hfx","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.1 spike protein in complex with white-tailed deer ACE2"},{"1":"8hhu","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with SY110"},{"1":"8h4y","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) F140L Mutant in Complex with Inhibitor Nirmatrelvir"},{"1":"8hc5","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 wildtype S1 in complex with YB9-258 Fab and R1-32 Fab"},{"1":"8hef","2":"2023","3":"X-ray diffraction","4":"The Crystal structure of deuterated S-217622 (Ensitrelvir) bound to the main protease (3CLpro/Mpro) of SARS-CoV-2"},{"1":"8crf","2":"2023","3":"X-ray diffraction","4":"Crystal structure of N-terminal SARS-CoV-2 nsp1 in complex with fragment hit 5E11 refined against anomalous diffraction data"},{"1":"8e1g","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 RBD in complex with Omicron-neutralizing antibody 2A10"},{"1":"8dkl","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8dig","2":"2023","3":"X-ray diffraction","4":"Virtual screening for novel SARS-CoV-2 main protease non-covalent and covalent inhibitors"},{"1":"8dza","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main protease A193T mutant in complex with Nirmatrelvir"},{"1":"8cz7","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro with compound C2"},{"1":"8cz4","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro with compound C3"},{"1":"8did","2":"2023","3":"X-ray diffraction","4":"Virtual screening for novel SARS-CoV-2 main protease non-covalent and covalent inhibitors"},{"1":"8b2t","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in complex with nirmatrelvir alkyne"},{"1":"7tel","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 3CLPro in complex with N-(4-(1H-imidazol-4-yl)phenyl)-N-(3-chloro-5-fluorobenzyl)-2-(isoquinolin-4-yl)acetamide"},{"1":"8dih","2":"2023","3":"X-ray diffraction","4":"Virtual screening for novel SARS-CoV-2 main protease non-covalent and covalent inhibitors"},{"1":"8dz0","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main protease in complex with Ensitrelvir"},{"1":"8dnt","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 specific T cell receptor"},{"1":"7qgi","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP14 in the absence of NSP10"},{"1":"8e69","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a m-fluorodimethyl oxybenzene inhibitor"},{"1":"8dfn","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) H164N Mutant"},{"1":"8e6a","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a p-chlorophenylethanol based inhibitor"},{"1":"8dv2","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Wuhan-hu-1-Spike-RBD bound to computationally engineered ACE2 mimetic CVD293"},{"1":"8dlb","2":"2023","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease in complex with compound Z2799209083"},{"1":"8dz2","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main protease in complex with Nirmatrelvir"},{"1":"8dkz","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8dfe","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) S144L Mutant"},{"1":"8ddm","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) E166R Mutant in Complex with Inhibitor GC376"},{"1":"8dzb","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor 11"},{"1":"8iov","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 XBB.1 spike RBD in complex with ACE2"},{"1":"8e64","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a benzimidazole dimethyl sulfane inhibitor"},{"1":"8e63","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a phenyl sulfane inhibitor"},{"1":"8dma","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.1 spike protein in complex with mouse ACE2 (focused refinement of RBD and ACE2)"},{"1":"8die","2":"2023","3":"X-ray diffraction","4":"Virtual screening for novel SARS-CoV-2 main protease non-covalent and covalent inhibitors"},{"1":"8iva","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike protein in complex with double nAbs XMA01 and 3E2 (local refinement)"},{"1":"8iho","2":"2023","3":"X-ray diffraction","4":"Crystal structures of SARS-CoV-2 papain-like protease in complex with covalent inhibitors"},{"1":"8dt8","2":"2023","3":"Electron Microscopy","4":"LM18/Nb136 bispecific tetra-nanobody immunoglobulin in complex with SARS-CoV-2-6P-Mut7 S protein (focused refinement)"},{"1":"8dox","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with an inhibitor TKB-245"},{"1":"8dm7","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 spike protein in complex with mouse ACE2"},{"1":"8dd1","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) H164N Mutant in Complex with Inhibitor GC376"},{"1":"8d47","2":"2023","3":"X-ray diffraction","4":"fp.006 Fab in complex with SARS-CoV-2 Fusion Peptide"},{"1":"8dz9","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main protease G143S mutant in complex with Nirmatrelvir"},{"1":"8dm3","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 spike protein in complex with Fab 4A8"},{"1":"8dmd","2":"2023","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease in complex with compound ZZ4461624291"},{"1":"8dmn","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8q95","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 BA.1 RBD with neutralizing-VHHs Ma16B06 and Ma3F05"},{"1":"8dic","2":"2023","3":"X-ray diffraction","4":"Virtual screening for novel SARS-CoV-2 main protease non-covalent and covalent inhibitors"},{"1":"8dv1","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Wuhan-hu-1-Spike-RBD bound to linker variant of affinity matured ACE2 mimetic CVD432"},{"1":"8aqs","2":"2023","3":"Electron Microscopy","4":"BA.4/5 SARS-CoV-2 Spike bound to human ACE2 (local)"},{"1":"8dzh","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Omicron BA.1.1.529 Spike trimer with two RBDs down in complex with the Fab fragment of human neutralizing antibody MB.02"},{"1":"8e68","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a p-fluorodimethyl oxybenzene inhibitor"},{"1":"8dm4","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 spike protein in complex with Fab 4A8 (focused refinement of NTD and 4A8)"},{"1":"8dm5","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 spike protein in complex with human ACE2"},{"1":"8d35","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (Mpro) C145A mutant in complex with peptide from human tRNA methyltransferase TRMT1"},{"1":"8dgu","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike stem helix peptide in complex with Fab of broadly neutralizing antibody CC25.106 isolated from a vaccinated COVID-19 convalescent"},{"1":"8dm1","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 spike protein"},{"1":"8dzi","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Omicron BA.1.1.529 Spike trimer with one RBD down in complex with the Fab fragment of human neutralizing antibody MB.02"},{"1":"8iv5","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike protein in complex with double nAbs 8H12 and 1C4 (local refinement)"},{"1":"8aqt","2":"2023","3":"Electron Microscopy","4":"Beta SARS-CoV-2 Spike bound to mouse ACE2 (local)"},{"1":"7tik","2":"2023","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 Omicron spike post-fusion bundle"},{"1":"8dcz","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) M165Y Mutant in Complex with Nirmatrelvir"},{"1":"7tek","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 3CLPro in complex with N-(4-(1H-pyrazol-4-yl)phenyl)-N-(3-chlorobenzyl)-2-(pyridin-3-yl)acetamide"},{"1":"7qif","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP14 in complex with 7MeGpppG."},{"1":"8djj","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8di3","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8d0z","2":"2023","3":"Electron Microscopy","4":"S728-1157 IgG in complex with SARS-CoV-2-6P-Mut7 Spike protein (focused refinement)"},{"1":"8cyu","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro with compound C5"},{"1":"8cmg","2":"2023","3":"X-ray diffraction","4":"Human Leukocyte Antigen class II allotype DR1 presenting SARS-CoV-2 nsp14 peptide (orf1ab)6420-6434"},{"1":"8uuo","2":"2023","3":"Electron Microscopy","4":"Prototypic SARS-CoV-2 spike (containing V417) in the open conformation"},{"1":"8c3d","2":"2023","3":"X-ray diffraction","4":"Sulfonated Calpeptin is a promising drug candidate against SARS-CoV-2 infections"},{"1":"8uun","2":"2023","3":"Electron Microscopy","4":"Prototypic SARS-CoV-2 spike (containing V417) in the closed conformation"},{"1":"8hfz","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 prototype spike protein in complex with white-tailed deer ACE2"},{"1":"8cmh","2":"2023","3":"X-ray diffraction","4":"Human Leukocyte Antigen class II allotype DR1 presenting SARS-CoV-2 Omicron (BA.1) Spike peptide S486-505"},{"1":"8c19","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 NSP3 macrodomain in complex with 1-methyl-4-[5-(morpholin-4-ylcarbonyl)-2-furyl]-1H-pyrrolo[2,3-b]pyridine"},{"1":"8h7w","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (Mpro) Mutant (S144A) in complex with protease inhibitor Nirmatrelvir"},{"1":"8h3d","2":"2023","3":"Electron Microscopy","4":"Structure of apo SARS-CoV-2 spike protein with one RBD up"},{"1":"8hg0","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 prototype spike protein receptor-binding domain in complex with white-tailed deer ACE2"},{"1":"8hc3","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike trimer (6P) in complex with 2 YB9-258 Fabs (2 RBD up)"},{"1":"8h08","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 BA.1 variants S ectodomain trimer in complex with neutralizing antibody 10-5B and 6-2C"},{"1":"8h3l","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) Double Mutant (T21I and E166V) in Complex with Inhibitor Enstrelvir"},{"1":"8hhy","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Delta Spike in complex with IS-9A"},{"1":"8h6n","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (Mpro) Mutant (T21I) in complex with protease inhibitor Nirmatrelvir"},{"1":"8h5u","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain in complex with neutralizing nanobody Nb-021"},{"1":"8hc2","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike trimer (6P) in complex with 1 YB9-258 Fab (1 RBD up)"},{"1":"8bwu","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nsp14 methyltransferase domain in complex with the SS148 inhibitor"},{"1":"8c3v","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Delta-RBD complexed with BA.2-13 Fab and C1 nanobody"},{"1":"8bbo","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Delta-RBD complexed with BA.2-36 Fab"},{"1":"8bbn","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Delta-RBD complexed with BA.2-10 and EY6A Fabs"},{"1":"8hhz","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 Spike in complex with IY-2A"},{"1":"8cmi","2":"2023","3":"X-ray diffraction","4":"Human Leukocyte Antigen class II allotype DR1 presenting SARS-CoV-2 Omicron (BA.1) Spike peptide S761-775"},{"1":"8cmb","2":"2023","3":"X-ray diffraction","4":"Human Leukocyte Antigen class II allotype DR1 presenting SARS-CoV-2 Spike peptide S486-505"},{"1":"8crk","2":"2023","3":"X-ray diffraction","4":"Crystal structure of N-terminal SARS-CoV-2 nsp1 in complex with fragment hit 7H2 refined against anomalous diffraction data"},{"1":"8c1v","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 S-trimer (3 RBDs up) bound to TriSb92, fitted into cryo-EM map"},{"1":"8c9n","2":"2023","3":"Electron Microscopy","4":"MiniCoV-ADDomer, a SARS-CoV-2 epitope presenting viral like particle"},{"1":"8bon","2":"2023","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the macrocyclic peptide S1B3inL1"},{"1":"8be1","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 RBD in complex with a Fab fragment of a neutralising antibody mRBD2"},{"1":"8fa1","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 HR1HR2 fusion core complex with N969K mutation"},{"1":"8eqj","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Orf3a in late endosome/lysosome-like membrane environment, MSP1D1 nanodisc"},{"1":"8f0g","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Omicron BA.1 spike in complex with antibody Fab 1C3"},{"1":"8f46","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a dimethyl phenyl sulfane inhibitor (cyano warhead)"},{"1":"8ey2","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Main protease C145S in complex with N-terminal peptide"},{"1":"8fah","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with SARS-CoV-2 reactive human antibody CR3022"},{"1":"8k45","2":"2023","3":"Electron Microscopy","4":"A potent and broad-spectrum neutralizing nanobody for SARS-CoV-2 viruses including all major Omicron strains"},{"1":"8dd6","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) H163A Mutant in Complex with GC376"},{"1":"8wox","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 prototype RBD in complex with rabbit ACE2 (local refinement)"},{"1":"8epp","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike trimer S2D14 with two RBDs in the open conformation"},{"1":"8gb0","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Spike H655Y variant, One RBD Open"},{"1":"8eoo","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing human antibodies WRAIR-2063 and WRAIR-2151"},{"1":"8eyj","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of uncleaved SARS-CoV-2 Main Protease C145S mutant in complex with Nirmatrelvir"},{"1":"8f2e","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of the CoV-Y domain of SARS-CoV-2 Nonstructural Protein 3"},{"1":"8g74","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike/Nb3 complex with 1 RBD up and 2 Nb3"},{"1":"8fa2","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 Omicron HR1-42G complex"},{"1":"8jop","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with 11a"},{"1":"8f0i","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with human antibody COVA309-22"},{"1":"8fdw","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 postfusion spike in membrane"},{"1":"8gop","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 specific private TCR RLQ7"},{"1":"8epn","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike trimer S2D14 in the 3-RBD Down conformation"},{"1":"8elj","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein in complex with the ICO-hu23 neutralizing antibody Fab fragment"},{"1":"8equ","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Orf3a in late endosome/lysosome-like environment, Saposin A nanodisc"},{"1":"8elp","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with antibody CC12.1 Fab and nanobody Nb-C4-240"},{"1":"8ej9","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8f44","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a dimethyl phenyl sulfane inhibitor"},{"1":"8gfk","2":"2023","3":"X-ray diffraction","4":"Room temperature X-ray structure of truncated SARS-CoV-2 main protease C145A mutant, residues 1-304"},{"1":"8fg2","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Nucleocapsid dimer structure determined from COVID-19 patients"},{"1":"8ej7","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8fy6","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in complex with covalent inhibitor"},{"1":"8t23","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of the RBD-ACE2 interface of the SARS-CoV-2 trimeric spike protein bound to ACE2 receptor after local refinement at upRBD conformation"},{"1":"8ens","2":"2023","3":"X-ray diffraction","4":"Crystal structure of beta'-COPI-WD40 domain in complex with SARS-CoV-2 spike tail hepta-peptide"},{"1":"8enw","2":"2023","3":"X-ray diffraction","4":"Crystal structure of beta'-COPI-WD40 domain in complex with SARS-CoV-2 clientized spike tail heptapeptide."},{"1":"8elo","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with antibody CC12.1 Fab and nanobody Nb-C4-225"},{"1":"8t25","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of the RBD-ACE2 interface of the SARS-CoV-2 trimeric spike protein bound to ACE2 receptor after local refinement at downRBD conformation."},{"1":"8th1","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of the G3BP1 NTF2-like domain bound to the IDR1 of SARS-CoV-2 nucleocapsid protein D3L mutant"},{"1":"8fiw","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor Jun10221"},{"1":"8fez","2":"2023","3":"Electron Microscopy","4":"Prefusion-stabilized SARS-CoV-2 spike protein"},{"1":"8gf2","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with antibodies eCR3022.20 and CC12.3"},{"1":"8gfn","2":"2023","3":"X-ray diffraction","4":"Room temperature X-ray structure of truncated SARS-CoV-2 main protease C145A mutant, residues 1-304, in complex with BBH1"},{"1":"8ftc","2":"2023","3":"X-ray diffraction","4":"Crystal structure of main protease of SARS-CoV-2 complexed with inhibitor"},{"1":"8fxb","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 XBB.1 spike RBD bound to the human ACE2 ectodomain and the S309 neutralizing antibody Fab fragment"},{"1":"8smt","2":"2023","3":"X-ray diffraction","4":"Crystal structure of antibody WRAIR-2134 in complex with SARS-CoV-2 receptor binding domain"},{"1":"8fxc","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 BQ.1.1 spike RBD bound to the human ACE2 ectodomain and the S309 neutralizing antibody Fab fragment"},{"1":"8elq","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with antibody CC12.1 Fab and nanobody Nb-C4-255"},{"1":"8gjn","2":"2023","3":"Electron Microscopy","4":"17B10 fab in complex with up-RBD of SARS-CoV-2 Spike G614 trimer"},{"1":"8f4s","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of the SARS-CoV-2 2'-O-Methyltransferase with Compound 5a bound to the Cryptic Pocket of nsp16"},{"1":"8gom","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 specific private TCR RLQ7 in complex with RLQ-HLA-A2"},{"1":"8gon","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 specific private TCR RLQ7 in complex with RLQ-T1006I-HLA-A2"},{"1":"8epq","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike trimer S2D14 with two RBDs exposed"},{"1":"8f4y","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 2'-O-Methyltransferase in Complex with Compound 5a covalently bound to nsp16 and nsp10"},{"1":"8fig","2":"2023","3":"X-ray diffraction","4":"Room-temperature X-ray structure of SARS-CoV-2 main protease double mutant E290A/R298A in complex with GC373"},{"1":"8gb6","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody 21B6"},{"1":"8fy7","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in complex with covalent inhibitor"},{"1":"8sk4","2":"2023","3":"X-ray diffraction","4":"Co-structure of SARS-CoV-2 (COVID-19 with covalent pyrazoline based inhibitors)"},{"1":"8t5q","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 ORF3a peptide in complex with TRAF2 TRAF domain"},{"1":"8eir","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 polyprotein substrate regulates the stepwise Mpro cleavage reaction"},{"1":"8elh","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of HLA-B*15:01 in complex with spike derived peptide NQKLIANQF from SARS-CoV-2 virus"},{"1":"7wnb","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain (RBD) in complex with NCV2SG48 Fab"},{"1":"8q71","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (MPro) in complex with the inhibitor GC-67"},{"1":"7wh8","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the ZB8 neutralizing antibody Fab (focused refinement on Fab-RBD)"},{"1":"7wn2","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain (RBD) in complex with NCV2SG53 Fab"},{"1":"8ios","2":"2023","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 XBB.1 spike glycoprotein (closed-1 state)"},{"1":"8iou","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 XBB.1 spike glycoprotein in complex with ACE2 (1-up state)"},{"1":"8uul","2":"2023","3":"Electron Microscopy","4":"Prototypic SARS-CoV-2 spike (containing K417) in the closed conformation"},{"1":"8cmd","2":"2023","3":"X-ray diffraction","4":"Human Leukocyte Antigen class II allotype DR1 presenting SARS-CoV-2 Spike peptide S761-775"},{"1":"8bzn","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 non-structural protein 10 (nsp10) variant T102I"},{"1":"8c8p","2":"2023","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the 10D12 heavy-chain-only antibody (local refinement)"},{"1":"8hi9","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with Robinetin"},{"1":"8hhx","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Delta Spike in complex with FP-12A"},{"1":"8hc6","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike trimer (6P) in complex with YB9-258 Fab, focused refinement of Fab region"},{"1":"8h5c","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Omicron BA.2.75 RBD in complex with human ACE2"},{"1":"8uys","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 5' proximal stem-loop 5"},{"1":"8hca","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike trimer (6P) in complex with 3 YB13-292 Fabs (1 RBD up)"},{"1":"8h6i","2":"2023","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 3C-like protease Double Mutant (L50F and E166V) in complex with a traditional Chinese Medicine Inhibitors"},{"1":"8e5z","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a dimethyl sulfonyl benzene inhibitor"},{"1":"8q7s","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD (Wuhan) with neutralizing VHHs Ma6F06 and Re21H01"},{"1":"8iot","2":"2023","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 XBB.1 spike glycoprotein (closed-2 state)"},{"1":"7whb","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the ZB8 neutralizing antibody Fab (3U)"},{"1":"7wd1","2":"2023","3":"X-ray diffraction","4":"Crystal structure of R14 bound to SARS-CoV-2 RBD"},{"1":"8ix3","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 BA.4/5 spike protein in complex with 1G11 (local refinement)"},{"1":"8iv4","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike protein in complex with double nAbs 8H12 and 3E2 (local refinement)"},{"1":"8iv8","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike protein in complex with double nAbs 3E2 and 1C4 (local refinement)"},{"1":"7wd2","2":"2023","3":"X-ray diffraction","4":"Crystal structure of S43 bound to SARS-CoV-2 RBD"},{"1":"8itu","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 Spike glycoprotein in complex with rabbit monoclonal antibody 1H1 IgG."},{"1":"8hgm","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike RBD in complex with neutralizing antibody NIV-11"},{"1":"8h01","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 Spike glycoprotein in complex with rabbit monoclonal antibody 1H1 Fab in class 2 conformation"},{"1":"8heb","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Spike trimer in complex with RmAb 9H1 Fab in the class 1 conformation"},{"1":"8h3n","2":"2023","3":"Electron Microscopy","4":"Conformation 2 of SARS-CoV-2 Omicron BA.1 Variant Spike protein complexed with MO1 Fab"},{"1":"8h00","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 Spike glycoprotein in complex with rabbit monoclonal antibody 1H1 Fab in the class 1 conformation"},{"1":"8h5f","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) L167F Mutant in Complex with Inhibitor Nirmatrelvir"},{"1":"8hcb","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike trimer (6P) in complex with 3 YB13-292 Fabs (2 RBD up)"},{"1":"8cme","2":"2023","3":"X-ray diffraction","4":"Human Leukocyte Antigen class II allotype DR1 presenting SARS-CoV-2 Membrane peptide M176-190"},{"1":"8hec","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Spike trimer in complex with RmAb 9H1 Fab in the class 2 conformation"},{"1":"8hfy","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike protein receptor-binding domain in complex with white-tailed deer ACE2"},{"1":"8crm","2":"2023","3":"X-ray diffraction","4":"Crystal structure of N-terminal SARS-CoV-2 nsp1 in complex with fragment hit 11C6 refined against anomalous diffraction data"},{"1":"8cmc","2":"2023","3":"X-ray diffraction","4":"Human Leukocyte Antigen class II allotype DR1 presenting SARS-CoV-2 Spike peptide S511-530"},{"1":"8hc7","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike trimer (6P) complex with YB9-258 Fab, focused refinement of RBD-dimer region"},{"1":"8hgl","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with neutralizing antibody NIV-11"},{"1":"8h3e","2":"2023","3":"Electron Microscopy","4":"Complex structure of a small molecule (SPC-14) bound SARS-CoV-2 spike protein, closed state"},{"1":"8aqw","2":"2023","3":"Electron Microscopy","4":"BA.4/5 SARS-CoV-2 Spike bound to mouse ACE2 (local)"},{"1":"7sh7","2":"2023","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI87"},{"1":"7ntq","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 Main Protease complexed with N-(pyridin-3-ylmethyl)thioformamide"},{"1":"8d55","2":"2023","3":"Electron Microscopy","4":"Closed state of SARS-CoV-2 BA.2 variant spike protein"},{"1":"7me0","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 NSP15 NendoU at pH 6.0"},{"1":"7qgw","2":"2023","3":"X-ray diffraction","4":"Sulfonated Calpeptin is a promising drug candidate against SARS-CoV-2 infections"},{"1":"8dkj","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8dm9","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.1 spike protein in complex with mouse ACE2"},{"1":"8e5x","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a dimethyl sulfinyl benzene inhibitor"},{"1":"8dxt","2":"2023","3":"X-ray diffraction","4":"Fab arm of antibody GAR12 bound to the receptor binding domain of SARS-CoV-2."},{"1":"8dqu","2":"2023","3":"X-ray diffraction","4":"Nanobody bound SARS-CoV-2 Nsp9"},{"1":"8dz1","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main protease mutant M49I in complex with Ensitrelvir"},{"1":"8e1y","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main protease A193S mutant in complex with Nirmatrelvir"},{"1":"8e26","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease N142S mutant in complex with Nirmatrelvir"},{"1":"8dkh","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8d48","2":"2023","3":"Electron Microscopy","4":"sd1.040 Fab in complex with SARS-CoV-2 Spike 2P glycoprotein"},{"1":"8czw","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with the cyclopropane based inhibitor 15d"},{"1":"8dxu","2":"2023","3":"X-ray diffraction","4":"Fab arms of antibodies GAR03 and 10G4 bound to the receptor binding domain of SARS-CoV-2 in a 1:1:1 complex."},{"1":"8dm8","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 spike protein in complex with mouse ACE2 (focused refinement of RBD and ACE2)"},{"1":"8dii","2":"2023","3":"X-ray diffraction","4":"Virtual screening for novel SARS-CoV-2 main protease non-covalent and covalent inhibitors"},{"1":"8dkk","2":"2023","3":"X-ray diffraction","4":"Polymorphism in SARS-CoV-2 Nsp5 main protease reveals differences in cleavage of viral and host substrates"},{"1":"8dd9","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) S144L Mutant in Complex with Inhibitor GC376"},{"1":"7qka","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Main Protease in complex with covalently bound GC376"},{"1":"8dm6","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 spike protein in complex with human ACE2 (focused refinement of RBD and ACE2)"},{"1":"8dif","2":"2023","3":"X-ray diffraction","4":"Virtual screening for novel SARS-CoV-2 main protease non-covalent and covalent inhibitors"},{"1":"8d5a","2":"2023","3":"Electron Microscopy","4":"Middle state of SARS-CoV-2 BA.2 variant spike protein"},{"1":"8aqu","2":"2023","3":"Electron Microscopy","4":"BA.1 SARS-CoV-2 Spike bound to mouse ACE2 (local)"},{"1":"8dzc","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor 17"},{"1":"7n7u","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 NendoU in complex with LIZA-7"},{"1":"8e61","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a m-chlorophenyl dimethyl sulfane inhibitor"},{"1":"7s4s","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 S receptor-binding domain (RBD) in complex CoV11 Fab"},{"1":"8ddi","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) E166N Mutant"},{"1":"8cyz","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro with compound C4"},{"1":"7t72","2":"2023","3":"X-ray diffraction","4":"Epitope-based selection of SARS-CoV-2 neutralizing antibodies from convalescent patients"},{"1":"8dm2","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 spike protein (focused refinement of NTD)"},{"1":"8ddl","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) H163A Mutant Apo Structure"},{"1":"8dl9","2":"2023","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease in complex with compound Z199538122"},{"1":"8dgb","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) Q192T Mutant in Complex with Inhibitor GC376"},{"1":"8dz6","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main protease mutant Q189K in complex with Nirmatrelvir"},{"1":"8dpr","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with inhibitor TKB-248"},{"1":"8dib","2":"2023","3":"X-ray diffraction","4":"Virtual screening for novel SARS-CoV-2 main protease non-covalent and covalent inhibitors"},{"1":"8e65","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with a p-chlorodimethyl oxybenzene inhibitor"},{"1":"8dnn","2":"2023","3":"X-ray diffraction","4":"Crystal structure of neutralizing antibody 80 in complex with SARS-CoV-2 receptor binding domain"},{"1":"7n7y","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 NendoU in complex with Z18197050"},{"1":"8k46","2":"2023","3":"Electron Microscopy","4":"A potent and broad-spectrum neutralizing nanobody for SARS-CoV-2 viruses including all major Omicron strains"},{"1":"8k47","2":"2023","3":"Electron Microscopy","4":"A potent and broad-spectrum neutralizing nanobody for SARS-CoV-2 viruses including all major Omicron strains"},{"1":"8czx","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with the cyclopropane based inhibitor 17d"},{"1":"8sgu","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain"},{"1":"8okm","2":"2023","3":"X-ray diffraction","4":"Crystal structure of F2F-2020197-00X bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"8i3u","2":"2023","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6P in complex with 14B1 Fab"},{"1":"7b0b","2":"2023","3":"X-ray diffraction","4":"Fab HbnC3t1p1_C6 bound to SARS-CoV-2 RBD"},{"1":"7gnq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NIR-WEI-dcc3321b-6 (Mpro-P2916)"},{"1":"7gmh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-37 (Mpro-P2183)"},{"1":"7gl4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-1981ceba-3 (Mpro-P1638)"},{"1":"7gjl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-UNI-3735e77e-1 (Mpro-P0600)"},{"1":"7ghy","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-c852c98b-10 (Mpro-P0031)"},{"1":"7gfw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-fa06b69f-6 (Mpro-x11894)"},{"1":"7geq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with VLA-UCB-1dbca3b4-15 (Mpro-x11498)"},{"1":"7gcf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-5a013bed-4 (Mpro-x10604)"},{"1":"7gaw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e194df51-1 (SARS2_MproA-x0862)"},{"1":"7gg7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with CHO-MSK-00c5269a-2 (Mpro-x12177)"},{"1":"7ges","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-bb423b95-7 (Mpro-x11501)"},{"1":"7gcz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-c59291d4-4 (Mpro-x10820)"},{"1":"7gbk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-95b75b4d-2 (Mpro-x10359)"},{"1":"7gnc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-8bb691af-8 (Mpro-P2487)"},{"1":"7glu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-6479a3a9-2 (Mpro-P2028)"},{"1":"7gka","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-5d232de5-8 (Mpro-P0816)"},{"1":"7giy","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BRU-CON-c4e3408a-1 (Mpro-P0145)"},{"1":"7ghl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with GAB-REV-70cc3ca5-18 (Mpro-x3366)"},{"1":"7ggd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-b5746674-38 (Mpro-x12350)"},{"1":"7geg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-a364e151-1 (Mpro-x11431)"},{"1":"7gcy","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-c59291d4-5 (Mpro-x10812)"},{"1":"7gb4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAR-DIA-23aa0b97-6 (Mpro-x10178)"},{"1":"7gng","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-705e09b8-1 (Mpro-P2607)"},{"1":"7gm4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-6f6ae286-5 (Mpro-P2080)"},{"1":"7gkd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-UNI-8d415491-3 (Mpro-P0850)"},{"1":"7gis","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-30 (Mpro-P0126)"},{"1":"7gh6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-916-2 (Mpro-x2910)"},{"1":"7gfp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-6b94ceba-5 (Mpro-x11801)"},{"1":"7gdt","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAK-UNK-0d6072ac-20 (Mpro-x11223)"},{"1":"7gc5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DUN-NEW-f8ce3686-24 (Mpro-x10506)"},{"1":"7gmw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e119ab4f-1 (Mpro-P2224)"},{"1":"7glc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-6f6ae286-3 (Mpro-P1858)"},{"1":"7gjm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-e4b030d8-11 (Mpro-P0601)"},{"1":"7gi2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-971238d3-5 (Mpro-P0039)"},{"1":"7ggg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ERI-UCB-d6de1f3c-2 (Mpro-x12582)"},{"1":"7get","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-0c2c77e1-1 (Mpro-x11507)"},{"1":"7gd0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-23 (Mpro-x10834)"},{"1":"7gb9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-11 (Mpro-x10248)"},{"1":"7gns","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-50a80394-1 (Mpro-P3050)"},{"1":"7gmb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-31 (Mpro-P2144)"},{"1":"7gkl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with SAM-UNK-2684b532-12 (Mpro-P0925)"},{"1":"7gj9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-c3a96089-4 (Mpro-P0186)"},{"1":"7ghk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAR-DIA-eace69ff-36 (Mpro-x3351)"},{"1":"7gga","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-e4b030d8-13 (Mpro-x12207)"},{"1":"7gei","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-8df914d1-2 (Mpro-x11454)"},{"1":"7gcl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-59 (Mpro-x10645)"},{"1":"7gfg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with CHO-MSK-6e55470f-5 (Mpro-x11723)"},{"1":"7gmq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-13 (Mpro-P2210)"},{"1":"7glb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-015fb6b4-2 (Mpro-P1835)"},{"1":"7gjv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-932d1078-3 (Mpro-P0661)"},{"1":"7gi4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-2eddb1ff-7 (Mpro-P0045)"},{"1":"7ggh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-c9c1e0d8-3 (Mpro-x12587)"},{"1":"7gex","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIC-UNK-66895286-1 (Mpro-x11532)"},{"1":"7gdf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-a3ef7265-23 (Mpro-x10995)"},{"1":"7gbx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-5a013bed-2 (Mpro-x10466)"},{"1":"7gmm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-20 (Mpro-P2204)"},{"1":"7gl8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-dc2604c4-1 (Mpro-P1788)"},{"1":"7gjz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-9739a092-6 (Mpro-P0764)"},{"1":"7gid","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-1 (Mpro-P0066)"},{"1":"7ggw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-2bb0cf2b-2 (Mpro-x12715)"},{"1":"7gcg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-21 (Mpro-x10606)"},{"1":"7gb8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-14 (Mpro-x10247)"},{"1":"7gk7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-ba1ac7b9-21 (Mpro-P0805)"},{"1":"7gfk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-044491d2-1 (Mpro-x11764)"},{"1":"7ge0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-6af13d92-2 (Mpro-x11276)"},{"1":"7gdr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LOR-NOR-30067bb9-2 (Mpro-x11208)"},{"1":"7gca","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-18 (Mpro-x10559)"},{"1":"7gmn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-14 (Mpro-P2205)"},{"1":"7gla","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-86c60949-2 (Mpro-P1812)"},{"1":"7gjn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-37aac4bd-4 (Mpro-P0602)"},{"1":"7gi9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-26 (Mpro-P0061)"},{"1":"7ggj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-4f474d93-1 (Mpro-x12659)"},{"1":"7gev","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-f13221e1-4 (Mpro-x11513)"},{"1":"7gd2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-3 (Mpro-x10862)"},{"1":"7gba","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ANT-OPE-d972fbad-1 (Mpro-x10296)"},{"1":"7gn3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-7fb4f80a-1 (Mpro-P2284)"},{"1":"7gle","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-13 (Mpro-P1889)"},{"1":"7gk1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-9e38fd34-1 (Mpro-P0766)"},{"1":"7gif","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-7 (Mpro-P0069)"},{"1":"7ggm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-afd4d4fd-2 (Mpro-x12677)"},{"1":"7gey","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIC-UNK-66895286-3 (Mpro-x11540)"},{"1":"7gdb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-968e8d9c-1 (Mpro-x10906)"},{"1":"7gbj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-32 (Mpro-x10355)"},{"1":"7gn4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-43f8f7d6-4 (Mpro-P2291)"},{"1":"7gls","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-bfb445d4-2 (Mpro-P2011)"},{"1":"7gkn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-611d11e7-4 (Mpro-P0978)"},{"1":"7giw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-f9802937-7 (Mpro-P0141)"},{"1":"7ggx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-971238d3-5 (Mpro-x12716)"},{"1":"7gfd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-2eddb1ff-8 (Mpro-x11641)"},{"1":"7gdk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0da5ad92-21 (Mpro-x11025)"},{"1":"7gbz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-11 (Mpro-x10474)"},{"1":"7gnf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIK-ENA-5d9157e9-5 (Mpro-P2606)"},{"1":"7glz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-14 (Mpro-P2067)"},{"1":"7gkp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-3 (Mpro-P1007)"},{"1":"7gjc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-BAS-c2bc0d80-6 (Mpro-P0207)"},{"1":"7ghw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-8b8a49e1-4 (Mpro-P0026)"},{"1":"7gg6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-477dc5b7-2 (Mpro-x12171)"},{"1":"7ge6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-a3ef7265-18 (Mpro-x11346)"},{"1":"7gnl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NIR-WEI-dcc3321b-1 (Mpro-P2757)"},{"1":"7gm6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-7889e8da-5 (Mpro-P2090)"},{"1":"7gks","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-11 (Mpro-P1062)"},{"1":"7gj8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-fce787c2-6 (Mpro-P0185)"},{"1":"7ghe","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-916a2c5a-4 (Mpro-x3303)"},{"1":"7gft","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAR-UCB-f313ec4d-2 (Mpro-x11813)"},{"1":"7gej","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-c9c1e0d8-2 (Mpro-x11458)"},{"1":"7gcw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NAU-LAT-445f63e5-6 (Mpro-x10800)"},{"1":"7gb2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MED-COV-4280ac29-25 (Mpro-x10155)"},{"1":"7gmy","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-e69ed63d-13 (Mpro-P2242)"},{"1":"7glk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-76744c27-4 (Mpro-P1986)"},{"1":"7gjy","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e9e99895-13 (Mpro-P0747)"},{"1":"7gik","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-1f3f1a6f-1 (Mpro-P0098)"},{"1":"7ghb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with GAB-REV-70cc3ca5-8 (Mpro-x3080)"},{"1":"7gfh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-f7918075-8 (Mpro-x11742)"},{"1":"7ge2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-6af13d92-1 (Mpro-x11313)"},{"1":"7gci","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-27 (Mpro-x10610)"},{"1":"7gd3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-c59291d4-3 (Mpro-x10870)"},{"1":"7gb5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0da5ad92-2 (Mpro-x10201)"},{"1":"7gn7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-1cbc2fae-2 (Mpro-P2381)"},{"1":"7gm5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-2 (Mpro-P2089)"},{"1":"7gk2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-ba1ac7b9-13 (Mpro-P0772)"},{"1":"7gic","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-13 (Mpro-P0065)"},{"1":"7ggn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ERI-UCB-d6de1f3c-1 (Mpro-x12679)"},{"1":"7gf4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-17 (Mpro-x11560)"},{"1":"7gdn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-0e996074-1 (Mpro-x11159)"},{"1":"7gbp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-10 (Mpro-x10395)"},{"1":"7gmv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-8bb691af-4 (Mpro-P2222)"},{"1":"7glf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-22 (Mpro-P1978)"},{"1":"7gjk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-4aa06b95-7 (Mpro-P0578)"},{"1":"7gi1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0e5afe9d-3 (Mpro-P0038)"},{"1":"7gge","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-6d04362c-1 (Mpro-x12419)"},{"1":"7geb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-93268d01-8 (Mpro-x11417)"},{"1":"7gcp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NAU-LAT-64f4b287-5 (Mpro-x10710)"},{"1":"7gay","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-031a96cc-8 (Mpro-x10022)"},{"1":"7gms","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-be9e6f63-3 (Mpro-P2215)"},{"1":"7glj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-4fff0a85-3 (Mpro-P1983)"},{"1":"7gju","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e9e99895-6 (Mpro-P0655)"},{"1":"7gib","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-10 (Mpro-P0064)"},{"1":"7ggu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0e5afe9d-3 (Mpro-x12699)"},{"1":"7gf6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-f42f3716-6 (Mpro-x11564)"},{"1":"7gdp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NAU-LAT-8502cac5-2 (Mpro-x11186)"},{"1":"7gc2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-93268d01-13 (Mpro-x10484)"},{"1":"7gmo","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-12c4873b-5 (Mpro-P2206)"},{"1":"7gl9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with VLA-UCB-50c39ae8-2 (Mpro-P1800)"},{"1":"7gk0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-BAS-c2bc0d80-7 (Mpro-P0765)"},{"1":"7gim","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-UNI-3735e77e-2 (Mpro-P0111)"},{"1":"7gh5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-714-12 (Mpro-x2908)"},{"1":"7gf9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TAT-ENA-80bfd3e5-4 (Mpro-x11590)"},{"1":"7gdq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LOR-NOR-30067bb9-15 (Mpro-x11204)"},{"1":"7gby","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-5 (Mpro-x10473)"},{"1":"7gko","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-30 (Mpro-P0996)"},{"1":"7gg9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ANT-DIA-62e4526e-1 (Mpro-x12204)"},{"1":"7kf4","2":"2023","3":"X-ray diffraction","4":"Crystal structure from SARS-CoV-2 NendoU NSP15"},{"1":"7giu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-05e671eb-10 (Mpro-P0130)"},{"1":"7gki","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RUB-POS-1325a9ea-4 (Mpro-P0887)"},{"1":"7giz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-5d232de5-4 (Mpro-P0148)"},{"1":"7ghd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-4e090d3a-57 (Mpro-x3298)"},{"1":"7gg3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with AAR-RCN-748c104b-1 (Mpro-x12080)"},{"1":"7gen","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-1e5f28a7-1 (Mpro-x11488)"},{"1":"7gd6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-38 (Mpro-x10888)"},{"1":"7gbm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-3 (Mpro-x10377)"},{"1":"7gmz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-c3ea9889-6 (Mpro-P2243)"},{"1":"7glq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-8df914d1-2 (Mpro-P2007)"},{"1":"7gkh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-UNI-8d415491-1 (Mpro-P0884)"},{"1":"7gnd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIK-UNK-78dbf1b8-1 (Mpro-P2601)"},{"1":"7glv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-a54ce14d-2 (Mpro-P2031)"},{"1":"7ghh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TOB-UNK-c2aba166-1 (Mpro-x3325)"},{"1":"7gfm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-1901c25b-1 (Mpro-x11790)"},{"1":"7gdw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BRU-THA-a358fbdd-2 (Mpro-x11233)"},{"1":"7gce","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-93268d01-7 (Mpro-x10598)"},{"1":"7gne","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIK-ENA-5d9157e9-6 (Mpro-P2605)"},{"1":"7gm0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-bfd29aac-1 (Mpro-P2070)"},{"1":"7gk3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-fb82b63d-3 (Mpro-P0776)"},{"1":"7gi7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-6c284e65-1 (Mpro-P0057)"},{"1":"7ggr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-c7771779-1 (Mpro-x12695)"},{"1":"7gf0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-bb423b95-1 (Mpro-x11542)"},{"1":"7gde","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0da5ad92-12 (Mpro-x10976)"},{"1":"7gbo","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-20 (Mpro-x10392)"},{"1":"7gnu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-133e7cd9-2 (Mpro-P3074)"},{"1":"7gmf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e119ab4f-3 (Mpro-P2178)"},{"1":"7gkx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-28 (Mpro-P1202)"},{"1":"7gjg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-UNI-8e43a71e-8 (Mpro-P0238)"},{"1":"7ghz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIC-UNK-91acba05-6 (Mpro-P0033)"},{"1":"7ggq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-92e193ae-1 (Mpro-x12692)"},{"1":"7gf8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TAT-ENA-80bfd3e5-37 (Mpro-x11587)"},{"1":"7gdm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-714a760b-16 (Mpro-x11044)"},{"1":"7gc7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-39 (Mpro-x10525)"},{"1":"7gn1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-8bb691af-6 (Mpro-P2263)"},{"1":"7glw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-1cbc2fae-1 (Mpro-P2036)"},{"1":"7gkb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-96f51285-5 (Mpro-P0831)"},{"1":"7giv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-de59a476-2 (Mpro-P0135)"},{"1":"7gh2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-090737b9-1 (Mpro-x12735)"},{"1":"7gfv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-2db6411e-2 (Mpro-x11852)"},{"1":"7gea","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-2 (Mpro-x11372)"},{"1":"7gc6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NIR-THE-c331be7a-6 (Mpro-x10513)"},{"1":"7ghs","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ERI-UCB-d6de1f3c-2 (Mpro-P0018)"},{"1":"7gg2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-8a69d52e-7 (Mpro-x12073)"},{"1":"7gel","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-cedd89ab-1 (Mpro-x11475)"},{"1":"7gmk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-d899bab6-1 (Mpro-P2201)"},{"1":"7gl2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-1981ceba-4 (Mpro-P1623)"},{"1":"7gjj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-9739a092-9 (Mpro-P0394)"},{"1":"7ghu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with VLA-UCB-29506327-1 (Mpro-P0022)"},{"1":"7ggb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-044491d2-7 (Mpro-x12300)"},{"1":"7gef","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-93268d01-11 (Mpro-x11428)"},{"1":"7gcj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-19 (Mpro-x10626)"},{"1":"7gmj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-477dc5b7-4 (Mpro-P2197)"},{"1":"7gl3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-1981ceba-2 (Mpro-P1624)"},{"1":"7gjr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e9e99895-11 (Mpro-P0630)"},{"1":"7gi6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-4aa06b95-1 (Mpro-P0056)"},{"1":"7ggo","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-8b8a49e1-4 (Mpro-x12682)"},{"1":"7gew","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-bb423b95-2 (Mpro-x11530)"},{"1":"7gd7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ADA-UNI-f8e79267-2 (Mpro-x10889)"},{"1":"7gaz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DUN-NEW-f8ce3686-14 (Mpro-x10049)"},{"1":"7gld","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-15 (Mpro-P1879)"},{"1":"7gjt","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-ba1ac7b9-11 (Mpro-P0642)"},{"1":"7gi5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BRU-THA-92256091-17 (Mpro-P0053)"},{"1":"7ggl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-ce760d3f-8 (Mpro-x12674)"},{"1":"7gf3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0da5ad92-5 (Mpro-x11557)"},{"1":"7gdh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LOR-NEU-c8f11034-6 (Mpro-x11001)"},{"1":"7gc0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-21 (Mpro-x10476)"},{"1":"7gnr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e48723dc-2 (Mpro-P3038)"},{"1":"7gm9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-1bed62cf-3 (Mpro-P2113)"},{"1":"7gkq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-18 (Mpro-P1010)"},{"1":"7gj3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-29afea89-2 (Mpro-P0157)"},{"1":"7ghf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-4e090d3a-47 (Mpro-x3305)"},{"1":"7gfr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-3c72d439-1 (Mpro-x11810)"},{"1":"7ge1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-6af13d92-3 (Mpro-x11294)"},{"1":"7gcs","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-18 (Mpro-x10733)"},{"1":"7gbc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-2eddb1ff-1 (Mpro-x10314)"},{"1":"7gn6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e119ab4f-5 (Mpro-P2358)"},{"1":"7glr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIC-UNK-08cd9c58-1 (Mpro-P2010)"},{"1":"7gk5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-5d232de5-7 (Mpro-P0793)"},{"1":"7gir","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-971238d3-1 (Mpro-P0125)"},{"1":"7gdc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-0a73fcb8-7 (Mpro-x10942)"},{"1":"7gn9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-12c4873b-2 (Mpro-P2402)"},{"1":"7gm1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-2e8b2191-10 (Mpro-P2072)"},{"1":"7gja","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-de59a476-4 (Mpro-P0187)"},{"1":"7ghr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-ce760d3f-8 (Mpro-P0017)"},{"1":"7glg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-18 (Mpro-P1980)"},{"1":"7gk6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-fb82b63d-1 (Mpro-P0800)"},{"1":"7gio","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-fce787c2-3 (Mpro-P0121)"},{"1":"7gh8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAR-DIA-23aa0b97-13 (Mpro-x2964)"},{"1":"7gmg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e119ab4f-2 (Mpro-P2182)"},{"1":"7gl1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-8293a91a-8 (Mpro-P1507)"},{"1":"7gji","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-d08626de-3 (Mpro-P0243)"},{"1":"7ghx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0e5afe9d-1 (Mpro-P0030)"},{"1":"7gez","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-6747fa38-1 (Mpro-x11541)"},{"1":"7gd9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DUN-NEW-f8ce3686-23 (Mpro-x10899)"},{"1":"7gbn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-MED-2de63afb-14 (Mpro-x10387)"},{"1":"7gmu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-22 (Mpro-P2219)"},{"1":"7gli","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-af1eef35-2 (Mpro-P1982)"},{"1":"7gjw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with KAD-UNI-80f122c8-2 (Mpro-P0743)"},{"1":"7gie","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-17 (Mpro-P0068)"},{"1":"7gf7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TAT-ENA-80bfd3e5-7 (Mpro-x11579)"},{"1":"7gbt","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-7e92b6ca-2 (Mpro-x10419)"},{"1":"7gnj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-976a33d5-1 (Mpro-P2724)"},{"1":"7gm8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-7889e8da-3 (Mpro-P2101)"},{"1":"7gkm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-e0fe77e5-13 (Mpro-P0950)"},{"1":"7gjb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5d65ec79-2 (Mpro-P0188)"},{"1":"7ghc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with GAB-REV-70cc3ca5-13 (Mpro-x3108)"},{"1":"7gfo","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ROB-IMP-e811baff-1 (Mpro-x11798)"},{"1":"7gdy","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LOR-NOR-30067bb9-6 (Mpro-x11258)"},{"1":"7gbv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-22 (Mpro-x10422)"},{"1":"7gnb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-ee636701-1 (Mpro-P2468)"},{"1":"7glo","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-5a013bed-2 (Mpro-P2001)"},{"1":"7gkg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-2f867453-1 (Mpro-P0878)"},{"1":"7git","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-fce787c2-5 (Mpro-P0129)"},{"1":"7gh0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-ce760d3f-4 (Mpro-x12723)"},{"1":"7gfe","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-6344a35d-1 (Mpro-x11642)"},{"1":"7gdu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-SGC-a8a902d9-1 (Mpro-x11225)"},{"1":"7gbw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-8 (Mpro-x10423)"},{"1":"7gn5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-968bafd9-1 (Mpro-P2295)"},{"1":"7gm3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-c20a539d-4 (Mpro-P2075)"},{"1":"7gkk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JIN-POS-6dc588a4-6 (Mpro-P0906)"},{"1":"7gj6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-fce787c2-4 (Mpro-P0178)"},{"1":"7gmx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-21 (Mpro-P2229)"},{"1":"7gfs","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAR-UCB-f313ec4d-6 (Mpro-x11812)"},{"1":"7gec","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with VLA-UCB-00f2c2b3-7 (Mpro-x11424)"},{"1":"7gcn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-41 (Mpro-x10679)"},{"1":"7gav","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-f7918075-2 (SARS2_MproA-x0854)"},{"1":"7ggi","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-14ad9fe9-1 (Mpro-x12640)"},{"1":"7gf1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-95b75b4d-5 (Mpro-x11543)"},{"1":"7gdd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ADA-UCB-6c2cb422-1 (Mpro-x10959)"},{"1":"7gbq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-23 (Mpro-x10396)"},{"1":"7gha","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAN-LON-a5fc619e-8 (Mpro-x3077)"},{"1":"7gfn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-7374c256-2 (Mpro-x11797)"},{"1":"7gdz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-c9973a83-1 (Mpro-x11271)"},{"1":"7gc4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0da5ad92-18 (Mpro-x10494)"},{"1":"7gmr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-7fb4f80a-2 (Mpro-P2214)"},{"1":"7gl5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-10fcb19e-1 (Mpro-P1661)"},{"1":"7gjq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-78e1d523-1 (Mpro-P0627)"},{"1":"7gij","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5d65ec79-1 (Mpro-P0097)"},{"1":"7gh1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-2bb0cf2b-1 (Mpro-x12731)"},{"1":"7gfc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ERI-UCB-a0b0dbcb-4 (Mpro-x11616)"},{"1":"7gbr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-7e92b6ca-4 (Mpro-x10403)"},{"1":"7ggv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIC-UNK-91acba05-6 (Mpro-x12710)"},{"1":"7gdj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0da5ad92-16 (Mpro-x11013)"},{"1":"7gkf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-UNI-8d415491-6 (Mpro-P0872)"},{"1":"7gii","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-8416115c-13 (Mpro-P0091)"},{"1":"7gfa","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-3b92565d-1 (Mpro-x11609)"},{"1":"7gdl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-15 (Mpro-x11041)"},{"1":"7gc1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-25 (Mpro-x10478)"},{"1":"7ghm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ERI-UCB-ce40166b-17 (Mpro-P0008)"},{"1":"7gfl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-bfb445d4-2 (Mpro-x11789)"},{"1":"7gdv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NAU-LAT-8502cac5-6 (Mpro-x11231)"},{"1":"7gc9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-22 (Mpro-x10555)"},{"1":"7gm2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-2e8b2191-12 (Mpro-P2074)"},{"1":"7gkj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-ba1ac7b9-19 (Mpro-P0904)"},{"1":"7gj2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-70ae9412-1 (Mpro-P0154)"},{"1":"7ghi","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAN-PUR-6788a628-2 (Mpro-x3333)"},{"1":"7gg0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-e44ffd04-1 (Mpro-x12026)"},{"1":"7geh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-06d94977-2 (Mpro-x11432)"},{"1":"7gcv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-2eddb1ff-7 (Mpro-x10789)"},{"1":"7gb0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-c0143b99-1 (Mpro-x10082)"},{"1":"7gcd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-23 (Mpro-x10575)"},{"1":"7gax","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with GAB-REV-70cc3ca5-4 (Mpro-x10019)"},{"1":"7gn0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-43f8f7d6-6 (Mpro-P2256)"},{"1":"7gll","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-4fff0a85-1 (Mpro-P1988)"},{"1":"7gk8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-dd3ad2b5-2 (Mpro-P0808)"},{"1":"7gip","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-8416115c-5 (Mpro-P0122)"},{"1":"7ggt","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-c852c98b-10 (Mpro-x12698)"},{"1":"7gf2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIC-UNK-08cd9c58-1 (Mpro-x11548)"},{"1":"7gdi","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-49816e9b-1 (Mpro-x11011)"},{"1":"7gbl","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-MED-2de63afb-1 (Mpro-x10371)"},{"1":"7gbu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ADA-UNI-f8e79267-5 (Mpro-x10421)"},{"1":"7gml","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-28 (Mpro-P2203)"},{"1":"7gi3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-090737b9-1 (Mpro-P0041)"},{"1":"7gg5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-87c86d55-1 (Mpro-x12143)"},{"1":"7f3q","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 RBD in complex with A5-10 Fab and A34-2 Fab"},{"1":"7gly","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-23 (Mpro-P2057)"},{"1":"7gkt","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e6dd326d-8 (Mpro-P1073)"},{"1":"7gj1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with VLA-UNK-82501c2c-1 (Mpro-P0153)"},{"1":"7ghn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-f2460aef-1 (Mpro-P0009)"},{"1":"7gfx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-199e2e7c-1 (Mpro-x12000)"},{"1":"7ge3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-714a760b-3 (Mpro-x11317)"},{"1":"7gcq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-2eddb1ff-2 (Mpro-x10723)"},{"1":"7gb1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with STE-KUL-2e0d2e88-2 (Mpro-x10150)"},{"1":"7gkv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-23 (Mpro-P1090)"},{"1":"7gjd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-3b97339c-2 (Mpro-P0208)"},{"1":"7gi0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-92e193ae-1 (Mpro-P0034)"},{"1":"7en9","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CLpro in complex with the non-covalent inhibitor WU-02"},{"1":"7gds","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LOR-NOR-30067bb9-18 (Mpro-x11212)"},{"1":"7gmt","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-19 (Mpro-P2218)"},{"1":"7glm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-21 (Mpro-P1990)"},{"1":"7gk4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-bb7ffe78-1 (Mpro-P0777)"},{"1":"7gia","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-33 (Mpro-P0063)"},{"1":"7ggk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-64a710fa-1 (Mpro-x12661)"},{"1":"7ge8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-7e92b6ca-16 (Mpro-x11366)"},{"1":"7ghq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-64a710fa-1 (Mpro-P0016)"},{"1":"7gg8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-cd485364-2 (Mpro-x12202)"},{"1":"7gem","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0da5ad92-15 (Mpro-x11485)"},{"1":"7gno","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NIR-WEI-dcc3321b-4 (Mpro-P2838)"},{"1":"7gme","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-38 (Mpro-P2177)"},{"1":"7gku","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-40 (Mpro-P1079)"},{"1":"7gj5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-5d232de5-5 (Mpro-P0171)"},{"1":"7ghj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with SIM-SYN-f15aaa3a-1 (Mpro-x3348)"},{"1":"7gg4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-52b62a6f-11 (Mpro-x12136)"},{"1":"7ged","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-cedd89ab-2 (Mpro-x11426)"},{"1":"7gcm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NIR-THE-c331be7a-2 (Mpro-x10678)"},{"1":"7gk9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-477dc5b7-5 (Mpro-P0811)"},{"1":"7gix","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with VLA-UNK-cf7facf1-1 (Mpro-P0143)"},{"1":"7gh7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-714-22 (Mpro-x2912)"},{"1":"7gfb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-b3e365b9-1 (Mpro-x11612)"},{"1":"7gdo","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-c0c213c9-14 (Mpro-x11164)"},{"1":"7gc3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-cedd89ab-4 (Mpro-x10488)"},{"1":"7gke","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-dd3ad2b5-3 (Mpro-P0851)"},{"1":"7giq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-c852c98b-5 (Mpro-P0124)"},{"1":"7gh9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-362d364a-10 (Mpro-x2971)"},{"1":"7gj7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with VLA-UCB-50c39ae8-7 (Mpro-P0179)"},{"1":"7gee","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-93268d01-5 (Mpro-x11427)"},{"1":"7gfq","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-b3e365b9-3 (Mpro-x11809)"},{"1":"7gbh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-49816e9b-2 (Mpro-x10334)"},{"1":"7gco","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-7e92b6ca-1 (Mpro-x10700)"},{"1":"7gbb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAN-LON-a5fc619e-3 (Mpro-x10306)"},{"1":"7gnn","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NIR-WEI-dcc3321b-3 (Mpro-P2775)"},{"1":"7gmd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-7174c657-6 (Mpro-P2176)"},{"1":"7gnh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-853c0ffa-9 (Mpro-P2649)"},{"1":"7gln","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-16 (Mpro-P1991)"},{"1":"7gnk","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-ecbed2ba-12 (Mpro-P2730)"},{"1":"7gma","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-7174c657-5 (Mpro-P2141)"},{"1":"7gkw","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e6dd326d-6 (Mpro-P1200)"},{"1":"7gfz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ERI-UCB-a0b0dbcb-8 (Mpro-x12025)"},{"1":"7ghp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-4f474d93-1 (Mpro-P0012)"},{"1":"7gct","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BRU-LEF-c49414a7-1 (Mpro-x10756)"},{"1":"7ge9","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-12 (Mpro-x11368)"},{"1":"7gnp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NIR-WEI-dcc3321b-5 (Mpro-P2889)"},{"1":"7gmc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5cd9ea36-17 (Mpro-P2147)"},{"1":"7gkr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-2 (Mpro-P1015)"},{"1":"7gj4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-5d232de5-1 (Mpro-P0160)"},{"1":"7gl6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-8695a11f-1 (Mpro-P1701)"},{"1":"7gjo","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-a13804f0-3 (Mpro-P0607)"},{"1":"7gj0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-70ae9412-2 (Mpro-P0151)"},{"1":"7ghg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAV-CRI-3edb475e-4 (Mpro-x3324)"},{"1":"7gfj","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-c0c213c9-1 (Mpro-x11757)"},{"1":"7gc8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-18 (Mpro-x10535)"},{"1":"7ggf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-6d04362c-2 (Mpro-x12423)"},{"1":"7geu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-50fe53e8-1 (Mpro-x11508)"},{"1":"7gd5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-d2866bdf-1 (Mpro-x10876)"},{"1":"7gbi","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-a3ef7265-3 (Mpro-x10338)"},{"1":"7gna","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-4483ae88-4 (Mpro-P2415)"},{"1":"7glt","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-4223bc15-12 (Mpro-P2017)"},{"1":"7gh4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-00c1612e-1 (Mpro-x12777)"},{"1":"7glx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-2e8b2191-11 (Mpro-P2039)"},{"1":"7gkc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-9c80c481-1 (Mpro-P0845)"},{"1":"7gih","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-16 (Mpro-P0075)"},{"1":"7ggy","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-966f8da6-2 (Mpro-x12717)"},{"1":"7gff","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-5d20d11c-1 (Mpro-x11708)"},{"1":"7gdg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAR-DIA-23aa0b97-14 (Mpro-x10996)"},{"1":"7gbs","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-95b75b4d-1 (Mpro-x10417)"},{"1":"7gd1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-58 (Mpro-x10856)"},{"1":"7gbd","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-MED-2de63afb-2 (Mpro-x10322)"},{"1":"7gl0","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-a13804f0-4 (Mpro-P1477)"},{"1":"7gjf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-8c98ee63-2 (Mpro-P0224)"},{"1":"7ght","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-c9c1e0d8-3 (Mpro-P0019)"},{"1":"7gfu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-bbbbc21a-3 (Mpro-x11831)"},{"1":"7ge7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ADA-UCB-6c2cb422-3 (Mpro-x11354)"},{"1":"7gcu","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-25 (Mpro-x10787)"},{"1":"7gb3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with DAV-CRI-3edb475e-6 (Mpro-x10172)"},{"1":"7gek","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-8df914d1-4 (Mpro-x11473)"},{"1":"7gcr","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with WAR-XCH-72a8c209-5 (Mpro-x10728)"},{"1":"7gb6","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JOR-UNI-2fc98d0b-12 (Mpro-x10236)"},{"1":"7gfi","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with WIL-MOD-03b86a88-6 (Mpro-x11743)"},{"1":"7gdx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LOR-NOR-30067bb9-7 (Mpro-x11254)"},{"1":"7gn2","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-e69ed63d-1 (Mpro-P2273)"},{"1":"7glh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-b38839dc-1 (Mpro-P1981)"},{"1":"7gil","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-6 (Mpro-P0108)"},{"1":"7gh3","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-3c65e9ce-4 (Mpro-x12740)"},{"1":"7gcb","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-13 (Mpro-x10565)"},{"1":"7gmp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-7 (Mpro-P2207)"},{"1":"7gl7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-6bf93aa8-1 (Mpro-P1783)"},{"1":"7gjs","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e9e99895-2 (Mpro-P0640)"},{"1":"7gin","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-869ac754-1 (Mpro-P0114)"},{"1":"7ggz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-3c65e9ce-2 (Mpro-x12719)"},{"1":"7gfy","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with LOR-NOR-c954e7ad-4 (Mpro-x12010)"},{"1":"7ge4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-714a760b-19 (Mpro-x11318)"},{"1":"7gcc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-95b75b4d-4 (Mpro-x10566)"},{"1":"7gd4","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-c59291d4-2 (Mpro-x10871)"},{"1":"7gbg","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-1 (Mpro-x10329)"},{"1":"7en8","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CLpro in complex with the non-covalent inhibitor WU-04"},{"1":"8okl","2":"2023","3":"X-ray diffraction","4":"Crystal structure of F2F-2020185-01X bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"8ifz","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.4/5 spike protein receptor-binding domain in complex with white-tailed deer ACE2"},{"1":"8hq7","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron Prototype RBD in complex with fab L4.65 and L5.34"},{"1":"8s9g","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 BN.1 spike RBD bound to the human ACE2 ectodomain and the S309 neutralizing antibody Fab fragment"},{"1":"8i5h","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 delta variant spike receptor-binding domain (RBD) in complex with NCV2SG48 Fab"},{"1":"6wji","2":"2023","3":"X-ray diffraction","4":"2.05 Angstrom Resolution Crystal Structure of C-terminal Dimerization Domain of Nucleocapsid Phosphoprotein from SARS-CoV-2"},{"1":"7gni","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-b1ef7fe3-1 (Mpro-P2660)"},{"1":"7gnt","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-50a80394-2 (Mpro-P3054)"},{"1":"7gmi","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-90fd5f68-2 (Mpro-P2185)"},{"1":"7gkz","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-afb6844f-1 (Mpro-P1474)"},{"1":"7gje","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-e69ad64a-2 (Mpro-P0213)"},{"1":"7gho","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with PET-UNK-c9c1e0d8-4 (Mpro-P0010)"},{"1":"7gg1","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ERI-UCB-a0b0dbcb-9 (Mpro-x12064)"},{"1":"7ge5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ERI-UCB-5b47150d-6 (Mpro-x11339)"},{"1":"7gck","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BAR-COM-0f94fc3d-48 (Mpro-x10638)"},{"1":"7gn8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-c7726e07-5 (Mpro-P2385)"},{"1":"7glp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ADA-UCB-6c2cb422-1 (Mpro-P2005)"},{"1":"7gjx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-3ccb8ef6-1 (Mpro-P0744)"},{"1":"7gi8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MIC-UNK-45817b9b-1 (Mpro-P0060)"},{"1":"7ggs","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0e5afe9d-1 (Mpro-x12696)"},{"1":"7geo","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDJ-MED-50fe53e8-3 (Mpro-x11493)"},{"1":"7gcx","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-590ac91e-24 (Mpro-x10801)"},{"1":"7gb7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JOR-UNI-2fc98d0b-6 (Mpro-x10237)"},{"1":"7gjp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-fe871b40-11 (Mpro-P0626)"},{"1":"7gig","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RAL-THA-2d450e86-12 (Mpro-P0074)"},{"1":"7ggp","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with VLA-UCB-29506327-1 (Mpro-x12686)"},{"1":"7gf5","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with EDG-MED-0da5ad92-7 (Mpro-x11562)"},{"1":"7gda","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with TRY-UNI-2eddb1ff-3 (Mpro-x10900)"},{"1":"7gnm","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with NIR-WEI-dcc3321b-2 (Mpro-P2761)"},{"1":"7gm7","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with ALP-POS-a577c8a2-1 (Mpro-P2099)"},{"1":"7gky","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with RUB-POS-1325a9ea-14 (Mpro-P1470)"},{"1":"7gjh","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with BEN-DND-f2e727cd-5 (Mpro-P0240)"},{"1":"7ghv","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-afd4d4fd-2 (Mpro-P0025)"},{"1":"7ggc","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-044491d2-3 (Mpro-x12321)"},{"1":"7ger","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with MAT-POS-f7918075-5 (Mpro-x11499)"},{"1":"7gd8","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-119787ef-1 (Mpro-x10898)"},{"1":"7gbf","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAN-GHE-83b26c96-9 (Mpro-x10327)"},{"1":"7gbe","2":"2023","3":"X-ray diffraction","4":"Group deposition SARS-CoV-2 main protease in complex with inhibitors from the COVID Moonshot -- Crystal Structure of SARS-CoV-2 main protease in complex with JAG-UCB-a3ef7265-20 (Mpro-x10324)"},{"1":"7keh","2":"2023","3":"X-ray diffraction","4":"Crystal structure from SARS-CoV-2 NendoU NSP15"},{"1":"7x93","2":"2023","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab765"},{"1":"7uob","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to GTP, in a pre-catalytic state"},{"1":"7uo7","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to ATP, in a pre-catalytic state"},{"1":"7uu8","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in its apo-form"},{"1":"7uo4","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to Remdesivir triphosphate, in a pre-catalytic state"},{"1":"7ukl","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of Antibody 12-16 in complex with prefusion SARS-CoV-2 Spike glycoprotein"},{"1":"7uo9","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to UTP, in a pre-catalytic state"},{"1":"7x2h","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain bound with 6-2C Fab"},{"1":"7ul1","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with the neutralizing IGHV3-53-encoded antibody EH3 isolated from a nonvaccinated pediatric patient"},{"1":"7uue","2":"2023","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI85"},{"1":"7uu7","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in its apo-form"},{"1":"7urs","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 S delta variant receptor-binding domain (RBD) in complex CoV11 Fab crystal form 2"},{"1":"7uu6","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in its apo-form"},{"1":"7xmn","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 ORF8"},{"1":"7xsb","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor binding domain bound with P5S-3B11 Fab"},{"1":"8aeb","2":"2023","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease complexed with N-(pyridin-3-ylmethyl)thioformamide"},{"1":"7xdk","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta Spike protein in complex with BA7054 and BA7125 fab"},{"1":"7xs8","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor binding domain bound with P5S-1H1 Fab"},{"1":"7xd2","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 S ectodomain trimer in complex with neutralizing antibody 10-5B"},{"1":"7xcz","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta RBD in complex with BA7054 and BA7125 fab (local refinement)"},{"1":"7x94","2":"2023","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab712"},{"1":"7urq","2":"2023","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 S delta variant receptor-binding domain (RBD) in complex CoV11 Fab crystal form 1"},{"1":"7rb2","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 NSP15 NendoU in BIS-Tris pH 6.0"},{"1":"7wqv","2":"2023","3":"X-ray diffraction","4":"Crystal structure of a neutralizing monoclonal antibody (Ab08) in complex with SARS-CoV-2 receptor-binding domain (RBD)"},{"1":"7vnu","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the N-terminal domain of SARS-CoV-2 nucleocapsid protein"},{"1":"7xda","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta RBD in complex with BA7208 and BA7125 fab (local refinement)"},{"1":"7x95","2":"2023","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab709"},{"1":"7xsa","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor binding domain bound with P2S-2E9 Fab"},{"1":"7x96","2":"2023","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab847"},{"1":"8aaa","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 S RBD in complex with a stapled peptide"},{"1":"7uua","2":"2023","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI8"},{"1":"7ul0","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with the ridge-binding nAb EH8 isolated from a nonvaccinated pediatric patient"},{"1":"7wzo","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 nucleocapsid protein N-terminal domain in complex with Ubl1"},{"1":"7uuc","2":"2023","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI19"},{"1":"7yvj","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S RBD in complex with TH236 Fab"},{"1":"7y7j","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer in complex with 1F Fab"},{"1":"7yvp","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S in complex with TH272/281 Fab"},{"1":"7y72","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein trimer complexed with Fab fragment of anti-RBD antibody E7 (focused refinement on Fab-RBD interface)"},{"1":"7yh6","2":"2023","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike RBD in complex with neutralizing antibody NIV-8"},{"1":"7yv8","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 RBD in complex with golden hamster ACE2 (local refinement)"},{"1":"7yvu","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 RBD in complex with mouse ACE2 (local refinement)"},{"1":"7u8e","2":"2023","3":"X-ray diffraction","4":"Crystal structure of antibody Ab246 in complex with SARS-CoV-2 receptor binding domain"},{"1":"7tq4","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with the cyclopropane based inhibitor 6c"},{"1":"7yh7","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with neutralizing antibody NIV-8 (state 2)"},{"1":"7y71","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein trimer complexed with Fab fragment of anti-RBD antibody E7"},{"1":"7yvl","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S RBD in complex with TH272 Fab"},{"1":"7y6l","2":"2023","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab816"},{"1":"7y7k","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 RBD in complex with 1F Fab"},{"1":"7yvn","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S in complex with TH281 Fab"},{"1":"7yck","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with FP-12A Fab"},{"1":"7ycn","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with IY-2A Fab"},{"1":"7z3u","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Main Protease after incubation with Sulfo-Calpeptin"},{"1":"7yow","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 omicron variant spike receptor-binding domain (RBD) in complex with NCV2SG48 Fab"},{"1":"7yc5","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with K202.B bispecific antibody"},{"1":"7yvi","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S in complex with TH236 Fab"},{"1":"7uu9","2":"2023","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in its apo-form"},{"1":"7tq6","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with the cyclopropane based inhibitor 13d"},{"1":"7rb0","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 NSP15 NendoU at pH 7.5"},{"1":"7uud","2":"2023","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI33"},{"1":"7uub","2":"2023","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI12"},{"1":"7yvf","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S RBD in complex with TH027 Fab"},{"1":"7yhw","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2.12.1 RBD in complex with human ACE2 (local refinement)"},{"1":"7y3o","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with human antibody BIOLS56"},{"1":"7y6d","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta variant spike proteins on intact virions: 3 Closed RBD"},{"1":"7z8o","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 S RBD in complex with a stapled peptide"},{"1":"7z4s","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro in complex with cyclic peptide GM4 including unnatural amino acids."},{"1":"7yvk","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S in complex with TH272 Fab"},{"1":"7ycl","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with IS-9A Fab"},{"1":"7yve","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S in complex with TH027 Fab"},{"1":"7tq5","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with the cyclopropane based inhibitor 10d"},{"1":"7tq2","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with the cyclopropane based inhibitor 1c"},{"1":"7xsc","2":"2023","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor binding domain bound with P5S-2B10"},{"1":"7uoe","2":"2023","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to CTP, in a pre-catalytic state"},{"1":"7yvm","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S RBD in complex with TH272 Fab"},{"1":"7yj3","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.2 RBD in complex with human ACE2 (local refinement)"},{"1":"7yvo","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S in complex with TH027/132 Fab"},{"1":"7yvh","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S RBD in complex with TH132 Fab"},{"1":"7xdl","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta Spike protein in complex with BA7208 and BA7125 fab"},{"1":"7y6n","2":"2023","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab803"},{"1":"7yvg","2":"2023","3":"Electron Microscopy","4":"Omicron BA.4/5 SARS-CoV-2 S in complex with TH132 Fab"},{"1":"7xdb","2":"2023","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron Spike protein in complex with BA7208 fab"},{"1":"7tq3","2":"2023","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with the cyclopropane based inhibitor 5c"},{"1":"7dk0","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with MW05 Fab"},{"1":"7dgb","2":"2022","3":"X-ray diffraction","4":"The co-crystal structure of SARS-CoV-2 main protease with (S)-2-cinnamamido-4-methyl-N-((S)-1-oxo-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)pentanamide"},{"1":"7e9p","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with 35B5 Fab(state2, local refinement of the RBD and 35B5 Fab)"},{"1":"7nij","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 main protease (Mpro) in a novel conformational state."},{"1":"7sd4","2":"2022","3":"Solid-state NMR","4":"SARS-CoV-2 Nucleocapsid N-terminal domain (N-NTD) protein"},{"1":"7s71","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI35"},{"1":"7mw6","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 Spike trimer with three RBDs up in complex with the Fab fragment of human neutralizing antibody clone 2"},{"1":"7mw3","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 Spike trimer with two RBDs down in complex with the Fab fragment of human neutralizing antibody clone 6"},{"1":"7t43","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 3c"},{"1":"7n1a","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 YLQ peptide binds to HLA-A2"},{"1":"8err","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 spike ectodomain trimer in complex with the S2X324 neutralizing antibody Fab fragment"},{"1":"7mky","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 frameshifting pseudoknot RNA"},{"1":"7ltj","2":"2022","3":"X-ray diffraction","4":"Room-temperature X-ray structure of SARS-CoV-2 main protease (3CL Mpro) in complex with a non-covalent inhibitor Mcule-5948770040"},{"1":"7mc5","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 ExoN-nsp10 complex"},{"1":"7b2u","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 1"},{"1":"7b2j","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 5"},{"1":"7b77","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 8"},{"1":"7bij","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 13"},{"1":"7k3t","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) at 1.2 A Resolution and a Possible Capture of Zinc Binding Intermediate"},{"1":"7k40","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in Complex with Covalent Inhibitor Boceprevir at 1.35 A Resolution"},{"1":"7k6d","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease Co-Crystal Structure with Telaprevir Determined from Crystals Grown with 40 nL Acoustically Ejected Mpro Droplets at 1.48 A Resolution (Cryo-protected)"},{"1":"7k6e","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease Co-Crystal Structure with Telaprevir Determined from Crystals Grown with 40 nL Acoustically Ejected Mpro Droplets at 1.63 A Resolution (Direct Vitrification)"},{"1":"7wk8","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron spike protein SD1 in complex with S3H3 Fab"},{"1":"7whz","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with three human neutralizing antibodies"},{"1":"7w9f","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-RBD-8D3"},{"1":"8gs6","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 BA.2.75 spike glycoprotein (closed state 1)"},{"1":"7p5s","2":"2022","3":"Solution NMR","4":"NMR structure of a peptide deriving from SARS-CoV-2 Lineages P.1 and B.1.351 S RBD 482-506 fragment in HFIP/H2O"},{"1":"7pbe","2":"2022","3":"X-ray diffraction","4":"Emergence of immune escape at dominant SARS-CoV-2 killer T-cell epitope"},{"1":"7p35","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 3CL protease in complex with rupintrivir"},{"1":"7p79","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with sybodyb#15 in a 1up/1up-out/1down conformation."},{"1":"7whc","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CLpro catalytic domain"},{"1":"7w99","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-ACE2-C2a"},{"1":"7s6y","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI32"},{"1":"7sc1","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, R40-1G8"},{"1":"7s5r","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibodies CV07-287 and COVA1-16"},{"1":"8cy9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 1-23"},{"1":"8cyd","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 2-45"},{"1":"7tc4","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 15/16"},{"1":"7skz","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of VN01H1 Fab in complex with SARS-CoV-2 S fusion peptide"},{"1":"7swn","2":"2022","3":"Electron Microscopy","4":"G32A4 Fab in complex with SARS-CoV-2 Spike 6P (RBD local reconstruction)"},{"1":"8d4j","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) H172Y Mutant"},{"1":"7tht","2":"2022","3":"Electron Microscopy","4":"CryoEM structure of SARS-CoV-2 S protein in complex with Receptor Binding Domain antibody DH1042"},{"1":"7tf3","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Kappa (B.1.617.1) Q484A spike protein"},{"1":"7ta4","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 9/10"},{"1":"7sn1","2":"2022","3":"X-ray diffraction","4":"Structure of human SARS-CoV-2 neutralizing antibody C1C-A3 Fab"},{"1":"7t9l","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike protein in complex with human ACE2 (focused refinement of RBD and ACE2)"},{"1":"8cxn","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 2-57"},{"1":"7tbf","2":"2022","3":"Electron Microscopy","4":"Locally refined region of SARS-CoV-2 spike in complex with antibodies B1-182.1 and A19-61.1"},{"1":"7t48","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 9c"},{"1":"7s73","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI37"},{"1":"7s6l","2":"2022","3":"Electron Microscopy","4":"J08 fragment antigen binding in complex with SARS-CoV-2-6P-Mut7 S protein (conformation 3)"},{"1":"7s72","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI36"},{"1":"7tey","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta (B.1.617.2) spike protein"},{"1":"7t9j","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 Omicron spike protein"},{"1":"7t7b","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with a cross-neutralizing antibody ADI-62113 Fab"},{"1":"7suo","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of the G3BP1 NTF2-like domain bound to the IDR1 of SARS-CoV-2 nucleocapsid protein"},{"1":"8dlo","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Gamma (P.1) spike protein"},{"1":"8dce","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Receptor-Binding Domain SPEEDesign Immunogen 1 Bound to C144 scFv"},{"1":"8d36","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 fusion peptide in complex with neutralizing antibody COV44-62"},{"1":"7tex","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta (B.1.617.2) spike protein in complex with human ACE2"},{"1":"7t8m","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 5/6"},{"1":"7si9","2":"2022","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with PF-07321332"},{"1":"8drv","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp8-nsp9 (C8) cut site sequence"},{"1":"7tgr","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 main protease in complex with GC376"},{"1":"7t70","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 4/5"},{"1":"7wk6","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike protein with human ACE2 (focus refinement on RBD-1/ACE2)"},{"1":"7w1s","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain in complex with neutralizing nanobody Nb-007"},{"1":"7we8","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike protein in complex with Fab XGv265"},{"1":"7vyr","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with the D27 neutralizing antibody Fab fragment"},{"1":"7wbl","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of human ACE2 complexed with SARS-CoV-2 Omicron RBD"},{"1":"7w9e","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-8D3"},{"1":"7wdf","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Beta spike in complex with two S3H3 Fabs"},{"1":"8gw1","2":"2022","3":"Electron Microscopy","4":"A mechanism for SARS-CoV-2 RNA capping and its inhibition by nucleotide analogue inhibitors"},{"1":"7wk4","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike protein with ACE2, C1 state"},{"1":"7wcd","2":"2022","3":"Electron Microscopy","4":"Cryo EM structure of SARS-CoV-2 spike in complex with TAU-2212 mAbs in conformation 4"},{"1":"7wgv","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein trimer in closed state"},{"1":"7wd7","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Beta spike in complex with three S5D2 Fabs"},{"1":"8gwg","2":"2022","3":"Electron Microscopy","4":"A mechanism for SARS-CoV-2 RNA capping and its inhibition by nucleotide analogue inhibitors"},{"1":"7p19","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD Q498Y complexed with human ACE2"},{"1":"8gwi","2":"2022","3":"Electron Microscopy","4":"A mechanism for SARS-CoV-2 RNA capping and its inhibition by nucleotide analogue inhibitors"},{"1":"7p2g","2":"2022","3":"X-ray diffraction","4":"Identification of low micromolar SARS-CoV-2 Mpro inhibitors from hits identified by in silico screens"},{"1":"8gwf","2":"2022","3":"Electron Microscopy","4":"A mechanism for SARS-CoV-2 RNA capping and its inhibition by nucleotide analogue inhibitors"},{"1":"8gwn","2":"2022","3":"Electron Microscopy","4":"A mechanism for SARS-CoV-2 RNA capping and its inhibitor of AT-527"},{"1":"8gwe","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 E-RTC complex with RNA-nsp9 and GMPPNP"},{"1":"7p7b","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with sybody no68 in a 1up/2down conformation"},{"1":"8gz5","2":"2022","3":"X-ray diffraction","4":"Crystal structure of neutralizing VHH P17 in complex with SARS-CoV-2 Alpha variant spike receptor-binding domain"},{"1":"8drr","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp4-nsp5 (C4) cut site sequence"},{"1":"8doy","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with an inhibitor TKB-198"},{"1":"8cyc","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 2-34"},{"1":"8cy6","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 2-65"},{"1":"7tf0","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Kappa (B.1.617.1) spike protein in complex with human ACE2"},{"1":"7t45","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 7c"},{"1":"8cya","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 2-67"},{"1":"8dln","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Beta (B.1.351) spike protein in complex with human ACE2 (focused refinement of RBD and ACE2)"},{"1":"8cyb","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 1-8"},{"1":"8cy7","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 2-34"},{"1":"7r4i","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 spike in complex with the 2.15 neutralizing nanobody"},{"1":"7r18","2":"2022","3":"Electron Microscopy","4":"Mink Variant SARS-CoV-2 Spike in Closed conformation"},{"1":"8dw9","2":"2022","3":"X-ray diffraction","4":"Crystal structure of neutralizing antibody D29 Fab in complex with SARS-CoV-2 spike receptor binding domain (RBD)"},{"1":"8d8q","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike RBD in complex with DMAbs 2130 and 2196"},{"1":"7qti","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S Omicron Spike B.1.1.529 - 3-P2G3 and 1-P5C3 Fabs (Global)"},{"1":"7the","2":"2022","3":"Electron Microscopy","4":"Structure of RBD directed antibody DH1042 in complex with SARS-CoV-2 spike: Local refinement of RBD-Fab interface"},{"1":"8b56","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (MPro) in complex with the inhibitor GD-9"},{"1":"8dlj","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Alpha (B.1.1.7) spike protein in complex with human ACE2"},{"1":"8b0t","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease adduct with Au(PEt3)Br"},{"1":"7tge","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron 1-RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-Omicron)"},{"1":"7t49","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 10c"},{"1":"7thm","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 nsp12/7/8 complex with a native N-terminus nsp9"},{"1":"7tca","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike in complex with antibody A19-46.1"},{"1":"8cxq","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with a pan-sarbecovirus nanobody 1-22"},{"1":"7tiw","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in complex with inhibitor EB54"},{"1":"7tez","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Kappa (B.1.617.1) spike protein in complex with human ACE2 (focused refinement of RBD and ACE2)"},{"1":"7t9y","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 8/9"},{"1":"8drw","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp9-nsp10 (C9) cut site sequence"},{"1":"7tf2","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Kappa (B.1.617.1) Q484I spike protein"},{"1":"7tj2","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 endoribonuclease Nsp15 bound to dsRNA"},{"1":"7tgx","2":"2022","3":"Electron Microscopy","4":"Prototypic SARS-CoV-2 G614 spike (open form)"},{"1":"8drz","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp13-nsp14 (C13) cut site sequence"},{"1":"7tj0","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in complex with inhibitor SL-4-241"},{"1":"7s3s","2":"2022","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease in complex with compound Z1530724813"},{"1":"7sbu","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with a highly potent antibody J08 Fab"},{"1":"7swp","2":"2022","3":"Electron Microscopy","4":"G32Q4 Fab in complex with SARS-CoV-2 Spike 6P (RBD local reconstruction)"},{"1":"7o36","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 N-CTD in complex with GTP (II)"},{"1":"7tf4","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Kappa (B.1.617.1) spike protein (focused refinement of RBD)"},{"1":"7tc9","2":"2022","3":"Electron Microscopy","4":"Locally refined region of SARS-CoV-2 spike in complex with antibody A19-46.1"},{"1":"7ta7","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 10/11"},{"1":"7s70","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI34"},{"1":"7s6k","2":"2022","3":"Electron Microscopy","4":"J08 fragment antigen binding in complex with SARS-CoV-2-6P-Mut2 S protein (conformation 2)"},{"1":"7sd5","2":"2022","3":"X-ray diffraction","4":"Crystallographic structure of neutralizing antibody 10-40 in complex with SARS-CoV-2 spike receptor binding domain"},{"1":"7sn2","2":"2022","3":"Electron Microscopy","4":"Structure of human SARS-CoV-2 neutralizing antibody C1C-A3 Fab"},{"1":"7s5q","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 B.1.351 variant receptor binding domain in complex with neutralizing antibodies CS44 and COVA1-16"},{"1":"7tb2","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 12/13"},{"1":"7t46","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 8c"},{"1":"7t8r","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 7/8"},{"1":"8gz6","2":"2022","3":"X-ray diffraction","4":"Crystal structure of neutralizing VHH P17 in complex with SARS-CoV-2 Alpha variant spike receptor-binding domain"},{"1":"7o05","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 N-CTD"},{"1":"7s75","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI42"},{"1":"7wk2","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron S-close"},{"1":"7wd9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Beta spike in complex with three S3H3 Fabs"},{"1":"7qck","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like Protease bound to N-(2,5-dihydroxybenzylidene)-thiosemicarbazone"},{"1":"7qny","2":"2022","3":"X-ray diffraction","4":"The receptor binding domain of SARS-CoV-2 spike glycoprotein in complex with COVOX-58 and COVOX-158 Fabs"},{"1":"8dry","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp12-nsp13 (C12) cut site sequence"},{"1":"8dlx","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Epsilon (B.1.429) spike protein in complex with VH ab6"},{"1":"7r17","2":"2022","3":"Electron Microscopy","4":"Beta Variant SARS-CoV-2 Spike with 2 Erect RBDs"},{"1":"7r11","2":"2022","3":"Electron Microscopy","4":"Dissociated S1 domain of Beta Variant SARS-CoV-2 Spike bound to ACE2 (Non-Uniform Refinement)"},{"1":"7qt6","2":"2022","3":"X-ray diffraction","4":"Room temperature In-situ SARS-CoV-2 MPRO with bound Z1367324110"},{"1":"7wgx","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein trimer in closed state after treatment with Cathepsin L"},{"1":"7wea","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike protein in complex with two XGv347 binding to one close state RBD and one open state RBD"},{"1":"7qus","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike, C3 symmetry"},{"1":"8dxs","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of RBD-directed neutralizing antibody P2B4 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"8dwa","2":"2022","3":"X-ray diffraction","4":"Crystal structure of neutralizing antibody P1D9 Fab in complex with SARS-CoV-2 spike receptor binding domain (RBD)"},{"1":"8drt","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp6-nsp7 (C6) cut site sequence (form 2)"},{"1":"8dlz","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 D614G spike protein in complex with VH ab6"},{"1":"7qtk","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S Omicron Spike B.1.1.529 - RBD down - 1-P2G3 Fab (Local)"},{"1":"7qnw","2":"2022","3":"X-ray diffraction","4":"The receptor binding domain of SARS-CoV-2 Omicron variant spike glycoprotein in complex with Beta-55 and EY6A Fabs"},{"1":"7qf1","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with the human antibody CV2.6264"},{"1":"7qdh","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S protein S:D614G mutant 1-up"},{"1":"7qci","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like Protease bound to N-(3,4-dihydroxybenzylidene)-thiosemicarbazone"},{"1":"7r1a","2":"2022","3":"Electron Microscopy","4":"Furin Cleaved Alpha Variant SARS-CoV-2 Spike in complex with 3 ACE2"},{"1":"7w98","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-ACE2-C1"},{"1":"7qt7","2":"2022","3":"X-ray diffraction","4":"Room temperature In-situ SARS-CoV-2 MPRO with bound Z4439011520"},{"1":"7qez","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with the ultrapotent antibody CV2.1169 and CR3022"},{"1":"8e4r","2":"2022","3":"X-ray diffraction","4":"Room-temperature X-ray structure of SARS-CoV-2 main protease H41A miniprecursor mutant in complex with GC373"},{"1":"8dtt","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike stem helix peptide in complex with neutralizing antibody COV93-03"},{"1":"8cwu","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with a cross-neutralizing nanobody 1-21"},{"1":"8dll","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Beta (B.1.351) spike protein"},{"1":"7tcq","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 neutralizing antibody WS6 in complex with spike S2 peptide"},{"1":"7tb8","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with antibodies B1-182.1 and A19-61.1"},{"1":"8dli","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Alpha (B.1.1.7) spike protein"},{"1":"8d4m","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) S144A Mutant in Complex with Inhibitor GC376"},{"1":"7tei","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron 1-RBD up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-Omicron)"},{"1":"7tas","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2K146 neutralizing antibody Fab fragment (local refinement of the RBD and S2K146)"},{"1":"8ds2","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Mpro in complex with the nsp13-nsp14 (C13) cut site sequence (form 2)"},{"1":"8dlr","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Gamma (P.1) spike protein in complex with Fab 4-8 (focused refinement of NTD and 4-8)"},{"1":"7r4r","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 spike in complex with the 1.10 neutralizing nanobody"},{"1":"8dlv","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Epsilon (B.1.429) spike protein in complex with human ACE2 (focused refinement of RBD and ACE2)"},{"1":"8ays","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 non-structural protein-1 (nsp1) in complex with 4-(2-aminothiazol-4-yl)phenol"},{"1":"7tia","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in complex with inhibitor NK01-14"},{"1":"7tcc","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike in complex with antibodies A19-46.1 and B1-182.1"},{"1":"8dlq","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Gamma (P.1) spike protein in complex with human ACE2 (focused refinement of RBD and ACE2)"},{"1":"7tat","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2K146 neutralizing antibody Fab fragment (two receptor-binding domains open)"},{"1":"8dlm","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Beta (B.1.351) spike protein in complex with human ACE2"},{"1":"8drs","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp6-nsp7 (C6) cut site sequence"},{"1":"8czi","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 HR1HR2 fusion core complex with extended HR2"},{"1":"7tiy","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in complex with inhibitor NK01-48"},{"1":"7tfr","2":"2022","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (3CL Mpro) in complex with NBH-2"},{"1":"8dtx","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike stem helix peptide in complex with neutralizing antibody COV89-22"},{"1":"8cyj","2":"2022","3":"Electron Microscopy","4":"RBD of SARS-CoV-2 Spike protein in complex with pan-sarbecovirus nanobodies 2-10, 2-67, 2-62 and 1-25"},{"1":"7s6w","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI29"},{"1":"7tgy","2":"2022","3":"Electron Microscopy","4":"Prototypic SARS-CoV-2 G614 spike (closed form)"},{"1":"7tdu","2":"2022","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Joint X-ray/neutron structure of SARS-CoV-2 main protease (3CL Mpro) in complex with BBH-1"},{"1":"7teh","2":"2022","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (3CL Mpro) in complex with BBH-2"},{"1":"7te1","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Receptor Binding Domain in Complex with Ab17"},{"1":"8dlt","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Epsilon (B.1.429) spike protein"},{"1":"8cwv","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with a cross-neutralizing nanobody 2-31 and a human antibody CC12.1 Fab"},{"1":"7tiz","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in complex with inhibitor NK01-63"},{"1":"8asy","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron BA.2.75 RBD in complex with ACE2"},{"1":"7tix","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in complex with inhibitor EB56"},{"1":"7s4b","2":"2022","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease in complex with compound Z1530724963"},{"1":"7t44","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 4c"},{"1":"8dls","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Gamma (P.1) spike protein in complex with Fab 4A8 (focused refinement of NTD and 4A8)"},{"1":"7tiv","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in complex with inhibitor EB48"},{"1":"7tf1","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Kappa (B.1.617.1) Q484I spike protein (focused refinement of RBD)"},{"1":"7t4a","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 11c"},{"1":"8bh5","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 BA.2.12.1 RBD in complex with Beta-27 Fab and C1 nanobody"},{"1":"7mhn","2":"2022","3":"X-ray diffraction","4":"Ensemble refinement structure of SARS-CoV-2 main protease (Mpro) at 277 K"},{"1":"7mmo","2":"2022","3":"X-ray diffraction","4":"LY-CoV1404 neutralizing antibody against SARS-CoV-2"},{"1":"7mb8","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) C145A in Complex with Cleavage Site Nsp8/9 (P6-P1)"},{"1":"7si2","2":"2022","3":"X-ray diffraction","4":"Crystal structure of neutralizing antibody 10-28 in complex with SARS-CoV-2 spike receptor binding domain (RBD)"},{"1":"7s6z","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI33"},{"1":"7o46","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 17"},{"1":"7sn3","2":"2022","3":"Electron Microscopy","4":"Structure of human SARS-CoV-2 spike glycoprotein trimer bound by neutralizing antibody C1C-A3 Fab (variable region)"},{"1":"8dcc","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Receptor-Binding Domain SPEEDesign Immunogen 3 Bound to P2B-2F6 Fab"},{"1":"7ns6","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike (dimers) in complex with six Fu2 nanobodies"},{"1":"7s6j","2":"2022","3":"Electron Microscopy","4":"J08 fragment antigen binding in complex with SARS-CoV-2-6P-Mut2 S protein (conformation 1)"},{"1":"7s6x","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI30"},{"1":"7t42","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 2c"},{"1":"7lme","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CLPro in complex with N-[4-[[2-(benzotriazol-1-yl)acetyl]-(3-thienylmethyl)amino]phenyl]cyclopropanecarboxamide"},{"1":"7mhh","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) at 277 K"},{"1":"7mb4","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) C145A in Complex with Cleavage Site Nsp4/5 (P6-P1)"},{"1":"7mhj","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) at 298 K and High Humidity"},{"1":"7mng","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in Complex with Covalent Inhibitor VBY-825 (Partial Occupancy)"},{"1":"7mhi","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) at 298 K"},{"1":"7lkd","2":"2022","3":"X-ray diffraction","4":"X-ray crystal structure of the SARS-CoV-2 main protease in space group P21."},{"1":"7lbn","2":"2022","3":"X-ray diffraction","4":"X-ray crystal structure of the SARS-CoV-2 main protease with Calpain I Inhibitor"},{"1":"7mlf","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) Covalently Bound to Compound C7"},{"1":"7mc6","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 ExoN-nsp10 complex containing Mg2+ ion"},{"1":"7mhq","2":"2022","3":"X-ray diffraction","4":"Ensemble refinement structure of SARS-CoV-2 main protease (Mpro) at 310 K"},{"1":"7mhm","2":"2022","3":"X-ray diffraction","4":"Ensemble refinement structure of SARS-CoV-2 main protease (Mpro) at 240 K"},{"1":"7mb7","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) C145A in Complex with Cleavage Site Nsp7/8 (P6-P1)"},{"1":"7mb9","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) C145A in Complex with Cleavage Site Nsp10/11 (P6-P1)"},{"1":"7mrr","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in Complex with Covalent Inhibitor Leupeptin"},{"1":"7mhg","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) at 240 K"},{"1":"7lmf","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CLPro in complex with 2-(benzotriazol-1-yl)-N-[4-(1H-imidazol-4-yl)phenyl]-N-(3-thienylmethyl)acetamide"},{"1":"7lmd","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CLPro in complex with 2-(benzotriazol-1-yl)-N-[4-(1H-pyrazol-4-yl)phenyl]-N-(3-thienylmethyl)acetamide"},{"1":"7lyj","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 frameshifting pseudoknot RNA"},{"1":"7n1b","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 RLQ peptide binds to HLA-A2"},{"1":"8erq","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.1 spike ectodomain trimer in complex with the S2X324 neutralizing antibody Fab fragment (local refinement of the RBD and S2X324)"},{"1":"7mw2","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 Spike trimer with all RBDs down in complex with the Fab fragment of human neutralizing antibody clone 6"},{"1":"7nfv","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like protease PLpro"},{"1":"7lke","2":"2022","3":"X-ray diffraction","4":"X-ray crystal structure of the SARS-CoV-2 main protease in space group C2"},{"1":"7mb5","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) C145A in Complex with Cleavage Site Nsp5/6 (P6-P1)"},{"1":"7mhk","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of Apo/Unliganded SARS-CoV-2 Main Protease (Mpro) at 310 K"},{"1":"7w9c","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-ACE2-C3"},{"1":"7web","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike protein with two XGv347 binding to two open state RBDs"},{"1":"7wk9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron open state spike protein in complex with S3H3 Fab"},{"1":"7wcr","2":"2022","3":"Electron Microscopy","4":"RBD-1 of SARS-CoV-2 Beta spike in complex with S5D2 Fab"},{"1":"7r4q","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 spike in complex with the 1.29 neutralizing nanobody"},{"1":"7r14","2":"2022","3":"Electron Microscopy","4":"Alpha Variant SARS-CoV-2 Spike with 1 Erect RBD"},{"1":"7r1t","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nsp10/nsp16 in complex with the SS148 inhibitor"},{"1":"7wlc","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike RBD in complex with Fab XGv282"},{"1":"7w9i","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-RBD-ACE2"},{"1":"7wkj","2":"2022","3":"X-ray diffraction","4":"A COVID-19 T-cell response detection method based on a newly identified human CD8+ T cell epitope from SARS-CoV-2-Hubei Province, 2021."},{"1":"7w9b","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-ACE2-C2b"},{"1":"7r0z","2":"2022","3":"Electron Microscopy","4":"Dissociated S1 domain of Alpha Variant SARS-CoV-2 Spike bound to ACE2 (Non-Uniform Refinement)"},{"1":"7qtj","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S Omicron Spike B.1.1.529 - RBD up - 1-P2G3 and 1-P5C3 Fabs (Local)"},{"1":"7qcg","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like Protease bound to N-(2-pyrrolidyl)-3,4,5-trihydroxybenzoylhydrazone"},{"1":"7w94","2":"2022","3":"Electron Microscopy","4":"Transition state of SARS-CoV-2 Delta variant spike protein"},{"1":"7r1u","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nsp10/nsp16 in complex with the WZ16 inhibitor"},{"1":"7qdg","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S protein S:A222V + S:D614G mutant 1-up"},{"1":"7qch","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like Protease bound to N-(3,5-dimethoxy-4-hydroxybenzyliden)thiosemicarbazone"},{"1":"7r2v","2":"2022","3":"X-ray diffraction","4":"Structure of nsp14 from SARS-CoV-2 in complex with SAH"},{"1":"7r13","2":"2022","3":"Electron Microscopy","4":"Alpha Variant SARS-CoV-2 Spike in Closed conformation"},{"1":"7wc0","2":"2022","3":"X-ray diffraction","4":"Crystal structure of Fab region of TAU-2212 neutralizing SARS-CoV-2"},{"1":"7wed","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike RBD in complex with Fab XGv347"},{"1":"7p78","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with sybody#15 and sybody#68 in a 1up/1up-out/1down conformation"},{"1":"7p7a","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with sybody#68 in a 2up/1flexible conformation"},{"1":"7qct","2":"2022","3":"X-ray diffraction","4":"PDZ2 of LNX2 with SARS-CoV-2_E PBM complex"},{"1":"7wi0","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike in complex with three human neutralizing antibodies"},{"1":"7wd8","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Beta spike SD1 in complex with S3H3 Fab"},{"1":"7r12","2":"2022","3":"Electron Microscopy","4":"Dissociated S1 domain of Mink Variant SARS-CoV-2 Spike bound to ACE2 (Non-Uniform Refinement)"},{"1":"7vu6","2":"2022","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 3CL protease in complex with compound 3"},{"1":"7wbp","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Omicron variant spike glycoprotein in complex with its receptor human ACE2"},{"1":"7w6u","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike receptor-binding domain complexed with its receptor equine ACE2"},{"1":"7qur","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike with ethylbenzamide-tri-iodo Siallyllactose, C3 symmetry"},{"1":"7vvt","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with a covalent inhibitor"},{"1":"7r15","2":"2022","3":"Electron Microscopy","4":"Alpha Variant SARS-CoV-2 Spike with 2 Erect RBDs"},{"1":"7wd0","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Beta spike in complex with two S5D2 Fabs"},{"1":"7w92","2":"2022","3":"Electron Microscopy","4":"Open state of SARS-CoV-2 Delta variant spike protein"},{"1":"7wbq","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Delta variant spike glycoprotein in complex with its receptor human ACE2"},{"1":"7we7","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike protein in complex with Fab XGv282"},{"1":"7we9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike protein in complex with Fab XGv289"},{"1":"8gwb","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 E-RTC complex with RNA-nsp9"},{"1":"7r40","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the 87G7 antibody Fab fragment"},{"1":"7vth","2":"2022","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 3CL protease in complex with compound 1"},{"1":"7wef","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike RBD in complex with Fab XGv289"},{"1":"7wcz","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Beta spike in complex with one S5D2 Fab"},{"1":"7w8s","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike receptor-binding domain Y453F mutation complexed with American mink ACE2"},{"1":"7wgz","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein trimer in open state"},{"1":"7wa1","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike receptor-binding domain F486L mutation complexed with American mink ACE2"},{"1":"7wec","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike protein with three XGv347 Fabs binding to three closed state RBDs"},{"1":"7p5q","2":"2022","3":"Solution NMR","4":"NMR structure of a peptide deriving from SARS-CoV-2 Lineage B.1.1.7 S RBD 482-506 fragment in HFIP/H2O"},{"1":"7wgy","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein trimer in Intermediate state"},{"1":"7wee","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike RBD in complex with Fab XGv265"},{"1":"8ctk","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 M protein in a lipid nanodisc"},{"1":"7wka","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron closed state spike protein in complex with S3H3 Fab"},{"1":"8gwo","2":"2022","3":"Electron Microscopy","4":"A mechanism for SARS-CoV-2 RNA capping and its inhibition by nucleotide analogue inhibitors"},{"1":"7owx","2":"2022","3":"X-ray diffraction","4":"Structure of coiled-coil tetramer from SARS-CoV-2 spike stalk region"},{"1":"7p77","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with sybody#15 and sybody#68 in a 3up conformation"},{"1":"7p5g","2":"2022","3":"Solution NMR","4":"NMR structure of a peptide deriving from SARS-CoV-2 S RBD 482-506 fragment in HFIP/H2O"},{"1":"7sn0","2":"2022","3":"X-ray diffraction","4":"Crystal structure of spike protein receptor binding domain of escape mutant SARS-CoV-2 from immunocompromised patient (d146*) in complex with human receptor ACE2"},{"1":"7s5p","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 B.1.351 variant receptor binding domain in complex with neutralizing antibody CS23"},{"1":"7swo","2":"2022","3":"Electron Microscopy","4":"C98C7 Fab in complex with SARS-CoV-2 Spike 6P (RBD local reconstruction)"},{"1":"7sl5","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of VP12E7 Fab in complex with SARS-CoV-2 S fusion peptide"},{"1":"7n1e","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 RLQ peptide-specific TCR pRLQ3 binds to RLQ-HLA-A2"},{"1":"7mhf","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) at 100 K"},{"1":"7te0","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor PF-07321332"},{"1":"7mlg","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) Covalently Bound to Compound C63"},{"1":"7mhl","2":"2022","3":"X-ray diffraction","4":"Ensemble refinement structure of SARS-CoV-2 main protease (Mpro) at 100 K"},{"1":"7mw5","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 Spike trimer with one RBD down in complex with the Fab fragment of human neutralizing antibody clone 2"},{"1":"7mb6","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) C145A in Complex with Cleavage Site Nsp6/7 (P6-P1)"},{"1":"7neo","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 15"},{"1":"7mhp","2":"2022","3":"X-ray diffraction","4":"Ensemble refinement structure of SARS-CoV-2 main protease (Mpro) at 298 K at high humidity"},{"1":"7n1c","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 RLQ peptide-specific TCR pRLQ3"},{"1":"7qcj","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like Protease bound to N-(2,4-dihydroxybenzylidene)-thiosemicarbazone"},{"1":"8d4k","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) H172Y Mutant in Complex with Inhibitor GC376"},{"1":"7qnx","2":"2022","3":"X-ray diffraction","4":"The receptor binding domain of SARS-CoV-2 spike glycoprotein in complex with Beta-55 and EY6A Fabs"},{"1":"7qcr","2":"2022","3":"X-ray diffraction","4":"MLLT4/Afadin PDZ domain in complex with the C-terminal peptide from protein E of SARS-CoV-2"},{"1":"8dlk","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Alpha (B.1.1.7) spike protein in complex with human ACE2 (focused refinement of RBD and ACE2)"},{"1":"8dlw","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Epsilon (B.1.429) spike protein in complex with Fab S2M11"},{"1":"8dru","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp7-nsp8 (C7) cut site sequence"},{"1":"8e4j","2":"2022","3":"X-ray diffraction","4":"Room-temperature X-ray structure of SARS-CoV-2 main protease H41A miniprecursor mutant"},{"1":"7r1b","2":"2022","3":"Electron Microscopy","4":"Mink Variant SARS-CoV-2 Spike with 1 Erect RBD"},{"1":"8d6z","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike stem fusion peptide in complex with neutralizing antibody COV91-27"},{"1":"8d4l","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) S144A Mutant"},{"1":"7r19","2":"2022","3":"Electron Microscopy","4":"Mink Variant SARS-CoV-2 Spike with 2 Erect RBDs"},{"1":"8dw2","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 RBD in complex with anti-SARS-CoV-2 DARPin,SR22, and two antibody Fabs, S309 and CR3022"},{"1":"8dlu","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Epsilon (B.1.429) spike protein in complex with human ACE2"},{"1":"8dao","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike stem fusion peptide in complex with neutralizing antibody COV44-79"},{"1":"8dw3","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 RBD in complex with anti-SARS-CoV-2 DARPin,SR16m, and two antibody Fabs, S309 and CR3022"},{"1":"8df5","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Beta RBD in complex with human ACE2 and S304 Fab and S309 Fab"},{"1":"8di5","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Beta (B.1.351) spike protein in complex with VH domain F6 (focused refinement of RBD and VH F6)"},{"1":"8dya","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein S2 subunit"},{"1":"8dm0","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 D614G spike protein in complex with VH ab6 (focused refinement of NTD and VH ab6)"},{"1":"7qcs","2":"2022","3":"X-ray diffraction","4":"PALS1/MPP5 PDZ domain in complex with SARS-CoV-2_E PBM peptide"},{"1":"8dly","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Epsilon (B.1.429) spike protein in complex with VH ab6 (focused refinement of NTD and VH ab6)"},{"1":"8d4n","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) E166Q Mutant"},{"1":"7qg7","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 macrodomain Nsp3b bound to the remdesivir nucleoside GS-441524"},{"1":"8ds0","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp14-nsp15 (C14) cut site sequence (form 2)"},{"1":"7oft","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like protease PLpro in complex with p-hydroxybenzaldehyde"},{"1":"7o35","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 N-CTD in complex with GTP (I)"},{"1":"7n1f","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 YLQ peptide-specific TCR pYLQ7 binds to YLQ-HLA-A2"},{"1":"7s74","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI38"},{"1":"7s3k","2":"2022","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease in complex with compound Z1530718726"},{"1":"7sjs","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike stem helix peptide in complex with neutralizing antibody CC40.8"},{"1":"7qt5","2":"2022","3":"X-ray diffraction","4":"Room temperature In-situ SARS-CoV-2 MPRO with bound Z31792168"},{"1":"7s6i","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2-6P-Mut2 S protein"},{"1":"7t3m","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S (Spike Glycoprotein) D614G with Three (3) RBDs Up, Bound to Antibody 2-7 scFv, composite map"},{"1":"7tf5","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Kappa (B.1.617.1) spike protein"},{"1":"7tbt","2":"2022","3":"X-ray diffraction","4":"Co-crystal structure of SARS-CoV-2 Mpro C145A with substrate peptide 13/14"},{"1":"7t4b","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 14c"},{"1":"8d8r","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike RBD in complex with DMAb 2196"},{"1":"7thk","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of prefusion SARS-CoV-2 spike omicron B.1.1.529 variant"},{"1":"8dtr","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike stem helix peptide in complex with neutralizing antibody COV30-14"},{"1":"8az8","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 non-structural protein-1 (nsp1) in complex with 2-(benzylamino)ethan-1-ol"},{"1":"7tf8","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron 3-RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-Omicron)"},{"1":"7t9k","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike protein in complex with human ACE2"},{"1":"7r10","2":"2022","3":"Electron Microscopy","4":"Dissociated S1 domain of Alpha Variant SARS-CoV-2 Spike bound to ACE2"},{"1":"7qt8","2":"2022","3":"X-ray diffraction","4":"Room temperature In-situ SARS-CoV-2 MPRO with bound ABT-957"},{"1":"7r16","2":"2022","3":"Electron Microscopy","4":"Beta Variant SARS-CoV-2 Spike with 1 Erect RBD"},{"1":"8drx","2":"2022","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with nsp10-nsp11 (C10) cut site sequence (form 2)"},{"1":"7qcm","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like Protease bound to N-(3-methoxy-4-hydroxy-acetophenone)thiosemicarbazone"},{"1":"7qt9","2":"2022","3":"X-ray diffraction","4":"Room temperature In-situ SARS-CoV-2 MPRO with bound Z4439011584"},{"1":"7qf0","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with the human antibody CV2.2325"},{"1":"7qbb","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 18"},{"1":"7nbt","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 21"},{"1":"7ofu","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like protease PLpro in complex with 3, 4-Dihydroxybenzoic acid, methyl ester"},{"1":"8ds1","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Mpro in complex with nsp12-nsp13 (C12) cut site sequence"},{"1":"8dlp","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Gamma (P.1) spike protein in complex with human ACE2"},{"1":"7ofs","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Papain-like protease PLpro in complex with 4-(2-hydroxyethyl)phenol"},{"1":"7tiu","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in complex with inhibitor EB46"},{"1":"7tew","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta (B.1.617.2) spike protein in complex with human ACE2 (focused refinement of RBD and ACE2)"},{"1":"7t67","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S (Spike Glycoprotein) D614G with One(1) RBD Up"},{"1":"7mw4","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 Spike trimer with one RBD down in complex with the Fab fragment of human neutralizing antibody clone 6"},{"1":"7nab","2":"2022","3":"X-ray diffraction","4":"Crystal structure of human neutralizing mAb CV3-25 binding to SARS-CoV-2 S MPER peptide 1140-1165"},{"1":"7nll","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike RBD (dimer) in complex with two Fu2 nanobodies"},{"1":"7n5h","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of broadly neutralizing antibody 2-36 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7n1d","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 YLQ peptide-specific TCR pYLQ7"},{"1":"7mho","2":"2022","3":"X-ray diffraction","4":"Ensemble refinement structure of SARS-CoV-2 main protease (Mpro) at 298 K"},{"1":"7b5z","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 6"},{"1":"7fce","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 A372T spike glycoprotein (closed)"},{"1":"7fcp","2":"2022","3":"X-ray diffraction","4":"Crystallographic structure of two neutralizing antibodies in complex with SARS-CoV-2 spike receptor-binding Domain (RBD)"},{"1":"7f7e","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 S protein RBD in complex with A5-10 Fab"},{"1":"7epx","2":"2022","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex with GW01"},{"1":"7enf","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with Fab30"},{"1":"7fjs","2":"2022","3":"X-ray diffraction","4":"Crystal structure of T6 Fab bound to theSARS-CoV-2 RBD of B.1.351"},{"1":"7kn3","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain complexed with a pre-pandemic antibody S-B8 Fab"},{"1":"7kn4","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain complexed with a pre-pandemic antibody S-E6 Fab"},{"1":"7fbk","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain N501Y mutant in complex with neutralizing nanobody 20G6"},{"1":"7dhj","2":"2022","3":"X-ray diffraction","4":"The co-crystal structure of SARS-CoV-2 main protease with the peptidomimetic inhibitor (S)-2-cinnamamido-N-((S)-1-oxo-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)pent-4-ynamide"},{"1":"7dvp","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro mutant (H41A) in complex with nsp4|5 peptidyl substrate"},{"1":"7d64","2":"2022","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 3CLpro with Zinc"},{"1":"7faz","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with Y180"},{"1":"7ed5","2":"2022","3":"Electron Microscopy","4":"A dual mechanism of action of AT-527 against SARS-CoV-2 polymerase"},{"1":"7fjo","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of South African (B.1.351) SARS-CoV-2 spike glycoprotein in complex with three T6 Fab"},{"1":"7fbj","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing nanobody 17F6"},{"1":"7eng","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with Fab30 (local refinement of the RBD and Fab30)"},{"1":"7ddc","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with Tafenoquine"},{"1":"7dw6","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro mutant (H41A) in complex with nsp15|16 peptidyl substrate"},{"1":"7dgi","2":"2022","3":"X-ray diffraction","4":"The co-crystal structure of SARS-CoV-2 main protease with peptidomimetic inhibitor N-((S)-3-methyl-1-(((S)-4-methyl-1-oxo-1-(((S)-1-oxo-3-((S)-2-oxopiperidin-3-yl)propan-2-yl)amino)pentan-2-yl)amino)-1-oxobutan-2-yl)-4-nitrobenzamide"},{"1":"7dgh","2":"2022","3":"X-ray diffraction","4":"The co-crystal structure of SARS-CoV-2 main protease with peptidomimetic inhibitor N-((S)-3-methyl-1-(((S)-4-methyl-1-oxo-1-(((S)-1-oxo-3-((S)-2-oxopiperidin-3-yl)propan-2-yl)amino)pentan-2-yl)amino)-1-oxobutan-2-yl)-2-naphthamide"},{"1":"7dvx","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro mutant (H41A) in complex with nsp6|7 peptidyl substrate"},{"1":"7dgf","2":"2022","3":"X-ray diffraction","4":"The co-crystal structure of SARS-CoV-2 main protease with peptidomimetic inhibitor (S)-2-cinnamamido-N-((S)-1-oxo-3-((S)-2-oxopiperidin-3-yl)propan-2-yl)hexanamide"},{"1":"7f2e","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 nucleocapsid protein C-terminal domain (dodecamer)"},{"1":"7fcd","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 A372T spike glycoprotein (open)"},{"1":"7fjn","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of South African (B.1.351) SARS-CoV-2 spike glycoprotein in complex with two T6 Fab"},{"1":"7fcq","2":"2022","3":"X-ray diffraction","4":"Crystallographic structure of neutralizing antibody P14-44 in complex with SARS-CoV-2 spike receptor-binding Domain (RBD)"},{"1":"7fay","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with (R)-1a"},{"1":"7e9o","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with 35B5 Fab(3 up RBDs, state2)"},{"1":"7fh0","2":"2022","3":"X-ray diffraction","4":"Crystallographic structure of two neutralizing nanobodies in complex with SARS-CoV-2 spike receptor-binding Domain (RBD)"},{"1":"7dw0","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro mutant (H41A) in complex with nsp14|15 peptidyl substrate"},{"1":"7f46","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with 35B5 Fab (state1, local refinement of the RBD, NTD and 35B5 Fab)"},{"1":"7f2b","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nucleocapsid protein C-terminal RNA binding domain at 2.0A resolution"},{"1":"7e9n","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with 35B5 Fab(1 down RBD, state1)"},{"1":"7dgg","2":"2022","3":"X-ray diffraction","4":"The co-crystal structure of SARS-CoV-2 main protease with (S)-2-cinnamamido-N-((S)-1-oxo-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)hexanamide"},{"1":"7dvy","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro mutant (H41A) in complex with nsp9|10 peptidyl substrate"},{"1":"7jyc","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (3CLpro/Mpro) in Complex with Covalent Inhibitor Narlaprevir"},{"1":"7djz","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with MW01 Fab"},{"1":"7dvw","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro mutant (H41A) in complex with nsp5|6 peptidyl substrate"},{"1":"7fc5","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD and horse ACE2"},{"1":"7e9q","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with 35B5 Fab(1 out RBD, state3)"},{"1":"7vpy","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the neutralizing nanobody P86 against SARS-CoV-2"},{"1":"7xwx","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 N-CTD"},{"1":"7xo5","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 Variant Spike Trimer with one mouse ACE2 Bound"},{"1":"7wz1","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron Spike trimer"},{"1":"7xu5","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 D614G Spike Protein with Engineered x3 Disulfide (x3(D427C, V987C) and single Arg S1/S2 cleavage site), Closed Conformation"},{"1":"8a4q","2":"2022","3":"X-ray diffraction","4":"crystal structures of diastereomer (R,S,S)-13b (13b-H) in complex with the SARS-CoV-2 Mpro."},{"1":"7wuh","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with Fab of m31A7"},{"1":"7wpe","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron Variant S Trimer complexed with two JMB2002 Fab"},{"1":"7wp5","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron S6P trimer in complex with neutralizing antibody VacW-209 (local refinement)"},{"1":"7upw","2":"2022","3":"Electron Microscopy","4":"Three RBD-down state of SARS-CoV-2 D614G spike in complex with the SP1-77 neutralizing antibody Fab fragment"},{"1":"7x92","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab445"},{"1":"7tz0","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with FSR22, an anti-SARS-CoV-2 DARPin (Local refinement of FSR22 and RBD)"},{"1":"7twi","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at 293 K (P43 crystal form, 539 kGy)"},{"1":"7rzt","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 HR1HR2 fusion core complex with S940F mutation"},{"1":"7uzb","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S S1 doamin in complex with the mouse antibody Fab fragment, HSW-2"},{"1":"7uhb","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with AHB2-2GS-SB175 (local refinement of the RBD and AHB2)"},{"1":"7wse","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike receptor-binding domain complexed with its receptor minke whale ACE2"},{"1":"7wp0","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta S6P trimer in complex with neutralizing antibody VacW-209 (local refinement)"},{"1":"7voa","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with aRBD5"},{"1":"7uw3","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Nucleocapsid Protein N-Terminal Domain"},{"1":"7uvj","2":"2022","3":"X-ray diffraction","4":"Rationally Designed ED1 Epitope-Scaffold Immunogen for SARS-CoV-2"},{"1":"7tov","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 1-RBD-up conformation; consensus state D2"},{"1":"7tl9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron 1-RBD up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-Omicron)"},{"1":"7uar","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the neutralizing antibody Fab fragment, C1717"},{"1":"7wo2","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CLPro Peptidomimetic Inhibitor TPM5"},{"1":"7upx","2":"2022","3":"Electron Microscopy","4":"Three RBD-down state of SARS-CoV-2 D614G spike in complex with the SP1-77 neutralizing antibody Fab fragment (local refinement of the RBD and Fab variable domains)"},{"1":"7wvo","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron S-open-2"},{"1":"7wp8","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 recombinant spike protein STFK1628x in complex with three neutralizing antibodies"},{"1":"7u9o","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike trimer RBD in complex with Fab NE12"},{"1":"7wpd","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron Variant S Trimer complexed with one JMB2002 Fab"},{"1":"7vgs","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 M protein dimer (short form) in complex with YN7717_9 Fab"},{"1":"7uxx","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nucleocapsid protein C-terminal domain"},{"1":"7wue","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Receptor Binding Domain in complex with the monoclonal antibody m31A7"},{"1":"7wrv","2":"2022","3":"Electron Microscopy","4":"The interface of JMB2002 Fab binds to SARS-CoV-2 Omicron Variant S"},{"1":"7wo1","2":"2022","3":"X-ray diffraction","4":"Discovery of SARS-CoV-2 3CLpro peptidomimetic inhibitors through H41-specific protein-ligand interactions"},{"1":"7twh","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at 293 K (P43 crystal form, 290 kGy)"},{"1":"7v26","2":"2022","3":"Electron Microscopy","4":"XG005-bound SARS-CoV-2 S"},{"1":"7wtf","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike in complex with Fab XGv051"},{"1":"7wo5","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with IgG 553-15 (S-553-15 trimer)"},{"1":"7ujg","2":"2022","3":"X-ray diffraction","4":"Room-temperature X-ray structure of monomeric SARS-CoV-2 main protease catalytic domain (MPro1-196) in complex with GC-376"},{"1":"7uzc","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 RBD in complex with the mouse antibody Fab fragment, M8a-34"},{"1":"7ub0","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron-BA.2 3-RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-Omicron-BA.2)"},{"1":"7xu0","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein with Engineered x3 Disulfide (x3(D427C, V987C) and single Arg S1/S2 cleavage site), Locked-211 Conformation"},{"1":"7xiz","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.3 variant spike (local)"},{"1":"7wti","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike in complex with Fab XGv264"},{"1":"7xa7","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor-binding domain in complex with intermediate horseshoe bat ACE2"},{"1":"7wpa","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron Variant SPIKE trimer complexed with ACE2"},{"1":"7xch","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike protein (S-6P-RRAR) in complex with human ACE2 ectodomain (two-RBD-up state)"},{"1":"7zr2","2":"2022","3":"X-ray diffraction","4":"Crystal structure of a chimeric protein mimic of SARS-CoV-2 Spike HR1 in complex with HR2"},{"1":"7x7u","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta variant spike protein in complex with three nAbs X01, X10 and X17"},{"1":"7wp1","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Mu S6P trimer in complex with neutralizing antibody VacW-209 (local refinement)"},{"1":"7xu1","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein with Engineered x3 Disulfide (x3(D427C, V987C) and single Arg S1/S2 cleavage site), Locked-122 Conformation"},{"1":"7xwz","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 N-NTD and dsRNA complex"},{"1":"7xrp","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike protein in complex with nanobody C5G2 (localized refinement)"},{"1":"7xoa","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 Variant Spike Trimer with one mouse ACE2 Bound"},{"1":"8a4y","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 non-structural protein-1 (nsp1) in complex with N-(2,3-dihydro-1H-inden-5-yl)acetamide"},{"1":"7xmz","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike glycoprotein in complex with three D2 Fab"},{"1":"7wnm","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Gamma variant receptor-binding domain complexed with high affinity human ACE2 mutant (T27F,R273Q)"},{"1":"7xu3","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein with Engineered x3 Disulfide (x3(D427C, V987C) and single Arg S1/S2 cleavage site), Closed Conformation"},{"1":"7x7t","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike protein in complex with three nAbs X01, X10 and X17"},{"1":"7xo8","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 Variant Spike Trimer with three human ACE2 Bound"},{"1":"7x6j","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 3af"},{"1":"7wpb","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron Variant RBD complexed with ACE2"},{"1":"7wo7","2":"2022","3":"Electron Microscopy","4":"Locally refined region of SARS-CoV-2 Spike in complex with IgG 553-15"},{"1":"7vgr","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 M protein dimer (long form) in complex with YN7756_1 Fab"},{"1":"7x6a","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2 variant spike protein in complex with Fab BD55-5840"},{"1":"7wt8","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike in complex with Fab 9A8 (State 2)"},{"1":"7xtz","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein with Engineered x3 Disulfide (x3(D427C, V987C) and single Arg S1/S2 cleavage site), Locked-1 Conformation"},{"1":"7x9e","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the 76E1 Fab in complex with a SARS-CoV-2 spike peptide"},{"1":"7x90","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab326"},{"1":"7x6k","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 3w"},{"1":"8acl","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (MPro) in complex with the non-covalent inhibitor GC-14"},{"1":"7xst","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike glycoprotein in complex with three F61 Fab and three D2 Fab"},{"1":"7xob","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 Variant Spike Trimer with two mouse ACE2 Bound"},{"1":"7xu2","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein with Engineered x3 Disulfide (x3(D427C, V987C) and single Arg S1/S2 cleavage site), Locked-2 Conformation"},{"1":"7xod","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 Variant Spike Trimer with three JMB2002 Fab Bound"},{"1":"7xiw","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 variant spike (state 1)"},{"1":"7xo4","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 Variant Spike Trimer with two mouse ACE2 Bound"},{"1":"7xnq","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.4 variant spike"},{"1":"7uap","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the neutralizing antibody Fab fragment, C1520"},{"1":"7tp7","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 1-RBD-up conformation; Subclassification D11 state"},{"1":"7tyz","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with FSR22, an anti-SARS-CoV-2 DARPin"},{"1":"7toy","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 3-RBD-down conformation; Subclassification D6 state"},{"1":"7rvw","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI23"},{"1":"7tow","2":"2022","3":"X-ray diffraction","4":"Antibody DH1058 Fab fragment bound to SARS-CoV-2 fusion peptide"},{"1":"7tl1","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron 3-RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-Omicron)"},{"1":"7rvm","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI11"},{"1":"7wp6","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 recombinant spike protein STFK in complex with three neutralizing antibodies"},{"1":"7ru8","2":"2022","3":"Electron Microscopy","4":"CC6.30 fragment antigen binding in complex with SARS-CoV-2-6P-Mut7 S protein (RBD/Fv local refinement)"},{"1":"7wo4","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with IgG 553-15 (S-553-15 dimer trimer )"},{"1":"7to4","2":"2022","3":"Electron Microscopy","4":"Structural and functional impact by SARS-CoV-2 Omicron spike mutations"},{"1":"7rvv","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI22"},{"1":"7ru3","2":"2022","3":"Electron Microscopy","4":"CC6.33 IgG in complex with SARS-CoV-2-6P-Mut7 S protein (non-uniform refinement)"},{"1":"7uz5","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the mouse antibody Fab fragment, M8a-6"},{"1":"7uz9","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the mouse antibody Fab fragment, M8a-34"},{"1":"7rzr","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 HR1HR2 fusion core complex with D936Y mutation"},{"1":"7ru4","2":"2022","3":"Electron Microscopy","4":"CC6.33 IgG in complex with SARS-CoV-2-6P-Mut7 S protein (RBD/Fv local refinement)"},{"1":"7wob","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with IgG 553-60 (2-up trimer)"},{"1":"7tou","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 3-RBD-down conformation; consensus state D1"},{"1":"7rvu","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI21"},{"1":"7x8w","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab354"},{"1":"7xc3","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 Macrodomain 3 (SARS-unique domain-M)"},{"1":"7wvq","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron Spike protein with human ACE2 receptor, C3 state"},{"1":"7woa","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with IgG 553-60 (1-up trimer)"},{"1":"7wri","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike receptor-binding domain in complex with mouse ACE2"},{"1":"7wog","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron S monomer complexed with 553-49"},{"1":"7wtg","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike RBD in complex with Fab XGv051"},{"1":"7wr8","2":"2022","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6P(B.1.1.529) in complex with BD55-3152 Fab"},{"1":"7woh","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CLpro"},{"1":"7v2a","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike trimer in complex with XG014 Fab"},{"1":"7xmx","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike glycoprotein in complex with three F61 Fab"},{"1":"7x2l","2":"2022","3":"X-ray diffraction","4":"Crystal structure of nanobody 3-2A2-4 with SARS-CoV-2 RBD"},{"1":"7xu4","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 D614G Spike Protein with Engineered x3 Disulfide (x3(D427C, V987C) and single Arg S1/S2 cleavage site), Locked-2 Conformation"},{"1":"7xo9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 Variant RBD complexed with human ACE2"},{"1":"7x7d","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Delta RBD and Nb22"},{"1":"7xco","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike protein (S-6P-RRAR) in complex with S309 fab"},{"1":"7xwa","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Omicron BA.4/5 variant spike protein in complex with its receptor ACE2"},{"1":"7zqv","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with AG7404"},{"1":"7xoc","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 Variant RBD complexed with mouse ACE2"},{"1":"7xci","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron RBD in complex with human ACE2 ectodomain (local refinement)"},{"1":"7wpf","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron Variant S Trimer complexed with three JMB2002 Fab"},{"1":"7xh8","2":"2022","3":"Electron Microscopy","4":"The structure of ZCB11 Fab against SARS-CoV-2 Omicron Spike"},{"1":"7xc4","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 Macrodomain 3 (SARS-unique domain-M) in complex with Oxaprozin"},{"1":"7xns","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.4 variant spike"},{"1":"7xo6","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.1 Variant RBD with mouse ACE2 Bound"},{"1":"7y0c","2":"2022","3":"X-ray diffraction","4":"Crystal structure of BD55-1403 and SARS-CoV-2 Omicron RBD"},{"1":"8a55","2":"2022","3":"X-ray diffraction","4":"Structure of N-terminal SARS-CoV-2 nonstructural protein 1 (nsp1) at atomic resolution"},{"1":"7zxu","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron BA.4/5 RBD in complex with Beta-27 Fab and C1 nanobody"},{"1":"7x91","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with an Fv-clasp form of a human neutralizing antibody Ab496"},{"1":"7xo7","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 Variant Spike Trimer with two human ACE2 Bound"},{"1":"7x7o","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 spike RBD in complex with neutralizing antibody UT28K"},{"1":"8acd","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (MPro) in complex with the non-covalent inhibitor GA-17S"},{"1":"7xsw","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 antibody S309 with GX/P2V/2017 RBD"},{"1":"7xiy","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.3 variant spike"},{"1":"7xby","2":"2022","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 Omicron BA.1 variant RBD in complex with equine ACE2"},{"1":"7vic","2":"2022","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 3C-like protease in complex with a traditional Chinese Medicine Inhibitors"},{"1":"7wz2","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 (D614G) Spike trimer"},{"1":"7wvl","2":"2022","3":"X-ray diffraction","4":"Structure of P4A2 Fab in complex with Spike-RBD from SARS-CoV-2"},{"1":"7wt9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike RBD in complex with Fab 9A8"},{"1":"7tws","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at pH 10 (P43 crystal form)"},{"1":"7wo3","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CLpro"},{"1":"7rzu","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 HR1HR2 fusion core complex with A942S mutation"},{"1":"7x08","2":"2022","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex with 2G1"},{"1":"7wof","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 3CLpro"},{"1":"7tpl","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the M1 conformation, D4"},{"1":"7tn0","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron RBD in complex with human ACE2 and S304 Fab and S309 Fab"},{"1":"7xck","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron RBD in complex with S309 fab (local refinement)"},{"1":"7uxz","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nucleocapsid protein C-terminal domain complexed with Chicoric acid"},{"1":"7won","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S2P trimer in complex with neutralizing antibody VacW-209 (local refinement)"},{"1":"7u2e","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody ADI-55688"},{"1":"7wxz","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the recombinant protein HR121 from the S2 protein of SARS-CoV-2"},{"1":"7wvp","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron Spike protein with human ACE2 receptor, C2 state"},{"1":"7wsh","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike receptor-binding domain in complex with sea lion ACE2"},{"1":"7wth","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike RBD in complex with Fab XGv264"},{"1":"7uaq","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 NTD in complex with C1520, local refinement"},{"1":"7ub6","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron-BA.2 3-RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-Omicron-BA.2)"},{"1":"7tpf","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 1-RBD-up conformation; Subclassification D17 state"},{"1":"7rdz","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(2)-RTC - apo class"},{"1":"7rvp","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI14"},{"1":"7ps4","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-38"},{"1":"7twn","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at pH 5 (P43 crystal form)"},{"1":"7rzq","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 HR1HR2 fusion core complex"},{"1":"7ru5","2":"2022","3":"Electron Microscopy","4":"CC6.30 fragment antigen binding in complex with SARS-CoV-2-6P-Mut7 S protein (non-uniform refinement)"},{"1":"7tm0","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S B.1.1.529 Omicron variant + S309 + S2L20 Global Refinement"},{"1":"7wtk","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike in complex with Fab XGv286"},{"1":"7vh8","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with protease inhibitor PF-07321332"},{"1":"7ub5","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron-BA.2 3-RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-Omicron-BA.2)"},{"1":"7twj","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at pH 4 (P43 crystal form)"},{"1":"7ps1","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-27 Fab"},{"1":"7tuq","2":"2022","3":"X-ray diffraction","4":"Crystal structure of BRD4 bromodomain 1 in complex with monoacetylated SARS-CoV-2 E"},{"1":"7tp9","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 1-RBD-up conformation; Subclassification D13 state"},{"1":"7re2","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(1)-RTC"},{"1":"7zfb","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron RBD in complex with nanobody C1, Omi-18 and Omi-31 Fabs"},{"1":"7q9m","2":"2022","3":"Electron Microscopy","4":"Beta-53 fab in complex with SARS-CoV-2 beta-Spike glycoprotein"},{"1":"7zf5","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron RBD in complex with Omi-12 and Beta-54 Fabs"},{"1":"7s0d","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with neutralizing antibody N-612-014"},{"1":"7re3","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(2)-RTC dimer"},{"1":"7zfd","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron RBD in complex with Omi-25 Fab"},{"1":"7pr0","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with FD-5D Fab"},{"1":"7u29","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Mpro mutant (K90R) in complex with Nirmatrelvir (PF-07321332)"},{"1":"7u0a","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of C77G12 Fab in complex with SARS-CoV-2 S fusion peptide"},{"1":"7raq","2":"2022","3":"X-ray diffraction","4":"Crystal structure of CV3-25 Fab bound to SARS-CoV-2 spike stem helix peptide"},{"1":"7q9f","2":"2022","3":"Electron Microscopy","4":"Beta-50 fab in complex with SARS-CoV-2 beta-Spike glycoprotein"},{"1":"7zfe","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron RBD in complex with Omi-32 Fab and nanobody C1"},{"1":"7u09","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of C13B8 Fab in complex with SARS-CoV-2 S fusion peptide"},{"1":"7ye8","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 refolded dimeric ORF9b"},{"1":"7q9k","2":"2022","3":"Electron Microscopy","4":"Beta-32 fab in complex with SARS-CoV-2 beta-Spike glycoprotein"},{"1":"7ps7","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-40 Fab"},{"1":"7re1","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(2)-RTC (composite)"},{"1":"7zf4","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron RBD in complex with Omi-9 Fab and nanobody F2"},{"1":"7tll","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Mpro Omicron P132H in complex with Nirmatrelvir (PF-07321332)"},{"1":"7zce","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with the single chain fragment scFv76"},{"1":"7q5f","2":"2022","3":"X-ray diffraction","4":"Crystal structure of F2F-2020216-01X bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"7toz","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 3-RBD-down conformation; Subclassification D7 state"},{"1":"7rw0","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI27"},{"1":"7zf7","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron BA.2 RBD in complex with ACE2"},{"1":"7zf3","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron RBD in complex with Omi-3 and EY6A Fabs"},{"1":"7two","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at pH 6 (P43 crystal form)"},{"1":"7tnw","2":"2022","3":"Electron Microscopy","4":"Structural and functional impact by SARS-CoV-2 Omicron spike mutations"},{"1":"7rvx","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI24"},{"1":"7q9j","2":"2022","3":"Electron Microscopy","4":"Beta-26 fab in complex with SARS-CoV-2 beta-Spike glycoprotein"},{"1":"7yr0","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer in complex with S309 (interface)"},{"1":"7yr1","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer in complex with XG2v024"},{"1":"7ytn","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Alpha RBD in complex with the D27LEY neutralizing antibody Fab fragment"},{"1":"7tp1","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 3-RBD-down conformation; Subclassification D9 state"},{"1":"7ye7","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 soluble dimeric ORF9b"},{"1":"7q9i","2":"2022","3":"Electron Microscopy","4":"Beta-43 fab in complex with SARS-CoV-2 beta-Spike glycoprotein"},{"1":"7q0i","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the N-terminal domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-43"},{"1":"7re0","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(2)-RTC - swiveled class"},{"1":"7zfa","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron RBD in complex with Omi-6 and COVOX-150 Fabs"},{"1":"7q0a","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike ectodomain with Fab FI3A"},{"1":"7tld","2":"2022","3":"Electron Microscopy","4":"Down-state locked, S2 stabilized rS2d-HexaPro SARS-CoV-2 spike ectodomain in the RBD-down conformation, State 2"},{"1":"7rnw","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease in complex with a cyclic peptide inhibitor"},{"1":"7tph","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 2-RBD-up conformation - D3"},{"1":"7tla","2":"2022","3":"Electron Microscopy","4":"Down-state locked rS2d SARS-CoV-2 spike ectodomain in the RBD-down conformation, State 1"},{"1":"7s0b","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 RBD in complex with neutralizing antibody N-612-056"},{"1":"7rzs","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 HR1HR2 fusion core complex with L938F mutation"},{"1":"7rvn","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI12"},{"1":"7zcf","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike RBD in complex with the single chain fragment scFv76 (Focused Refinement)"},{"1":"7zf9","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron BA.2 RBD in complex with COVOX-150 Fab (P21)"},{"1":"7yqy","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer in complex with S309 (state2)"},{"1":"7yeg","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike (6P) in complex with 3 R1-32 Fabs and 3 ACE2"},{"1":"7s0c","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with neutralizing antibody N-612-017"},{"1":"7tqv","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 endoribonuclease Nsp15 bound to dsRNA"},{"1":"7y9s","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of apo SARS-CoV-2 Omicron spike protein (S-2P-GSAS)"},{"1":"7z0x","2":"2022","3":"X-ray diffraction","4":"THSC20.HVTR26 Fab bound to SARS-CoV-2 Receptor Binding Domain"},{"1":"7zfc","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Beta RBD in complex with nanobody C1, Omi-18 and Omi-31 Fabs"},{"1":"7rq6","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with non-neutralizing NTD-directed CV3-13 Fab isolated from convalescent individual"},{"1":"7ydi","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike (6P) in complex with 3 R1-32 Fabs and 3 ACE2, focused refinement of RBD region"},{"1":"7yqx","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer in complex with S309 (state1)"},{"1":"7tox","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 3-RBD-down conformation; Subclassification D5 state"},{"1":"7yr2","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer in complex with ACE2(state1)"},{"1":"7ya0","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of hACE2-bound SARS-CoV-2 Omicron spike protein with L371S, P373S and F375S mutations (S-6P-RRAR)"},{"1":"7yqz","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer in complex with S309 (state3)"},{"1":"7zf8","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Omicron BA.2 RBD in complex with COVOX-150 Fab"},{"1":"7q1z","2":"2022","3":"Electron Microscopy","4":"Structure of formaldehyde cross-linked SARS-CoV-2 S glycoprotein"},{"1":"7rvq","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI16"},{"1":"7z0y","2":"2022","3":"X-ray diffraction","4":"THSC20.HVTR04 Fab bound to SARS-CoV-2 Receptor Binding Domain"},{"1":"7y9n","2":"2022","3":"X-ray diffraction","4":"an engineered 5-helix bundle derived from SARS-CoV-2 S2 in complex with HR2P"},{"1":"7ye9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike (6P) in complex with 3 R1-32 Fabs"},{"1":"7tlb","2":"2022","3":"Electron Microscopy","4":"Down-state locked rS2d SARS-CoV-2 spike ectodomain in the RBD-down conformation, State 2"},{"1":"7rvs","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI19"},{"1":"7tob","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 Omicron main protease (Mpro) in complex with inhibitor GC376"},{"1":"7tp4","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody K398.22"},{"1":"7tx3","2":"2022","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Neutron crystal structure of SARS-CoV-2 NSP3 macrodomain at 293 K (P43 crystal form)"},{"1":"7pry","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with COVOX-45 and beta-6 Fabs"},{"1":"7q0g","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-49 and FI-3A Fabs"},{"1":"7u9p","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike trimer RBD in complex with Fab NA8"},{"1":"7twr","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at pH 8 (P43 crystal form)"},{"1":"7pqy","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with FI-3A Fab"},{"1":"7q5e","2":"2022","3":"X-ray diffraction","4":"Crystal structure of F2F-2020209-00X bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"7ps2","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-29 and Beta-53 Fabs"},{"1":"7u2d","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody ADG20"},{"1":"7tx5","2":"2022","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Neutron crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose at 293 K (C2 crystal form)"},{"1":"7s0e","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S1 subunit in complex with antibody N-612-004"},{"1":"7tlz","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S NTD B.1.1.529 Omicron variant + S309 Local Refinement"},{"1":"7twq","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at pH 9 (P43 crystal form)"},{"1":"7rw1","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI28"},{"1":"7rbu","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with PVI.V6-14 Fab"},{"1":"7rbv","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with PVI.V6-14 Fab"},{"1":"7twp","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at pH 7 (P43 crystal form)"},{"1":"7tuu","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with halicin"},{"1":"7rzv","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 HR1HR2 fusion core complex with V1176F mutation"},{"1":"7ws4","2":"2022","3":"Electron Microscopy","4":"Ultrapotent SARS-CoV-2 neutralizing antibodies with protective efficacy against newly emerged mutational variants"},{"1":"7u0e","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of C13C9 Fab in complex with SARS-CoV-2 S fusion peptide"},{"1":"7prz","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with beta-22 Fabs"},{"1":"7tp8","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 1-RBD-up conformation; Subclassification D12 state"},{"1":"7q0h","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-50 and Beta-54"},{"1":"7ps6","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-44 and Beta-54 Fabs"},{"1":"7tx4","2":"2022","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Neutron crystal structure of SARS-CoV-2 NSP3 macrodomain at 293 K (P21 crystal form)"},{"1":"7ps5","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with Beta-47 Fab"},{"1":"7rvr","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI18"},{"1":"7wvn","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron S-open"},{"1":"7vq0","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike protein (2-up RBD) bound to neutralizing nanobodies P86"},{"1":"7v7m","2":"2022","3":"X-ray diffraction","4":"crystal structure of SARS-CoV-2 3CL protease"},{"1":"7q9p","2":"2022","3":"Electron Microscopy","4":"Beta-06 fab in complex with SARS-CoV-2 beta-Spike glycoprotein"},{"1":"7pki","2":"2022","3":"X-ray diffraction","4":"Crystal structure of human ACE2 bound to the spike receptor-binding domain from a cave bat sarbecovirus closely related to SARS-CoV-2."},{"1":"7yqt","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer (1 RBD Up)"},{"1":"7yqu","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer (3 RBD Down)"},{"1":"7y9z","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron spike protein (S-6P-RRAR) in complex with human ACE2 ectodomain (one-RBD-up state)"},{"1":"7yr3","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer in complex with ACE2(state2)"},{"1":"7rvo","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI13"},{"1":"7z59","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 main protease (Mpro) covalently modified with a penicillin derivative"},{"1":"7tp2","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 3-RBD-down conformation; Subclassification D10 state"},{"1":"7tpe","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 1-RBD-up conformation; Subclassification D16 state"},{"1":"7ru2","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2-6P-Mut7 S protein (asymmetric)"},{"1":"7ya1","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of hACE2-bound SARS-CoV-2 Omicron spike protein with L371S, P373S and F375S mutations (local refinement)"},{"1":"7wt7","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike in complex with Fab 9A8 (State 1)"},{"1":"7uza","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the mouse antibody Fab fragment, HSW-1"},{"1":"7twg","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at 293 K (P43 crystal form, 153 kGy)"},{"1":"7tp0","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 3-RBD-down conformation; Subclassification D8 state"},{"1":"7wp9","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron Variant SPIKE trimer, all RBDs down"},{"1":"7uzd","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 RBD in complex with the mouse antibody Fab fragment, HSW-2"},{"1":"7tp3","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody K288.2"},{"1":"7uz6","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the mouse antibody Fab fragment, M8a-28"},{"1":"7tpc","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 1-RBD-up conformation; Subclassification D15 state"},{"1":"7xu6","2":"2022","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein with Engineered x3 Disulfide (x3(D427C, V987C) and single Arg S1/S2 cleavage site), incubated in Low pH after 40-Day Storage in PBS, Locked-2 Conformation"},{"1":"7tlt","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 Spike-derived peptide S489-497 (YFPLQSYGF) presented by HLA-A*29:02"},{"1":"7rvt","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI20"},{"1":"7ru1","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2-6P-Mut7 S protein (C3 symmetry)"},{"1":"7uj9","2":"2022","3":"X-ray diffraction","4":"Room-temperature X-ray structure of monomeric SARS-CoV-2 main protease catalytic domain (MPro1-199)"},{"1":"7u28","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Mpro Lambda (G15S) in complex with Nirmatrelvir (PF-07321332)"},{"1":"7tv0","2":"2022","3":"X-ray diffraction","4":"Crystal structure of BRD4 bromodomain 1 in complex with dual-acetylated SARS-CoV-2 E"},{"1":"7wsk","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Omicron spike receptor-binding domain in complex with civet ACE2"},{"1":"7y42","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein in complex with all-trans retinoic acid"},{"1":"7yqv","2":"2022","3":"Electron Microscopy","4":"pH 5.5 SARS-CoV-2 BA.2.75 S Trimer (1 RBD Up)"},{"1":"7ydy","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike (6P) in complex with 1 R1-32 Fab"},{"1":"7pku","2":"2022","3":"Solution NMR","4":"Structure of SARS-CoV-2 nucleoprotein in dynamic complex with its viral partner nsp3a"},{"1":"7tpa","2":"2022","3":"Electron Microscopy","4":"Delta (B.1.617.2) SARS-CoV-2 variant spike protein (S-GSAS-Delta) in the 1-RBD-up conformation; Subclassification D14 state"},{"1":"7q6e","2":"2022","3":"Electron Microscopy","4":"Beta049 fab in complex with SARS-CoV2 beta-Spike glycoprotein, The Beta mAb response underscores the antigenic distance to other SARS-CoV-2 variants"},{"1":"7pqz","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with FI-3A and FD-11A Fabs"},{"1":"7twf","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain at 293 K (P43 crystal form, 73 kGy)"},{"1":"7ps0","2":"2022","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 beta variant spike glycoprotein in complex with beta-24 Fabs"},{"1":"7tlc","2":"2022","3":"Electron Microscopy","4":"Down-state locked, S2 stabilized rS2d-HexaPro SARS-CoV-2 spike ectodomain in the RBD-down conformation, State 1"},{"1":"7rvy","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI25"},{"1":"7uju","2":"2022","3":"X-ray diffraction","4":"Room-temperature X-ray structure of monomeric SARS-CoV-2 main protease catalytic domain (MPro1-196) in complex with nirmatrelvir"},{"1":"7rdy","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(2)-RTC - engaged class"},{"1":"7xnr","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2.13 variant spike"},{"1":"7x8z","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab188"},{"1":"7xix","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron BA.2 variant spike (state 2)"},{"1":"7x7e","2":"2022","3":"X-ray diffraction","4":"SARS-CoV-2 RBD and Nb22"},{"1":"7xx1","2":"2022","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 N-NTD"},{"1":"7au4","2":"2022","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease (Nsp5) in complex with compound 3"},{"1":"7rdx","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(2)-RTC - open class"},{"1":"7ye5","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Spike (6P) in complex with 2 R1-32 Fabs"},{"1":"7q9g","2":"2022","3":"Electron Microscopy","4":"COVOX-222 fab in complex with SARS-CoV-2 beta-Spike glycoprotein"},{"1":"7xnf","2":"2022","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 antibody P2C-1F11 with GX/P2V/2017 RBD"},{"1":"8a4t","2":"2022","3":"X-ray diffraction","4":"crystal structures of diastereomer (S,S,S)-13b (13b-K) in complex with the SARS-CoV-2 Mpro"},{"1":"7xar","2":"2022","3":"X-ray diffraction","4":"Crystal structure of 3C-like protease from SARS-CoV-2 in complex with covalent inhibitor"},{"1":"7x2k","2":"2022","3":"X-ray diffraction","4":"Crystal structure of nanobody Nb70 with antibody 1F11 fab and SARS-CoV-2 RBD"},{"1":"7uz7","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the mouse antibody Fab fragment, M8a-31"},{"1":"7wp2","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 C.1.2 S6P trimer in complex with neutralizing antibody VacW-209 (local refinement)"},{"1":"7uhc","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with AHB2-2GS-SB175"},{"1":"7x8y","2":"2022","3":"Electron Microscopy","4":"The SARS-CoV-2 receptor binding domain bound with the Fab fragment of a human neutralizing antibody Ab159"},{"1":"7wpc","2":"2022","3":"Electron Microscopy","4":"The second RBD of SARS-CoV-2 Omicron Variant in complexed with RBD-ACE2"},{"1":"7wrh","2":"2022","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Omicron BA.1 spike protein in complex with mouse ACE2"},{"1":"7woc","2":"2022","3":"Electron Microscopy","4":"Locally refined region of SARS-CoV-2 Spike in complex with IgG 553-60"},{"1":"7uz8","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 Omicron BA.1 S 6P trimer in complex with the mouse antibody Fab fragment, M8a-31"},{"1":"7wtj","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron variant spike RBD in complex with Fab XGv286"},{"1":"7uz4","2":"2022","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the mouse antibody Fab fragment, M8a-3"},{"1":"7ukk","2":"2022","3":"X-ray diffraction","4":"Room-temperature X-ray structure of SARS-CoV-2 main protease in complex with GC-376"},{"1":"7x2m","2":"2022","3":"X-ray diffraction","4":"Crystal structure of nanobody 1-2C7 with SARS-CoV-2 RBD"},{"1":"7tly","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 S B.1.1.529 Omicron variant (RBD + S309 Local Refinement)"},{"1":"7rvz","2":"2022","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI26"},{"1":"7zbu","2":"2022","3":"Electron Microscopy","4":"CryoEM structure of SARS-CoV-2 spike monomer in complex with neutralising antibody P008_60"},{"1":"7yqw","2":"2022","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer (3 RBD Down)"},{"1":"5s4f","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF003"},{"1":"5s48","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1741982125"},{"1":"5s3z","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0140"},{"1":"5s2m","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z56827661"},{"1":"5rur","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000017744334"},{"1":"5s2s","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434894"},{"1":"5rv4","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000039224"},{"1":"5s2y","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z19727416"},{"1":"5rvd","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000263980802"},{"1":"5s3d","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z30820160"},{"1":"5rv5","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000008578948"},{"1":"5rs8","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000001601"},{"1":"5s2k","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z445856640"},{"1":"5ruf","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000016989831"},{"1":"5s2h","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434920"},{"1":"5rv0","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000039994"},{"1":"5s3c","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434937"},{"1":"5rvc","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000933940912"},{"1":"5rtk","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000164504"},{"1":"5rvn","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332748"},{"1":"5rtn","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000013514509"},{"1":"5s3s","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0103"},{"1":"5s2b","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z373769142"},{"1":"5rut","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000161958"},{"1":"5s31","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1741959530"},{"1":"5rvg","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000400552187_N3"},{"1":"5s3o","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z102768020"},{"1":"5s20","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with PB1827975385"},{"1":"5run","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000194295"},{"1":"5s1u","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-52144"},{"1":"5ru5","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000098208711"},{"1":"5s1s","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1613477500"},{"1":"5rvv","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000020269197"},{"1":"5ru4","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001688638"},{"1":"5s3e","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z274553586"},{"1":"5s1k","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-105873"},{"1":"5rtv","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001698894"},{"1":"5rss","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000006691828"},{"1":"5rlv","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2467208649"},{"1":"5s3v","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0120"},{"1":"5s32","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z26781943"},{"1":"5ru9","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000165882"},{"1":"5s2f","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z44592329"},{"1":"5rvj","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001612349"},{"1":"5rtp","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001679336"},{"1":"5rmg","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z285675722"},{"1":"5s4i","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF051"},{"1":"5s4c","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1954800348"},{"1":"5s46","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z57131035"},{"1":"5s3l","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z54628578"},{"1":"5s26","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z605596346"},{"1":"5ruu","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000438614"},{"1":"5rl6","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z198195770"},{"1":"5s3q","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0013"},{"1":"5s2d","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z369936976"},{"1":"5rua","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000033986325"},{"1":"5rt6","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000156509"},{"1":"5rlz","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2293643386"},{"1":"5s28","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z409974522"},{"1":"5rux","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002020050"},{"1":"5rsf","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000026180281"},{"1":"5rlg","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z19739650"},{"1":"5rto","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388302"},{"1":"5rvt","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002582714"},{"1":"5rsh","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000274438208"},{"1":"5rlj","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1407673036"},{"1":"5s43","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00024661"},{"1":"5s3b","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1741966151"},{"1":"5ru2","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000331715"},{"1":"5rsy","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000004787230"},{"1":"5rlq","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z285782452"},{"1":"5rsu","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000002055"},{"1":"5rv3","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000057162"},{"1":"5s3i","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z50145861"},{"1":"5s1i","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-301084"},{"1":"5rul","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000163774"},{"1":"5s34","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434941"},{"1":"5rtl","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388056"},{"1":"5s4e","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2301685688"},{"1":"5s47","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z940713508"},{"1":"5s3u","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0041"},{"1":"5s1m","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with STK497968"},{"1":"5rtq","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000019015078"},{"1":"5rm2","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1741964527"},{"1":"5s3t","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0128"},{"1":"5s24","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-697611"},{"1":"5ruq","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000032199226"},{"1":"5rtf","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002047514"},{"1":"5s42","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00023833"},{"1":"5s30","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z65532537"},{"1":"5rvl","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000149580"},{"1":"5ruj","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000404314"},{"1":"5rt7","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000015442276"},{"1":"5rm1","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z426041412"},{"1":"5s1o","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with STL414928"},{"1":"5rty","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000157088"},{"1":"5rsv","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000340465"},{"1":"5rlw","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z45705015"},{"1":"5s3g","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z384468096"},{"1":"5s1c","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z3034471507"},{"1":"5rvf","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000082473428_N3"},{"1":"5rsz","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000004218283"},{"1":"5rm7","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z69118333"},{"1":"5ruv","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000019015194"},{"1":"5rtj","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332752"},{"1":"5rt8","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000161908"},{"1":"5rmm","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with POB0066"},{"1":"5s2t","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z26781964"},{"1":"5rvi","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000084843283"},{"1":"5s4h","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF048"},{"1":"5s4b","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z3219959731"},{"1":"5s44","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00024890"},{"1":"5s38","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1745658474"},{"1":"5rtz","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000404062"},{"1":"5rlm","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1650168321"},{"1":"5rle","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1429867185"},{"1":"7dx8","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 2 (2 up RBD and 2 PD bound)"},{"1":"7drv","2":"2021","3":"X-ray diffraction","4":"Structural basis of SARS-CoV-2-closely-related bat coronavirus RaTG13 to hACE2"},{"1":"7ekc","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Gamma variant spike receptor-binding domain complexed with human ACE2"},{"1":"7e7y","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-623 Fab"},{"1":"7dx9","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 3 (3 up RBD and 2 PD bound)"},{"1":"7diy","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nsp10 bound to nsp14-exoribonuclease domain"},{"1":"7cwm","2":"2021","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 spike protein and Fab P17 with one RBD in open state and two RBD in closed state"},{"1":"7cwt","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike protein in complex with hb27 and fc05 Fab cocktail"},{"1":"7eb3","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, one RBD-up conformation 3"},{"1":"7dx2","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2 D614G mutant"},{"1":"7e7d","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 wild-type S-Trimer from a subunit vaccine candidate"},{"1":"7dx5","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 2 (1 up RBD and 1 PD bound)"},{"1":"7e3l","2":"2021","3":"Electron Microscopy","4":"Ultrapotent SARS-CoV-2 neutralizing antibodies with protective efficacy against newly emerged mutational variants"},{"1":"7dx7","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 1 (1 up RBD and 1 PD bound)"},{"1":"7e88","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-515 Fab"},{"1":"7czx","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B9"},{"1":"7cwu","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike proteins trimer in complex with P17 and FC05 Fabs cocktail"},{"1":"7cyv","2":"2021","3":"X-ray diffraction","4":"Crystal structure of FD20, a neutralizing single-chain variable fragment (scFv) in complex with SARS-CoV-2 Spike receptor-binding domain (RBD)"},{"1":"7cut","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with Z-VAD-FMK"},{"1":"5s2w","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1407672867"},{"1":"5s4g","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF005"},{"1":"5s41","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00023825"},{"1":"5s33","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z906021418"},{"1":"5rsn","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000064576"},{"1":"5s1y","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with STK346965"},{"1":"5ru0","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388514"},{"1":"5rso","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000000226"},{"1":"5rlr","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z822382694"},{"1":"5rsc","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000003888754"},{"1":"5rlf","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z235341991"},{"1":"5s3m","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z45656995"},{"1":"5s1q","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-17035"},{"1":"5ruh","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000123600"},{"1":"5rt4","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000051581"},{"1":"5rlp","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z166605480"},{"1":"5s3a","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1562205518"},{"1":"5rv8","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000039575"},{"1":"5rsj","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000089254160_N3"},{"1":"5rlb","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z216450634"},{"1":"5rvb","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000014419577"},{"1":"5rth","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000156863"},{"1":"5s18","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-321461"},{"1":"5rvq","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002508153"},{"1":"5rts","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000159004"},{"1":"5rtm","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000002005"},{"1":"5rse","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000336438345"},{"1":"5s1e","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with AB-601_30915014"},{"1":"5rva","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000016343276"},{"1":"5ru8","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000154817"},{"1":"5rsi","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000374420934"},{"1":"5rlc","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z56923284"},{"1":"5s3j","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1324853681"},{"1":"5s2c","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z45612755"},{"1":"5ruk","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000161692"},{"1":"5s2a","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1263529624"},{"1":"5ruo","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001683100"},{"1":"5rsk","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000901381520_N3"},{"1":"5rly","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2027049478"},{"1":"5rte","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000013283576"},{"1":"5rmh","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1101755952"},{"1":"5s1g","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-108952"},{"1":"5rmj","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z68299550"},{"1":"5rmd","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z57614330"},{"1":"5s49","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z56866006"},{"1":"5rve","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000736709772"},{"1":"5s2z","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z126932614"},{"1":"5ru7","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000003591110"},{"1":"5rsp","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002560357"},{"1":"5rli","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z45617795"},{"1":"5s4j","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with SF054"},{"1":"5s4a","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z955123498"},{"1":"5s3w","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0135"},{"1":"5s2v","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1186029914"},{"1":"5rvm","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000157088"},{"1":"5s1w","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z838838708"},{"1":"5rvo","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000013514509"},{"1":"5rsd","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000331945"},{"1":"5rld","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z19735981"},{"1":"5s2u","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z85956652"},{"1":"5rvh","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000265642"},{"1":"5s3y","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0012"},{"1":"5s2l","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2234920345"},{"1":"5rvp","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000154817"},{"1":"5rtt","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000873830"},{"1":"5rsg","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000263392672"},{"1":"5rt5","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000058111"},{"1":"5rmi","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z53860899"},{"1":"5rmk","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1273312153"},{"1":"5rst","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332673"},{"1":"5rm5","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z373768900"},{"1":"5rm3","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1745658474"},{"1":"5rm0","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1492796719"},{"1":"5rsb","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001674697"},{"1":"5rt9","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388280"},{"1":"5rma","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z321318226"},{"1":"5rus","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388081"},{"1":"5rsl","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000365052868"},{"1":"5rl8","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z53825177"},{"1":"5rsw","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000337835"},{"1":"5rm8","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1614545742"},{"1":"5ruz","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000019685960"},{"1":"5rs9","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000007636250"},{"1":"5rup","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000004976927"},{"1":"5s2x","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1139246057"},{"1":"5rvu","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002506130"},{"1":"5rtx","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000090873"},{"1":"5rsm","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000001099"},{"1":"5rlt","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z53116498"},{"1":"5rm6","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z396380540"},{"1":"5s2i","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z57299529"},{"1":"5ru6","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000001442764"},{"1":"5rm4","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1639162606"},{"1":"5rt2","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000008652361"},{"1":"5rlu","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z744754722"},{"1":"5rtr","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000018169763"},{"1":"5rl7","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z364321922"},{"1":"5rtb","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000006534965"},{"1":"5rm9","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2856434942"},{"1":"5rs7","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000034618676"},{"1":"5s3p","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1238477790"},{"1":"5s29","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z199959602"},{"1":"5rum","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000008861082"},{"1":"5s2p","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z927746322"},{"1":"5rtu","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000159056"},{"1":"5rl9","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1703168683"},{"1":"5rvs","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000159004"},{"1":"5rtw","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000164777"},{"1":"5rsr","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000158650"},{"1":"5rln","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z364328788"},{"1":"5s35","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z68404778"},{"1":"5rmb","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2856434920"},{"1":"5rta","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332540"},{"1":"5rmc","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z24758179"},{"1":"5s3f","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z57446103"},{"1":"5rvk","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002977810"},{"1":"5rue","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000000922"},{"1":"5rt0","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000002582714"},{"1":"5rll","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z425387594"},{"1":"5rsx","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388262"},{"1":"5rlo","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1454310449"},{"1":"5s2g","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z321318226"},{"1":"5ruw","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000045014941"},{"1":"5rme","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z26333434"},{"1":"5rml","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z85956652"},{"1":"5s2j","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z509756472"},{"1":"5ruy","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000013517187"},{"1":"5rv1","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000251609"},{"1":"5rlk","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z1509882419"},{"1":"5ruc","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000005878"},{"1":"5rsq","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000158490"},{"1":"5rlh","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z2856434778"},{"1":"5s2n","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1787627869"},{"1":"5s40","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00023824"},{"1":"5s37","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1954800564"},{"1":"5rui","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000332651"},{"1":"5rt3","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000039281982"},{"1":"5rmf","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z54226006"},{"1":"5s3k","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z219104216"},{"1":"5s27","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1262398530"},{"1":"5rv6","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000158540"},{"1":"5rtg","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000395673"},{"1":"5rtc","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000006490906"},{"1":"5s3r","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0014"},{"1":"5s2r","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z57292369"},{"1":"5rv2","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000311783"},{"1":"5s39","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z165170770"},{"1":"5rv9","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000388150"},{"1":"5s1a","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with EN300-43406"},{"1":"5rvr","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000016052862"},{"1":"5ru3","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000161696"},{"1":"5s36","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434938"},{"1":"5s4k","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with FMOOA000509a"},{"1":"5s4d","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1741982441"},{"1":"5s45","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with NCL-00024773"},{"1":"5s2e","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z1152242726"},{"1":"5rud","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000008615114"},{"1":"5s3x","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with POB0136"},{"1":"5s2q","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z26781952"},{"1":"5rv7","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000003954002"},{"1":"5rti","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000004219237"},{"1":"5rt1","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000039810"},{"1":"5rls","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 helicase in complex with Z59181945"},{"1":"5s3n","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z287484230"},{"1":"5s22","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z145120524"},{"1":"5ru1","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000034687"},{"1":"5rtd","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000157108"},{"1":"5s3h","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z2856434892"},{"1":"5s2o","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 Nsp3 macrodomain in complex with Z645232558"},{"1":"5rug","2":"2021","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ZINC000000038389"},{"1":"7n62","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike (2P) in complex with C12C9 Fab (NTD local reconstruction)"},{"1":"7mzj","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab WCSL 129 and Fab PDI 93"},{"1":"7sbp","2":"2021","3":"Electron Microscopy","4":"Closed state of pre-fusion SARS-CoV-2 Kappa variant spike protein"},{"1":"7sbq","2":"2021","3":"Electron Microscopy","4":"One RBD-up 1 of pre-fusion SARS-CoV-2 Kappa variant spike protein"},{"1":"7neh","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-269 Fab"},{"1":"7n5z","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease C145S mutant"},{"1":"7np1","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of the SARS-CoV-2 Receptor Binding Domain in Complex with Antibody ION-360"},{"1":"7nd7","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-316 Fab"},{"1":"7n9t","2":"2021","3":"Electron Microscopy","4":"CryoEM structure of SARS-CoV-2 Spike in complex with Nb17"},{"1":"7sy0","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,L452R mutant spike protein ectodomain bound to human ACE2 ectodomain (focused refinement of RBD and ACE2)"},{"1":"7nxb","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 P.1 variant Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7n0c","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the monomeric form of SARS-CoV-2 nsp10-nsp14 (E191A)-RNA complex"},{"1":"7mzg","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab PDI 42"},{"1":"7nd8","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-384 Fab"},{"1":"7lws","2":"2021","3":"Electron Microscopy","4":"UK (B.1.1.7) SARS-CoV-2 S-GSAS-D614G variant spike protein in the 3-RBD-down conformation"},{"1":"7lss","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab 2-7"},{"1":"7lkv","2":"2021","3":"X-ray diffraction","4":"1.55 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 3c"},{"1":"7m00","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 13c"},{"1":"7cjd","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 PLpro C111S mutant"},{"1":"7jyy","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer in Complex with (m7GpppA)pUpUpApApA (Cap-0) and S-Adenosylmethionine (SAM)."},{"1":"7chs","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P22A-1D1 with RBD"},{"1":"7can","2":"2021","3":"X-ray diffraction","4":"Structure of sybody MR17-K99Y in complex with the SARS-CoV-2 S Receptor-binding domain (RBD)"},{"1":"7lhq","2":"2021","3":"Solution NMR","4":"Solution structure of SARS-CoV-2 nonstructural protein 7 at pH 7.0"},{"1":"7l7e","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike RBD in complex with human monoclonal antibodies AZD8895 and AZD1061"},{"1":"7k0e","2":"2021","3":"X-ray diffraction","4":"1.90 A resolution structure of SARS-CoV-2 3CL protease in complex with deuterated GC376"},{"1":"7jst","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL in apo form"},{"1":"7jq2","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI5"},{"1":"7cdi","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P2C-1F11 with RBD"},{"1":"7c8v","2":"2021","3":"X-ray diffraction","4":"Structure of sybody SR4 in complex with the SARS-CoV-2 S Receptor Binding domain (RBD)"},{"1":"7mdw","2":"2021","3":"Electron Microscopy","4":"CryoEM structure of SARS-CoV-2 RBD in complex with nanobodies Nb21 and Nb105"},{"1":"7m7b","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike:Fab 3D11 complex focused refinement"},{"1":"7lwu","2":"2021","3":"Electron Microscopy","4":"UK (B.1.1.7) SARS-CoV-2 spike protein variant (S-GSAS-B.1.1.7) in the 1-RBD-up conformation"},{"1":"7ld1","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S protein in complex with Receptor Binding Domain antibody DH1047"},{"1":"7l7d","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike RBD in complex with human monoclonal antibody AZD8895"},{"1":"7mf1","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody 47D1"},{"1":"7m6e","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, BG10-19"},{"1":"7m01","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 14c"},{"1":"7lyo","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 1-RBD-up conformation"},{"1":"7lwk","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 3-RBD down conformation"},{"1":"7llz","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-69"},{"1":"7lks","2":"2021","3":"X-ray diffraction","4":"1.70 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 2f"},{"1":"7lv2","2":"2021","3":"X-ray diffraction","4":"GSQASS segment from the Nucleoprotein of SARS-CoV-2, residues 179-184"},{"1":"7lka","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD-targeting antibody COV107-23"},{"1":"7lzz","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 5c"},{"1":"7mjh","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to VH ab8"},{"1":"7m2p","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 3CL protease in complex with inhibitor 18"},{"1":"7chp","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P5A-3C8 with RBD"},{"1":"7com","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with Boceprevir (space group P212121)"},{"1":"7b3b","2":"2021","3":"Electron Microscopy","4":"Structure of elongating SARS-CoV-2 RNA-dependent RNA polymerase with Remdesivir at position -3 (structure 1)"},{"1":"7aww","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Clonidine"},{"1":"7bf3","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 macrodomain in complex with adenosine"},{"1":"7bnv","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of the SARS-CoV-2 Receptor Binding Domain in Complex with Antibody ION-300"},{"1":"7bf6","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 macrodomain in complex with remdesivir metabolite GS-441524"},{"1":"7ar5","2":"2021","3":"X-ray diffraction","4":"Structure of apo SARS-CoV-2 Main Protease with small beta angle, space group C2."},{"1":"7k0f","2":"2021","3":"X-ray diffraction","4":"1.65 A resolution structure of SARS-CoV-2 3CL protease in complex with a deuterated GC376 alpha-ketoamide analog (compound 5)"},{"1":"7c53","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 HR1 motif in complex with pan-CoVs inhibitor EK1"},{"1":"7bei","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-150 Fab"},{"1":"7ar6","2":"2021","3":"X-ray diffraction","4":"Structure of apo SARS-CoV-2 Main Protease with large beta angle, space group C2."},{"1":"7b3c","2":"2021","3":"Electron Microscopy","4":"Structure of elongating SARS-CoV-2 RNA-dependent RNA polymerase with Remdesivir at position -4 (structure 2)"},{"1":"7jrn","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the wild type SARS-CoV-2 papain-like protease (PLPro) with inhibitor GRL0617"},{"1":"7jpy","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in its apo-form"},{"1":"7ju7","2":"2021","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 Main Protease in complex with masitinib"},{"1":"7lym","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the RBD-down conformation"},{"1":"7lww","2":"2021","3":"Electron Microscopy","4":"Triple mutant (K417N-E484K-N501Y) SARS-CoV-2 spike protein in the 1-RBD-up conformation (S-GSAS-D614G-K417N-E484K-N501Y)"},{"1":"7lc8","2":"2021","3":"Solution NMR","4":"SARS-CoV-2 spike Protein TM domain"},{"1":"7l5b","2":"2021","3":"X-ray diffraction","4":"Crystallographic structure of neutralizing antibody 2-15 in complex with SARS-CoV-2 spike receptor-binding Domain (RBD)."},{"1":"7jq5","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI8"},{"1":"7jpe","2":"2021","3":"X-ray diffraction","4":"Room Temperature Structure of SARS-CoV-2 Nsp10/Nsp16 Methyltransferase in a Complex with m7GpppA Cap-0 and SAM Determined by Fixed-Target Serial Crystallography"},{"1":"7k0r","2":"2021","3":"Electron Microscopy","4":"Nucleotide bound SARS-CoV-2 Nsp15"},{"1":"7ci3","2":"2021","3":"X-ray diffraction","4":"The crystal structure of the SARS-CoV-2 ORF7a ectodomain"},{"1":"7b62","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein N-terminal domain in complex with biliverdin"},{"1":"7cbt","2":"2021","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 main protease in complex with GC376"},{"1":"7bnn","2":"2021","3":"Electron Microscopy","4":"Open conformation of D614G SARS-CoV-2 spike with 1 Erect RBD"},{"1":"7bal","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease treated with ebselen derivative of MR6-31-2"},{"1":"7c7p","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with Telaprevir"},{"1":"7b3d","2":"2021","3":"Electron Microscopy","4":"Structure of elongating SARS-CoV-2 RNA-dependent RNA polymerase with AMP at position -4 (structure 3)"},{"1":"7bel","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in a ternary complex with COVOX-88 and COVOX-45 Fabs"},{"1":"7jq0","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI3"},{"1":"7jq1","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI4"},{"1":"7jtl","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 ORF8 accessory protein"},{"1":"7jw8","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with compound 4 in space group P1"},{"1":"7jlt","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 NSP7-NSP8 complex."},{"1":"7k3n","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of NSP1 from SARS-CoV-2"},{"1":"7jpz","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI1"},{"1":"7k7p","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 nonstuctural protein 1"},{"1":"7jt0","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with MAC5576"},{"1":"7cjf","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with a neutralizing antibody Fab"},{"1":"7cmd","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 PLpro with GRL0617"},{"1":"7c8w","2":"2021","3":"X-ray diffraction","4":"Structure of sybody MR17 in complex with the SARS-CoV-2 S receptor-binding domain (RBD)"},{"1":"7cho","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P5A-1D2 with RBD"},{"1":"7be7","2":"2021","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"7jt7","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with compound 4"},{"1":"7jib","2":"2021","3":"X-ray diffraction","4":"Room Temperature Crystal Structure of Nsp10/Nsp16 from SARS-CoV-2 with Substrates and Products of 2'-O-methylation of the Cap-1"},{"1":"7jkv","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease in complex with an inhibitor GRL-2420"},{"1":"7jsu","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 3CL protease in complex with GC376"},{"1":"7lyl","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the RBD-down conformation"},{"1":"7lwi","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 3-RBD down conformation"},{"1":"7laa","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S protein in complex with Receptor Binding Domain antibody DH1041"},{"1":"7jq3","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI6"},{"1":"7jhe","2":"2021","3":"X-ray diffraction","4":"Room Temperature Structure of SARS-CoV-2 Nsp10/Nsp16 Methyltransferase in a Complex with 2'-O-methylated m7GpppA Cap-1 and SAH Determined by Fixed-Target Serial Crystallography"},{"1":"7orb","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the L452R mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-75 and COVOX-253 Fabs"},{"1":"7ozu","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RdRp with Molnupiravir/ NHC in the template strand base-paired with A"},{"1":"7or9","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and COVOX-278 Fabs"},{"1":"7ozv","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RdRp with Molnupiravir/ NHC in the template strand base-paired with G"},{"1":"7sy4","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K mutant spike protein ectodomain bound to human ACE2 ectodomain (focused refinement of RBD and ACE2)"},{"1":"7sob","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S B.1.617.1 kappa variant + S309 + S2L20 Global Refinement"},{"1":"7sbs","2":"2021","3":"Electron Microscopy","4":"One RBD-up 1 of pre-fusion SARS-CoV-2 Gamma variant spike protein"},{"1":"7sxw","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K,K417T mutant spike protein ectodomain"},{"1":"7soa","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S NTD B.1.617.2 delta variant + S2L20 Local Refinement"},{"1":"7s3n","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 S stem helix peptide bound to Fab22"},{"1":"7sof","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S NTD B.1.617.1 kappa variant S2X303 Local Refinement"},{"1":"7sy2","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y mutant spike protein ectodomain bound to human ACE2 ectodomain (focused refinement of RBD and ACE2)"},{"1":"7sxz","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,L452R mutant spike protein ectodomain bound to human ACE2 ectodomain (global refinement)"},{"1":"7sbl","2":"2021","3":"Electron Microscopy","4":"One RBD-up 1 of pre-fusion SARS-CoV-2 Delta variant spike protein"},{"1":"7vxm","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with ACE2, Beta variant, C3 state"},{"1":"7vx5","2":"2021","3":"Electron Microscopy","4":"ACE2-RBD in SARS-CoV-2 Kappa variant S-ACE2 complex"},{"1":"7olz","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD with neutralizing-VHHs Re5D06 and Re9F06"},{"1":"7vxe","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Kappa variant spike protein in open state"},{"1":"7vxf","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with ACE2, Beta variant, C2B state"},{"1":"7on5","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 neutralizing nanobody Re5D06"},{"1":"7vx9","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Kappa variant spike protein in complex wth ACE2, state C1"},{"1":"7sxr","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G mutant spike protein ectodomain"},{"1":"7sy3","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K mutant spike protein ectodomain bound to human ACE2 ectodomain (global refinement)"},{"1":"7sxs","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,L452R mutant spike protein ectodomain"},{"1":"7sy8","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K,K417T mutant spike protein ectodomain bound to human ACE2 ectodomain (focused refinement of RBD and ACE2)"},{"1":"7spo","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain in complex with VNAR 3B4"},{"1":"7soe","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S B.1.617.1 kappa variant + S2X303 Global Refinement"},{"1":"7sy6","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K,K417N mutant spike protein ectodomain bound to human ACE2 ectodomain (focused refinement of RBD and ACE2)"},{"1":"7sy5","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K,K417N mutant spike protein ectodomain bound to human ACE2 ectodomain (global refinement)"},{"1":"7nx9","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the N501Y mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7sxt","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y mutant spike protein ectodomain"},{"1":"7qo7","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S Omicron Spike B.1.1.529"},{"1":"7vx4","2":"2021","3":"Electron Microscopy","4":"ACE2-RBD in SARS-CoV-2 Beta variant S-ACE2 complex"},{"1":"7qo9","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S Omicron Spike B.1.1.529 - RBD and NTD (Local)"},{"1":"7so9","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S B.1.617.2 delta variant + S2M11 + S2L20 Global Refinement"},{"1":"8csa","2":"2021","3":"Electron Microscopy","4":"Triple mutant (K417N-E484K-N501Y) SARS-CoV-2 spike protein in the 3-RBD-Down conformation (S-GSAS-D614G-K417N-E484K-N501Y)"},{"1":"7o81","2":"2021","3":"Electron Microscopy","4":"Rabbit 80S ribosome colliding in another ribosome stalled by the SARS-CoV-2 pseudoknot"},{"1":"7nx7","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the K417N mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7ndd","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein (one RBD up) in complex with COVOX-159"},{"1":"7m4r","2":"2021","3":"Electron Microscopy","4":"Structural basis for SARS-CoV-2 envelope protein in recognition of human cell junction protein PALS1"},{"1":"7m8u","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-B*35:01 in complex with IPFAMQMAY, an 9-mer epitope from SARS-CoV-2 spike (S896-904)"},{"1":"7nev","2":"2021","3":"X-ray diffraction","4":"Structure of the hemiacetal complex between the SARS-CoV-2 Main Protease and Leupeptin"},{"1":"7o7z","2":"2021","3":"Electron Microscopy","4":"Rabbit 80S ribosome stalled close to the mutated SARS-CoV-2 slippery site by a pseudoknot (classified for pseudoknot)"},{"1":"7o80","2":"2021","3":"Electron Microscopy","4":"Rabbit 80S ribosome in complex with eRF1 and ABCE1 stalled at the STOP codon in the mutated SARS-CoV-2 slippery site"},{"1":"7nx8","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the K417T mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7nt9","2":"2021","3":"Electron Microscopy","4":"Trimeric SARS-CoV-2 spike ectodomain in complex with biliverdin (closed conformation)"},{"1":"7nxa","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 B.1.351 variant Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7ntk","2":"2021","3":"X-ray diffraction","4":"PALS1 PDZ1 domain with SARS-CoV-2_E PBM complex"},{"1":"7nxc","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 P.1 variant Spike glycoprotein in complex with ACE2"},{"1":"7nx6","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-222 and EY6A Fabs"},{"1":"7nn0","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 helicase in complex with AMP-PNP"},{"1":"7nd5","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-150 Fab"},{"1":"7n6n","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease C145S mutant in complex with N and C-terminal residues"},{"1":"7n4l","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing human antibody WRAIR-2125."},{"1":"7n33","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp15 endoribonuclease pre-cleavage state"},{"1":"7n1t","2":"2021","3":"Electron Microscopy","4":"Structural basis for enhanced infectivity and immune evasion of SARS-CoV-2 variants"},{"1":"7ntc","2":"2021","3":"Electron Microscopy","4":"Trimeric SARS-CoV-2 spike ectodomain bound to P008_056 Fab"},{"1":"7sod","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S NTD B.1.617.1 kappa variant S2L20 Local Refinement"},{"1":"7mlx","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 programmed -1 frameshifting element three stem H-type pseudoknot"},{"1":"7lx5","2":"2021","3":"Electron Microscopy","4":"The SARS-CoV-2 spike protein receptor binding domain bound to neutralizing nanobodies WNb 2 and WNb 10"},{"1":"7l8i","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Rupintrivir (P21)"},{"1":"7lzu","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 12b"},{"1":"7lwn","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 1-RBD up conformation"},{"1":"7lkr","2":"2021","3":"X-ray diffraction","4":"1.65 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 2a"},{"1":"7lbr","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-89"},{"1":"7ly2","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2M28 Global Refinement"},{"1":"7l8j","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Rupintrivir (P21212)"},{"1":"7mjj","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to Fab ab1 (class 1)"},{"1":"7mbg","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease in orthorhombic space group"},{"1":"7mzk","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab WCSL 129 and Fab PDI 96"},{"1":"7lwp","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 2-RBD up conformation"},{"1":"7lkx","2":"2021","3":"X-ray diffraction","4":"1.60 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 3e"},{"1":"7lco","2":"2021","3":"X-ray diffraction","4":"Improved Feline Drugs as SARS-CoV-2 Mpro Inhibitors: Structure-Activity Studies & Micellar Solubilization for Enhanced Bioavailability"},{"1":"7mgs","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in complex with N-terminal autoprocessing substrate"},{"1":"7lkw","2":"2021","3":"X-ray diffraction","4":"1.65 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 3d (deuterated analog of inhibitor 3c)"},{"1":"7ldl","2":"2021","3":"X-ray diffraction","4":"Improved Feline Drugs as SARS-CoV-2 Mpro Inhibitors: Structure-Activity Studies & Micellar Solubilization for Enhanced Bioavailability"},{"1":"7lzy","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 3c"},{"1":"7ly3","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 S NTD bound to S2M28 Fab"},{"1":"7lrs","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with neutralizing antibody A23-58.1 that targets the receptor-binding domain"},{"1":"7lkt","2":"2021","3":"X-ray diffraction","4":"1.50 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 2k"},{"1":"7mm0","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with neutralizing antibody B1-182.1 that targets the receptor-binding domain"},{"1":"7mjg","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain"},{"1":"7mlz","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with neutralizing antibody B1-182.1 that targets the receptor-binding domain"},{"1":"7me7","2":"2021","3":"Electron Microscopy","4":"CryoEM structure of SARS-CoV-2 RBD in complex with nanobodies Nb17 and Nb105"},{"1":"7ljr","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Protein Trimer bound to DH1043 fab"},{"1":"7lfe","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - F2X Entry Library E03 fragment"},{"1":"7m6h","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, BG7-20"},{"1":"7mkm","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike RBD in complex with neutralizing Fab SARS2-38 (local refinement)"},{"1":"7m6f","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, BG1-22"},{"1":"7m0j","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 u1S2q All Down RBD State Spike Protein Trimer - asymmetric refinement"},{"1":"7lyk","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 2-RBD-up conformation"},{"1":"7lw4","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 nsp16/nsp10 complex in presence of S-adenosyl-L-homocysteine (SAH)"},{"1":"7m04","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 21c"},{"1":"7lyi","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAWJ9-36-3"},{"1":"7lxy","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2X333 Global Refinement"},{"1":"7llf","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-83"},{"1":"7ldj","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain in complex with WNb-2"},{"1":"7lqw","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 2-17 Fab in complex with SARS-CoV-2 S2P spike"},{"1":"7mjk","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to Fab ab1 (class 2)"},{"1":"7m7w","2":"2021","3":"X-ray diffraction","4":"Antibodies to the SARS-CoV-2 receptor-binding domain that maximize breadth and resistance to viral escape"},{"1":"7lyn","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 1-RBD-up conformation"},{"1":"7lab","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S protein in complex with N-terminal domain antibody DH1052"},{"1":"7l6r","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer in Complex with (m7GpppA2m)pUpUpApApA (Cap-1), S-Adenosyl-L-homocysteine (SAH) and Manganese (Mn)."},{"1":"7mji","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to VH ab8 (focused refinement of RBD and VH ab8)"},{"1":"7m8j","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S-NTD + Fab CM25"},{"1":"7lzt","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 8b"},{"1":"7lwj","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 3-RBD down conformation"},{"1":"7m6g","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, BG7-15"},{"1":"7lwm","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 1-RBD up conformation"},{"1":"7luz","2":"2021","3":"X-ray diffraction","4":"GQTVTK segment from the Nucleoprotein of SARS-CoV-2, residues 243-248"},{"1":"7m71","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike:5A6 Fab complex I focused refinement"},{"1":"7m42","2":"2021","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 receptor binding domain with the Fab fragments of neutralizing antibodies REGN10985 and REGN10989"},{"1":"7mjm","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to human ACE2 ectodomain"},{"1":"7ndc","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein (all RBD down) in complex with COVOX-159"},{"1":"7n4j","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing human antibody WRAIR-2173."},{"1":"7lm8","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with two cross-neutralizing antibodies CV38-142 and COVA1-16 Fabs isolated from COVID-19 patients"},{"1":"7lzx","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 1c"},{"1":"7lyh","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAWJ9-36-1"},{"1":"7lwo","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 1-RBD up conformation"},{"1":"7lux","2":"2021","3":"X-ray diffraction","4":"AALALL segment from the Nucleoprotein of SARS-CoV-2, residues 217-222, crystal form 2"},{"1":"7lyp","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 1-RBD-up conformation"},{"1":"7lcs","2":"2021","3":"X-ray diffraction","4":"Improved Feline Drugs as SARS-CoV-2 Mpro Inhibitors: Structure-Activity Studies & Micellar Solubilization for Enhanced Bioavailability"},{"1":"7lop","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with antibodies CV05-163 and CR3022"},{"1":"7lcn","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S protein in complex with N-terminal domain antibody DH1050.1"},{"1":"7lfp","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - F2X Entry Library G05 fragment"},{"1":"7lk9","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD-targeting antibody COV107-23 HC + COVD21-C8 LC"},{"1":"7m02","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 17c"},{"1":"7ldx","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - F2X Entry Library E06 fragment"},{"1":"7n1x","2":"2021","3":"Electron Microscopy","4":"Structural basis for enhanced infectivity and immune evasion of SARS-CoV-2 variants"},{"1":"7mzl","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab PDI 210"},{"1":"7m3i","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike protein receptor binding domain in complex with a neutralizing antibody, CV2-75 Fab"},{"1":"7lo4","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 spike receptor-binding domain with a G485R mutation in complex with human ACE2"},{"1":"7lku","2":"2021","3":"X-ray diffraction","4":"1.65 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 3b (deuterated analog of inhibitor 2a)"},{"1":"7m03","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 18c"},{"1":"7lw3","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 nsp16/nsp10 complex in presence of Cap-1 analog (m7GpppAmU) and SAH"},{"1":"7lxz","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2L28 Global Refinement"},{"1":"7ls9","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of neutralizing antibody 1-57 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7lb7","2":"2021","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Joint X-ray/neutron structure of SARS-CoV-2 main protease (3CL Mpro) in complex with Telaprevir"},{"1":"7sy1","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y mutant spike protein ectodomain bound to human ACE2 ectodomain (global refinement)"},{"1":"7n9c","2":"2021","3":"Electron Microscopy","4":"Potent neutralizing nanobodies resist convergent circulating variants of SARS-CoV-2 by targeting novel and conserved epitopes-CovS with NB95"},{"1":"7n1y","2":"2021","3":"Electron Microscopy","4":"Structural basis for enhanced infectivity and immune evasion of SARS-CoV-2 variants"},{"1":"7mzi","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab WCSL 129"},{"1":"7mzm","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab PDI 215"},{"1":"7nd3","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-40 Fab"},{"1":"7n1v","2":"2021","3":"Electron Microscopy","4":"Structural basis for enhanced infectivity and immune evasion of SARS-CoV-2 variants"},{"1":"7n3g","2":"2021","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment C098"},{"1":"7n1q","2":"2021","3":"Electron Microscopy","4":"Structural basis for enhanced infectivity and immune evasion of SARS-CoV-2 variants"},{"1":"7n3i","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain in complex with the human neutralizing antibody Fab fragment C098"},{"1":"7mxp","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody LP5 Fab in complex with SARS-CoV-2 S2P spike"},{"1":"7ng3","2":"2021","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in spacegroup P1."},{"1":"7n0r","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 N protein RNA-binding domain bound to single-domain antibody C2"},{"1":"7nd4","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-88 Fab"},{"1":"7n0h","2":"2021","3":"Electron Microscopy","4":"CryoEM structure of SARS-CoV-2 spike protein (S-6P, 2-up) in complex with sybodies (Sb45)"},{"1":"7n64","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike (2P) in complex with G32R7 Fab (RBD and NTD local reconstruction)"},{"1":"7n4i","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing human antibody WRAIR-2057."},{"1":"7lxx","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2L28 Local Refinement"},{"1":"7lq7","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with antibodies CV503 and COVA1-16"},{"1":"7l6t","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer in Complex with (m7GpppA2m)pUpUpApApA (Cap-1), S-Adenosyl-L-homocysteine (SAH) and two Magnesium (Mg) ions."},{"1":"7lrt","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with neutralizing antibody A23-58.1 that targets the receptor-binding domain"},{"1":"7lxw","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2X333 Local Refinement"},{"1":"7n3e","2":"2021","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment C032"},{"1":"7lcr","2":"2021","3":"X-ray diffraction","4":"Improved Feline Drugs as SARS-CoV-2 Mpro Inhibitors: Structure-Activity Studies & Micellar Solubilization for Enhanced Bioavailability"},{"1":"7lzw","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 20b (deuterated analog of 19b)"},{"1":"7lwl","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein (S-GSAS-D614G-delFV) in the 3-RBD down conformation"},{"1":"7lwq","2":"2021","3":"Electron Microscopy","4":"Mink Cluster 5-associated SARS-CoV-2 spike protein(S-GSAS-D614G-delFV) missing the S1 subunit and SD2 subdomain of one protomer"},{"1":"7lgd","2":"2021","3":"X-ray diffraction","4":"HLA-B*07:02 in complex with SARS-CoV-2 nucleocapsid peptide N105-113"},{"1":"7ly0","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S/S2M11/S2M28 Local Refinement"},{"1":"7mjl","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to Fab ab1 (focused refinement of RBD and Fab ab1)"},{"1":"7m8k","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of Brazil (P.1) SARS-CoV-2 spike glycoprotein variant in the prefusion state (1 RBD up)"},{"1":"7m6i","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, BG1-24"},{"1":"7lbs","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-24"},{"1":"7lqv","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 4-8 Fab in complex with SARS-CoV-2 S2P spike"},{"1":"7lct","2":"2021","3":"X-ray diffraction","4":"Improved Feline Drugs as SARS-CoV-2 Mpro Inhibitors: Structure-Activity Studies & Micellar Solubilization for Enhanced Bioavailability"},{"1":"7m53","2":"2021","3":"X-ray diffraction","4":"B6 Fab fragment bound to the SARS-CoV/SARS-CoV-2 spike stem helix peptide"},{"1":"7lzv","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 3CL protease in complex with inhibitor 19b"},{"1":"7lyq","2":"2021","3":"Electron Microscopy","4":"South African (B.1.351) SARS-CoV-2 spike protein variant (S-GSAS-B.1.351) in the 1-RBD-up conformation"},{"1":"7lwv","2":"2021","3":"Electron Microscopy","4":"UK (B.1.1.7) SARS-CoV-2 spike protein variant (S-GSAS-B.1.1.7) in the 1-RBD-up conformation"},{"1":"7los","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 papain-like protease (PLpro) bound to inhibitor XR8-65"},{"1":"7mfu","2":"2021","3":"X-ray diffraction","4":"Crystal structure of synthetic nanobody (Sb14+Sb68) complexes with SARS-CoV-2 receptor binding domain"},{"1":"7vxa","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Kappa variant spike protein in complex with ACE2, state C2a"},{"1":"7oyg","2":"2021","3":"Electron Microscopy","4":"Dimeric form of SARS-CoV-2 RNA-dependent RNA polymerase"},{"1":"7ora","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the T478K mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-45 and COVOX-253 Fabs"},{"1":"7vxc","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Kappa variant spike protein in C3 state"},{"1":"7vxb","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Kappa variant spike protein in C2b state"},{"1":"7vxk","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with ACE2, Beta variant, C2A state"},{"1":"7vxd","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with ACE2, Beta variant, C1 state"},{"1":"7vxi","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Kappa variant spike protein in transition state"},{"1":"7vx1","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Beta variant spike protein in open state"},{"1":"7sbk","2":"2021","3":"Electron Microscopy","4":"Closed state of pre-fusion SARS-CoV-2 Delta variant spike protein"},{"1":"7sbr","2":"2021","3":"Electron Microscopy","4":"One RBD-up 2 of pre-fusion SARS-CoV-2 Kappa variant spike protein"},{"1":"7sxy","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G mutant spike protein ectodomain bound to human ACE2 ectodomain (focused refinement of RBD and ACE2)"},{"1":"7n44","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (2019-NCoV) main protease in complex with 5-(3-{3-chloro-5-[(5-methyl-1,3-thiazol-4-yl)methoxy]phenyl}-2-oxo-2H-[1,3'-bipyridin]-5-yl)pyrimidine-2,4(1H,3H)-dione (compound 13)"},{"1":"7msq","2":"2021","3":"X-ray diffraction","4":"Complex between the Fab arm of AB-3467 and the SARS-CoV-2 receptor binding domain (RBD)"},{"1":"7tb4","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the spike of SARS-CoV-2 Omicron variant of concern"},{"1":"7mkl","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing Fab SARS2-38 (three down conformation)"},{"1":"7mgr","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in complex with nsp8/9 substrate peptide"},{"1":"7neg","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the N501Y mutant receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-269 Fab"},{"1":"7n06","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp15 endoribonuclease post-cleavage state"},{"1":"7o7y","2":"2021","3":"Electron Microscopy","4":"Rabbit 80S ribosome stalled close to the mutated SARS-CoV-2 slippery site by a pseudoknot (high resolution)"},{"1":"7n8i","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S (B.1.429 / epsilon variant) + S2M11 + S2L20 (Local Refinement of the NTD/S2L20)"},{"1":"7n9e","2":"2021","3":"Electron Microscopy","4":"Potent neutralizing nanobodies resist convergent circulating variants of SARS-CoV-2 by targeting novel and conserved epitopes-CovS with NB34"},{"1":"7n3k","2":"2021","3":"X-ray diffraction","4":"Oridonin-bound SARS-CoV-2 Nsp9"},{"1":"7m6d","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 RBD in complex with neutralizing antibodies BG4-25 and CR3022"},{"1":"7mej","2":"2021","3":"Electron Microscopy","4":"CryoEM structure of SARS-CoV-2 RBD in complex with nanobodies Nb21 and Nb36"},{"1":"7m8t","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*11:01 in complex with NSASFSTFK, an 9-mer epitope from SARS-CoV-2 spike (S370-378)"},{"1":"7nw2","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-47"},{"1":"7ng6","2":"2021","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in spacegroup P1 in absence of DTT."},{"1":"7n0b","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 nsp10-nsp14 (WT)-RNA complex"},{"1":"7sbo","2":"2021","3":"Electron Microscopy","4":"One RBD-up 2 of pre-fusion SARS-CoV-2 Delta variant spike protein"},{"1":"7nts","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 Main Protease with oxidized C145"},{"1":"7nta","2":"2021","3":"Electron Microscopy","4":"Trimeric SARS-CoV-2 spike ectodomain in complex with biliverdin (one RBD erect)"},{"1":"7n8c","2":"2021","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Joint X-ray/neutron structure of SARS-CoV-2 main protease (Mpro) in complex with Mcule5948770040"},{"1":"7n4m","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing human antibody WRAIR-2151."},{"1":"7n0g","2":"2021","3":"Electron Microscopy","4":"CryoEm structure of SARS-CoV-2 spike protein (S-6P, 1-up) in complex with sybodies (Sb45)"},{"1":"7sbt","2":"2021","3":"Electron Microscopy","4":"One RBD-up 2 of pre-fusion SARS-CoV-2 Gamma variant spike protein"},{"1":"7sy7","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K,K417T mutant spike protein ectodomain bound to human ACE2 ectodomain (global refinement)"},{"1":"7sxx","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G mutant spike protein ectodomain bound to human ACE2 ectodomain (global refinement)"},{"1":"7spp","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain in complex with VNAR 2C02"},{"1":"7soc","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S RBD B.1.617.1 kappa variant S309 Local Refinement"},{"1":"7sxv","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K,K417N mutant spike protein ectodomain"},{"1":"7nda","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein (all RBD down) in complex with COVOX-253H55L Fab"},{"1":"7n89","2":"2021","3":"X-ray diffraction","4":"Room-temperature X-ray structure of SARS-CoV-2 main protease C145A mutant in complex with substrate Ac-SAVLQSGF-CONH2"},{"1":"7nng","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 helicase in complex with Z2327226104"},{"1":"7t01","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S-RBD + Fab 54042-4"},{"1":"7sxu","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 D614G,N501Y,E484K mutant spike protein ectodomain"},{"1":"7n9b","2":"2021","3":"Electron Microscopy","4":"Potent neutralizing nanobodies resist convergent circulating variants of SARS-CoV-2 by targeting novel and conserved epitopes-CovS with NB21"},{"1":"7n3h","2":"2021","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment C099"},{"1":"7n0d","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the tetrameric form of SARS-CoV-2 nsp10-nsp14 (E191A)-RNA complex"},{"1":"7nd9","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein (one RBD up) in complex with COVOX-253H55L Fab"},{"1":"7n3f","2":"2021","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment C080"},{"1":"7n0i","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 N protein C-terminal domain bound to single-domain antibody E2"},{"1":"7mzn","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab PDI 231"},{"1":"7m8s","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*02:01 in complex with KLNDLCFTNV, an 10-mer epitope from SARS-CoV-2 Spike (S386-395)"},{"1":"7nio","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 helicase APO form"},{"1":"7ndb","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-253H165L Fab"},{"1":"7n1u","2":"2021","3":"Electron Microscopy","4":"Structural basis for enhanced infectivity and immune evasion of SARS-CoV-2 variants"},{"1":"7nf5","2":"2021","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in spacegroup C2."},{"1":"7nd6","2":"2021","3":"Electron Microscopy","4":"EM structure of SARS-CoV-2 Spike glycoprotein in complex with COVOX-40 Fab"},{"1":"7mzh","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab WCSL 119"},{"1":"7my8","2":"2021","3":"Solution NMR","4":"Fusion Peptide of SARS-CoV-2 Spike Rearranges into a Wedge Inserted in Bilayered Micelles"},{"1":"7mtc","2":"2021","3":"Electron Microscopy","4":"Structure of freshly purified SARS-CoV-2 S2P spike at pH 7.4"},{"1":"7mte","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 S2P spike at pH 7.4 refolded by low-pH treatment"},{"1":"7n8h","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S (B.1.429 / epsilon variant) + S2M11 + S2L20 Global Refinement"},{"1":"7n1w","2":"2021","3":"Electron Microscopy","4":"Structural basis for enhanced infectivity and immune evasion of SARS-CoV-2 variants"},{"1":"7mzf","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 receptor binding domain bound to Fab PDI 37"},{"1":"7mtd","2":"2021","3":"Electron Microscopy","4":"Structure of aged SARS-CoV-2 S2P spike at pH 7.4"},{"1":"7mjn","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 N501Y mutant spike protein ectodomain bound to human ACE2 ectodomain (focused refinement of RBD and ACE2)"},{"1":"7aha","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Maleate."},{"1":"7bh9","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RBD-62 in complex with ACE2 peptidase domain"},{"1":"7baj","2":"2021","3":"X-ray diffraction","4":"Crystal structure of ligand-free SARS-CoV-2 main protease"},{"1":"7arf","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to thioglucose."},{"1":"7bb2","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the main protease (3CLpro/Mpro) of SARS-CoV-2 at 1.6A resolution (spacegroup P2(1)2(1)2(1))"},{"1":"7akd","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the 47D11 neutralizing antibody Fab fragment"},{"1":"7ali","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the main protease (3CLpro/Mpro) of SARS-CoV-2 at 1.65A resolution (spacegroup P2(1))."},{"1":"7aph","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Tofogliflozin."},{"1":"7ad1","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of a prefusion stabilized SARS-CoV-2 Spike (D614N, R682S, R685G, A892P, A942P and V987P)(One up trimer)"},{"1":"7adw","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to 2,4'-Dimethylpropiophenone."},{"1":"7aeg","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in a covalent complex with SDZ 224015 derivative, compound 5"},{"1":"7ak4","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Tretazicar."},{"1":"7b3o","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with STE90-C11 Fab"},{"1":"7bep","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in a ternary complex with COVOX-384 and S309 Fabs"},{"1":"7abu","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to RS102895"},{"1":"7b83","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to pyrithione zinc"},{"1":"7aol","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Climbazole"},{"1":"7ans","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Adrafinil."},{"1":"7bek","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-158 Fab (crystal form 2)"},{"1":"7aqj","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Triglycidyl isocyanurate."},{"1":"7aqi","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Ifenprodil"},{"1":"7bem","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-269 scFv"},{"1":"7bf5","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 macrodomain in complex with ADP-ribose-phosphate (ADP-ribose-2'-phosphate, ADPRP)"},{"1":"7bgp","2":"2021","3":"X-ray diffraction","4":"Crystal structure of MG-132 covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in absence of DTT."},{"1":"7b3e","2":"2021","3":"X-ray diffraction","4":"Crystal structure of myricetin covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2"},{"1":"7beh","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-316 Fab"},{"1":"7bak","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease treated with ebselen"},{"1":"7aws","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to TH-302."},{"1":"7a4n","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of a prefusion stabilized SARS-CoV-2 Spike (D614N, R682S, R685G, A892P, A942P and V987P)(S-closed trimer)"},{"1":"7bf4","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 macrodomain in complex with GMP"},{"1":"7awu","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to LSN2463359."},{"1":"7axm","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Pelitinib"},{"1":"7l5d","2":"2021","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 Main Protease in complex with demethylated analog of masitinib"},{"1":"7axo","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to AR-42."},{"1":"7aeh","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in a covalent complex with a pyridine derivative of ABT-957, compound 1"},{"1":"7avd","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to SEN1269 ligand"},{"1":"7aku","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Calpeptin."},{"1":"7bej","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with COVOX-158 Fab (crystal form 1)"},{"1":"7aqe","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to UNC-2327"},{"1":"7awr","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Tegafur"},{"1":"7a1u","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Fusidic Acid."},{"1":"6yvf","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to AZD6482."},{"1":"6xdc","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 ORF3a"},{"1":"7beo","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in a ternary complex with COVOX-253H55L and COVOX-75 Fabs"},{"1":"7ax6","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Glutathione isopropyl ester"},{"1":"6zco","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of C-terminal Dimerization Domain of Nucleocapsid Phosphoprotein from SARS-CoV-2, crystal form II"},{"1":"6woj","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 macrodomain (NSP3) in complex with ADP-ribose"},{"1":"6ynq","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to 2-Methyl-1-tetralone."},{"1":"6xkm","2":"2021","3":"X-ray diffraction","4":"Room Temperature Structure of SARS-CoV-2 NSP10/NSP16 Methyltransferase in a Complex with SAM Determined by Fixed-Target Serial Crystallography"},{"1":"7l2e","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 4-18 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7kn7","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain complexed with nanobody VHH W and antibody Fab CC12.3"},{"1":"7klw","2":"2021","3":"X-ray diffraction","4":"Crystal structure of synthetic nanobody (Sb45+Sb68) complexes with SARS-CoV-2 receptor binding domain"},{"1":"7kgt","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201 in complex with SARS-CoV-2 N226-234"},{"1":"7kqp","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose (P43 crystal form)"},{"1":"7keb","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1RBD up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub-classification)"},{"1":"7kml","2":"2021","3":"Electron Microscopy","4":"cryo-EM structure of SARS-CoV-2 spike in complex with Fab 15033-7, three RBDs bound"},{"1":"7kgr","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201in complex with SARS-CoV-2 N159-167"},{"1":"7ey5","2":"2021","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6PV2 in complex with BD-771 Fab and BD-821 Fab"},{"1":"7exm","2":"2021","3":"X-ray diffraction","4":"The N-terminal crystal structure of SARS-CoV-2 NSP2"},{"1":"7fae","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P36-5D2(state2)"},{"1":"7faf","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P36-5D2 (state1)"},{"1":"7km5","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD complexed with Nanosota-1"},{"1":"7kph","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease in mature form"},{"1":"7kfi","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - apo structure"},{"1":"7krp","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 backtracked complex complex bound to nsp13 helicase - BTC (local refinement)"},{"1":"7kn5","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain complexed with nanobodies VHH E and U"},{"1":"7ksg","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with nanobodies E"},{"1":"7kx5","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with noncovalent inhibitor Jun8-76-3A"},{"1":"7ke9","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD-up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub-classification)"},{"1":"7kj4","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with three ACE2 Bound"},{"1":"7l4z","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike RBD in complex with cyclic peptide"},{"1":"7krr","2":"2021","3":"Electron Microscopy","4":"Structural impact on SARS-CoV-2 spike protein by D614G substitution"},{"1":"7krq","2":"2021","3":"Electron Microscopy","4":"Structural impact on SARS-CoV-2 spike protein by D614G substitution"},{"1":"7kn6","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain complexed with nanobody VHH V and antibody Fab CC12.3"},{"1":"7kmk","2":"2021","3":"Electron Microscopy","4":"cryo-EM structure of SARS-CoV-2 spike in complex with Fab 15033-7, two RBDs bound"},{"1":"7kjr","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 ORF3a"},{"1":"7k9k","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike RBD in complex with neutralizing Fab 2H04 (local refinement)"},{"1":"7kri","2":"2021","3":"X-ray diffraction","4":"FR6-bound SARS-CoV-2 Nsp9 RNA-replicase"},{"1":"7l0n","2":"2021","3":"X-ray diffraction","4":"Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity"},{"1":"7kgs","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201 in complex with SARS-CoV-2 N138-146"},{"1":"7kro","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 backtracked complex complex bound to nsp13 helicase - nsp13(2)-BTC"},{"1":"7kgp","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201 in complex with SARS-CoV-2 N316-324"},{"1":"7kdi","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer fully cleaved by furin without the P986-P987 stabilizing mutations (S-RRAR-D614G)"},{"1":"7l57","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab 2-15"},{"1":"7kgo","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201in complex with SARS-CoV-2 N351-359"},{"1":"7l06","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 2P S ectodomain bound to two copies of domain-swapped antibody 2G12"},{"1":"7k9i","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike RBD in complex with neutralizing Fab 2B04 (local refinement)"},{"1":"7l2d","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 1-87 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7kqw","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain (C2 crystal form, methylated)"},{"1":"7kj3","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with two ACE2 Bound"},{"1":"7ke7","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3-RBD-down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G Sub-Classification)"},{"1":"7l2c","2":"2021","3":"X-ray diffraction","4":"Crystallographic structure of neutralizing antibody 2-51 in complex with SARS-CoV-2 spike N-terminal domain (NTD)"},{"1":"7krn","2":"2021","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 backtracked complex bound to nsp13 helicase - nsp13(1)-BTC"},{"1":"7ke4","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G Sub-class)"},{"1":"7k9z","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with the Fab fragments of neutralizing antibodies 298 and 52"},{"1":"7kea","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD-up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub classification)"},{"1":"7k9j","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing Fab 2H04 (three down conformation)"},{"1":"7kmg","2":"2021","3":"X-ray diffraction","4":"LY-CoV555 neutralizing antibody against SARS-CoV-2"},{"1":"7l2f","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 5-24 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7l09","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 2P S ectodomain bound domain-swapped antibody 2G12 from masked 3D refinement"},{"1":"7kj2","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with one ACE2 Bound"},{"1":"7kmi","2":"2021","3":"X-ray diffraction","4":"LY-CoV481 neutralizing antibody against SARS-CoV-2"},{"1":"7dcd","2":"2021","3":"X-ray diffraction","4":"Nonstructural protein 7 and 8 complex of SARS-CoV-2"},{"1":"7cwl","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein and P17 fab complex with one RBD in close state"},{"1":"7edj","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-UK variant (B.1.1.7) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain"},{"1":"7egq","2":"2021","3":"Electron Microscopy","4":"Co-transcriptional capping machineries in SARS-CoV-2 RTC: Coupling of N7-methyltransferase and 3'-5' exoribonuclease with polymerase reveals mechanisms for capping and proofreading"},{"1":"7d7k","2":"2021","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 papain-like protease in apo form"},{"1":"7cyp","2":"2021","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 spike trimer with its neutralizing antibody HB27"},{"1":"7d00","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with FabP5A-1B8"},{"1":"7cr5","2":"2021","3":"X-ray diffraction","4":"Complex structure of a human monoclonal antibody with SARS-CoV-2 nucleocapsid protein NTD"},{"1":"7d0c","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-3A1"},{"1":"7cwo","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein RBD and P17 fab complex"},{"1":"7d4g","2":"2021","3":"Electron Microscopy","4":"A proof of concept for neutralizing antibody-guided vaccine design against SARS-CoV-2"},{"1":"7dx3","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 1 (1 up RBD and no PD bound)"},{"1":"7eb5","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, two RBD-up conformation 2"},{"1":"7e3o","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody nCoV617"},{"1":"7ey4","2":"2021","3":"Electron Microscopy","4":"Local CryoEM of the SARS-CoV-2 S6PV2 in complex with BD-667"},{"1":"7ey0","2":"2021","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6PV2 in complex with BD-813 Fab and BD-744 Fab"},{"1":"7ekg","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Beta variant spike receptor-binding domain complexed with human ACE2"},{"1":"7eya","2":"2021","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6PV2 in complex with BD-804 Fab"},{"1":"7eq4","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of the N-terminus of Nonstructural protein 1 from SARS-CoV-2"},{"1":"7f63","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with a neutralizing antibody chAb-45 (Focused refinement of S-RBD and chAb-45 region)"},{"1":"7fet","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 B.1.1.7 Spike Glycoprotein trimer"},{"1":"7eam","2":"2021","3":"X-ray diffraction","4":"immune complex of SARS-CoV-2 RBD and cross-neutralizing antibody 7D6"},{"1":"7dk1","2":"2021","3":"X-ray diffraction","4":"Crystal structure of Zinc bound SARS-CoV-2 main protease"},{"1":"7d6h","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 papain-like protease (PLPro) C112S mutant"},{"1":"7czp","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P2B-1A1"},{"1":"7cwn","2":"2021","3":"Electron Microscopy","4":"P17-H014 Fab cocktail in complex with SARS-CoV-2 spike protein"},{"1":"7df4","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S-ACE2 complex"},{"1":"7czv","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B6_3B"},{"1":"7ct5","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with T-ACE2"},{"1":"7cyq","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of an extended SARS-CoV-2 replication and transcription complex reveals an intermediate state in cap synthesis"},{"1":"7dhx","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD binding to pangolin ACE2"},{"1":"7d03","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with FabP5A-2G7"},{"1":"7cws","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Proteins Trimer in Complex with FC05 and H014 Fabs Cocktail"},{"1":"7dhg","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Orf9b complex with human TOM70"},{"1":"7dpv","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with 7-O-methyl-dihydromyricetin"},{"1":"7czw","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-2G7"},{"1":"7dk3","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer, S-open"},{"1":"7czs","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B8_3B"},{"1":"7cuu","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with MG132"},{"1":"7dmu","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike receptor-binding domain complexed with high affinity ACE2 mutant 3N39"},{"1":"7eb0","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, one RBD-up conformation 2"},{"1":"7e3k","2":"2021","3":"Electron Microscopy","4":"Ultrapotent SARS-CoV-2 neutralizing antibodies with protective efficacy against newly emerged mutational variants"},{"1":"7d0d","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-3C12_2B"},{"1":"7df3","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer, S-closed"},{"1":"7czr","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B8_2B"},{"1":"7cwb","2":"2021","3":"X-ray diffraction","4":"Ambient-Temperature Serial Femtosecond X-ray Crystal structure of SARS-CoV-2 Main Protease at 1.9 A Resolution (C121)"},{"1":"7e23","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the CA521 neutralizing antibody Fab (focused refinement on Fab-RBD)"},{"1":"7e9t","2":"2021","3":"Electron Microscopy","4":"Nanometer resolution in situ structure of SARS-CoV-2 post-fusion spike"},{"1":"7czt","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-2G9"},{"1":"7dwy","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in the locked conformation"},{"1":"7ekh","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike receptor-binding domain Y453F mutation complexed with human ACE2"},{"1":"7eh5","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-D614G variant in complex with neutralizing antibodies, RBD-chAb15 and RBD-chAb45"},{"1":"7e86","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-508 Fab"},{"1":"7dwz","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in the active conformation"},{"1":"7czu","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-1B6_2B"},{"1":"7ein","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 main proteinase complex with microbial metabolite leupeptin"},{"1":"7efp","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike receptor-binding domain in complex with high affinity ACE2 mutant (S19W,N330Y)"},{"1":"7d2z","2":"2021","3":"X-ray diffraction","4":"Structure of sybody SR31 in complex with the SARS-CoV-2 S Receptor Binding domain (RBD)"},{"1":"7f62","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with a neutralizing antibody chAb-25 (Focused refinement of S-RBD and chAb-25 region)"},{"1":"7kvl","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - FMAX Library E01 fragment"},{"1":"7l58","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab H4"},{"1":"7ks9","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of prefusion SARS-CoV-2 spike glycoprotein in complex with 910-30 Fab"},{"1":"7dx6","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 bound with PD of ACE2 in the conformation 3 (2 up RBD and 1 PD bound)"},{"1":"7eu2","2":"2021","3":"X-ray diffraction","4":"Complex structure of HLA0201 with recognizing SARS-CoV-2 epitope S1"},{"1":"7eke","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike receptor-binding domain F486L mutation complexed with human ACE2"},{"1":"7edg","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-UK variant (B.1.1.7), one RBD-up conformation 2"},{"1":"7e3j","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD binding to dog ACE2"},{"1":"7e8f","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 NTD in complex with N9 Fab"},{"1":"7eiz","2":"2021","3":"Electron Microscopy","4":"Coupling of N7-methyltransferase and 3'-5' exoribonuclease with SARS-CoV-2 polymerase reveals mechanisms for capping and proofreading"},{"1":"7ezv","2":"2021","3":"Electron Microscopy","4":"local CryoEM structure of the SARS-CoV-2 S6PV2 in complex with BD-812 Fab and BD-836 Fab"},{"1":"7edi","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-UK variant (B.1.1.7), two RBD-up conformation"},{"1":"7e8m","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P2C-1F11 with mutated RBD"},{"1":"7k9h","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing Fab 2B04 (one up, two down conformation)"},{"1":"7e3c","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the Ab1 neutralizing antibody (focused refinement on Fab-RBD)"},{"1":"7ctt","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of Favipiravir bound to replicating polymerase complex of SARS-CoV-2 in the pre-catalytic state."},{"1":"7czz","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-2F11_3B"},{"1":"7djr","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (no ligand)"},{"1":"7cwc","2":"2021","3":"X-ray diffraction","4":"Ambient-Temperature Serial Femtosecond X-ray Crystal structure of SARS-CoV-2 Main Protease at 2.1 A Resolution (P212121)"},{"1":"7deo","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with a neutralizing antibody scFv"},{"1":"7det","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with a neutralizing antibody scFv"},{"1":"7czy","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-2F11_2B"},{"1":"7klh","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 RBD in complex with Fab 15033-7"},{"1":"7kj5","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein, prefusion with one RBD up conformation"},{"1":"7kqo","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain (P43 crystal form)"},{"1":"7kec","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD-up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G Sub-Classification)"},{"1":"7kgk","2":"2021","3":"X-ray diffraction","4":"Crystal structure of synthetic nanobody (Sb16) complexes with SARS-CoV-2 receptor binding domain"},{"1":"7ke6","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub-classification)"},{"1":"7kqb","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein:Fab 5A6 complex I"},{"1":"7klg","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 RBD in complex with Fab 15033"},{"1":"7l0d","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML188"},{"1":"7ke8","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G sub-classification)"},{"1":"7kxb","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Nsp3 Macrodomain complex with PARG329"},{"1":"7e5s","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer with four-antibody cocktail complex"},{"1":"7dx0","2":"2021","3":"Electron Microscopy","4":"Trypsin-digested S protein of SARS-CoV-2"},{"1":"7f4w","2":"2021","3":"X-ray diffraction","4":"Complex structure of HLA2402 with recognizing SARS-CoV-2 epitope pep4"},{"1":"7e7x","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Spike Protein N terminal domain in Complex with N11 Fab"},{"1":"7eaz","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, one RBD-up conformation 1"},{"1":"7dpm","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with MW06 Fab"},{"1":"7eb4","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike D614G variant, two RBD-up conformation 1"},{"1":"7e5y","2":"2021","3":"X-ray diffraction","4":"Molecular basis for neutralizing antibody 2B11 targeting SARS-CoV-2 RBD"},{"1":"7f5f","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 ORF8 S84"},{"1":"7dpu","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with 7-O-methyl-myricetin"},{"1":"7d3i","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with MI-23"},{"1":"7d30","2":"2021","3":"X-ray diffraction","4":"Structure of sybody MR17-SR31 fusion in complex with the SARS-CoV-2 S Receptor Binding domain (RBD)"},{"1":"7edf","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-UK variant (B.1.1.7), one RBD-up conformation 1"},{"1":"7d7l","2":"2021","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 papain-like protease in complex with YM155"},{"1":"7deu","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with a neutralizing antibody scFv"},{"1":"7e5o","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with antibody NT-193"},{"1":"7d0b","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P5A-3C12_1B"},{"1":"7ap6","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to MUT056399."},{"1":"7alh","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the main protease (3CLpro/Mpro) of SARS-CoV-2 at 1.65A resolution (spacegroup C2)."},{"1":"7af0","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Ipidacrine."},{"1":"7aga","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to AT7519"},{"1":"6ywk","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 (Covid-19) NSP3 macrodomain in complex with HEPES"},{"1":"6yun","2":"2021","3":"X-ray diffraction","4":"1.45 Angstrom Resolution Crystal Structure of C-terminal Dimerization Domain of Nucleocapsid Phosphoprotein from SARS-CoV-2"},{"1":"7jq4","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI7"},{"1":"7lwt","2":"2021","3":"Electron Microscopy","4":"UK (B.1.1.7) SARS-CoV-2 spike protein variant (S-GSAS-B.1.1.7) in the 1-RBD-up conformation"},{"1":"7cdj","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 antibody P2C-1A3 with RBD"},{"1":"7ben","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the receptor binding domain of SARS-CoV-2 Spike glycoprotein in a ternary complex with COVOX-253 and COVOX-75 Fabs"},{"1":"7bno","2":"2021","3":"Electron Microscopy","4":"Open conformation of D614G SARS-CoV-2 spike with 2 Erect RBDs"},{"1":"7ay7","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to Isofloxythepin"},{"1":"7kdh","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RBD up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS)"},{"1":"7l02","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 2P S ectodomain bound to one copy of domain-swapped antibody 2G12"},{"1":"7kr0","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain (C2 crystal form, 100 K)"},{"1":"7l56","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab 2-43"},{"1":"6ywm","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 (Covid-19) NSP3 macrodomain in complex with MES"},{"1":"6wqd","2":"2021","3":"X-ray diffraction","4":"The 1.95 A Crystal Structure of the Co-factor Complex of NSP7 and the C-terminal Domain of NSP8 from SARS-CoV-2"},{"1":"7kxk","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with Fab 15033-7, 2-\"up\"-1-\"down\" conformation"},{"1":"7kmh","2":"2021","3":"X-ray diffraction","4":"LY-CoV488 neutralizing antibody against SARS-CoV-2"},{"1":"7kgq","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of HLA-A*0201in complex with SARS-CoV-2 N222-230"},{"1":"7l1f","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RdRp in complex with 4 Remdesivir monophosphate"},{"1":"7kdl","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD up Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G)"},{"1":"7czq","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with P2B-1A10"},{"1":"7ean","2":"2021","3":"X-ray diffraction","4":"immune complex of SARS-CoV-2 RBD and cross-neutralizing antibody 6D6"},{"1":"7e8c","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S-6P in complex with 9 Fabs"},{"1":"7fem","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 B.1.1.7 S-ACE2 complex"},{"1":"7bnm","2":"2021","3":"Electron Microscopy","4":"Closed conformation of D614G SARS-CoV-2 spike protein"},{"1":"7amj","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Main Protease bound to PD 168568."},{"1":"6zsl","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 helicase at 1.94 Angstrom resolution"},{"1":"6ywl","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 (Covid-19) NSP3 macrodomain in complex with ADP-ribose"},{"1":"7dx1","2":"2021","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 D614G mutant"},{"1":"7e35","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 papain-like protease (PLPro) C112S mutant bound to compound S43"},{"1":"7e7b","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 furin site mutant S-Trimer from a subunit vaccine candidate"},{"1":"7ekf","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Alpha variant spike receptor-binding domain complexed with human ACE2"},{"1":"7e5r","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer with three-antibody cocktail complex"},{"1":"7dpp","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with myricetin"},{"1":"7edh","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-UK variant (B.1.1.7), one RBD-up conformation 3"},{"1":"7efr","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike receptor-binding domain in complex with high affinity ACE2 mutant (T27W,N330Y)"},{"1":"7kdk","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 3 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS-D614G)"},{"1":"7e39","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the Ab4 neutralizing antibody (State 3)"},{"1":"7dte","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RdRP catalytic complex with T33-1 RNA"},{"1":"7kdj","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 D614G 1-RBD-up Spike Protein Trimer fully cleaved by furin without the P986-P987 stabilizing mutations (S-RRAR-D614G)"},{"1":"7kgj","2":"2021","3":"X-ray diffraction","4":"Crystal structure of synthetic nanobody (Sb45) complexes with SARS-CoV-2 receptor binding domain"},{"1":"7kdg","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 RBD down Spike Protein Trimer without the P986-P987 stabilizing mutations (S-GSAS)"},{"1":"7krs","2":"2021","3":"Electron Microscopy","4":"Structural impact on SARS-CoV-2 spike protein by D614G substitution"},{"1":"7kr1","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain (C2 crystal form, 310 K)"},{"1":"7kxj","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in complex with Fab 15033-7, 3-\"up\", asymmetric"},{"1":"7kqe","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike glycoprotein:Fab 3D11 complex"},{"1":"7kvr","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Main protease immature form - FMAX Library E09 fragment"},{"1":"7vk1","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 1.93 A resolution-5"},{"1":"7vjw","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.20 A resolution-10"},{"1":"7vne","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with a human single domain antibody n3113.1 (UUU-state)"},{"1":"7vnb","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with a human single domain antibody n3113"},{"1":"7vk2","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.0 A resolution -9"},{"1":"7vk3","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.10 A resolution-2"},{"1":"7vk6","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.25 A resolution-13"},{"1":"7vk4","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.10 A resolution-3"},{"1":"7rmt","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with HL-3-70"},{"1":"7r8n","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with neutralizing antibody C051"},{"1":"7vph","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the C-terminal tail of SARS-CoV-2 Orf6 complex with human nucleoporin pair Rae1-Nup98"},{"1":"7vnd","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with a human single domain antibody n3113 (UUD-state, state 2)"},{"1":"7vk5","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.10 A resolution-8"},{"1":"7vk8","2":"2021","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro at 2.4 A Resolution"},{"1":"7vnc","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with a human single domain antibody n3113 (UDD-state, state 1)"},{"1":"7rr0","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 receptor binding domain bound to Fab PDI 222"},{"1":"7r95","2":"2021","3":"Solution NMR","4":"Membrane bound structure of HR1 domain of SARS-CoV-2 spike protein"},{"1":"7vk7","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.4 A resolution-11"},{"1":"7rfu","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 main protease in complex with a covalent inhibitor"},{"1":"7ral","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S bound to S2X259 Fab (local refinement of the RBD/S2X259 variable domains)"},{"1":"7r6x","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 spike receptor-binding domain (RBD) in complex with S2E12 Fab, S309 Fab, and S304 Fab"},{"1":"7vjz","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 1.90 A resolution-7"},{"1":"7rku","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 receptor binding domain in complex with the human neutralizing antibody Fab fragment, C022"},{"1":"7rme","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with HL-3-52"},{"1":"7rfw","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 main protease in complex with a covalent inhibitor"},{"1":"7rmb","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with HL-3-78"},{"1":"7r7h","2":"2021","3":"X-ray diffraction","4":"Peptidomimetic nitrile warheads as SARS-CoV-2 3CL protease inhibitors"},{"1":"7r6w","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 spike receptor-binding domain (RBD) in complex with S2X35 Fab and S309 Fab"},{"1":"7r7n","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2D106 neutralizing antibody Fab fragment (local refinement of the RBD and S2D106)"},{"1":"7rm2","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with Mcule-CSR-494190-S1"},{"1":"7ra8","2":"2021","3":"Electron Microscopy","4":"SARS-CoV-2 S glycoprotein in complex with S2X259 Fab"},{"1":"7rkv","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with neutralizing antibody C118 (State 1)"},{"1":"7rnh","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with HL-3-45"},{"1":"7rw2","2":"2021","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed neutralizing antibody 5-7 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"7rpv","2":"2021","3":"X-ray diffraction","4":"Crystal structure of affinity-enhancing and catalytically inactive ACE2 in complex with SARS-CoV-2 RBD"},{"1":"7rn4","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with HL-3-69"},{"1":"7rls","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with HL-3-68"},{"1":"7rtd","2":"2021","3":"X-ray diffraction","4":"SARS-CoV-2 Spike-derived peptide S269-277 (YLQPRTFLL) presented by HLA-A*02:01"},{"1":"7rfs","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 main protease in complex with a covalent inhibitor"},{"1":"7r8o","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with neutralizing antibody C548"},{"1":"7rc0","2":"2021","3":"X-ray diffraction","4":"X-ray Structure of SARS-CoV-2 main protease covalently modified by compound GRL-091-20"},{"1":"7r8m","2":"2021","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with neutralizing antibody C032"},{"1":"7r98","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 N protein RNA-binding domain bound to single-domain antibody B6"},{"1":"7rn1","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor Jun9-62-2R"},{"1":"7r8l","2":"2021","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 RBD in complex with neutralizing antibody C099 and CR3022"},{"1":"7rtr","2":"2021","3":"X-ray diffraction","4":"YLQ-SG3 TCR in complex with SARS-CoV-2 Spike-derived peptide S269-277 (YLQPRTFLL) presented by HLA-A*02:01"},{"1":"7rnk","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with HL-3-71"},{"1":"7vk0","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.10 A resolution-6"},{"1":"7rnj","2":"2021","3":"X-ray diffraction","4":"S2P6 Fab fragment bound to the SARS-CoV/SARS-CoV-2 spike stem helix peptide"},{"1":"7rbz","2":"2021","3":"X-ray diffraction","4":"X-ray Structure of SARS-CoV-2 main protease covalently modified by compound GRL-017-20"},{"1":"7vjx","2":"2021","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Mpro at 2.20 A resolution-12"},{"1":"7rmz","2":"2021","3":"X-ray diffraction","4":"Room temperature X-ray structure of SARS-CoV-2 main protease (Mpro) in complex with HL-3-63"},{"1":"7rn0","2":"2021","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor Jun9-57-3R"},{"1":"7rfr","2":"2021","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 main protease in complex with a covalent inhibitor"},{"1":"7tpr","2":"2021","3":"Electron Microscopy","4":"Camel nanobodies 7A3 and 8A2 broadly neutralize SARS-CoV-2 variants"},{"1":"5rev","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0103072"},{"1":"5rfn","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102868"},{"1":"5rff","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102704"},{"1":"5rgj","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1401276297 (Mpro-x0425)"},{"1":"5rel","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102340"},{"1":"5rfl","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102389"},{"1":"5rgg","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434890 (Mpro-x0165)"},{"1":"5res","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102281"},{"1":"5rfm","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102539"},{"1":"5rgq","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1849009686 (Mpro-x1086)"},{"1":"5ren","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102425"},{"1":"5ref","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z24758179"},{"1":"5rft","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102432"},{"1":"5rhd","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with SF013 (Mpro-x2193)"},{"1":"5rfk","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102575"},{"1":"5rfz","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102274"},{"1":"5rex","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102287"},{"1":"5rej","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102241"},{"1":"5rfi","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102353"},{"1":"5rgh","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1619978933 (Mpro-x0395)"},{"1":"5reo","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102578"},{"1":"5rfo","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102972"},{"1":"5rg3","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with NCL-00025412"},{"1":"5rhc","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Cov_HetLib053 (Mpro-x2119)"},{"1":"5rfa","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2643472210"},{"1":"5r80","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z18197050"},{"1":"7cz4","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 macro domain in complex with ADP-ribose"},{"1":"5rfe","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z509756472"},{"1":"5rgi","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z369936976 (Mpro-x0397)"},{"1":"5rem","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0103016"},{"1":"5r7z","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1220452176"},{"1":"5rfy","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102974"},{"1":"5rec","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1587220559"},{"1":"5rfv","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102306"},{"1":"5re4","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1129283193"},{"1":"5rep","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102201"},{"1":"5rfs","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102739"},{"1":"5rf4","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1741982125"},{"1":"5rfu","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102121"},{"1":"5rf0","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with POB0073"},{"1":"5rfw","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102243"},{"1":"5re8","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2737076969"},{"1":"5red","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434865"},{"1":"5rg1","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with NCL-00024905"},{"1":"5re6","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z54571979"},{"1":"5rf6","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1348371854"},{"1":"5rfb","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1271660837"},{"1":"5reh","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z111507846"},{"1":"5rgs","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1259086950 (Mpro-x1163)"},{"1":"5reu","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102395"},{"1":"5ret","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102269"},{"1":"5rfd","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z126932614"},{"1":"5rgp","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102628 (Mpro-x0771)"},{"1":"5rf7","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z316425948_minor"},{"1":"5rhb","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Cov_HetLib030 (Mpro-x2097)"},{"1":"5rf1","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with NCL-00023830"},{"1":"5rf9","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z217038356"},{"1":"5rf2","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1741969146"},{"1":"5rek","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102327"},{"1":"5rfj","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0103067"},{"1":"5rgm","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102142 (Mpro-x0708)"},{"1":"5ree","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2217052426"},{"1":"5rgn","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102759 (Mpro-x0731)"},{"1":"5rea","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z31432226"},{"1":"5rfq","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102179"},{"1":"5rew","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102275"},{"1":"5rey","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102911"},{"1":"5rer","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102615"},{"1":"5reg","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1545313172"},{"1":"5rgr","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z328695024 (Mpro-x1101)"},{"1":"5rfx","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102254"},{"1":"5rfg","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102372"},{"1":"5rfp","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102190"},{"1":"5rfr","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102169"},{"1":"5re7","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z30932204"},{"1":"5rg2","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with NCL-00025058"},{"1":"5re9","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434836"},{"1":"5rf5","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z3241250482"},{"1":"5rfh","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102277"},{"1":"5rgo","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102248 (Mpro-x0736)"},{"1":"5rez","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with POB0129"},{"1":"5rf8","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z271004858"},{"1":"5rfc","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z979145504"},{"1":"5rei","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434856"},{"1":"5rg0","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102535"},{"1":"5rhe","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PG-COV-42 (Mpro-x2052)"},{"1":"5rf3","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1741970824"},{"1":"5rgk","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1310876699 (Mpro-x0426)"},{"1":"5re5","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z33545544"},{"1":"5rgl","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PCM-0102962 (Mpro-x0705)"},{"1":"5reb","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2856434899"},{"1":"5r83","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 main protease in complex with Z44592329"},{"1":"5rhf","2":"2020","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with PG-COV-34 (Mpro-x2754)"},{"1":"7k8p","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C110"},{"1":"7k3g","2":"2020","3":"Solid-state NMR","4":"SARS-CoV-2 Envelope Protein Transmembrane Domain: Pentameric Structure Determined by Solid-State NMR"},{"1":"7che","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with BD-236 Fab and BD-368-2 Fab"},{"1":"7cb7","2":"2020","3":"X-ray diffraction","4":"1.7A resolution structure of SARS-CoV-2 main protease (Mpro) in complex with broad-spectrum coronavirus protease inhibitor GC376"},{"1":"7k8o","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C002"},{"1":"7jzl","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with LCB1 (2RBDs open)"},{"1":"7jme","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 NSP3 Macro X domain in complex with cyclic AMP"},{"1":"7k4n","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2E12 neutralizing antibody Fab fragment"},{"1":"7jqb","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 and rabbit 40S ribosome complex"},{"1":"7chh","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 S-6P in complex with BD-368-2 Fabs"},{"1":"7jz0","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer in Complex with (m7GpppA2m)pUpUpApApA (Cap-1) and S-Adenosyl-L-homocysteine (SAH)."},{"1":"7jva","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2A4 neutralizing antibody Fab fragment (local refinement of the receptor-binding domain and Fab variable domains)"},{"1":"7cak","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer with three RBD in the open state and complexed with three H014 Fab"},{"1":"7bw4","2":"2020","3":"Electron Microscopy","4":"Structure of the RNA-dependent RNA polymerase from SARS-CoV-2"},{"1":"7chc","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-629 Fab and BD-368-2 Fab"},{"1":"7jzu","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with LCB1 (local refinement of the RBD and LCB1)"},{"1":"7bwj","2":"2020","3":"X-ray diffraction","4":"crystal structure of SARS-CoV-2 antibody with RBD"},{"1":"7btf","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 RNA-dependent RNA polymerase in complex with cofactors in reduced condition"},{"1":"7c6u","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 complexed with GC376"},{"1":"7jmp","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody COVA2-39"},{"1":"7c8d","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of cat ACE2 and SARS-CoV-2 RBD"},{"1":"7k8u","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, C104"},{"1":"7k43","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2M11 neutralizing antibody Fab fragment"},{"1":"7ch5","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-629 Fab"},{"1":"7cah","2":"2020","3":"Electron Microscopy","4":"The interface of H014 Fab binds to SARS-CoV-2 S"},{"1":"7jv4","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2H13 neutralizing antibody (one RBD open)"},{"1":"7k8y","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C121 (State 2)"},{"1":"7jji","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 3Q-2P full-length prefusion spike trimer (C3 symmetry)"},{"1":"7k8n","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C102"},{"1":"7jjj","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 3Q-2P full-length dimers of spike trimers"},{"1":"7jzm","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with LCB3 (local refinement of the RBD and LCB3)"},{"1":"7jun","2":"2020","3":"c(\"X-ray diffraction\", \"Neutron Diffraction\", \"Hybrid\")","4":"Joint neutron/X-ray structure of SARS-CoV-2 3CL Mpro at room temperature"},{"1":"7k8t","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, C002 (State 2)"},{"1":"7jmw","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike protein receptor-binding domain in complex with cross-neutralizing antibody COVA1-16 Fab"},{"1":"7cab","2":"2020","3":"Electron Microscopy","4":"Structural basis for neutralization of SARS-CoV-2 and SARS-CoV by a potent therapeutic antibody"},{"1":"7ch4","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with BD-604 Fab"},{"1":"7cac","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer with one RBD in the open state and complexed with one H014 Fab."},{"1":"7c22","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the C-terminal domain of SARS-CoV-2 nucleocapsid protein"},{"1":"7c2i","2":"2020","3":"X-ray diffraction","4":"Crystal structure of nsp16-nsp10 heterodimer from SARS-CoV-2 in complex with SAM (with additional SAM during crystallization)"},{"1":"7jxc","2":"2020","3":"X-ray diffraction","4":"Mapping neutralizing and immunodominant sites on the SARS-CoV-2 spike receptor-binding domain by structure-guided high-resolution serology"},{"1":"7jwy","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 4.5"},{"1":"7jw0","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S304 neutralizing antibody Fab fragment"},{"1":"7k8x","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C121 (State 1)"},{"1":"7k8z","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C135"},{"1":"7jx3","2":"2020","3":"X-ray diffraction","4":"Mapping neutralizing and immunodominant sites on the SARS-CoV-2 spike receptor-binding domain by structure-guided high-resolution serology"},{"1":"7jp1","2":"2020","3":"X-ray diffraction","4":"Structure of wild-type substrate free SARS-CoV-2 Mpro."},{"1":"7k8q","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C121"},{"1":"7k3q","2":"2020","3":"X-ray diffraction","4":"An ultra-potent human neutralizing antibody locks the SARS-CoV-2 spike in the closed conformation"},{"1":"7c2j","2":"2020","3":"X-ray diffraction","4":"Crystal structure of nsp16-nsp10 heterodimer from SARS-CoV-2 in complex with SAM (without additional SAM during crystallization)"},{"1":"7chf","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with BD-604 Fab and BD-368-2 Fab"},{"1":"7k45","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2E12 neutralizing antibody Fab fragment (local refinement of the RBD and Fab variable domains)"},{"1":"7k8w","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C119"},{"1":"7c2l","2":"2020","3":"Electron Microscopy","4":"S protein of SARS-CoV-2 in complex bound with 4A8"},{"1":"7cai","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 S trimer with two RBDs in the open state and complexed with two H014 Fab"},{"1":"7jvb","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 spike receptor-binding domain (RBD) with nanobody Nb20"},{"1":"7k90","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the human neutralizing antibody Fab fragment, C144"},{"1":"7jqc","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1, CrPV IRES and rabbit 40S ribosome complex"},{"1":"7k8s","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C002 (state 1)"},{"1":"7cn9","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike ectodomain"},{"1":"7c6s","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease complexed with Boceprevir"},{"1":"7cam","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 main protease (Mpro) apo structure (space group P212121)"},{"1":"7joy","2":"2020","3":"X-ray diffraction","4":"Product structure of SARS-CoV-2 Mpro C145A mutant in complex with its C-terminal autoprocessing sequence."},{"1":"7k8m","2":"2020","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 receptor binding domain in complex with the human neutralizing antibody Fab fragment, C102"},{"1":"7jv2","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2H13 neutralizing antibody Fab fragment (local refinement of the receptor-binding motif and Fab variable domains)"},{"1":"7jmo","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody COVA2-04"},{"1":"7jmx","2":"2020","3":"X-ray diffraction","4":"Crystal structure of a SARS-CoV-2 cross-neutralizing antibody COVA1-16 Fab"},{"1":"7jzn","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with LCB3 (2RBDs open)"},{"1":"7jxd","2":"2020","3":"X-ray diffraction","4":"Mapping neutralizing and immunodominant sites on the SARS-CoV-2 spike receptor-binding domain by structure-guided high-resolution serology"},{"1":"7jxe","2":"2020","3":"X-ray diffraction","4":"Mapping neutralizing and immunodominant sites on the SARS-CoV-2 spike receptor-binding domain by structure-guided high-resolution serology"},{"1":"7jvc","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2A4 neutralizing antibody Fab fragment"},{"1":"7jjc","2":"2020","3":"X-ray diffraction","4":"Crystal structure of neuropilin-1 b1 domain in complex with SARS-CoV-2 S1 C-end rule (CendR) peptide"},{"1":"7k8r","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C135"},{"1":"7k8v","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 2P trimer in complex with the human neutralizing antibody Fab fragment, C110"},{"1":"7a5r","2":"2020","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 spike and CR3022 Fab (Non-Uniform Refinement)"},{"1":"6zp2","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein Trimer (K986P, V987P, single Arg S1/S2 cleavage site) in Locked State"},{"1":"7acs","2":"2020","3":"Solution NMR","4":"The SARS-CoV-2 nucleocapsid phosphoprotein N-terminal domain in complex with 7mer dsRNA"},{"1":"7a95","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 1 ACE2 Bound and 1 RBD Erect in Clockwise Direction"},{"1":"7bwq","2":"2020","3":"X-ray diffraction","4":"Structure of nonstructural protein Nsp9 from SARS-CoV-2"},{"1":"7a97","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 2 ACE2 Bound"},{"1":"7a96","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 1 ACE2 Bound and 1 RBD Erect in Anticlockwise Direction"},{"1":"7a5s","2":"2020","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 spike and CR3022 Fab (Homogeneous Refinement)"},{"1":"7c33","2":"2020","3":"X-ray diffraction","4":"Macro domain of SARS-CoV-2 in complex with ADP-ribose"},{"1":"7c01","2":"2020","3":"X-ray diffraction","4":"Molecular basis for a potent human neutralizing antibody targeting SARS-CoV-2 RBD"},{"1":"7a92","2":"2020","3":"Electron Microscopy","4":"Dissociated S1 domain of SARS-CoV-2 Spike bound to ACE2 (Unmasked Refinement)"},{"1":"7a91","2":"2020","3":"Electron Microscopy","4":"Dissociated S1 domain of SARS-CoV-2 Spike bound to ACE2 (Non-Uniform Refinement)"},{"1":"7act","2":"2020","3":"Solution NMR","4":"The SARS-CoV-2 nucleocapsid phosphoprotein N-terminal domain in complex with 10mer ssRNA"},{"1":"6wtm","2":"2020","3":"X-ray diffraction","4":"Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication"},{"1":"6xc3","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with antibodies CC12.1 and CR3022"},{"1":"6xca","2":"2020","3":"X-ray diffraction","4":"Crystal structure of an anti-SARS-CoV-2 human neutralizing antibody Fab fragment, C105"},{"1":"6vxx","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein (closed state)"},{"1":"6xb0","2":"2020","3":"X-ray diffraction","4":"Room temperature X-ray crystallography reveals catalytic cysteine in the SARS-CoV-2 3CL Mpro is highly reactive: Insights for enzyme mechanism and drug design"},{"1":"6w4h","2":"2020","3":"X-ray diffraction","4":"1.80 Angstrom Resolution Crystal Structure of NSP16 - NSP10 Complex from SARS-CoV-2"},{"1":"6xra","2":"2020","3":"Electron Microscopy","4":"Distinct conformational states of SARS-CoV-2 spike protein"},{"1":"6yi3","2":"2020","3":"Solution NMR","4":"The N-terminal RNA-binding domain of the SARS-CoV-2 nucleocapsid phosphoprotein"},{"1":"6zmt","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to a pre-40S-like ribosome complex"},{"1":"6zer","2":"2020","3":"X-ray diffraction","4":"Crystal structure of receptor binding domain of SARS-CoV-2 Spike glycoprotein in complex with EY6A Fab"},{"1":"6zxn","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike protein bound to neutralizing nanobodies (Ty1)"},{"1":"6z97","2":"2020","3":"Electron Microscopy","4":"Structure of the prefusion SARS-CoV-2 spike glycoprotein"},{"1":"6zgg","2":"2020","3":"Electron Microscopy","4":"Furin Cleaved Spike Protein of SARS-CoV-2 with One RBD Erect"},{"1":"6yor","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike S1 protein in complex with CR3022 Fab"},{"1":"6ym0","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain in complex with CR3022 Fab (crystal form 1)"},{"1":"6y2f","2":"2020","3":"X-ray diffraction","4":"Crystal structure (monoclinic form) of the complex resulting from the reaction between SARS-CoV-2 (2019-nCoV) main protease and tert-butyl (1-((S)-1-(((S)-4-(benzylamino)-3,4-dioxo-1-((S)-2-oxopyrrolidin-3-yl)butan-2-yl)amino)-3-cyclopropyl-1-oxopropan-2-yl)-2-oxo-1,2-dihydropyridin-3-yl)carbamate (alpha-ketoamide 13b)"},{"1":"6zgi","2":"2020","3":"Electron Microscopy","4":"Furin Cleaved Spike Protein of SARS-CoV-2 in Closed Conformation"},{"1":"6z6i","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 Macrodomain in complex with ADP-HPD"},{"1":"6z5t","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 Macrodomain in complex with ADP-ribose"},{"1":"6xc7","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with antibodies CC12.3 and CR3022"},{"1":"6xch","2":"2020","3":"X-ray diffraction","4":"Room-temperature X-ray Crystal structure of SARS-CoV-2 main protease in complex with Leupeptin"},{"1":"6xbg","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAW246"},{"1":"6x6p","2":"2020","3":"Electron Microscopy","4":"Characterization of the SARS-CoV-2 S Protein: Biophysical, Biochemical, Structural, and Antigenic Analysis"},{"1":"6x29","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 rS2d Down State Spike Protein Trimer"},{"1":"6xhm","2":"2020","3":"X-ray diffraction","4":"Covalent complex of SARS-CoV-2 main protease with N-[(2S)-1-({(2S,3S)-3,4-dihydroxy-1-[(3S)-2-oxopyrrolidin-3-yl]butan-2-yl}amino)-4-methyl-1-oxopentan-2-yl]-4-methoxy-1H-indole-2-carboxamide"},{"1":"6x2c","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 u1S2q All Down RBD State Spike Protein Trimer"},{"1":"6wxd","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 Nsp9 RNA-replicase"},{"1":"6xfn","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with UAW243"},{"1":"6wps","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the S309 neutralizing antibody Fab fragment"},{"1":"6xrz","2":"2020","3":"Electron Microscopy","4":"The 28-kDa Frameshift Stimulation Element from the SARS-CoV-2 RNA Genome"},{"1":"6wqf","2":"2020","3":"X-ray diffraction","4":"Structural Plasticity of the SARS-CoV-2 3CL Mpro Active Site Cavity Revealed by Room Temperature X-ray Crystallography"},{"1":"6wvn","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of Nsp16-Nsp10 from SARS-CoV-2 in Complex with 7-methyl-GpppA and S-Adenosylmethionine."},{"1":"6m2n","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CL pro) in complex with a novel inhibitor"},{"1":"6zp5","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in prefusion state (flexibility analysis, 1-up closed conformation)"},{"1":"6zon","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to a human 43S preinitiation ribosome complex - state 1"},{"1":"6zox","2":"2020","3":"Electron Microscopy","4":"Structure of Disulphide-stabilized SARS-CoV-2 Spike Protein Trimer (x2 disulphide-bond mutant, G413C, V987C, single Arg S1/S2 cleavage site)"},{"1":"6zlw","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human 40S ribosomal subunit"},{"1":"6zp4","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to a human 43S preinitiation ribosome complex - state 2"},{"1":"6zoz","2":"2020","3":"Electron Microscopy","4":"Structure of Disulphide-stabilized SARS-CoV-2 Spike Protein Trimer (x1 disulphide-bond mutant, S383C, D985C, K986P, V987P, single Arg S1/S2 cleavage site) in Locked State"},{"1":"6zlr","2":"2020","3":"X-ray diffraction","4":"Soaking competent crystal form of the SARS-CoV-2 Receptor Binding Domain (RBD):CR3022 complex."},{"1":"6xa4","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with UAW241"},{"1":"6wks","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 nsp16/nsp10 in complex with RNA cap analogue (m7GpppA) and S-adenosylmethionine"},{"1":"6xb2","2":"2020","3":"X-ray diffraction","4":"Room temperature X-ray crystallography reveals catalytic cysteine in the SARS-CoV-2 3CL Mpro is highly reactive: Insights for enzyme mechanism and drug design"},{"1":"6wrz","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of Nsp16-Nsp10 Heterodimer from SARS-CoV-2 with 7-methyl-GpppA and S-adenosyl-L-homocysteine in the Active Site and Sulfates in the mRNA Binding Groove."},{"1":"6w9q","2":"2020","3":"X-ray diffraction","4":"Peptide-bound SARS-CoV-2 Nsp9 RNA-replicase"},{"1":"6xc4","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody CC12.3"},{"1":"6wcf","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of ADP ribose phosphatase of NSP3 from SARS-CoV-2 in complex with MES"},{"1":"6w41","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with human antibody CR3022"},{"1":"6vw1","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 chimeric receptor-binding domain complexed with its receptor human ACE2"},{"1":"6xcm","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the C105 neutralizing antibody Fab fragment (state 1)"},{"1":"6xhu","2":"2020","3":"X-ray diffraction","4":"Room temperature X-ray crystallography reveals oxidation and reactivity of cysteine residues in SARS-CoV-2 3CL Mpro: Insights for enzyme mechanism and drug design"},{"1":"6y2g","2":"2020","3":"X-ray diffraction","4":"Crystal structure (orthorhombic form) of the complex resulting from the reaction between SARS-CoV-2 (2019-nCoV) main protease and tert-butyl (1-((S)-1-(((S)-4-(benzylamino)-3,4-dioxo-1-((S)-2-oxopyrrolidin-3-yl)butan-2-yl)amino)-3-cyclopropyl-1-oxopropan-2-yl)-2-oxo-1,2-dihydropyridin-3-yl)carbamate (alpha-ketoamide 13b)"},{"1":"6xlu","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 4.0"},{"1":"6wtk","2":"2020","3":"X-ray diffraction","4":"Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication"},{"1":"6xc2","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody CC12.1"},{"1":"6wtj","2":"2020","3":"X-ray diffraction","4":"Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication"},{"1":"6wkq","2":"2020","3":"X-ray diffraction","4":"1.98 Angstrom Resolution Crystal Structure of NSP16-NSP10 Heterodimer from SARS-CoV-2 in Complex with Sinefungin"},{"1":"6xm5","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 5.5, all RBDs down"},{"1":"6xbi","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAW248"},{"1":"6y2e","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the free enzyme of the SARS-CoV-2 (2019-nCoV) main protease"},{"1":"6wtt","2":"2020","3":"X-ray diffraction","4":"Crystals Structure of the SARS-CoV-2 (COVID-19) main protease with inhibitor GC-376"},{"1":"6wjt","2":"2020","3":"X-ray diffraction","4":"2.0 Angstrom Resolution Crystal Structure of Nsp16-Nsp10 Heterodimer from SARS-CoV-2 in Complex with S-Adenosyl-L-Homocysteine"},{"1":"6vyb","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike ectodomain structure (open state)"},{"1":"6wzq","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Nucleocapsid dimerization domain, P21 form"},{"1":"7kkj","2":"2020","3":"X-ray diffraction","4":"Structure of anti-SARS-CoV-2 Spike nanobody mNb6"},{"1":"7kkk","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing nanobody Nb6"},{"1":"7kfy","2":"2020","3":"X-ray diffraction","4":"Structural basis for a germline-biased antibody response to SARS-CoV-2 (RBD:C1A-F10 Fab)"},{"1":"7kmz","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of double ACE2-bound SARS-CoV-2 trimer Spike at pH 7.4"},{"1":"7knb","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of single ACE2-bound SARS-CoV-2 trimer spike at pH 7.4"},{"1":"7kne","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of single ACE2-bound SARS-CoV-2 trimer spike at pH 5.5"},{"1":"7kfx","2":"2020","3":"X-ray diffraction","4":"Structural basis for a germline-biased antibody response to SARS-CoV-2 (RBD:C1A-C2 Fab)"},{"1":"7knh","2":"2020","3":"Electron Microscopy","4":"Cryo-EM Structure of Double ACE2-Bound SARS-CoV-2 Trimer Spike at pH 5.5"},{"1":"7khp","2":"2020","3":"X-ray diffraction","4":"Acyl-enzyme intermediate structure of SARS-CoV-2 Mpro in complex with its C-terminal autoprocessing sequence."},{"1":"7kkl","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing nanobody mNb6"},{"1":"6wey","2":"2020","3":"X-ray diffraction","4":"High-resolution structure of the SARS-CoV-2 NSP3 Macro X domain"},{"1":"7kmb","2":"2020","3":"Electron Microscopy","4":"ACE2-RBD Focused Refinement Using Symmetry Expansion of Applied C3 for Triple ACE2-bound SARS-CoV-2 Trimer Spike at pH 7.4"},{"1":"7cxm","2":"2020","3":"Electron Microscopy","4":"Architecture of a SARS-CoV-2 mini replication and transcription complex"},{"1":"7de1","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nucleocapsid protein C-terminal RNA binding domain"},{"1":"7d1m","2":"2020","3":"X-ray diffraction","4":"CRYSTAL STRUCTURE OF THE SARS-CoV-2 MAIN PROTEASE COMPLEXED WITH GC376"},{"1":"7kni","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of Triple ACE2-bound SARS-CoV-2 Trimer Spike at pH 5.5"},{"1":"7cxn","2":"2020","3":"Electron Microscopy","4":"Architecture of a SARS-CoV-2 mini replication and transcription complex"},{"1":"6x79","2":"2020","3":"Electron Microscopy","4":"Prefusion SARS-CoV-2 S ectodomain trimer covalently stabilized in the closed conformation"},{"1":"6zn5","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to a pre-40S-like ribosome complex - state 2"},{"1":"6xb1","2":"2020","3":"X-ray diffraction","4":"Room temperature X-ray crystallography reveals catalytic cysteine in the SARS-CoV-2 3CL Mpro is highly reactive: Insights for enzyme mechanism and drug design"},{"1":"6xkp","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody CV07-270"},{"1":"6yz1","2":"2020","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 nsp10-nsp16 methyltransferase complex with Sinefungin"},{"1":"6xs6","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike D614G variant, minus RBD"},{"1":"6xf6","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of a biotinylated SARS-CoV-2 spike probe in the prefusion state (1 RBD up)"},{"1":"6xmk","2":"2020","3":"X-ray diffraction","4":"1.70 A resolution structure of SARS-CoV-2 3CL protease in complex with inhibitor 7j"},{"1":"6xe1","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 spike protein receptor binding domain in complex with a potent neutralizing antibody, CV30 Fab"},{"1":"7a93","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 2 RBDs Erect"},{"1":"6yyt","2":"2020","3":"Electron Microscopy","4":"Structure of replicating SARS-CoV-2 polymerase"},{"1":"6zol","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2-Nsp1-40S complex, focused on head"},{"1":"6zm7","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human CCDC124-80S-EBP1 ribosome complex"},{"1":"7a25","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike protein bound to neutralizing sybodies (Sb23)"},{"1":"6zp1","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein Trimer (K986P, V987P, single Arg S1/S2 cleavage site) in Closed State"},{"1":"6yz7","2":"2020","3":"X-ray diffraction","4":"H11-D4, SARS-CoV-2 RBD, CR3022 ternary complex"},{"1":"6zdh","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike glycoprotein in complex with a neutralizing antibody EY6A Fab"},{"1":"6zge","2":"2020","3":"Electron Microscopy","4":"Uncleavable Spike Protein of SARS-CoV-2 in Closed Conformation"},{"1":"6zoy","2":"2020","3":"Electron Microscopy","4":"Structure of Disulphide-stabilized SARS-CoV-2 Spike Protein Trimer (x1 disulphide-bond mutant, S383C, D985C, K986P, V987P, single Arg S1/S2 cleavage site) in Closed State"},{"1":"6zoj","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2-Nsp1-40S complex, composite map"},{"1":"6m2q","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CL pro) apo structure (space group C21)"},{"1":"6zow","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in prefusion state"},{"1":"6zh9","2":"2020","3":"X-ray diffraction","4":"Ternary complex CR3022 H11-H4 and RBD (SARS-CoV-2)"},{"1":"6zgh","2":"2020","3":"Electron Microscopy","4":"Furin Cleaved Spike Protein of SARS-CoV-2 in Intermediate Conformation"},{"1":"6zok","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2-Nsp1-40S complex, focused on body"},{"1":"6m3m","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nucleocapsid protein N-terminal RNA binding domain"},{"1":"6zp0","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 Spike Protein Trimer (single Arg S1/S2 cleavage site) in Closed State"},{"1":"7a98","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 3 ACE2 Bound"},{"1":"7a94","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Spike Glycoprotein with 1 ACE2 Bound"},{"1":"6z72","2":"2020","3":"X-ray diffraction","4":"SARS-CoV-2 Macrodomain in complex with ADP-HPM"},{"1":"6zwv","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike Proteins on intact virions: 3 Closed RBDs"},{"1":"6zp7","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in prefusion state (flexibility analysis, 1-up open conformation)"},{"1":"6xr8","2":"2020","3":"Electron Microscopy","4":"Distinct conformational states of SARS-CoV-2 spike protein"},{"1":"6xbh","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor UAW247"},{"1":"6xqu","2":"2020","3":"X-ray diffraction","4":"Room-temperature X-ray Crystal structure of SARS-CoV-2 main protease in complex with Boceprevir"},{"1":"6xkl","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 HexaPro S One RBD up"},{"1":"6xf5","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of a biotinylated SARS-CoV-2 spike probe in the prefusion state (RBDs down)"},{"1":"6xcn","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the C105 neutralizing antibody Fab fragment (state 2)"},{"1":"6yla","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 receptor binding domain in complex with CR3022 Fab"},{"1":"6zmi","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human LYAR-80S ribosome complex"},{"1":"6m0j","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain bound with ACE2"},{"1":"6zcz","2":"2020","3":"X-ray diffraction","4":"Crystal structure of receptor binding domain of SARS-CoV-2 Spike glycoprotein in ternary complex with EY6A Fab and a nanobody."},{"1":"7jv6","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 spike in complex with the S2H13 neutralizing antibody (closed conformation)"},{"1":"6xqs","2":"2020","3":"X-ray diffraction","4":"Room-temperature X-ray Crystal structure of SARS-CoV-2 main protease in complex with Telaprevir"},{"1":"6zmo","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human LYAR-80S-eEF1a ribosome complex"},{"1":"6xm3","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 5.5, single RBD up, conformation 1"},{"1":"6xm4","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 spike at pH 5.5, single RBD up, conformation 2"},{"1":"6zme","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp1 bound to the human CCDC124-80S-eERF1 ribosome complex"},{"1":"7chb","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 RBD in complex with BD-236 Fab"},{"1":"6x2b","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 u1S2q 2-RBD Up Spike Protein Trimer"},{"1":"6wpt","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 spike glycoprotein in complex with the S309 neutralizing antibody Fab fragment (open state)"},{"1":"6x2a","2":"2020","3":"Electron Microscopy","4":"SARS-CoV-2 u1S2q 1-RBD Up Spike Protein Trimer"},{"1":"6xdg","2":"2020","3":"Electron Microscopy","4":"Complex of SARS-CoV-2 receptor binding domain with the Fab fragments of two neutralizing antibodies"},{"1":"6wen","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of ADP ribose phosphatase of NSP3 from SARS-CoV-2 in the apo form"},{"1":"6wzo","2":"2020","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 Nucleocapsid dimerization domain, P1 form"},{"1":"6xkq","2":"2020","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 receptor binding domain in complex with neutralizing antibody CV07-250"},{"1":"6wq3","2":"2020","3":"X-ray diffraction","4":"Crystal Structure of Nsp16-Nsp10 Heterodimer from SARS-CoV-2 in Complex with 7-methyl-GpppA and S-adenosyl-L-homocysteine."},{"1":"6w75","2":"2020","3":"X-ray diffraction","4":"1.95 Angstrom Resolution Crystal Structure of NSP10 - NSP16 Complex from SARS-CoV-2"},{"1":"7kl9","2":"2020","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 S 6P trimer in complex with the ACE2 protein decoy, CTC-445.2 (State 4)"},{"1":"7kfv","2":"2020","3":"X-ray diffraction","4":"Structural basis for a germline-biased antibody response to SARS-CoV-2 (RBD:C1A-B12 Fab)"},{"1":"7a29","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike protein bound to neutralizing sybodies (Sb23) 2-up conformation"},{"1":"6yhu","2":"2020","3":"X-ray diffraction","4":"Crystal structure of the nsp7-nsp8 complex of SARS-CoV-2"},{"1":"6xez","2":"2020","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 replication-transcription complex bound to nsp13 helicase - nsp13(2)-RTC"},{"1":"6xqt","2":"2020","3":"X-ray diffraction","4":"Room-temperature X-ray Crystal structure of SARS-CoV-2 main protease in complex with Narlaprevir"},{"1":"6xm0","2":"2020","3":"Electron Microscopy","4":"Consensus structure of SARS-CoV-2 spike at pH 5.5"},{"1":"6xey","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of the SARS-CoV-2 spike glycoprotein bound to Fab 2-4"},{"1":"7kfw","2":"2020","3":"X-ray diffraction","4":"Structural basis for a germline-biased antibody response to SARS-CoV-2 (RBD:C1A-B3 Fab)"},{"1":"7kms","2":"2020","3":"Electron Microscopy","4":"Cryo-EM structure of triple ACE2-bound SARS-CoV-2 trimer spike at pH 7.4"},{"1":"5sm1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z68277692"},{"1":"5smi","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z71580604"},{"1":"5sl3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z223688272"},{"1":"5slc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1849009686"},{"1":"5rl1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-27 (Mpro-x3113)"},{"1":"5skz","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z57258487"},{"1":"5sli","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1003146540"},{"1":"5sm6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1899842917"},{"1":"5s73","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 Nsp3 macrodomain"},{"1":"5sl9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z54571979"},{"1":"5smb","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z419995480"},{"1":"5slt","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1816233707"},{"1":"5slv","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2856434942"},{"1":"5smc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2033637875"},{"1":"5sm3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z943693514"},{"1":"5smf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z56791867"},{"1":"5smh","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2856434938"},{"1":"5sl1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1273312153"},{"1":"5rh3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1264525706 (Mpro-x2649)"},{"1":"5slj","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1430613393"},{"1":"5slh","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z65532537"},{"1":"5slo","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z56983806"},{"1":"5rha","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z147647874 (Mpro-x2779)"},{"1":"5r8t","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 main protease screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries"},{"1":"5sll","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z54615640"},{"1":"5sl4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z383202616"},{"1":"5sle","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z56880342"},{"1":"5smn","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of SARS-CoV-2 main protease ligands identified from single sequence-guideddeep generative framework -- Crystal structure of SARS-CoV-2 main protease in complex with Z1365651030 (Mpro-IBM0078)"},{"1":"5sky","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z466628048"},{"1":"5slr","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2073741691"},{"1":"5sln","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z57299529"},{"1":"5rl5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-30 (Mpro-x3359)"},{"1":"5rgw","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4444621910 (Mpro-x2569)"},{"1":"5smd","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z274575916"},{"1":"5slz","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2072621991"},{"1":"5rh9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4438424255 (Mpro-x2776)"},{"1":"5smk","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 NSP14"},{"1":"7ej5","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with a neutralizing antibody RBD-chAb-45"},{"1":"7d47","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Papain-like protease C111S"},{"1":"5sl7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1186029914"},{"1":"5sm9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2234920345"},{"1":"5slk","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1354370680"},{"1":"5sly","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1526504764"},{"1":"5rh1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z2010253653 (Mpro-x2643)"},{"1":"5sm4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2856434944"},{"1":"5sm5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2856434807"},{"1":"5sl5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z32014663"},{"1":"5rgy","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1535580916 (Mpro-x2581)"},{"1":"5sl8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2856434762"},{"1":"5sma","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2856434890"},{"1":"5rob","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 helicase"},{"1":"5rgz","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1343543528 (Mpro-x2600)"},{"1":"5sla","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1003207278"},{"1":"5slb","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z744930860"},{"1":"5rh0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1286870272 (Mpro-x2608)"},{"1":"5rl2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-26 (Mpro-x3115)"},{"1":"5s74","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of ground-state model of SARS-CoV-2 Nsp3 macrodomain"},{"1":"5sme","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z437584380"},{"1":"5rl4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-3 (Mpro-x3124)"},{"1":"5rgt","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4439011607 (Mpro-x2540)"},{"1":"5rh6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4439011588 (Mpro-x2703)"},{"1":"5rgx","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1344037997 (Mpro-x2572)"},{"1":"5rl3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-39 (Mpro-x3117)"},{"1":"5rh4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1530425063 (Mpro-x2659)"},{"1":"5rgu","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4444622180 (Mpro-x2562)"},{"1":"5rh2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z1129289650 (Mpro-x2646)"},{"1":"5sml","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of SARS-CoV-2 main protease ligands identified from single sequence-guideddeep generative framework -- Crystal structure of SARS-CoV-2 main protease in complex with Z68337194 (Mpro-IBM0045)"},{"1":"5skw","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1272494722"},{"1":"5sls","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1373445602"},{"1":"5sm0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z32665176"},{"1":"5sld","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1246465616"},{"1":"5rh7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4439011584 (Mpro-x2705)"},{"1":"5smm","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of SARS-CoV-2 main protease ligands identified from single sequence-guideddeep generative framework -- Crystal structure of SARS-CoV-2 main protease in complex with Z1633315555 (Mpro-IBM0058)"},{"1":"5slw","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1310876699"},{"1":"5rh8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4444621965 (Mpro-x2764)"},{"1":"5slp","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z373768898"},{"1":"5slu","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1796014543"},{"1":"5sl6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z256709556"},{"1":"5rgv","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4444622066 (Mpro-x2563)"},{"1":"5rh5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition SARS-CoV-2 main protease fragment screen -- Crystal Structure of SARS-CoV-2 main protease in complex with Z4439011520 (Mpro-x2694)"},{"1":"5rl0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition of computational designs of SARS-CoV-2 main protease covalent inhibitors -- Crystal Structure of SARS-CoV-2 main protease in complex with LON-WEI-adc59df6-2 (Mpro-x3110)"},{"1":"5slm","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z28290384"},{"1":"5slx","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z752989138"},{"1":"5sm8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2027158783"},{"1":"5slf","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z198195770"},{"1":"5sm7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z1247413608"},{"1":"5slq","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2856434829"},{"1":"5slg","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z32400357"},{"1":"5sm2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z3006151474"},{"1":"5sl0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z57260516"},{"1":"5sl2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z100643660"},{"1":"5smg","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z2092370954"},{"1":"5skx","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal Structure of SARS-CoV-2 NSP14 in complex with Z126932614"},{"1":"8uh5","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with an inhibitor TKB-272"},{"1":"7mx9","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 ORF8 accessory protein"},{"1":"8ka8","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Delta RBD in complex with golden hamster ACE2 (local refinement)"},{"1":"8uh9","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease E166V mutant in complex with an inhibitor TKB-272"},{"1":"8uab","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease (Mpro) complex with AC1115"},{"1":"7sf1","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML1001"},{"1":"8u1g","2":"NA","3":"X-ray diffraction","4":"Prefusion-stabilized SARS-CoV-2 S2 subunit"},{"1":"8uh8","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease E166V (Apo structure)"},{"1":"8ufm","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of L516C/Y647C Mutant of SARS-Unique Domain (SUD) from SARS-CoV-2"},{"1":"8kc2","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 BA.3 RBD in complex with golden hamster ACE2 (local refinement)"},{"1":"8stz","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI37"},{"1":"8ifp","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 1"},{"1":"8hrk","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-RBD-ACE2 complex"},{"1":"8ifq","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 2"},{"1":"8spj","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) N28T Mutant"},{"1":"8otr","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with SAM analog BDH 33959089"},{"1":"8jyk","2":"NA","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 XBB.1.5 spike glycoprotein (closed state 1)"},{"1":"8tyj","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nsp10/nsp16 complex with bound SAH"},{"1":"8sty","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI60"},{"1":"8f02","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML2006a4"},{"1":"8eyg","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein complexed with two nanobodies"},{"1":"8ers","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z4718398507 - (R,S) isomer"},{"1":"7lgo","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the nucleic acid binding domain (NAB) of Nsp3 from SARS-CoV-2"},{"1":"7msx","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Nsp2"},{"1":"8sxo","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) H164I Mutant"},{"1":"8f5i","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 S2 helix epitope scaffold bound by antibody DH1057.1"},{"1":"8fwn","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 papain-like protease C111S mutant"},{"1":"8r12","2":"NA","3":"X-ray diffraction","4":"Structure of compound 8 bound to SARS-CoV-2 main protease"},{"1":"7c8t","2":"NA","3":"X-ray diffraction","4":"Complex Structure of SARS-CoV-2 3CL Protease with TG-0205221"},{"1":"8ifs","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 7"},{"1":"8ov2","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with Sangivamycin"},{"1":"8hri","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Delta variant spike protein"},{"1":"8ot0","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with MTA and glycine"},{"1":"8hom","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Omicron Main Protease (Mpro) in Complex with Ensitrelvir"},{"1":"7cot","2":"NA","3":"X-ray diffraction","4":"Structure of post fusion core of SARS-CoV-2 S2 subunit"},{"1":"8ift","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 10"},{"1":"8hn4","2":"NA","3":"X-ray diffraction","4":"Complex structure of HLA2402 with recognizing SARS-CoV-2 epitope QYIKWPWYI"},{"1":"8hrj","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Delta variant spike protein"},{"1":"7jfq","2":"NA","3":"X-ray diffraction","4":"The crystal structure of 3CL MainPro of SARS-CoV-2 with de-oxidized C145"},{"1":"8r19","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro (Omicron, P132H) free enzyme"},{"1":"8r14","2":"NA","3":"X-ray diffraction","4":"Structure of compound 11 bound to SARS-CoV-2 main protease"},{"1":"7bfb","2":"NA","3":"X-ray diffraction","4":"Crystal structure of ebselen covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"8fwo","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 papain-like protease"},{"1":"7jx6","2":"NA","3":"X-ray diffraction","4":"STRUCTURE OF THE SARS-CoV-2 ORF8 ENCODED ACCESSORY PROTEIN"},{"1":"8hoz","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Omicron Main Protease (Mpro) in Complex with Nirmatrelvir"},{"1":"7jr3","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease crystallized under reducing conditions"},{"1":"8hol","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Omicron Main Protease (Mpro)"},{"1":"8oto","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with AMP"},{"1":"8ov3","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with 5-Iodotubercidin"},{"1":"8r0v","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro (Omicron, P132H) in complex with alpha-ketoamide 13b-K at pH 6.5"},{"1":"8ov1","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with ADP"},{"1":"8r11","2":"NA","3":"X-ray diffraction","4":"Structure of compound 7 bound to SARS-CoV-2 main protease"},{"1":"7c8r","2":"NA","3":"X-ray diffraction","4":"Complex Structure of SARS-CoV-2 3CL Protease with TG-0203770"},{"1":"8r16","2":"NA","3":"X-ray diffraction","4":"Structure of compound 12 bound to SARS-CoV-2 main protease"},{"1":"8ifr","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 3"},{"1":"8ov4","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with Toyocamycin"},{"1":"8osx","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with ATP"},{"1":"8bs1","2":"NA","3":"X-ray diffraction","4":"Room-temperature structure of SARS-CoV-2 Main protease at atmospheric pressure"},{"1":"8gzb","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CLpro"},{"1":"7wck","2":"NA","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6P(B.1.617.2) in complex with SWA9 Fab"},{"1":"8gy6","2":"NA","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 RNA-dependent RNA polymerase with gossypol binding"},{"1":"8bfq","2":"NA","3":"X-ray diffraction","4":"Structure of the apo form of Mpro from SARS-CoV-2"},{"1":"7wcp","2":"NA","3":"Electron Microscopy","4":"CryoEM structure of the SARS-CoV-2 S6P(B.1.617.2) in complex with SWC11 Fab"},{"1":"7wk3","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron S-open"},{"1":"7sis","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Spike-derived peptide S976-984 (VLNDILSRL) presented by HLA-A*02:01"},{"1":"7sfh","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML102"},{"1":"7sfi","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML104"},{"1":"7sd9","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI48"},{"1":"8ign","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with RAY1216"},{"1":"7sww","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Spike NTD in complex with neutralizing Fab SARS2-57 (local refinement)"},{"1":"7sh8","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI88"},{"1":"8dpz","2":"NA","3":"Electron Microscopy","4":"Local refinement of SARS-CoV-2 vaccine induced antibody DH1338 bound to SARS-CoV-2 HexaPro RBD Spike ectodomain"},{"1":"8wrh","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 XBB.1.5.70 in complex with ACE2"},{"1":"8gw4","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro 1-302/C145A in complex with peptide 8-1"},{"1":"8gxh","2":"NA","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 main protease in complex with 14b"},{"1":"8bsd","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with tubercidin"},{"1":"8bs2","2":"NA","3":"X-ray diffraction","4":"Room-temperature structure of SARS-CoV-2 Main protease at 104 MPa helium gas pressure in a sapphire capillary"},{"1":"8h91","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 spike receptor-binding domain in complex with nanobody N19"},{"1":"8gxg","2":"NA","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 main protease in complex with 14a"},{"1":"8h7l","2":"NA","3":"Electron Microscopy","4":"Cryo-EM Structure of SARS-CoV-2 BA.2 Spike protein in complex with BA7535"},{"1":"8hda","2":"NA","3":"X-ray diffraction","4":"Crystal structure of Ubl1 (residues 18-111) of SARS-CoV-2"},{"1":"8ehk","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) T135I Mutant"},{"1":"8azc","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 NSP3 macrodomain in the apo form"},{"1":"8d4p","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease in complex with inhibitor Jun10-90-3-C1"},{"1":"7ti9","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the ubiquitin-like domain 1 (Ubl1) of Nsp3 from SARS-CoV-2, form 2"},{"1":"7swx","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Spike in complex with neutralizing Fab SARS2-57 (three down conformation)"},{"1":"7sh9","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI86"},{"1":"7sgh","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML124N"},{"1":"7vzt","2":"NA","3":"X-ray diffraction","4":"A human neutralizing antibody targeting SARS-CoV-2 RBD"},{"1":"8dtk","2":"NA","3":"Electron Microscopy","4":"Structure of RBD directed antibody DH1047 in complex with SARS-CoV-2 spike: Local refinement of RBD-Fab interace"},{"1":"8ehm","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) S144F Mutant"},{"1":"8igy","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with nirmatrelvir"},{"1":"8azd","2":"NA","3":"X-ray diffraction","4":"Structure of SARS-CoV-2 NSP3 macrodomain in complex with ADPR"},{"1":"8aiz","2":"NA","3":"X-ray diffraction","4":"Mpro of SARS-CoV-2 in complex with the RK-68 inhibitor"},{"1":"7qip","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Nucleocapsid phosphopeptide 201-210 bound to human 14-3-3 sigma"},{"1":"7sa2","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 spike-derived peptide S1060-1068 (VVFLHVTYV) presented by HLA-A*02:01"},{"1":"7qik","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Nucleocapsid phosphopeptide 193-200 bound to human 14-3-3 sigma"},{"1":"7t9w","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the Nsp3 bSM (Betacoronavirus-Specific Marker) domain from SARS-CoV-2"},{"1":"8bec","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with pT1375 scFV"},{"1":"8bgd","2":"NA","3":"X-ray diffraction","4":"Structure of Mpro from SARS-CoV-2 in complex with FGA147"},{"1":"8bfo","2":"NA","3":"X-ray diffraction","4":"Structure of the apo form of Mpro from SARS-CoV-2"},{"1":"8gx9","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD with P2C-1F11 and P2B-1G5"},{"1":"8h7z","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 BA.2 RBD in complex with BA7535 fab (local refinement)"},{"1":"8gxi","2":"NA","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 main protease in complex with 14c"},{"1":"8c5m","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with MTA"},{"1":"8h6f","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Spike protein in complex with A6 repebody"},{"1":"8bg1","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with pT1511 scFV"},{"1":"8bg6","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 S protein in complex with pT1644 Fab"},{"1":"8bzv","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 nsp10-16 methyltransferase in complex with adenosine"},{"1":"8bg3","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with pT1610 scFV"},{"1":"8cbe","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Delta-RBD complexed with BA.4/5-2 and Beta-49 Fabs"},{"1":"8bg8","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 S protein in complex with pT1696 Fab"},{"1":"8c9l","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro-S144A mutant, free enzyme"},{"1":"8bg5","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with pT1631 scFV"},{"1":"8csj","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of NTD-directed non-neutralizing antibody 4-33 in complex with prefusion SARS-CoV-2 spike glycoprotein"},{"1":"8bgg","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike (Omicron BA.1 variant) in complex with nanobody W25 (map 5, focus refinement on RBD, W25 and adjacent NTD)"},{"1":"8bg4","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with pT1611 scFV"},{"1":"8bg2","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 S RBD in complex with pT1580 scFV"},{"1":"8juh","2":"NA","3":"X-ray diffraction","4":"Crystal structure of ScFv against the receptor binding domine of SARS-CoV-2 S-spike protein"},{"1":"7maw","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor SM129"},{"1":"7mb2","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor SM144"},{"1":"8jpq","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro in complex with D-5-96"},{"1":"7mb0","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor SM141"},{"1":"7ntt","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 Main Protease"},{"1":"7set","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML1000"},{"1":"7nbr","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (Mpro) in complex with the HCV NS3/4A inhibitor boceprevir"},{"1":"7s82","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 Main protease C145S in complex with N-terminal peptide"},{"1":"7sda","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI49"},{"1":"7sdc","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MI-09"},{"1":"8ezv","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML2006a"},{"1":"8f5h","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 S2 helix epitope scaffold"},{"1":"8f2v","2":"NA","3":"X-ray diffraction","4":"Antibody Fab directed against SARS-CoV-2 Spike Protein Receptor Binding Domain (RBD)"},{"1":"8f2c","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML3006a"},{"1":"7mav","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor dFFCit-yne"},{"1":"7mb1","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor SM143"},{"1":"8eyh","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein bound with a nanobody"},{"1":"8u2x","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of NendoU (Uridylate-specific endoribonuclease, nsp15) from Betacoronavirus SARS-CoV-2 (H235A mutant)"},{"1":"8ezz","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML2006a2"},{"1":"8f2j","2":"NA","3":"X-ray diffraction","4":"Crystal structure of antibody WRAIR-2134 in complex with SARS-CoV-2 receptor binding domain"},{"1":"7mb3","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor SM145"},{"1":"8f4p","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein trimer (down conformation) bound with a nanobody"},{"1":"8gqc","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-unique domain (SUD) of SARS-CoV-2 (1.35 angstrom resolution)"},{"1":"7msw","2":"NA","3":"Electron Microscopy","4":"Full length SARS-CoV-2 Nsp2"},{"1":"8fdo","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 fusion peptide epitope scaffold FP15 bound to DH1058"},{"1":"8ftl","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 (COVID-19) main protease (Mpro) in complex with inhibitor Jun89-3-C1"},{"1":"7mat","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor dFFR"},{"1":"7n9a","2":"NA","3":"Electron Microscopy","4":"Potent neutralizing nanobodies resist convergent circulating variants of SARS-CoV-2 by targeting novel and conserved epitopes-CovS RBD with NB21"},{"1":"7nby","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SU3327 (halicin) covalently bound to the main protease (3CLpro/Mpro) of SARS-CoV-2."},{"1":"7max","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor SM137"},{"1":"8k5g","2":"NA","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 BA.1 RBD with UT28-RD"},{"1":"8f2d","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML4006a"},{"1":"8eqf","2":"NA","3":"Electron Microscopy","4":"cryoEM structure of a broadly neutralizing anti-SARS-CoV-2 antibody STI-9167"},{"1":"8eoy","2":"NA","3":"X-ray diffraction","4":"Structure of the SARS-CoV-2 main protease in complex with inhibitor MPI37"},{"1":"7ntw","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 Main Protease with a Zinc ion coordinated in the active site"},{"1":"8f2t","2":"NA","3":"X-ray diffraction","4":"Antibody Fab directed against SARS-CoV-2 Spike Protein Receptor Binding Domain (RBD)"},{"1":"7wch","2":"NA","3":"Electron Microscopy","4":"CryoEM structure of the SARS-CoV-2 S6P(B.1.617.2) in complex with SWA9 Fab"},{"1":"7wcu","2":"NA","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6P(B.1.617.2) in complex with SWC11 Fab"},{"1":"8cbf","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Delta-RBD complexed with Omi-42 and Beta-49 Fabs"},{"1":"8x1h","2":"NA","3":"X-ray diffraction","4":"Crystal structure of N-terminal domain of Nucleocapsid protein of SARS-CoV-2"},{"1":"8vci","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Frameshift Stimulatory Element with Upstream Multibranch Loop"},{"1":"7whh","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 omicron RBD and human ACE2"},{"1":"8gtv","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound JZD-07"},{"1":"8xab","2":"NA","3":"X-ray diffraction","4":"Crystal structure of Ubl1 domain of nonstructural protein 3 of SARS-CoV-2"},{"1":"8gws","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro 1-302 c145a in complex with peptide 4"},{"1":"8gvd","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Mpro in complex with D-4-38"},{"1":"8hbl","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-unique domain (SUD) of SARS-CoV-2 (1.58 angstrom resolution)"},{"1":"8cbd","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Delta-RBD complexed with BA.4/5-1 and EY6A Fabs"},{"1":"8gtw","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound JZD-26"},{"1":"8bev","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike (HexaPro variant) in complex with nanobody W25 (map 3, focus refinement on RBD, W25 and adjacent NTD)"},{"1":"8c9o","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro-S144A mutant in complex with 13b-K"},{"1":"7p2o","2":"NA","3":"Solution NMR","4":"NMR solution structure of SUD-C domain of SARS-CoV-2"},{"1":"7sf3","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML1006m"},{"1":"8jyp","2":"NA","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 XBB.1.5 spike RBD in complex with ACE2"},{"1":"7nbs","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease (Mpro) in complex with the HCV NS3/4A inhibitor telaprevir"},{"1":"7maz","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor SM139"},{"1":"8ehl","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) S144M Mutant"},{"1":"8igx","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease (3CLpro) in complex with compound 9 (simnotrelvir, SIM0417, SSD8432)"},{"1":"8dad","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 receptor binding domain in complex with AZ090 Fab"},{"1":"8igo","2":"NA","3":"X-ray diffraction","4":"Crystal structure of apo SARS-CoV-2 main protease"},{"1":"8ilc","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SetMet CoV-Y domain of Nsp3 in SARS-CoV-2"},{"1":"8ehj","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Main Protease (Mpro) H172Q Mutant"},{"1":"8jyo","2":"NA","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 XBB.1.5 spike glycoprotein in complex with ACE2 (2-up state)"},{"1":"8jym","2":"NA","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 XBB.1.5 spike glycoprotein (closed state 2)"},{"1":"8k5h","2":"NA","3":"Electron Microscopy","4":"Structure of the SARS-CoV-2 BA.1 spike with UT28-RD"},{"1":"7mau","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with Covalent Inhibitor dFFR-yne"},{"1":"7sfb","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML101"},{"1":"8jyn","2":"NA","3":"Electron Microscopy","4":"Structure of SARS-CoV-2 XBB.1.5 spike glycoprotein in complex with ACE2 (1-up state)"},{"1":"7mkb","2":"NA","3":"X-ray diffraction","4":"Human leukocyte antigen A*0201 in complex with SARS-CoV-2 epitope YLQPRTFLL"},{"1":"8hqt","2":"NA","3":"X-ray diffraction","4":"The complex structure of COPI cargo sorting module with SARS-CoV-2 Spike KxHxx sorting motif"},{"1":"6wc1","2":"NA","3":"X-ray diffraction","4":"Peptide-bound SARS-CoV-2 Nsp9 RNA-replicase"},{"1":"6xkf","2":"NA","3":"X-ray diffraction","4":"The crystal structure of 3CL MainPro of SARS-CoV-2 with oxidized Cys145 (Sulfenic acid cysteine)."},{"1":"6z2e","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Mpro in complex with the activity-based probe, biotin-PEG(4)-Abu-Tle-Leu-Gln-vinylsulfone"},{"1":"6xdh","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of NendoU (Uridylate-specific endoribonuclease, nsp15) from Betacoronavirus SARS-CoV-2"},{"1":"6zbp","2":"NA","3":"X-ray diffraction","4":"H11-H4 complex with SARS-CoV-2"},{"1":"6w37","2":"NA","3":"X-ray diffraction","4":"STRUCTURE OF THE SARS-CoV-2 ORF7A ENCODED ACCESSORY PROTEIN"},{"1":"6wnp","2":"NA","3":"X-ray diffraction","4":"X-ray Structure of SARS-CoV-2 main protease bound to Boceprevir at 1.45 A"},{"1":"6w7y","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV and SARS-CoV-2 reactive human antibody CR3022"},{"1":"7kyu","2":"NA","3":"X-ray diffraction","4":"The crystal structure of SARS-CoV-2 Main Protease with the formation of Cys145-1H-indole-5-carboxylate"},{"1":"7frb","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5551426009 - (S) isomer"},{"1":"7fb4","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in two-RBD weak state after CTSL-treatment"},{"1":"7fr9","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z1367095370"},{"1":"7fr5","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2890189003"},{"1":"7f7h","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 S protein RBD in complex with A8-1 Fab"},{"1":"7fr4","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with PTERA_A26A - (S) isomer"},{"1":"7fr7","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z431872694"},{"1":"7fr1","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z1423250928"},{"1":"7fra","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z1343520564"},{"1":"7kag","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the ubiquitin-like domain 1 (Ubl1) of Nsp3 from SARS-CoV-2"},{"1":"7fr0","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2890182452"},{"1":"7kzc","2":"NA","3":"X-ray diffraction","4":"Potent SARS-CoV-2 binding and neutralization through maturation of iconic SARS-CoV-1antibodies"},{"1":"7fr8","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2890408258 - (R) isomer"},{"1":"7fr6","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z2890147894"},{"1":"7fb1","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in one-RBD open state"},{"1":"7dk2","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Spike RBD in complex with MW07 Fab"},{"1":"7cyh","2":"NA","3":"Electron Microscopy","4":"Binding interface of SARS-CoV-2 RBD and its neutralizing antibody HB27"},{"1":"7fr2","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with Z5551425673 - (S) isomer"},{"1":"7frc","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with PTERA_A05 - (R) isomer"},{"1":"7frd","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with PTERA_A25A - (S) isomer"},{"1":"7kza","2":"NA","3":"X-ray diffraction","4":"Potent SARS-CoV-2 binding and neutralization through maturation of iconic SARS-CoV-1antibodies"},{"1":"7f12","2":"NA","3":"X-ray diffraction","4":"A SARS-CoV-2 neutralizing antibody"},{"1":"7f5r","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Y453F-RBD bound to mink ACE2"},{"1":"7d6i","2":"NA","3":"X-ray diffraction","4":"A neutralizing MAb targeting receptor-binding-domain of SARS-CoV-2"},{"1":"7cx9","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with INZ-1"},{"1":"6z2m","2":"NA","3":"X-ray diffraction","4":"H11-D4, SARS-CoV-2 RBD, CR3022 ternary complex"},{"1":"6y84","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease with unliganded active site (2019-nCoV, coronavirus disease 2019, COVID-19)"},{"1":"6z43","2":"NA","3":"Electron Microscopy","4":"Cryo-EM Structure of SARS-CoV-2 Spike : H11-D4 Nanobody Complex"},{"1":"7c8b","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 main protease in complex with Z-VAD(OMe)-FMK"},{"1":"6xoa","2":"NA","3":"X-ray diffraction","4":"The crystal structure of 3CL MainPro of SARS-CoV-2 with C145S mutation"},{"1":"6yz5","2":"NA","3":"X-ray diffraction","4":"H11-D4 complex with SARS-CoV-2 RBD"},{"1":"6z4u","2":"NA","3":"X-ray diffraction","4":"X-ray Crystallographic Structure of Orf9b from SARS-CoV-2"},{"1":"7jr4","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 3CL protease with alternative conformation of the active site promoted by methylene-bridged cysteine and lysine residues"},{"1":"8hrl","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Delta S-RBD-ACE2"},{"1":"6xqb","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 RdRp/RNA complex"},{"1":"6yb7","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease with unliganded active site (2019-nCoV, coronavirus disease 2019, COVID-19)."},{"1":"7kzb","2":"NA","3":"X-ray diffraction","4":"Potent SARS-CoV-2 binding and neutralization through maturation of iconic SARS-CoV-1antibodies"},{"1":"7fjc","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Beta RBD complexed with P36-5D2 Fab"},{"1":"7fb0","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in closed state"},{"1":"7f15","2":"NA","3":"X-ray diffraction","4":"A SARS-CoV-2 neutralizing antibody"},{"1":"6xr3","2":"NA","3":"X-ray diffraction","4":"X-ray Structure of SARS-CoV-2 main protease bound to GRL-024-20 at 1.45 A"},{"1":"7ej4","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with a neutralizing antibody RBD-chAb-25"},{"1":"7f0x","2":"NA","3":"X-ray diffraction","4":"A SARS-CoV-2 neutralizing antibody"},{"1":"7fb3","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 spike protein in one-RBD weak state after CTSL-treatment"},{"1":"7fr3","2":"NA","3":"X-ray diffraction","4":"PanDDA analysis group deposition -- Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with PTERA_A01A - (S) isomer"},{"1":"7koa","2":"NA","3":"X-ray diffraction","4":"Room Temperature Structure of SARS-CoV-2 Nsp10/16 Methyltransferase in a Complex with Cap-0 and SAM Determined by Pink-Beam Serial Crystallography"},{"1":"7vbf","2":"NA","3":"X-ray diffraction","4":"1.3 Angstrom Resolution Crystal Structure of SARS-CoV-2 Nucleocapsid dimerization domain, pH 8.5"},{"1":"7v7a","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Gamma variant (P.1), two RBD-up conformation"},{"1":"7v85","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, two ACE2-bound form"},{"1":"7v7g","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1), two RBD-up conformation"},{"1":"7um2","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Spike-derived peptide S417-425 K417T mutant (TIADYNYKL) presented by HLA-A*02:01"},{"1":"7v8b","2":"NA","3":"Electron Microscopy","4":"Local refinement of SARS-CoV-2 S-Delta variant (B.1.617.2) RBD and Angiotensin-converting enzyme 2 (ACE2) ectodomain"},{"1":"7v76","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Beta variant (B.1.351), uncleavable form, one RBD-up conformation"},{"1":"7ur1","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Spike-derived peptide S1215-1224 (YIWLGFIAGL) presented by HLA-A*02:01"},{"1":"7wqk","2":"NA","3":"X-ray diffraction","4":"wild-type SARS-CoV-2 main protease in complex with MG-132"},{"1":"7v7v","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), two RBD-up conformation 3"},{"1":"7v87","2":"NA","3":"Electron Microscopy","4":"Local refinement of SARS-CoV-2 S-Kappa variant (B.1.617.1) RBD and Angiotensin-converting enzyme 2 (ACE2) ectodomain"},{"1":"7xj8","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 BA.1 Spike trimer in complex with 55A8 Fab and 58G6 Fab in the class 2 conformation"},{"1":"7wwi","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 BA.1 Spike trimer in complex with 55A8 Fab in the class 1 conformation"},{"1":"7v7p","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), one RBD-up conformation 2"},{"1":"7xxk","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 N-CTD in complex with GMP"},{"1":"7x66","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2-Omicron-RBD and BD-236-GWP/P-VK antibody complex"},{"1":"7v8c","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Beta variant (B.1.351), Cleavable form, one RBD-up conformation"},{"1":"7xj9","2":"NA","3":"Electron Microscopy","4":"Local refinement of the SARS-CoV-2 BA.1 Spike trimer in complex with 55A8 Fab and 58G6 Fab"},{"1":"7wqa","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease in complex with Z-VAD-FMK"},{"1":"7wq9","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with Z-IETD-FMK"},{"1":"7wq8","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 main protease in complex with Z-DEVD-FMK"},{"1":"7v7q","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), one RBD-up conformation 3"},{"1":"7v8a","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, three ACE2-bound form conformation 2"},{"1":"7xil","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2-Beta-RBD and B38-GWP/P-VK antibody complex"},{"1":"7y0o","2":"NA","3":"Electron Microscopy","4":"Interface of SARS-CoV-2 WT Spike in complex with R15 Fab and P14 Nanobody"},{"1":"7xik","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2-Omicron-RBD and B38-GWP/P-VK antibody complex"},{"1":"7twv","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose at pH 5 (P43 crystal form)"},{"1":"7v7z","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Beta variant (B.1.351) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, three ACE2-bound form"},{"1":"7wqb","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 main protease mutant (P168A) in complex with MG-132"},{"1":"7xei","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2-prototyped-RBD and CB6-092-Fab complex"},{"1":"7v7r","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), one RBD-up conformation 4"},{"1":"7tx1","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose at pH 10 (P43 crystal form)"},{"1":"7tww","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose at pH 6 (P43 crystal form)"},{"1":"7v7n","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), all RBD-down conformation"},{"1":"7v78","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Gamma variant (P.1), one RBD-up conformation 1"},{"1":"7y0n","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 WT Spike in complex with R15 Fab and P14 Nanobody"},{"1":"7xy3","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with VHH14"},{"1":"7xj6","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 BA.1 Spike trimer in complex with 55A8 Fab and 58G6 Fab in the class 1 conformation"},{"1":"7v7s","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), one RBD-up conformation 5"},{"1":"7vhn","2":"NA","3":"Electron Microscopy","4":"Spike of SARS-CoV-2 spike protein(1 up)"},{"1":"7ult","2":"NA","3":"X-ray diffraction","4":"Crystal Structure of SARS-CoV-2 Nsp16/10 Heterodimer Apo-Form."},{"1":"7v7e","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1), one RBD-up conformation 1"},{"1":"7twy","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose at pH 8 (P43 crystal form)"},{"1":"7wrj","2":"NA","3":"Electron Microscopy","4":"Local CryoEM structure of the SARS-CoV-2 S6P(B.1.1.529) in complex with BD55-4637 Fab"},{"1":"7vhj","2":"NA","3":"Electron Microscopy","4":"Furin Site deletion of SARS-CoV-2 spike"},{"1":"7v79","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Gamma variant (P.1), one RBD-up conformation 2"},{"1":"7v7u","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), two RBD-up conformation 2"},{"1":"7v7d","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1), all RBD-down conformation"},{"1":"7ukm","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of Antibody 12-19 in complex with prefusion SARS-CoV-2 Spike glycoprotein"},{"1":"7vhh","2":"NA","3":"Electron Microscopy","4":"Delta variant of SARS-CoV-2 Spike protein"},{"1":"7v7j","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1), dimer of S trimer conformation 3"},{"1":"7v7o","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), one RBD-up conformation 1"},{"1":"8a23","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 nsp10/nsp16 methyltransferase in complex with TO383"},{"1":"7uup","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease S144A (Mpro S144A) in Complex with Nirmatrelvir (PF-07321332)"},{"1":"7rby","2":"NA","3":"X-ray diffraction","4":"Crystal structure of Nanobody nb112 and SARS-CoV-2 RBD"},{"1":"7pzq","2":"NA","3":"X-ray diffraction","4":"Oxidized form of SARS-CoV-2 Main Protease determined by XFEL radiation"},{"1":"7twt","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose at pH 4 (P43 crystal form)"},{"1":"7wwj","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 BA.1 Spike trimer in complex with 55A8 Fab in the class 2 conformation"},{"1":"7rqg","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the Nsp3 Y3 domain from SARS-CoV-2"},{"1":"7q3q","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with the neutralizing nanobody VHH-12"},{"1":"7zjl","2":"NA","3":"Electron Microscopy","4":"Delta SARS-CoV-2 spike protein in complex with REGN10987 Fab homologue."},{"1":"7tpk","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 E406W mutant RBD - Local Refinement"},{"1":"7ybh","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 lambda variant spike"},{"1":"7twx","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose at pH 7 (P43 crystal form)"},{"1":"7yco","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Receptor Binding Domain bound to A6 repebody"},{"1":"7ybn","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 C.1.2 variant spike (Open state)"},{"1":"7ybl","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 B.1.620 variant spike (close state)"},{"1":"7u92","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease (Mpro) in Complex with ML1006a"},{"1":"7u1r","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Spike-derived peptide S1185-1193 K1191N mutant (RLNEVANNL) presented by HLA-A*02:01"},{"1":"7tpi","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 E406W mutant Spike ectodomain"},{"1":"7ycz","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron 2-RBD up Spike trimer complexed with three XG005 molecules"},{"1":"7z2k","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 Main Protease in orthorhombic space group p212121"},{"1":"7ylb","2":"NA","3":"X-ray diffraction","4":"Two monobodies recognizing the conserved epitopes of SARS-CoV-2 N antigen applicable to the broad COVID-19 diagnosis"},{"1":"7yd1","2":"NA","3":"Electron Microscopy","4":"Local refinement of SARS-CoV-2 Omicron S trimer complexed with XG005"},{"1":"7y6k","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 receptor binding domain in complex with K202.B bispecific antibody"},{"1":"7tvs","2":"NA","3":"X-ray diffraction","4":"The Crystal Structure of SARS-CoV-2 Omicron Mpro (P132H) in complex with demethylated analog of masitinib"},{"1":"7tvx","2":"NA","3":"X-ray diffraction","4":"The Crystal Structure of SARS-CoV-2 Omicron Mpro (P132H) in complex with masitinib"},{"1":"7v82","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Gamma variant (P.1) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, three ACE2-bound form conformation 1"},{"1":"7phz","2":"NA","3":"X-ray diffraction","4":"Crystal structure of X77 bound to the main protease (3CLpro/Mpro) of SARS-CoV-2 in spacegroup P2(1)2(1)2(1)."},{"1":"7v7h","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1), dimer of S trimer conformation 1"},{"1":"7ybi","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Mu variant spike (open state)"},{"1":"7ybk","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 B.1.620 variant spike (open state)"},{"1":"7ycy","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron 1-RBD up Spike trimer complexed with three XG005 molecules"},{"1":"7v84","2":"NA","3":"Electron Microscopy","4":"Local refinement of SARS-CoV-2 S-Gamma variant (P.1) RBD and Angiotensin-converting enzyme 2 (ACE2) ectodomain"},{"1":"7v81","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Gamma variant (P.1) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, two ACE2-bound form"},{"1":"7tx0","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 NSP3 macrodomain in complex with ADP-ribose at pH 9 (P43 crystal form)"},{"1":"7vbe","2":"NA","3":"X-ray diffraction","4":"1.6 Angstrom Resolution Crystal Structure of SARS-CoV-2 Nucleocapsid dimerization domain, pH 5.0"},{"1":"7v86","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, three ACE2-bound form"},{"1":"7v77","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Beta variant (B.1.351), uncleavable form, two RBD-up conformation"},{"1":"7v7f","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1), one RBD-up conformation 2"},{"1":"7x63","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2-Beta-RBD and BD-236-GWP/P-VK antibody complex"},{"1":"7q3r","2":"NA","3":"X-ray diffraction","4":"Crystal structure of SARS-CoV-2 RBD in complex with the neutralizing nanobodies VHH-F04 and VHH-G09"},{"1":"7v80","2":"NA","3":"Electron Microscopy","4":"Local refinement of SARS-CoV-2 S-Beta variant (B.1.351) RBD and Angiotensin-converting enzyme 2 (ACE2) ectodomain"},{"1":"7pxz","2":"NA","3":"X-ray diffraction","4":"Reduced form of SARS-CoV-2 Main Protease determined by XFEL radiation"},{"1":"7xeg","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2-Beta-RBD and CB6-092-Fab complex"},{"1":"7xy4","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 spike in complex with VHH21"},{"1":"7yr4","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 BA.2.75 S Trimer in complex with ACE2(interface)"},{"1":"7ywr","2":"NA","3":"Solution NMR","4":"NMR structure of the N-terminal domain of Nsp8 from SARS-CoV-2"},{"1":"7y8j","2":"NA","3":"X-ray diffraction","4":"3D1 in complex with 6-mer HR1 peptide from SARS-CoV-2"},{"1":"7v7t","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2), two RBD-up conformation 1"},{"1":"7wwk","2":"NA","3":"Electron Microscopy","4":"Local refinement of the SARS-CoV-2 BA.1 Spike trimer in complex with 55A8 Fab"},{"1":"7upy","2":"NA","3":"Electron Microscopy","4":"One RBD-up state of SARS-CoV-2 D614G spike in complex with the SP1-77 neutralizing antibody Fab fragment"},{"1":"7uug","2":"NA","3":"X-ray diffraction","4":"SARS-CoV-2 Main Protease S144A (Mpro S144A) in Complex with ML1006a"},{"1":"7v89","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, three ACE2-bound form conformation 1"},{"1":"7v83","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Gamma variant (P.1) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, three ACE2-bound form conformation 2"},{"1":"7v88","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Delta variant (B.1.617.2) in complex with Angiotensin-converting enzyme 2 (ACE2) ectodomain, two ACE2-bound form"},{"1":"7v7i","2":"NA","3":"Electron Microscopy","4":"Cryo-EM structure of SARS-CoV-2 S-Kappa variant (B.1.617.1), dimer of S trimer conformation 2"},{"1":"7ybm","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 C.1.2 variant spike (Close state)"},{"1":"7ybg","2":"NA","3":"X-ray diffraction","4":"Crystal structure of the SARS-CoV-2 papain-like protease (C111S mutant)"},{"1":"7yd0","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Omicron 1-RBD up spike trimer complexed with two XG005 Fab"},{"1":"7ybj","2":"NA","3":"Electron Microscopy","4":"SARS-CoV-2 Mu variant spike(close state)"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p><a href="datasets/sars-cov2-structures.json"><strong>Download raw
dataset in JSON format</strong></a></p>
</div>

<div id="rmd-source-code">LS0tCnRpdGxlOiAiU0FSUy1Db1YtMiBzdHJ1Y3R1cmVzIgotLS0KCmBgYHtyIHNldHVwLCBpbmNsdWRlPUZBTFNFfQprbml0cjo6b3B0c19jaHVuayRzZXQobWVzc2FnZSA9IEZBTFNFLCB3YXJuaW5nID0gRkFMU0UpCmBgYAoKKipMYXN0IHVwZGF0ZWQgb24gYHIgZGF0ZSgpYC4qKgoKIyMgSG93IHRvIHJlLXVzZSB0aGlzIHdvcmsKCklmIHlvdSB1c2UgdGhlc2UgZmlndXJlcyBpbiB5b3VyIG93biB3b3JrLCBwbGVhc2UgY2l0ZSB0aGlzIHdlYnNpdGU6CjxodHRwczovL2RvaS5vcmcvMTAuNTI4MS96ZW5vZG8uMzQ3MDExOT4KCmBgYHtyIExvYWQgcGFja2FnZXMsIGRvd25sb2FkIGRhdGEgYW5kIHByZXBhcmUgZGF0YXNldH0KIyBMb2FkIHJlcXVpcmVkIHBhY2thZ2VzCmxpYnJhcnkobWFncml0dHIpCmxpYnJhcnkoZHBseXIpCmxpYnJhcnkoanNvbmxpdGUpCmxpYnJhcnkoZm9yY2F0cykKbGlicmFyeShnZ3Bsb3QyKQpsaWJyYXJ5KHBsb3RseSkKbGlicmFyeShoZXJlKQpsaWJyYXJ5KGx1YnJpZGF0ZSkKCiMgV2UgbmVlZCB0d28gcXVlcmllczogb25lIGZvciB1cHBlcmNhc2UgdGl0bGVzLCB0aGUgb3RoZXIgZm9yIGxvd2VyY2FzZSBvbmVzCnBkYl9xdWVyeSA8LSAnaHR0cHM6Ly93d3cuZWJpLmFjLnVrL3BkYmUvc2VhcmNoL3BkYi9zZWxlY3Q/cT10aXRsZToqU0FSUy1Db1YtMiolMjBBTkQlMjBzdGF0dXM6UkVMJmZsPXBkYl9pZCxkZXBvc2l0aW9uX2RhdGUsY2l0YXRpb25feWVhcix0aXRsZSxleHBlcmltZW50YWxfbWV0aG9kLHJlc29sdXRpb24sb3JnYW5pc21fc2NpZW50aWZpY19uYW1lLG1vbGVjdWxlX25hbWUsbW9sZWN1bGVfdHlwZSZyb3dzPTEwMDAwMDAmd3Q9anNvbicKCiMgVGhpcyBpcyBhIGhlbHBlciBwaXBlbGluZSB0byBleHRyYWN0IGRhdGEKZGlnX3VwX2RhdGEgPC0gLiAlPiUKICAgIC4kcmVzcG9uc2UgJT4lCiAgICAuJGRvY3MgJT4lCiAgICBhc190aWJibGUoKQoKIyBRdWVyeSB0aGUgUERCIGFuZCBjbGVhbiB1cCBkYXRhCnBkYl9kYXRhIDwtIHBkYl9xdWVyeSAlPiUgCiAgICBmcm9tSlNPTigpICU+JSAKICAgIGRpZ191cF9kYXRhKCkgJT4lIAogICAgYmluZF9yb3dzKCkgJT4lIAogICAgbXV0YXRlKAogICAgICBleHBlcmltZW50YWxfbWV0aG9kICAgPSBhc19mYWN0b3IoYXMuY2hhcmFjdGVyKGV4cGVyaW1lbnRhbF9tZXRob2QpKSwKICAgICAgZGVwb3NpdGlvbl9kYXRlICAgICAgID0gYXNfZGF0ZSh5bWRfaG1zKGRlcG9zaXRpb25fZGF0ZSkpLAogICAgICBjaXRhdGlvbl95ZWFyICAgICAgICAgPSBhcy5pbnRlZ2VyKGNpdGF0aW9uX3llYXIpLAogICAgICBtb2xlY3VsZV9uYW1lICAgICAgICAgPSBhcy5jaGFyYWN0ZXIobW9sZWN1bGVfbmFtZSksCiAgICAgIG1vbGVjdWxlX3R5cGUgICAgICAgICA9IGFzX2ZhY3Rvcihhcy5jaGFyYWN0ZXIobW9sZWN1bGVfdHlwZSkpCiAgICAgICkgJT4lIAogICAgZGlzdGluY3QocGRiX2lkLCAua2VlcF9hbGwgPSBUUlVFKQpgYGAKCiMjIEFib3V0IHRoaXMgc2VjdGlvbgoKV2l0aCB0aGUgb25nb2luZyBbQ09WSUQtMTkKcGFuZGVtaWNdKGh0dHBzOi8vZW4ud2lraXBlZGlhLm9yZy93aWtpL0NPVklELTE5X3BhbmRlbWljKSwgbmV3IHN0cnVjdHVyZXMgb2YKcHJvdGVpbnMgZnJvbQpbU0FSUy1Db1YtMl0oaHR0cHM6Ly9lbi53aWtpcGVkaWEub3JnL3dpa2kvU2V2ZXJlX2FjdXRlX3Jlc3BpcmF0b3J5X3N5bmRyb21lX2Nvcm9uYXZpcnVzXzIpIGFyZSBkZXBvc2l0ZWQgdG8gdGhlIFBEQiBhdCBhIGZhc3QgcGFjZS4gVGhlIHZpc3VhbGl6YXRpb25zIHByZXNlbnRlZCBoZXJlCndvbid0IGhlbHAgY3VyYiB0aGUgcGFuZGVtaWMgKGFuZCBhcmUgbm90IG1lZGljYWwgYWR2aWNlLCBvYnZpb3VzbHkpLCBidXQgSSBob3BlCnRoZXkgd2lsbCBtYWtlIGN1cmlvdXMgYW5kIGhlbHBsZXNzIG1pbmRzIChsaWtlIG15c2VsZikgZ2V0IGEgZmVlbCBmb3IgdGhlCnJlc2VhcmNoIGVmZm9ydCBjdXJyZW50bHkgYmVpbmcgZGVwbG95ZWQgYWdhaW5zdCBpdC4KClRoZSBmb2xsb3dpbmcgcmVzb3VyY2VzIGFyZSBhIGxvdCBtb3JlIHJlbGV2YW50IGFuZCBhY3Rpb25hYmxlOgoKLSBbQ29yb25hdmlydXMgU3RydWN0dXJhbCBUYXNrIEZvcmNlXShodHRwczovL2luc2lkZWNvcm9uYS5uZXQvKQotIFtQREJlIENPVklELTE5IERhdGEgUG9ydGFsXShodHRwczovL3d3dy5lYmkuYWMudWsvcGRiZS9jb3ZpZC0xOSkKLSBbQ3Jvd2RmaWdodCBDT1ZJRC0xOV0oaHR0cHM6Ly9jcm93ZGZpZ2h0Y292aWQxOS5vcmcvKQoKIyMgU0FSUy1Db1YtMiBzdHJ1Y3R1cmVzIGJ5IGRlcG9zaXRpb24gZGF0ZQoKQWxsIGZpZ3VyZXMgYXJlIGludGVyYWN0aXZlICh5b3UgY2FuIHpvb20gaW4sIGFuZCBob3ZlcmluZyBvdmVyIGVsZW1lbnRzIHdpbGwKc2hvdyBtb3JlIGluZm9ybWF0aW9uKS4KCmBgYHtyIFNBUlMtQ29WLTIgc3RydWN0dXJlcyBieSB5ZWFyfQpzYXJzX2NvdjJfc3RydWN0dXJlc195ZWFyIDwtIHBkYl9kYXRhICU+JSAKICAgIGdncGxvdCgpICsKICAgIGdlb21fYmFyKG1hcHBpbmcgPSBhZXMoeCA9IGRlcG9zaXRpb25fZGF0ZSkpICsKICAgIGdndGl0bGUoIlN0cnVjdHVyZXMgb2YgU0FSUy1Db1YtMiBieSBkZXBvc2l0aW9uIGRhdGUiKSArCiAgICB4bGFiKCJEZXBvc2l0aW9uIGRhdGUiKSArCiAgICB5bGFiKCJOdW1iZXIgb2YgUERCIGVudHJpZXMiKSArCiAgICB0aGVtZV9idygpCmdncGxvdGx5KHNhcnNfY292Ml9zdHJ1Y3R1cmVzX3llYXIpCmBgYAoKTm90aWNlYWJsZSB2ZXJ5IGxhcmdlIG51bWJlcnMgb2YgZGVwb3NpdGlvbnMgb24gYSBzYW1lIGRheSAoZm9yIGV4YW1wbGUsIDY4CnN0cnVjdHVyZXMgZGVwb3NpdGVkIG9uIE1hcmNoIDE1dGggYW5kIDE0IHN0cnVjdHVyZXMgZGVwb3NpdGVkIG9uIEFwcmlsIDd0aCkKY29tZSBmcm9tIGxhcmdlLXNjYWxlIGZyYWdtZW50IHNjcmVlbmluZyBleHBlcmltZW50cyBwZXJmb3JtZWQgYXQgRGlhbW9uZCBMaWdodApTb3VyY2U6CgotIFttYWluIHByb3RlYXNlXShodHRwczovL3d3dy5kaWFtb25kLmFjLnVrL2NvdmlkLTE5L2Zvci1zY2llbnRpc3RzL01haW4tcHJvdGVhc2Utc3RydWN0dXJlLWFuZC1YQ2hlbS5odG1sKQotIFttYWNybyBkb21haW5dKGh0dHBzOi8vd3d3LmRpYW1vbmQuYWMudWsvY292aWQtMTkvZm9yLXNjaWVudGlzdHMvTlNQMy1tYWNyb2RvbWFpbi1zdHJ1Y3R1cmUtYW5kLVhDaGVtLmh0bWwpCi0gW2hlbGljYXNlXShodHRwczovL3d3dy5kaWFtb25kLmFjLnVrL2NvdmlkLTE5L2Zvci1zY2llbnRpc3RzL05TUDEzLUhlbGljYXNlLVN0cnVjdHVyZS1hbmQtWENoZW0uaHRtbCkKCmBgYHtyIFNhdmUgU1ZHIGZpbGUgb2YgeWVhciBncmFwaCwgaW5jbHVkZT1GQUxTRX0KIyBTYXZlIGZpZ3VyZSBmb3IgZG93bmxvYWQKaWYgKCFkaXIuZXhpc3RzKGhlcmUoImZpZ3VyZXMiKSkpIHsKICAgIGRpci5jcmVhdGUoaGVyZSgiZmlndXJlcyIpKQp9Cmdnc2F2ZShmaWxlbmFtZSA9ICJzYXJzLWNvdjItc3RydWN0dXJlcy1ieS15ZWFyLnN2ZyIsCiAgICAgICBwbG90ID0gc2Fyc19jb3YyX3N0cnVjdHVyZXNfeWVhciwKICAgICAgIGRldmljZSA9ICJzdmciLAogICAgICAgcGF0aCA9IGhlcmUoImZpZ3VyZXMiKSkKYGBgCgpbKipEb3dubG9hZCBmaWd1cmUgaW4gU1ZHIGZvcm1hdCoqXShmaWd1cmVzL3NhcnMtY292Mi1zdHJ1Y3R1cmVzLWJ5LXllYXIuc3ZnKQoKIyMgU0FSUy1Db1YtMiBzdHJ1Y3R1cmVzIGJ5IGV4cGVyaW1lbnRhbCBtZXRob2QKCmBgYHtyIFNBUlMtQ29WLTIgc3RydWN0dXJlcyBieSBleHBlcmltZW50YWwgbWV0aG9kfQpzYXJzX2NvdjJfc3RydWN0dXJlc19tZXRob2QgPC0gcGRiX2RhdGEgJT4lIAogICAgZ2dwbG90KCkgKwogICAgZ2VvbV9iYXIobWFwcGluZyA9IGFlcyh4ID0gZXhwZXJpbWVudGFsX21ldGhvZCwgZmlsbCA9IGV4cGVyaW1lbnRhbF9tZXRob2QpKSArCiAgICBndWlkZXMoZmlsbCA9IGd1aWRlX2xlZ2VuZCh0aXRsZSA9ICJFeHBlcmltZW50YWwgbWV0aG9kIikpICsKICAgIGdndGl0bGUoIlN0cnVjdHVyZXMgb2YgU0FSUy1Db1YtMiBieSBleHBlcmltZW50YWwgbWV0aG9kIikgKwogICAgeGxhYigiIikgKwogICAgeWxhYigiTnVtYmVyIG9mIFBEQiBlbnRyaWVzIikgKwogICAgdGhlbWVfYncoKQpnZ3Bsb3RseShzYXJzX2NvdjJfc3RydWN0dXJlc19tZXRob2QpCmBgYAoKYGBge3IgU2F2ZSBTVkcgZmlsZSBvZiBtZXRob2QgZ3JhcGgsIGluY2x1ZGU9RkFMU0V9CiMgU2F2ZSBmaWd1cmUgZm9yIGRvd25sb2FkCmlmICghZGlyLmV4aXN0cyhoZXJlKCJmaWd1cmVzIikpKSB7CiAgICBkaXIuY3JlYXRlKGhlcmUoImZpZ3VyZXMiKSkKfQpnZ3NhdmUoZmlsZW5hbWUgPSAic2Fycy1jb3YyLXN0cnVjdHVyZXMtYnktbWV0aG9kLnN2ZyIsCiAgICAgICBwbG90ID0gc2Fyc19jb3YyX3N0cnVjdHVyZXNfbWV0aG9kLAogICAgICAgZGV2aWNlID0gInN2ZyIsCiAgICAgICBwYXRoID0gaGVyZSgiZmlndXJlcyIpKQpgYGAKClsqKkRvd25sb2FkIGZpZ3VyZSBpbiBTVkcgZm9ybWF0KipdKGZpZ3VyZXMvc2Fycy1jb3YyLXN0cnVjdHVyZXMtYnktbWV0aG9kLnN2ZykKCgojIyBEYXRhc2V0CgpUaGUgZ3JhcGhzIHByZXNlbnRlZCBhYm92ZSBhcmUgZGVyaXZlZCBmcm9tIHRoZSBmb2xsb3dpbmcgZGF0YXNldDoKCmBgYHtyIEVudGlyZSBkYXRhc2V0fQojIEZvcm1hdCB0YWJsZSBmb3IgZGlzcGxheQpwZGJfdGFibGUgPC0gcGRiX2RhdGEgJT4lIAogICAgYXJyYW5nZShkZXNjKGNpdGF0aW9uX3llYXIpKSAlPiUgCiAgICBzZWxlY3QoYFBEQiBjb2RlYCA9IHBkYl9pZCwKICAgICAgICAgICBgQ2l0YXRpb24geWVhcmAgPSBjaXRhdGlvbl95ZWFyLAogICAgICAgICAgIGBFeHBlcmltZW50YWwgbWV0aG9kYCA9IGV4cGVyaW1lbnRhbF9tZXRob2QsCiAgICAgICAgICAgVGl0bGUgPSB0aXRsZSkKcGRiX3RhYmxlCmBgYAoKYGBge3IgU2F2ZSBkYXRhc2V0IGFzIEpTT04sIGluY2x1ZGU9RkFMU0V9CiMgU2F2ZSBkYXRhc2V0IGZvciBkb3dubG9hZAppZiAoIWRpci5leGlzdHMoaGVyZSgiZGF0YXNldHMiKSkpIHsKICAgIGRpci5jcmVhdGUoaGVyZSgiZGF0YXNldHMiKSkKfQoKd3JpdGVfanNvbihwZGJfZGF0YSwgaGVyZSgiZGF0YXNldHMiLCAic2Fycy1jb3YyLXN0cnVjdHVyZXMuanNvbiIpKQpgYGAKClsqKkRvd25sb2FkIHJhdyBkYXRhc2V0IGluIEpTT04gZm9ybWF0KipdKGRhdGFzZXRzL3NhcnMtY292Mi1zdHJ1Y3R1cmVzLmpzb24pCg==</div>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeSourceEmbed("sars-cov-2.Rmd");
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
